// @carbon/pictograms-react/lib/chart--area/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const ChartArea = /*#__PURE__*/React__default["default"].forwardRef(function ChartArea(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31,31.36H1c-0.199,0-0.36-0.161-0.36-0.36V1h0.72v6.5l11.526-3.842\tc0.138-0.045,0.292-0.003,0.387,0.107l5.83,6.801l11.758-4.899c0.111-0.047,0.238-0.034,0.338,0.033C31.3,5.767,31.36,5.879,31.36,6\tv25C31.36,31.199,31.199,31.36,31,31.36z M1.36,30.64h29.28v-7.058l-7.479,3.74c-0.118,0.059-0.259,0.049-0.367-0.027l-9.787-6.851\tL1.36,29.18V30.64z M1.36,22.149v6.131l11.424-8.568c0.125-0.094,0.295-0.095,0.422-0.007l9.825,6.877l7.608-3.805v-8.142\tl-9.454,5.673c-0.154,0.094-0.354,0.058-0.467-0.084l-7.749-9.686L1.36,22.149z M1.36,8.259v12.872L12.746,9.746\tc0.072-0.073,0.173-0.114,0.274-0.105c0.102,0.005,0.197,0.055,0.261,0.134l7.804,9.754l9.555-5.733V6.54l-11.501,4.792\tc-0.145,0.059-0.311,0.021-0.412-0.098l-5.842-6.816L1.36,8.259z"
  })), children);
});

module.exports = ChartArea;
