// @carbon/pictograms-react/lib/server--rack/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const ServerRack = /*#__PURE__*/React__default["default"].forwardRef(function ServerRack(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M27,31.36H5c-0.199,0-0.36-0.161-0.36-0.36V1c0-0.199,0.161-0.36,0.36-0.36h22\tc0.199,0,0.36,0.161,0.36,0.36v30C27.36,31.199,27.199,31.36,27,31.36z M5.36,30.64h21.28v-4.28H5.36V30.64z M5.36,25.64h21.28\tv-4.28H5.36V25.64z M5.36,20.64h21.28v-4.28H5.36V20.64z M5.36,15.64h21.28v-4.28H5.36V15.64z M5.36,10.64h21.28V6.36H5.36V10.64z M5.36,5.64h21.28V1.36H5.36V5.64z M25,28.36h-6v-0.72h6V28.36z M12,28.36h-1v-0.72h1V28.36z M10,28.36H9v-0.72h1V28.36z M8,28.36H7\tv-0.72h1V28.36z M25,23.36h-6v-0.72h6V23.36z M12,23.36h-1v-0.72h1V23.36z M10,23.36H9v-0.72h1V23.36z M8,23.36H7v-0.72h1V23.36z M25,18.36h-6v-0.72h6V18.36z M12,18.36h-1v-0.72h1V18.36z M10,18.36H9v-0.72h1V18.36z M8,18.36H7v-0.72h1V18.36z M25,13.36h-6\tv-0.72h6V13.36z M12,13.36h-1v-0.72h1V13.36z M10,13.36H9v-0.72h1V13.36z M8,13.36H7v-0.72h1V13.36z M25,8.36h-6V7.64h6V8.36z M12,8.36h-1V7.64h1V8.36z M10,8.36H9V7.64h1V8.36z M8,8.36H7V7.64h1V8.36z M25,3.36h-6V2.64h6V3.36z M12,3.36h-1V2.64h1V3.36z M10,3.36H9V2.64h1V3.36z M8,3.36H7V2.64h1V3.36z"
  })), children);
});

module.exports = ServerRack;
