// @carbon/pictograms-react/es/control-panel/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const ControlPanel = /*#__PURE__*/React.forwardRef(function ControlPanel(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M30,30.36H2c-0.199,0-0.36-0.161-0.36-0.36V2\tc0-0.199,0.161-0.36,0.36-0.36h28c0.199,0,0.36,0.161,0.36,0.36v28C30.36,30.199,30.199,30.36,30,30.36z M2.36,29.64h27.28V2.36\tH2.36V29.64z M9,25.36c-0.75,0-1.36-0.61-1.36-1.36c0-0.625,0.424-1.153,1-1.312V7h0.72v15.688c0.576,0.158,1,0.687,1,1.312\tC10.36,24.75,9.75,25.36,9,25.36z M9,23.36c-0.353,0-0.64,0.287-0.64,0.64S8.647,24.64,9,24.64S9.64,24.353,9.64,24\tS9.353,23.36,9,23.36z M23.36,25h-0.72V9.312c-0.575-0.158-1-0.686-1-1.312c0-0.75,0.61-1.36,1.36-1.36S24.36,7.25,24.36,8\tc0,0.625-0.425,1.153-1,1.312V25z M23,7.36c-0.353,0-0.64,0.287-0.64,0.64S22.647,8.64,23,8.64S23.64,8.353,23.64,8\tS23.353,7.36,23,7.36z M16.36,25h-0.72v-7.688c-0.576-0.158-1-0.686-1-1.312s0.424-1.153,1-1.312V7h0.72v7.688\tc0.576,0.158,1,0.686,1,1.312c0,0.626-0.425,1.153-1,1.312V25z M16,15.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64\ts0.64-0.287,0.64-0.64S16.353,15.36,16,15.36z"
  })), children);
});

export { ControlPanel as default };
