// @carbon/pictograms-react/es/banking/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Banking = /*#__PURE__*/React.forwardRef(function Banking(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M31,31.36H1c-0.199,0-0.36-0.161-0.36-0.36v-3\tc0-0.199,0.161-0.36,0.36-0.36h1.64V25c0-0.199,0.161-0.36,0.36-0.36h1.64v-8.28H3c-0.199,0-0.36-0.161-0.36-0.36v-2.64H1\tc-0.153,0-0.289-0.096-0.34-0.241c-0.05-0.144-0.004-0.305,0.115-0.4l15-12c0.132-0.104,0.318-0.104,0.45,0l15,12\tc0.12,0.096,0.166,0.256,0.115,0.4c-0.051,0.145-0.187,0.241-0.34,0.241h-1.64V16c0,0.199-0.161,0.36-0.36,0.36h-1.64v8.28H29\tc0.199,0,0.36,0.161,0.36,0.36v2.64H31c0.199,0,0.36,0.161,0.36,0.36v3C31.36,31.199,31.199,31.36,31,31.36z M1.36,30.64h29.28\tv-2.28H1.36V30.64z M3.36,27.64h25.28v-2.28H3.36V27.64z M23.36,24.64h3.279v-8.28H23.36V24.64z M18.36,24.64h4.279v-8.28H18.36\tV24.64z M14.36,24.64h3.28v-8.28h-3.28V24.64z M9.36,24.64h4.28v-8.28H9.36V24.64z M5.36,24.64h3.28v-8.28H5.36V24.64z M3.36,15.64\th25.28v-2.28H3.36V15.64z M2.026,12.64h27.947L16,1.461L2.026,12.64z M16.36,11h-0.72V9.86H14V9.14h3c0.353,0,0.64-0.287,0.64-0.64\tS17.353,7.86,17,7.86h-2c-0.75,0-1.36-0.61-1.36-1.36S14.25,5.14,15,5.14h0.64V4h0.72v1.14H18v0.72h-3\tc-0.353,0-0.64,0.287-0.64,0.64S14.647,7.14,15,7.14h2c0.75,0,1.36,0.61,1.36,1.36S17.75,9.86,17,9.86h-0.64V11z"
  })), children);
});

export { Banking as default };
