// @carbon/pictograms-react/lib/chart--bubble--line/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const ChartBubbleLine = /*#__PURE__*/React__default["default"].forwardRef(function ChartBubbleLine(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31,31.36H1c-0.199,0-0.36-0.161-0.36-0.36V1h0.72v14.64h4.308c0.174-1.131,1.154-2,2.333-2\tc1.301,0,2.36,1.059,2.36,2.36c0,0.503-0.158,0.971-0.428,1.354l4.714,4.714c0.619-0.436,1.517-0.545,2.23-0.26L23.71,9.976\tC23.066,9.554,22.64,8.826,22.64,8c0-1.301,1.059-2.36,2.36-2.36c1.179,0,2.159,0.869,2.333,2H31v0.72h-3.667\tc-0.174,1.131-1.154,2-2.333,2c-0.219,0-0.43-0.03-0.631-0.085l-6.873,11.901c0.527,0.434,0.864,1.09,0.864,1.824\tc0,1.302-1.059,2.36-2.36,2.36s-2.36-1.059-2.36-2.36c0-0.539,0.181-1.036,0.486-1.434l-4.693-4.692\tC9.036,18.179,8.539,18.36,8,18.36c-1.179,0-2.159-0.869-2.333-2H1.36v14.28H31V31.36z M16,22.36c-0.904,0-1.64,0.735-1.64,1.64\ts0.736,1.64,1.64,1.64c0.904,0,1.64-0.735,1.64-1.64S16.904,22.36,16,22.36z M6.36,16c0,0.904,0.736,1.64,1.64,1.64\tS9.64,16.904,9.64,16S8.904,14.36,8,14.36S6.36,15.096,6.36,16z M25,6.36c-0.904,0-1.64,0.736-1.64,1.64S24.096,9.64,25,9.64\tS26.64,8.904,26.64,8S25.904,6.36,25,6.36z"
  })), children);
});

module.exports = ChartBubbleLine;
