// @carbon/pictograms-react/lib/carbon--for-ibm-dotcom/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const CarbonForIbmDotcom = /*#__PURE__*/React__default["default"].forwardRef(function CarbonForIbmDotcom(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M4,2.5C4,2.776,3.776,3,3.5,3S3,2.776,3,2.5S3.224,2,3.5,2S4,2.224,4,2.5z M5.5,2\tC5.224,2,5,2.224,5,2.5S5.224,3,5.5,3S6,2.776,6,2.5S5.776,2,5.5,2z M7.5,2C7.224,2,7,2.224,7,2.5S7.224,3,7.5,3S8,2.776,8,2.5\tS7.776,2,7.5,2z M23.681,31.312l7.5-4.336c0.11-0.064,0.18-0.183,0.18-0.312v-8.672c0-0.129-0.069-0.247-0.18-0.312l-7.5-4.336\tc-0.113-0.064-0.248-0.064-0.361,0l-7.5,4.336c-0.111,0.064-0.18,0.183-0.18,0.312v8.672c0,0.129,0.068,0.247,0.18,0.312l7.5,4.336\tc0.057,0.032,0.118,0.049,0.181,0.049S23.624,31.344,23.681,31.312z M16.36,18.2l7.14-4.128l7.14,4.128v8.256l-7.14,4.128\tl-7.14-4.128V18.2z M13,22.64H1.36V4.36h29.28V15h0.721V1c0-0.199-0.161-0.36-0.36-0.36H1C0.801,0.64,0.64,0.801,0.64,1v22\tc0,0.199,0.161,0.36,0.36,0.36h12V22.64z M1.36,1.36h29.28v2.28H1.36V1.36z"
  })), children);
});

module.exports = CarbonForIbmDotcom;
