// @carbon/pictograms-react/es/mobile--phone/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const MobilePhone = /*#__PURE__*/React.forwardRef(function MobilePhone(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M25,31.36H7c-0.199,0-0.36-0.161-0.36-0.36V1c0-0.199,0.161-0.36,0.36-0.36h18\tc0.199,0,0.36,0.161,0.36,0.36v30C25.36,31.199,25.199,31.36,25,31.36z M7.36,30.64h17.28v-6.28H7.36V30.64z M7.36,23.64h17.28V1.36\tH7.36V23.64z M17,27.5c0-0.552-0.448-1-1-1s-1,0.448-1,1s0.448,1,1,1S17,28.052,17,27.5z"
  })), children);
});

export { MobilePhone as default };
