// @carbon/pictograms-react/lib/compute/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Compute = /*#__PURE__*/React__default["default"].forwardRef(function Compute(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M30,30.36H2c-0.199,0-0.36-0.161-0.36-0.36V2\tc0-0.199,0.161-0.36,0.36-0.36h28c0.199,0,0.36,0.161,0.36,0.36v28C30.36,30.199,30.199,30.36,30,30.36z M16.36,29.64h13.28V16.36\tH16.36V29.64z M2.36,29.64h13.28V16.36H2.36V29.64z M16.36,15.64h13.28V2.36H16.36V15.64z M2.36,15.64h13.28V2.36H2.36V15.64z M23,27.36c-0.75,0-1.36-0.61-1.36-1.36s0.61-1.36,1.36-1.36s1.36,0.61,1.36,1.36S23.75,27.36,23,27.36z M23,25.36\tc-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S23.353,25.36,23,25.36z M11.746,26.255L9,23.509\tl-2.746,2.746l-0.509-0.51L8.491,23l-2.745-2.745l0.509-0.51L9,22.491l2.746-2.746l0.509,0.51L9.509,23l2.745,2.745L11.746,26.255z M27,23.36h-8v-0.72h8V23.36z M23,21.36c-0.75,0-1.36-0.61-1.36-1.36s0.61-1.36,1.36-1.36s1.36,0.61,1.36,1.36S23.75,21.36,23,21.36\tz M23,19.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S23.353,19.36,23,19.36z M9.36,13H8.64V9.36H5\tV8.64h3.64V5h0.72v3.64H13v0.72H9.36V13z M27,9.36h-8V8.64h8V9.36z"
  })), children);
});

module.exports = Compute;
