// @carbon/pictograms-react/es/tornado/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Tornado = /*#__PURE__*/React.forwardRef(function Tornado(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M17.5,30.36h-4v-0.72h4V30.36z M23.5,26.36h-8v-0.72h8V26.36z M25.5,22.36h-12v-0.72h12V22.36z M21.5,18.36\th-12v-0.72h12V18.36z M28.5,14.36h-17v-0.72h17V14.36z M25.5,10.36h-17V9.64h17V10.36z M23.5,6.36h-20V5.64h20V6.36z M26.5,2.36h-20\tV1.64h20V2.36z"
  })), children);
});

export { Tornado as default };
