// @carbon/pictograms-react/lib/start-for-free/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const StartForFree = /*#__PURE__*/React__default["default"].forwardRef(function StartForFree(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,31.36C7.53,31.36,0.64,24.47,0.64,16\tS7.53,0.64,16,0.64S31.36,7.53,31.36,16S24.47,31.36,16,31.36z M9.36,29.046c1.994,1.019,4.251,1.594,6.64,1.594\ts4.646-0.575,6.64-1.594V16.36h-5.328c-0.158,0.576-0.686,1-1.312,1c-0.75,0-1.36-0.61-1.36-1.36s0.61-1.36,1.36-1.36\tc0.625,0,1.153,0.424,1.312,1h5.328V7.36H9.36V29.046z M23.36,16.36v12.292c4.253-2.484,7.148-7.052,7.275-12.292H23.36z M16,1.36\tC7.927,1.36,1.36,7.927,1.36,16c0,5.392,2.93,10.111,7.28,12.652V7c0-0.199,0.161-0.36,0.36-0.36h14c0.199,0,0.36,0.161,0.36,0.36\tv8.64h7.275C30.444,7.733,23.952,1.36,16,1.36z M16,15.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64\tS16.353,15.36,16,15.36z"
  })), children);
});

module.exports = StartForFree;
