// @carbon/pictograms-react/lib/big-data/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const BigData = /*#__PURE__*/React__default["default"].forwardRef(function BigData(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,31.36c-0.059,0-0.117-0.015-0.171-0.043l-13-7\tC2.713,24.254,2.64,24.133,2.64,24V8c0-0.132,0.073-0.254,0.189-0.317l13-7c0.107-0.058,0.234-0.058,0.342,0l13,7\tC29.287,7.746,29.36,7.868,29.36,8v16c0,0.133-0.073,0.254-0.189,0.317l-13,7C16.117,31.346,16.059,31.36,16,31.36z M10.36,27.555\tL16,30.591l5.64-3.036v-8.332l-4.628-2.314c-0.499,0.555-1.525,0.555-2.023,0l-4.629,2.314C10.36,19.223,10.36,27.555,10.36,27.555z M17.334,16.265l4.827,2.413C22.283,18.74,22.36,18.864,22.36,19v8.167l2.279-1.229V14.212l-7.668-4.261\tc-0.494,0.505-1.448,0.505-1.942,0L7.36,14.212v11.727l2.28,1.229V19c0-0.137,0.077-0.261,0.199-0.322l4.827-2.413\tC14.649,16.179,14.64,16.091,14.64,16c0-0.75,0.61-1.36,1.36-1.36s1.36,0.61,1.36,1.36C17.36,16.091,17.352,16.179,17.334,16.265z M17.321,9.322l7.854,4.363c0.114,0.064,0.186,0.184,0.186,0.315v11.551l3.279-1.766V8.215L16,1.409L3.36,8.215v15.57l3.28,1.766V14\tc0-0.131,0.071-0.251,0.185-0.315l7.854-4.363C14.653,9.219,14.64,9.111,14.64,9c0-0.75,0.61-1.36,1.36-1.36S17.36,8.25,17.36,9\tC17.36,9.111,17.347,9.219,17.321,9.322z M16,15.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64\tS16.353,15.36,16,15.36z M16,8.36c-0.353,0-0.64,0.287-0.64,0.64S15.647,9.64,16,9.64c0.353,0,0.64-0.287,0.64-0.64\tS16.353,8.36,16,8.36z"
  })), children);
});

module.exports = BigData;
