// @carbon/pictograms-react/lib/hard--drive/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const HardDrive = /*#__PURE__*/React__default["default"].forwardRef(function HardDrive(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M26,6c-0.552,0-1-0.448-1-1s0.448-1,1-1s1,0.448,1,1S26.552,6,26,6z M27,16c0-0.552-0.448-1-1-1\ts-1,0.448-1,1s0.448,1,1,1S27,16.552,27,16z M27,27c0-0.552-0.448-1-1-1s-1,0.448-1,1s0.448,1,1,1S27,27.552,27,27z M31,31.36H1\tc-0.199,0-0.36-0.161-0.36-0.36v-8c0-0.199,0.161-0.36,0.36-0.36h30c0.199,0,0.36,0.161,0.36,0.36v8\tC31.36,31.199,31.199,31.36,31,31.36z M1.36,30.64h29.28v-7.28H1.36V30.64z M31,20.36H1c-0.199,0-0.36-0.161-0.36-0.36v-8\tc0-0.199,0.161-0.36,0.36-0.36h30c0.199,0,0.36,0.161,0.36,0.36v8C31.36,20.199,31.199,20.36,31,20.36z M1.36,19.64h29.28v-7.28\tH1.36V19.64z M31,9.36H1C0.801,9.36,0.64,9.199,0.64,9V1c0-0.199,0.161-0.36,0.36-0.36h30c0.199,0,0.36,0.161,0.36,0.36v8\tC31.36,9.199,31.199,9.36,31,9.36z M1.36,8.64h29.28V1.36H1.36V8.64z"
  })), children);
});

module.exports = HardDrive;
