// @carbon/pictograms-react/es/help-desk/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const HelpDesk = /*#__PURE__*/React.forwardRef(function HelpDesk(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M31,31.36H1c-0.199,0-0.36-0.161-0.36-0.36V17\tc0-0.199,0.161-0.36,0.36-0.36h8.64V16c0-2.415,1.397-4.627,3.535-5.692C12.204,9.485,11.64,8.289,11.64,7\tc0-2.404,1.956-4.36,4.36-4.36S20.36,4.596,20.36,7c0,1.289-0.564,2.485-1.535,3.308c2.138,1.064,3.535,3.277,3.535,5.692v0.64H31\tc0.199,0,0.36,0.161,0.36,0.36v14C31.36,31.199,31.199,31.36,31,31.36z M1.36,30.64h29.28V17.36H1.36V30.64z M10.36,16.64h11.28V16\tc0-2.333-1.471-4.453-3.66-5.275c-0.128-0.048-0.218-0.165-0.231-0.301s0.052-0.269,0.168-0.341C18.995,9.409,19.64,8.257,19.64,7\tc0-2.007-1.633-3.64-3.64-3.64S12.36,4.993,12.36,7c0,1.257,0.645,2.409,1.724,3.082c0.116,0.073,0.181,0.205,0.168,0.341\tc-0.014,0.136-0.104,0.253-0.232,0.301c-2.189,0.822-3.66,2.942-3.66,5.275C10.36,15.999,10.36,16.64,10.36,16.64z"
  })), children);
});

export { HelpDesk as default };
