// @carbon/pictograms-react/es/chart--high-low/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const ChartHighLow = /*#__PURE__*/React.forwardRef(function ChartHighLow(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M31,31.36H1c-0.199,0-0.36-0.161-0.36-0.36V1h0.72v29.64H31V31.36z M24.36,26h-0.72v-2.64H21v-0.72\th2.64V7h0.721v1.64H27v0.72h-2.64V26z M16.36,26h-0.72v-6.64H13v-0.72h2.64V7h0.72v5.64H19v0.72h-2.64V26z M8.36,26H7.64v-4.64H5\tv-0.72h2.64V7h0.72v3.64H11v0.72H8.36V26z"
  })), children);
});

export { ChartHighLow as default };
