// @carbon/pictograms-react/lib/envelope/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Envelope = /*#__PURE__*/React__default["default"].forwardRef(function Envelope(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31,4.64H1C0.801,4.64,0.64,4.801,0.64,5v22c0,0.199,0.161,0.36,0.36,0.36h30\tc0.199,0,0.36-0.161,0.36-0.36V5C31.36,4.801,31.199,4.64,31,4.64z M30.64,26.181L19.638,16.097L30.64,5.829V26.181z M15.754,19.263\tc0.139,0.131,0.353,0.131,0.491,0l2.865-2.674L30.074,26.64H1.926L12.89,16.59L15.754,19.263z M30.087,5.36L16,18.508L1.913,5.36\tH30.087z M12.362,16.097L1.36,26.182V5.829L12.362,16.097z"
  })), children);
});

module.exports = Envelope;
