// @carbon/pictograms-react/lib/windows-hosting/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const WindowsHosting = /*#__PURE__*/React__default["default"].forwardRef(function WindowsHosting(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M30,30.36H8c-0.199,0-0.36-0.161-0.36-0.36\tV19.36H2c-0.199,0-0.36-0.161-0.36-0.36V2c0-0.199,0.161-0.36,0.36-0.36h22c0.199,0,0.36,0.161,0.36,0.36v10.64H30\tc0.199,0,0.36,0.161,0.36,0.36v17C30.36,30.199,30.199,30.36,30,30.36z M8.36,29.64h21.28V13.36h-5.28v4.329\tc0.575,0.158,1,0.687,1,1.312c0,0.75-0.61,1.36-1.36,1.36s-1.36-0.61-1.36-1.36c0-0.625,0.425-1.153,1-1.312V13.36H12v-0.72h11.64\tV2.36H2.36v16.28h5.28v-4.328c-0.576-0.158-1-0.686-1-1.312c0-0.75,0.61-1.36,1.36-1.36S9.36,12.25,9.36,13\tc0,0.625-0.424,1.153-1,1.312v4.328H20v0.721H8.36V29.64z M24,18.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64\ts0.64-0.287,0.64-0.64S24.353,18.36,24,18.36z M8,12.36c-0.353,0-0.64,0.287-0.64,0.64S7.647,13.64,8,13.64\tc0.353,0,0.64-0.287,0.64-0.64S8.353,12.36,8,12.36z"
  })), children);
});

module.exports = WindowsHosting;
