// @carbon/pictograms-react/lib/partnership/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Partnership = /*#__PURE__*/React__default["default"].forwardRef(function Partnership(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,30.36C8.082,30.36,1.64,23.918,1.64,16\tS8.082,1.64,16,1.64S30.36,8.082,30.36,16S23.918,30.36,16,30.36z M2.36,16c0,7.521,6.119,13.64,13.64,13.64\tc5.452,0,10.167-3.215,12.349-7.849C26.634,23.963,23.977,25.36,21,25.36c-4.679,0-8.608-3.429-9.262-8.025\tC11.113,17.212,10.64,16.66,10.64,16c0-0.75,0.61-1.36,1.36-1.36s1.36,0.61,1.36,1.36c0,0.589-0.377,1.093-0.902,1.281\tc0.625,4.218,4.24,7.358,8.542,7.358c4.764,0,8.64-3.876,8.64-8.64c0-7.521-6.118-13.64-13.64-13.64\tc-5.452,0-10.167,3.215-12.349,7.85C5.367,8.037,8.023,6.64,11,6.64c4.679,0,8.607,3.429,9.263,8.025\tC20.888,14.788,21.36,15.34,21.36,16c0,0.75-0.61,1.36-1.36,1.36s-1.36-0.61-1.36-1.36c0-0.589,0.378-1.093,0.903-1.281\tC18.918,10.5,15.303,7.36,11,7.36C6.236,7.36,2.36,11.236,2.36,16z M20,15.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64\ts0.64-0.287,0.64-0.64S20.353,15.36,20,15.36z M12,15.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64\tS12.353,15.36,12,15.36z"
  })), children);
});

module.exports = Partnership;
