// @carbon/pictograms-react/lib/apartment/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Apartment = /*#__PURE__*/React__default["default"].forwardRef(function Apartment(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M30,31.36H2c-0.199,0-0.36-0.161-0.36-0.36V13c0-0.199,0.161-0.36,0.36-0.36h11.64V1\tc0-0.199,0.161-0.36,0.36-0.36h16c0.199,0,0.36,0.161,0.36,0.36v30C30.36,31.199,30.199,31.36,30,31.36z M14.36,30.64h15.28V1.36\tH14.36V13c0,0.199-0.161,0.36-0.36,0.36H2.36v17.28h7.28V23c0-0.199,0.161-0.36,0.36-0.36h4c0.199,0,0.36,0.161,0.36,0.36V30.64z M10.36,30.64h3.28v-7.28h-3.28V30.64z M26.36,25h-0.72v-2h0.721L26.36,25L26.36,25z M22.36,25h-0.72v-2h0.721L22.36,25L22.36,25z M18.36,25h-0.72v-2h0.721L18.36,25L18.36,25z M6.36,25H5.64v-2h0.72C6.36,23,6.36,25,6.36,25z M26.36,19h-0.72v-2h0.721L26.36,19\tL26.36,19z M22.36,19h-0.72v-2h0.721L22.36,19L22.36,19z M18.36,19h-0.72v-2h0.721L18.36,19L18.36,19z M14.36,19h-0.72v-2h0.72V19z M10.36,19H9.64v-2h0.72C10.36,17,10.36,19,10.36,19z M6.36,19H5.64v-2h0.72C6.36,17,6.36,19,6.36,19z M26.36,13h-0.72v-2h0.721\tL26.36,13L26.36,13z M22.36,13h-0.72v-2h0.721L22.36,13L22.36,13z M18.36,13h-0.72v-2h0.721L18.36,13L18.36,13z M26.36,7h-0.72V5\th0.721L26.36,7L26.36,7z M22.36,7h-0.72V5h0.721L22.36,7L22.36,7z M18.36,7h-0.72V5h0.721L18.36,7L18.36,7z"
  })), children);
});

module.exports = Apartment;
