// @carbon/pictograms-react/es/face--very--satisfied/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const FaceVerySatisfied = /*#__PURE__*/React.forwardRef(function FaceVerySatisfied(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M10.75,13c0,0.414-0.336,0.75-0.75,0.75S9.25,13.414,9.25,13s0.336-0.75,0.75-0.75\tS10.75,12.586,10.75,13z M22,12.25c-0.414,0-0.75,0.336-0.75,0.75s0.336,0.75,0.75,0.75s0.75-0.336,0.75-0.75S22.414,12.25,22,12.25\tz M16,31.36C7.53,31.36,0.64,24.47,0.64,16S7.53,0.64,16,0.64S31.36,7.53,31.36,16S24.47,31.36,16,31.36z M16,1.36\tC7.927,1.36,1.36,7.927,1.36,16c0,8.072,6.567,14.64,14.64,14.64c8.072,0,14.64-6.567,14.64-14.64S24.072,1.36,16,1.36z M16,24.36\tc-3.507,0-6.36-2.854-6.36-6.36h0.72c0,3.11,2.53,5.64,5.64,5.64s5.64-2.529,5.64-5.64h0.721C22.36,21.507,19.507,24.36,16,24.36z"
  })), children);
});

export { FaceVerySatisfied as default };
