// @carbon/pictograms-react/lib/video--chat/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const VideoChat = /*#__PURE__*/React__default["default"].forwardRef(function VideoChat(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M29,2.5C29,2.776,28.776,3,28.5,3S28,2.776,28,2.5S28.224,2,28.5,2S29,2.224,29,2.5z M26.5,2\tC26.224,2,26,2.224,26,2.5S26.224,3,26.5,3S27,2.776,27,2.5S26.776,2,26.5,2z M24.5,2C24.224,2,24,2.224,24,2.5S24.224,3,24.5,3\tS25,2.776,25,2.5S24.776,2,24.5,2z M31,31.36H1c-0.199,0-0.36-0.161-0.36-0.36V1c0-0.199,0.161-0.36,0.36-0.36h30\tc0.199,0,0.36,0.161,0.36,0.36v30C31.36,31.199,31.199,31.36,31,31.36z M27.36,30.64h3.279V4.36H1.36v26.28h3.28V30\tc0-4.712,3.286-8.985,7.932-10.478c-1.854-1.168-3.01-3.218-3.01-5.445c0-3.55,2.888-6.438,6.437-6.438\tc3.55,0,6.438,2.888,6.438,6.438c0,2.228-1.156,4.277-3.01,5.445C24.073,21.015,27.36,25.289,27.36,30V30.64z M5.359,30.64h21.28V30\tc0-4.673-3.473-8.888-8.257-10.021c-0.15-0.036-0.261-0.163-0.275-0.317c-0.015-0.153,0.071-0.299,0.212-0.362\tc2.063-0.918,3.396-2.968,3.396-5.222c0-3.153-2.564-5.718-5.717-5.718s-5.717,2.565-5.717,5.718c0,2.254,1.333,4.304,3.397,5.222\tc0.141,0.063,0.226,0.209,0.212,0.362c-0.014,0.154-0.125,0.281-0.275,0.317C8.832,21.112,5.359,25.326,5.359,30V30.64z M1.36,3.64\th29.28V1.36H1.36V3.64z"
  })), children);
});

module.exports = VideoChat;
