// @carbon/pictograms-react/es/cics--transaction-gateway/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const CicsTransactionGateway = /*#__PURE__*/React.forwardRef(function CicsTransactionGateway(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M28,30.36\tc-1.302,0-2.36-1.059-2.36-2.36s1.059-2.36,2.36-2.36s2.36,1.059,2.36,2.36S29.302,30.36,28,30.36z M28,26.36\tc-0.904,0-1.64,0.735-1.64,1.64s0.735,1.64,1.64,1.64s1.64-0.735,1.64-1.64S28.904,26.36,28,26.36z M23,28.36H8\tc-3.507,0-6.36-2.854-6.36-6.36S4.493,15.64,8,15.64v0.72c-3.11,0-5.64,2.53-5.64,5.64S4.89,27.64,8,27.64h15V28.36z M17,22.36h-4\tv-0.72h4c2.559,0,4.64-2.081,4.64-4.64v-2h0.721v2C22.36,19.955,19.955,22.36,17,22.36z M16,18.36c-1.301,0-2.36-1.059-2.36-2.36\ts1.059-2.36,2.36-2.36c1.302,0,2.36,1.059,2.36,2.36C18.36,17.302,17.302,18.36,16,18.36z M16,14.36c-0.904,0-1.64,0.736-1.64,1.64\tc0,0.904,0.736,1.64,1.64,1.64c0.904,0,1.64-0.735,1.64-1.64S16.904,14.36,16,14.36z M10.36,17H9.64v-2\tc0-2.956,2.404-5.36,5.36-5.36h4v0.72h-4c-2.559,0-4.64,2.082-4.64,4.64C10.36,15,10.36,17,10.36,17z M24,16.36v-0.72\tc3.11,0,5.64-2.53,5.64-5.64S27.11,4.36,24,4.36H9V3.64h15c3.507,0,6.36,2.853,6.36,6.36S27.507,16.36,24,16.36z M4,6.36\tC2.699,6.36,1.64,5.301,1.64,4S2.699,1.64,4,1.64S6.36,2.699,6.36,4S5.301,6.36,4,6.36z M4,2.36C3.096,2.36,2.36,3.096,2.36,4\tS3.096,5.64,4,5.64S5.64,4.904,5.64,4S4.904,2.36,4,2.36z"
  })), children);
});

export { CicsTransactionGateway as default };
