// @carbon/pictograms-react/lib/expand--horz/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const ExpandHorz = /*#__PURE__*/React__default["default"].forwardRef(function ExpandHorz(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31,1.37c-0.199,0-0.36-0.161-0.36-0.36S30.801,0.645,31,0.645S31.36,0.801,31.36,1v0.01\tC31.36,1.208,31.199,1.37,31,1.37z M31.36,29.01v-0.02c0-0.199-0.161-0.351-0.36-0.351s-0.36,0.171-0.36,0.37s0.161,0.36,0.36,0.36\tS31.36,29.209,31.36,29.01z M31.36,27.01v-0.02c0-0.199-0.161-0.351-0.36-0.351s-0.36,0.171-0.36,0.37s0.161,0.36,0.36,0.36\tS31.36,27.209,31.36,27.01z M31.36,25.01v-0.02c0-0.199-0.161-0.351-0.36-0.351s-0.36,0.171-0.36,0.37s0.161,0.36,0.36,0.36\tS31.36,25.209,31.36,25.01z M31.36,23.01v-0.02c0-0.199-0.161-0.351-0.36-0.351s-0.36,0.171-0.36,0.37s0.161,0.36,0.36,0.36\tS31.36,23.209,31.36,23.01z M31.36,21.01v-0.02c0-0.199-0.161-0.351-0.36-0.351s-0.36,0.171-0.36,0.37s0.161,0.36,0.36,0.36\tS31.36,21.209,31.36,21.01z M31.36,19.01v-0.02c0-0.199-0.161-0.351-0.36-0.351s-0.36,0.171-0.36,0.37s0.161,0.36,0.36,0.36\tS31.36,19.209,31.36,19.01z M31.36,17.01v-0.02c0-0.199-0.161-0.35-0.36-0.35s-0.36,0.171-0.36,0.37c0,0.199,0.161,0.36,0.36,0.36\tS31.36,17.209,31.36,17.01z M31.36,15.01v-0.02c0-0.199-0.161-0.35-0.36-0.35s-0.36,0.171-0.36,0.37s0.161,0.36,0.36,0.36\tS31.36,15.208,31.36,15.01z M31.36,13.01v-0.02c0-0.199-0.161-0.35-0.36-0.35s-0.36,0.171-0.36,0.37s0.161,0.36,0.36,0.36\tS31.36,13.208,31.36,13.01z M31.36,11.01v-0.02c0-0.199-0.161-0.35-0.36-0.35s-0.36,0.171-0.36,0.37s0.161,0.36,0.36,0.36\tS31.36,11.208,31.36,11.01z M31.36,9.01V8.99c0-0.199-0.161-0.35-0.36-0.35s-0.36,0.171-0.36,0.37S30.801,9.37,31,9.37\tS31.36,9.208,31.36,9.01z M31.36,7.01V6.99c0-0.199-0.161-0.35-0.36-0.35s-0.36,0.171-0.36,0.37S30.801,7.37,31,7.37\tS31.36,7.208,31.36,7.01z M31.36,5.01V4.99c0-0.199-0.161-0.35-0.36-0.35s-0.36,0.171-0.36,0.37S30.801,5.37,31,5.37\tS31.36,5.209,31.36,5.01z M31.36,3.01V2.99c0-0.199-0.161-0.35-0.36-0.35s-0.36,0.171-0.36,0.37S30.801,3.37,31,3.37\tS31.36,3.209,31.36,3.01z M31.36,31v-0.01c0-0.199-0.161-0.355-0.36-0.355S30.64,30.801,30.64,31s0.161,0.36,0.36,0.36\tS31.36,31.199,31.36,31z M1.36,1H0.64v30h0.72V1z M28.509,16l-3.254-3.254l-0.51,0.509l2.386,2.386H4v0.72h23.131l-2.386,2.385\tl0.51,0.51L28.509,16z"
  })), children);
});

module.exports = ExpandHorz;
