// @carbon/pictograms-react/es/networking--02/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Networking_02 = /*#__PURE__*/React.forwardRef(function Networking_02(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M28,31.36c-1.731,0-3.161-1.316-3.341-3H16c-0.199,0-0.36-0.161-0.36-0.36V16.36H7.341\tc-0.18,1.684-1.61,3-3.341,3c-1.853,0-3.36-1.508-3.36-3.36S2.147,12.64,4,12.64c1.731,0,3.161,1.316,3.341,3h8.299V4\tc0-0.199,0.161-0.36,0.36-0.36h8.659c0.18-1.684,1.609-3,3.341-3c1.853,0,3.36,1.507,3.36,3.36S29.853,7.36,28,7.36\tc-1.731,0-3.161-1.316-3.341-3H16.36v11.28h8.299c0.18-1.684,1.609-3,3.341-3c1.853,0,3.36,1.507,3.36,3.36s-1.508,3.36-3.36,3.36\tc-1.731,0-3.161-1.316-3.341-3H16.36v11.28h8.299c0.18-1.684,1.609-3,3.341-3c1.853,0,3.36,1.508,3.36,3.36S29.853,31.36,28,31.36z M28,25.36c-1.456,0-2.64,1.184-2.64,2.64s1.184,2.64,2.64,2.64s2.64-1.184,2.64-2.64S29.456,25.36,28,25.36z M28,13.36\tc-1.456,0-2.64,1.185-2.64,2.64c0,1.456,1.184,2.64,2.64,2.64s2.64-1.184,2.64-2.64S29.456,13.36,28,13.36z M4,13.36\tc-1.456,0-2.64,1.185-2.64,2.64c0,1.456,1.185,2.64,2.64,2.64S6.64,17.456,6.64,16S5.456,13.36,4,13.36z M28,1.36\tc-1.456,0-2.64,1.185-2.64,2.64S26.544,6.64,28,6.64S30.64,5.456,30.64,4S29.456,1.36,28,1.36z"
  })), children);
});

export { Networking_02 as default };
