// @carbon/pictograms-react/es/credit--card/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const CreditCard = /*#__PURE__*/React.forwardRef(function CreditCard(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M29,26.36H3c-1.301,0-2.36-1.059-2.36-2.36V8c0-1.301,1.059-2.36,2.36-2.36h26\tc1.302,0,2.36,1.059,2.36,2.36v16C31.36,25.302,30.302,26.36,29,26.36z M1.36,13.36V24c0,0.904,0.736,1.64,1.64,1.64h26\tc0.904,0,1.64-0.735,1.64-1.64V13.36H1.36z M1.36,12.64h29.28v-2.28H1.36V12.64z M1.36,9.64h29.28V8c0-0.904-0.735-1.64-1.64-1.64H3\tC2.096,6.36,1.36,7.096,1.36,8V9.64z"
  })), children);
});

export { CreditCard as default };
