// @carbon/pictograms-react/es/hyper-protect--virtual-servers/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const HyperProtectVirtualServers = /*#__PURE__*/React.forwardRef(function HyperProtectVirtualServers(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M16,31.36\tC8.082,31.36,1.64,24.918,1.64,17V1c0-0.199,0.161-0.36,0.36-0.36h28c0.199,0,0.36,0.161,0.36,0.36v16\tC30.36,24.918,23.918,31.36,16,31.36z M2.36,1.36V17c0,7.521,6.119,13.64,13.64,13.64S29.64,24.522,29.64,17V1.36H2.36z M25,20.36H7\tc-0.199,0-0.36-0.161-0.36-0.36v-6c0-0.199,0.161-0.36,0.36-0.36h13.64v-1.28H7c-0.199,0-0.36-0.161-0.36-0.36V6\tc0-0.199,0.161-0.36,0.36-0.36h18c0.199,0,0.36,0.161,0.36,0.36v6c0,0.199-0.161,0.36-0.36,0.36h-3.64v1.28H25\tc0.199,0,0.36,0.161,0.36,0.36v6C25.36,20.199,25.199,20.36,25,20.36z M7.36,19.64h17.28v-5.28h-3.28v1.329\tc0.575,0.158,1,0.686,1,1.312c0,0.75-0.61,1.36-1.36,1.36s-1.36-0.61-1.36-1.36c0-0.625,0.425-1.153,1-1.312V14.36H7.36V19.64z M21,16.36c-0.353,0-0.64,0.287-0.64,0.64c0,0.353,0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64C21.64,16.647,21.353,16.36,21,16.36z M21.36,11.64h3.279V6.36H7.36v5.28h13.28v-1.329c-0.575-0.158-1-0.686-1-1.312c0-0.75,0.61-1.36,1.36-1.36S22.36,8.25,22.36,9\tc0,0.625-0.425,1.153-1,1.312V11.64z M21,8.36c-0.353,0-0.64,0.287-0.64,0.64S20.647,9.64,21,9.64S21.64,9.353,21.64,9\tS21.353,8.36,21,8.36z"
  })), children);
});

export { HyperProtectVirtualServers as default };
