// @carbon/pictograms-react/es/cloud--native--01/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const CloudNative_01 = /*#__PURE__*/React.forwardRef(function CloudNative_01(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M27,28.36H17c-0.199,0-0.36-0.161-0.36-0.36\tv-3.64H13c-0.199,0-0.36-0.161-0.36-0.36v-9c0-0.199,0.161-0.36,0.36-0.36h10c0.199,0,0.36,0.161,0.36,0.36v3.64H27\tc0.199,0,0.36,0.161,0.36,0.36v1.465c1.873-0.885,3.279-3.218,3.279-5.558c0-2.863-2.233-5.259-5.084-5.454\tC25.373,9.44,25.23,9.294,25.22,9.112c-0.197-3.787-3.322-6.753-7.113-6.753c-2.753,0-5.288,1.614-6.457,4.111\tc-0.082,0.174-0.288,0.254-0.463,0.181C10.323,6.296,9.412,6.116,8.48,6.116c-3.927,0-7.122,3.2-7.122,7.133\tc0,3.684,2.634,6.93,5.996,7.389L11,20.635v0.721H7.307C3.485,20.837,0.64,17.354,0.64,13.25c0-4.33,3.518-7.853,7.841-7.853\tc0.914,0,1.809,0.157,2.667,0.468c1.345-2.577,4.04-4.225,6.959-4.225c4.07,0,7.442,3.105,7.81,7.125\tc3.077,0.368,5.443,3.008,5.443,6.143c0,2.727-1.738,5.452-4,6.341V28C27.36,28.199,27.199,28.36,27,28.36z M17.36,27.64h9.279\tv-8.28H17.36V27.64z M13.36,23.64h3.28V19c0-0.199,0.161-0.36,0.36-0.36h5.64v-3.28h-9.28V23.64z M21,21.36h-2v-0.72h2V21.36z M17,17.36h-2v-0.72h2V17.36z"
  })), children);
});

export { CloudNative_01 as default };
