// @carbon/pictograms-react/lib/systems-devops--test/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const SystemsDevopsTest = /*#__PURE__*/React__default["default"].forwardRef(function SystemsDevopsTest(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M23.25,30.36c-3.921,0-7.11-3.189-7.11-7.11s3.189-7.11,7.11-7.11s7.11,3.189,7.11,7.11\tS27.171,30.36,23.25,30.36z M23.25,16.86c-3.523,0-6.39,2.867-6.39,6.39s2.867,6.39,6.39,6.39s6.39-2.866,6.39-6.39\tS26.773,16.86,23.25,16.86z M15.819,30.311l-12-7C3.708,23.246,3.64,23.128,3.64,23V9h0.72v13.793l11.821,6.896L15.819,30.311z M23,26.36c-0.092,0-0.185-0.035-0.255-0.105L20.189,23.7l0.51-0.51L23,25.491l4.002-4.004l0.51,0.51l-4.257,4.258\tC23.185,26.325,23.092,26.36,23,26.36z M8.36,16H7.64V6h0.72V16z M28.36,15h-0.72V9.207L16,2.417l-3.64,2.116V13h-0.72V4.325\tc0-0.128,0.068-0.247,0.179-0.311l4-2.325c0.112-0.065,0.251-0.065,0.362,0l12,7C28.292,8.753,28.36,8.872,28.36,9V15z"
  })), children);
});

module.exports = SystemsDevopsTest;
