// @carbon/pictograms-react/lib/hills/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Hills = /*#__PURE__*/React__default["default"].forwardRef(function Hills(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M30.192,29.64L16.36,8.891V5.36h4.509l-1.86-1.86l1.86-1.86H15.64v7.251L1.807,29.64H1v0.721h30V29.64H30.192\tz M16.36,2.36h2.771l-1.14,1.14l1.14,1.14H16.36V2.36z M2.673,29.64L8,21.649l5.327,7.99L2.673,29.64L2.673,29.64z M14.193,29.64\tL8.433,21L12,15.649l9.327,13.991H14.193z M22.192,29.64L12.433,15L16,9.649L29.327,29.64H22.192z"
  })), children);
});

module.exports = Hills;
