// @carbon/pictograms-react/es/silence/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Silence = /*#__PURE__*/React.forwardRef(function Silence(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M28.36,31h-0.72v-6.75c0-0.491-0.398-0.89-0.89-0.89\ts-0.89,0.398-0.89,0.89V28h-0.72v-4.75c0-0.491-0.398-0.89-0.89-0.89s-0.89,0.398-0.89,0.89V28h-0.72v-4.75\tc0-0.491-0.398-0.89-0.89-0.89s-0.89,0.398-0.89,0.89V28h-0.72v-8.75c0-0.491-0.398-0.89-0.89-0.89s-0.89,0.398-0.89,0.89V31h-0.72\tV19.25c0-0.888,0.723-1.61,1.61-1.61s1.61,0.723,1.61,1.61v2.659c0.255-0.17,0.561-0.27,0.89-0.27c0.504,0,0.954,0.233,1.25,0.597\tc0.296-0.363,0.746-0.597,1.25-0.597c0.778,0,1.43,0.556,1.578,1.291c0.262-0.183,0.579-0.291,0.922-0.291\tc0.888,0,1.61,0.723,1.61,1.61V31z M5.36,31H4.64v-2.999c0-0.199,0.161-0.36,0.36-0.36h4.999c1.456,0,2.641-1.185,2.641-2.641v-1\tc0-0.056,0.013-0.111,0.038-0.161L13.098,23l-0.419-0.839c-0.051-0.102-0.051-0.221,0-0.322L13.098,21l-0.419-0.839\tc-0.025-0.05-0.038-0.105-0.038-0.161v-1c0-0.199,0.161-0.36,0.36-0.36h1.601c0.374,0,0.709-0.193,0.897-0.517\tc0.187-0.321,0.188-0.706,0.004-1.027l-2.815-4.917C12.656,12.125,12.64,12.063,12.64,12v-2c0-4.764-3.876-8.64-8.64-8.64V0.64\tc5.161,0,9.36,4.199,9.36,9.36v1.904l2.768,4.834c0.312,0.548,0.31,1.2-0.007,1.746c-0.318,0.548-0.887,0.876-1.52,0.876H13.36\tv0.555l0.462,0.924c0.051,0.102,0.051,0.221,0,0.322L13.402,22l0.419,0.839c0.051,0.102,0.051,0.221,0,0.322l-0.462,0.924V25\tc0,1.854-1.508,3.361-3.361,3.361H5.36V31z"
  })), children);
});

export { Silence as default };
