// @carbon/pictograms-react/es/graphic-intensive-workloads/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const GraphicIntensiveWorkloads = /*#__PURE__*/React.forwardRef(function GraphicIntensiveWorkloads(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M24,31.36h-2\tc-0.199,0-0.36-0.161-0.36-0.36v-0.64H3.36V31c0,0.199-0.161,0.36-0.36,0.36H1c-0.199,0-0.36-0.161-0.36-0.36v-2\tc0-0.199,0.161-0.36,0.36-0.36h0.64V3.36H1C0.801,3.36,0.64,3.199,0.64,3V1c0-0.199,0.161-0.36,0.36-0.36h2\tc0.199,0,0.36,0.161,0.36,0.36v0.64H30c0.199,0,0.36,0.161,0.36,0.36v28c0,0.199-0.161,0.36-0.36,0.36h-5.64V31\tC24.36,31.199,24.199,31.36,24,31.36z M22.36,30.64h1.279v-1.28H22.36V30.64z M1.36,30.64h1.28v-1.28H1.36V30.64z M24.36,29.64\th5.279V2.36H3.36V3c0,0.199-0.161,0.36-0.36,0.36H2.36v5.28h19.28V8c0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2\tc0,0.199-0.161,0.36-0.36,0.36h-0.64v18.28H24c0.199,0,0.36,0.161,0.36,0.36V29.64z M16.36,29.64h5.28V29\tc0-0.199,0.161-0.36,0.36-0.36h0.64V10.36H22c-0.199,0-0.36-0.161-0.36-0.36V9.36H2.36v6.28H16c0.199,0,0.36,0.161,0.36,0.36V29.64z M9.36,29.64h6.28V16.36H2.36v6.28H9c0.199,0,0.36,0.161,0.36,0.36V29.64z M3.36,29.64h5.28v-6.28H2.36v5.279H3\tc0.199,0,0.36,0.161,0.36,0.36C3.36,28.999,3.36,29.64,3.36,29.64z M22.36,9.64h1.279V8.36H22.36V9.64z M1.36,2.64h1.28V1.36H1.36\tV2.64z"
  })), children);
});

export { GraphicIntensiveWorkloads as default };
