// @carbon/pictograms-react/lib/receipt/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Receipt = /*#__PURE__*/React__default["default"].forwardRef(function Receipt(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M26,31.36H7c-0.199,0-0.36-0.161-0.36-0.36V1c0-0.199,0.161-0.36,0.36-0.36h19\tc0.199,0,0.36,0.161,0.36,0.36v30C26.36,31.199,26.199,31.36,26,31.36z M7.36,30.64h18.28V1.36H7.36V30.64z M23,26.36h-2v-0.72h2\tV26.36z M23,22.36H10v-0.72h13V22.36z M23,18.36h-2v-0.72h2V18.36z M17,18.36h-7v-0.72h7V18.36z M23,14.36h-2v-0.72h2V14.36z M17,14.36h-7v-0.72h7V14.36z M23,10.36h-2V9.64h2V10.36z M17,10.36h-7V9.64h7V10.36z M23,6.36h-2V5.64h2V6.36z M17,6.36h-7V5.64h7\tV6.36z"
  })), children);
});

module.exports = Receipt;
