// @carbon/pictograms-react/lib/refresh/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Refresh = /*#__PURE__*/React__default["default"].forwardRef(function Refresh(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M28,31.36H4c-1.301,0-2.36-1.059-2.36-2.36V16h0.72v13c0,0.904,0.736,1.64,1.64,1.64h24 c0.904,0,1.64-0.735,1.64-1.64V9c0-0.904-0.735-1.64-1.64-1.64H7.869l5.385,5.386l-0.509,0.509L6.491,7l6.255-6.254l0.509,0.509 L7.869,6.64H28c1.302,0,2.36,1.059,2.36,2.36v20C30.36,30.302,29.302,31.36,28,31.36z"
  })), children);
});

module.exports = Refresh;
