// @carbon/pictograms-react/es/connected--devices/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const ConnectedDevices = /*#__PURE__*/React.forwardRef(function ConnectedDevices(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M16,30.36c-7.381,0-13.59-5.684-14.281-13H1\tc-0.199,0-0.36-0.161-0.36-0.36v-2c0-0.199,0.161-0.36,0.36-0.36h0.719c0.691-7.317,6.9-13,14.281-13c7.382,0,13.59,5.683,14.28,13\tH31c0.199,0,0.36,0.161,0.36,0.36v2c0,0.199-0.161,0.36-0.36,0.36h-0.72C29.59,24.677,23.382,30.36,16,30.36z M3.094,20.37\tc1.835,5.402,6.972,9.27,12.906,9.27c6.988,0,12.869-5.363,13.558-12.279H29c-0.199,0-0.36-0.161-0.36-0.36v-2\tc0-0.199,0.161-0.36,0.36-0.36h0.558C28.869,7.723,22.988,2.36,16,2.36c-5.934,0-11.071,3.868-12.906,9.271\tC4.447,9.809,6.615,8.64,9,8.64c1.256,0,2.452,0.324,3.5,0.897C13.548,8.964,14.744,8.64,16,8.64c3.527,0,6.58,2.558,7.227,6H24\tc0.199,0,0.36,0.161,0.36,0.36v2c0,0.199-0.161,0.36-0.36,0.36h-0.773c-0.646,3.441-3.699,6-7.227,6c-1.256,0-2.452-0.324-3.5-0.897\tc-1.048,0.573-2.244,0.897-3.5,0.897C6.615,23.36,4.447,22.19,3.094,20.37z M13.213,22.019c0.853,0.399,1.799,0.621,2.787,0.621\tc3.136,0,5.856-2.242,6.492-5.279H22c-0.199,0-0.36-0.161-0.36-0.36v-2c0-0.199,0.161-0.36,0.36-0.36h0.492\tC21.856,11.602,19.136,9.36,16,9.36c-0.987,0-1.934,0.222-2.786,0.622c1.533,1.079,2.65,2.73,3.012,4.659H17\tc0.199,0,0.36,0.161,0.36,0.36v2c0,0.199-0.161,0.36-0.36,0.36h-0.774C15.864,19.289,14.747,20.939,13.213,22.019z M2.508,17.36\tC3.143,20.397,5.863,22.64,9,22.64c0.987,0,1.934-0.222,2.787-0.621c-1.534-1.079-2.65-2.729-3.012-4.658H8\tc-0.199,0-0.36-0.161-0.36-0.36v-2c0-0.199,0.161-0.36,0.36-0.36h0.774c0.361-1.929,1.479-3.58,3.012-4.659\tC10.934,9.582,9.987,9.36,9,9.36c-3.137,0-5.857,2.242-6.492,5.28H3c0.199,0,0.36,0.161,0.36,0.36v2c0,0.199-0.161,0.36-0.36,0.36\tH2.508z M9.508,17.36C9.886,19.165,11,20.69,12.5,21.63c1.5-0.939,2.614-2.465,2.992-4.27H15c-0.199,0-0.36-0.161-0.36-0.36v-2\tc0-0.199,0.161-0.36,0.36-0.36h0.492C15.114,12.834,14,11.31,12.5,10.37c-1.5,0.939-2.614,2.464-2.992,4.27H10\tc0.199,0,0.36,0.161,0.36,0.36v2c0,0.199-0.161,0.36-0.36,0.36H9.508z M29.36,16.64h1.279v-1.28H29.36V16.64z M22.36,16.64h1.279\tv-1.28H22.36V16.64z M15.36,16.64h1.28v-1.28h-1.28V16.64z M8.36,16.64h1.28v-1.28H8.36V16.64z M1.36,16.64h1.28v-1.28H1.36V16.64z"
  })), children);
});

export { ConnectedDevices as default };
