// @carbon/pictograms-react/lib/population--diagram/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const PopulationDiagram = /*#__PURE__*/React__default["default"].forwardRef(function PopulationDiagram(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16.36,31h-0.72v-2.64H1c-0.199,0-0.36-0.161-0.36-0.36v-4c0-0.199,0.161-0.36,0.36-0.36h14.64\tv-2.28H4c-0.199,0-0.36-0.161-0.36-0.36v-4c0-0.199,0.161-0.36,0.36-0.36h11.64v-2.28H7c-0.199,0-0.36-0.161-0.36-0.36v-4\tc0-0.199,0.161-0.36,0.36-0.36h8.64V7.36H10C9.801,7.36,9.64,7.199,9.64,7V3c0-0.199,0.161-0.36,0.36-0.36h5.64V1h0.72v1.64H22\tc0.199,0,0.36,0.161,0.36,0.36v4c0,0.199-0.161,0.36-0.36,0.36h-5.64v2.28H25c0.199,0,0.36,0.161,0.36,0.36v4\tc0,0.199-0.161,0.36-0.36,0.36h-8.64v2.28H28c0.199,0,0.36,0.161,0.36,0.36v4c0,0.199-0.161,0.36-0.36,0.36H16.36v2.279H31\tc0.199,0,0.36,0.161,0.36,0.36v4c0,0.199-0.161,0.36-0.36,0.36H16.36V31z M16.36,27.64h14.28v-3.28H16.36V27.64z M1.36,27.64h14.28\tv-3.28H1.36V27.64z M16.36,20.64h11.28v-3.28H16.36V20.64z M4.36,20.64h11.28v-3.28H4.36V20.64z M16.36,13.64h8.28v-3.28h-8.28\tV13.64z M7.36,13.64h8.28v-3.28H7.36V13.64z M16.36,6.64h5.28V3.36h-5.28V6.64z M10.36,6.64h5.28V3.36h-5.28\tC10.36,3.36,10.36,6.64,10.36,6.64z"
  })), children);
});

module.exports = PopulationDiagram;
