// @carbon/pictograms-react/es/station--hydration/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const StationHydration = /*#__PURE__*/React.forwardRef(function StationHydration(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M21.122,31.36H10.878\tc-0.681,0-1.261-0.51-1.349-1.185L6.643,8.046l0.714-0.093l1.167,8.946c0.13,0.268,0.965,1.74,3.476,1.74\tc1.4,0,2.573-0.703,3.814-1.448c1.271-0.763,2.585-1.551,4.185-1.551c1.896,0,2.977,0.384,3.549,0.706l1.095-8.393l0.715,0.093\tl-2.887,22.129C22.383,30.851,21.804,31.36,21.122,31.36z M8.701,18.255l1.542,11.827c0.042,0.318,0.314,0.558,0.635,0.558h10.244\tc0.32,0,0.593-0.239,0.634-0.557l1.688-12.943c-0.291-0.222-1.242-0.78-3.445-0.78c-1.4,0-2.572,0.704-3.814,1.449\tC14.914,18.571,13.6,19.36,12,19.36C10.37,19.36,9.332,18.809,8.701,18.255z M15.998,11.36c-1.853,0-3.36-1.507-3.36-3.36\tc0-0.486,0.102-0.955,0.302-1.395l2.735-5.76C15.734,0.72,15.861,0.64,16,0.64l0,0c0.139,0,0.266,0.08,0.325,0.206l2.728,5.755\tc0.001,0.001,0.001,0.003,0.002,0.004C19.256,7.044,19.358,7.514,19.358,8C19.358,9.853,17.851,11.36,15.998,11.36z M16,1.84\tl-2.407,5.069C13.438,7.25,13.357,7.618,13.357,8c0,1.456,1.185,2.64,2.64,2.64s2.64-1.185,2.64-2.64\tc0-0.381-0.079-0.749-0.236-1.093L16,1.84z"
  })), children);
});

export { StationHydration as default };
