// @carbon/pictograms-react/lib/calendar/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Calendar = /*#__PURE__*/React__default["default"].forwardRef(function Calendar(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M30,31.36H2c-0.199,0-0.36-0.161-0.36-0.36V4c0-0.199,0.161-0.36,0.36-0.36h5.64V1h0.72v2.64h7.28V1h0.72\tv2.64h7.28V1h0.721v2.64H30c0.199,0,0.36,0.161,0.36,0.36v27C30.36,31.199,30.199,31.36,30,31.36z M2.36,30.64h27.28V4.36h-5.28V7\th-0.72V4.36h-7.28V7h-0.72V4.36H8.36V7H7.64V4.36H2.36V30.64z M25,27.36h-2c-0.199,0-0.36-0.161-0.36-0.36v-2\tc0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2C25.36,27.199,25.199,27.36,25,27.36z M23.36,26.64h1.279v-1.28\tH23.36V26.64z M17,27.36h-2c-0.199,0-0.36-0.161-0.36-0.36v-2c0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2\tC17.36,27.199,17.199,27.36,17,27.36z M15.36,26.64h1.28v-1.28h-1.28V26.64z M9,27.36H7c-0.199,0-0.36-0.161-0.36-0.36v-2\tc0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2C9.36,27.199,9.199,27.36,9,27.36z M7.36,26.64h1.28v-1.28H7.36\tV26.64z M25,20.36h-2c-0.199,0-0.36-0.161-0.36-0.36v-2c0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2\tC25.36,20.199,25.199,20.36,25,20.36z M23.36,19.64h1.279v-1.28H23.36V19.64z M17,20.36h-2c-0.199,0-0.36-0.161-0.36-0.36v-2\tc0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2C17.36,20.199,17.199,20.36,17,20.36z M15.36,19.64h1.28v-1.28\th-1.28V19.64z M9,20.36H7c-0.199,0-0.36-0.161-0.36-0.36v-2c0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2\tC9.36,20.199,9.199,20.36,9,20.36z M7.36,19.64h1.28v-1.28H7.36V19.64z M25,13.36h-2c-0.199,0-0.36-0.161-0.36-0.36v-2\tc0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2C25.36,13.199,25.199,13.36,25,13.36z M23.36,12.64h1.279v-1.28\tH23.36V12.64z M17,13.36h-2c-0.199,0-0.36-0.161-0.36-0.36v-2c0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2\tC17.36,13.199,17.199,13.36,17,13.36z M15.36,12.64h1.28v-1.28h-1.28V12.64z M9,13.36H7c-0.199,0-0.36-0.161-0.36-0.36v-2\tc0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2C9.36,13.199,9.199,13.36,9,13.36z M7.36,12.64h1.28v-1.28H7.36\tV12.64z"
  })), children);
});

module.exports = Calendar;
