// @carbon/pictograms-react/lib/electric/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Electric = /*#__PURE__*/React__default["default"].forwardRef(function Electric(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M25,8.64c-3.507,0-6.36,2.853-6.36,6.36v10c0,3.11-2.53,5.64-5.64,5.64S7.36,28.11,7.36,25v-5.658\tc3.339-0.188,6-2.957,6-6.342V5c0-0.199-0.161-0.36-0.36-0.36h-2.64V1c0-0.199-0.161-0.36-0.36-0.36S9.64,0.801,9.64,1v3.64H4.36V1\tc0-0.199-0.161-0.36-0.36-0.36S3.64,0.801,3.64,1v3.64H1C0.801,4.64,0.64,4.801,0.64,5v8c0,3.385,2.661,6.154,6,6.342V25\tc0,3.507,2.853,6.36,6.36,6.36s6.36-2.854,6.36-6.36V15c0-3.11,2.529-5.64,5.64-5.64s5.64,2.53,5.64,5.64v10h0.721V15\tC31.36,11.493,28.507,8.64,25,8.64z M1.36,5.36h11.28v3.28H1.36V5.36z M1.36,13V9.36h11.28V13c0,3.11-2.53,5.64-5.64,5.64\tS1.36,16.11,1.36,13z"
  })), children);
});

module.exports = Electric;
