// @carbon/pictograms-react/es/person--08/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Person_08 = /*#__PURE__*/React.forwardRef(function Person_08(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M18.36,30h-0.72v-8h0.721L18.36,30L18.36,30z M14.36,30h-0.72v-8h0.72V30z M21,20.36h-2\tc-0.75,0-1.36-0.61-1.36-1.36v-9c0-0.75,0.61-1.36,1.36-1.36h2c0.75,0,1.36,0.61,1.36,1.36v9C22.36,19.75,21.75,20.36,21,20.36z M19,9.36c-0.353,0-0.64,0.287-0.64,0.64v9c0,0.353,0.287,0.64,0.64,0.64h2c0.353,0,0.64-0.287,0.64-0.64v-9\tc0-0.353-0.287-0.64-0.64-0.64H19z M13,20.36h-2c-0.75,0-1.36-0.61-1.36-1.36v-9c0-0.75,0.61-1.36,1.36-1.36h2\tc0.75,0,1.36,0.61,1.36,1.36v9C14.36,19.75,13.75,20.36,13,20.36z M11,9.36c-0.353,0-0.64,0.287-0.64,0.64v9\tc0,0.353,0.287,0.64,0.64,0.64h2c0.353,0,0.64-0.287,0.64-0.64v-9c0-0.353-0.287-0.64-0.64-0.64H11z M16,7.36\tc-1.577,0-2.86-1.283-2.86-2.86S14.423,1.64,16,1.64s2.86,1.283,2.86,2.86S17.577,7.36,16,7.36z M16,2.36\tc-1.18,0-2.14,0.96-2.14,2.14S14.82,6.64,16,6.64s2.14-0.96,2.14-2.14S17.18,2.36,16,2.36z"
  })), children);
});

export { Person_08 as default };
