// @carbon/pictograms-react/lib/sap/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Sap = /*#__PURE__*/React__default["default"].forwardRef(function Sap(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M8,25.36c-0.625,0-1.153-0.425-1.312-1H1\tc-0.199,0-0.36-0.161-0.36-0.36V8c0-0.199,0.161-0.36,0.36-0.36h18.688c0.158-0.576,0.687-1,1.312-1s1.153,0.424,1.312,1H31\tc0.139,0,0.266,0.08,0.325,0.205s0.042,0.274-0.046,0.382l-13,16C18.211,24.312,18.108,24.36,18,24.36H9.312\tC9.153,24.936,8.625,25.36,8,25.36z M8,23.36c-0.353,0-0.64,0.287-0.64,0.64S7.647,24.64,8,24.64c0.353,0,0.64-0.287,0.64-0.64\tS8.353,23.36,8,23.36z M9.312,23.64h8.518L30.244,8.36h-7.933c-0.158,0.576-0.687,1-1.312,1s-1.153-0.424-1.312-1H1.36v15.28h5.329\tc0.158-0.575,0.686-1,1.312-1S9.153,23.064,9.312,23.64z M21,7.36c-0.353,0-0.64,0.287-0.64,0.64S20.647,8.64,21,8.64\tS21.64,8.353,21.64,8S21.353,7.36,21,7.36z"
  })), children);
});

module.exports = Sap;
