// @carbon/pictograms-react/lib/expand--vert/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const ExpandVert = /*#__PURE__*/React__default["default"].forwardRef(function ExpandVert(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M1.01,1.36C0.811,1.36,0.645,1.199,0.645,1S0.801,0.64,1,0.64h0.01c0.199,0,0.36,0.161,0.36,0.36\tS1.208,1.36,1.01,1.36z M29.37,1c0-0.199-0.161-0.36-0.36-0.36h-0.02c-0.199,0-0.351,0.161-0.351,0.36s0.171,0.36,0.37,0.36\tS29.37,1.199,29.37,1z M27.37,1c0-0.199-0.161-0.36-0.36-0.36h-0.02c-0.199,0-0.351,0.161-0.351,0.36s0.171,0.36,0.37,0.36\tS27.37,1.199,27.37,1z M25.37,1c0-0.199-0.161-0.36-0.36-0.36h-0.02c-0.199,0-0.351,0.161-0.351,0.36s0.171,0.36,0.37,0.36\tS25.37,1.199,25.37,1z M23.37,1c0-0.199-0.161-0.36-0.36-0.36h-0.02c-0.199,0-0.351,0.161-0.351,0.36s0.171,0.36,0.37,0.36\tS23.37,1.199,23.37,1z M21.37,1c0-0.199-0.161-0.36-0.36-0.36h-0.02c-0.199,0-0.351,0.161-0.351,0.36s0.171,0.36,0.37,0.36\tS21.37,1.199,21.37,1z M19.37,1c0-0.199-0.161-0.36-0.36-0.36h-0.02c-0.199,0-0.351,0.161-0.351,0.36s0.171,0.36,0.37,0.36\tS19.37,1.199,19.37,1z M17.37,1c0-0.199-0.161-0.36-0.36-0.36h-0.02c-0.199,0-0.35,0.161-0.35,0.36s0.171,0.36,0.37,0.36\tC17.209,1.36,17.37,1.199,17.37,1z M15.37,1c0-0.199-0.161-0.36-0.36-0.36h-0.02c-0.199,0-0.35,0.161-0.35,0.36\ts0.171,0.36,0.37,0.36S15.37,1.199,15.37,1z M13.37,1c0-0.199-0.161-0.36-0.36-0.36h-0.02c-0.199,0-0.35,0.161-0.35,0.36\ts0.171,0.36,0.37,0.36S13.37,1.199,13.37,1z M11.37,1c0-0.199-0.161-0.36-0.36-0.36h-0.02c-0.199,0-0.35,0.161-0.35,0.36\ts0.171,0.36,0.37,0.36S11.37,1.199,11.37,1z M9.37,1c0-0.199-0.161-0.36-0.36-0.36H8.99C8.792,0.64,8.64,0.801,8.64,1\ts0.171,0.36,0.37,0.36S9.37,1.199,9.37,1z M7.37,1c0-0.199-0.161-0.36-0.36-0.36H6.99C6.792,0.64,6.64,0.801,6.64,1\ts0.171,0.36,0.37,0.36S7.37,1.199,7.37,1z M5.37,1c0-0.199-0.161-0.36-0.36-0.36H4.99C4.792,0.64,4.64,0.801,4.64,1\ts0.171,0.36,0.37,0.36S5.37,1.199,5.37,1z M3.37,1c0-0.199-0.161-0.36-0.36-0.36H2.99C2.792,0.64,2.64,0.801,2.64,1\ts0.171,0.36,0.37,0.36S3.37,1.199,3.37,1z M31.36,1c0-0.199-0.161-0.36-0.36-0.36h-0.01c-0.199,0-0.355,0.161-0.355,0.36\tS30.801,1.36,31,1.36S31.36,1.199,31.36,1z M31,30.64H1v0.721h30V30.64z M16.36,4.869l2.385,2.385l0.51-0.509L16,3.491l-3.254,3.255\tl0.509,0.509l2.386-2.385V28h0.72L16.36,4.869L16.36,4.869z"
  })), children);
});

module.exports = ExpandVert;
