// @carbon/pictograms-react/lib/insurance--02/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Insurance_02 = /*#__PURE__*/React__default["default"].forwardRef(function Insurance_02(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M14,31.36c-1.301,0-2.36-1.059-2.36-2.36h0.72\tc0,0.904,0.736,1.64,1.64,1.64s1.64-0.735,1.64-1.64V18.312c-0.576-0.158-1-0.687-1-1.312c0-0.75,0.61-1.36,1.36-1.36\tc0.75,0,1.36,0.61,1.36,1.36c0,0.625-0.425,1.153-1,1.312V29C16.36,30.302,15.301,31.36,14,31.36z M16,16.36\tc-0.353,0-0.64,0.287-0.64,0.64c0,0.353,0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64C16.64,16.647,16.353,16.36,16,16.36z M30.36,17\th-0.72c0-7.521-6.118-13.64-13.64-13.64C8.479,3.36,2.36,9.479,2.36,17H1.64c0-7.798,6.248-14.164,14-14.355V1h0.72v1.645\tC24.112,2.836,30.36,9.202,30.36,17z M25.36,17h-0.72c0-4.764-3.876-8.64-8.64-8.64S7.36,12.236,7.36,17H6.64\tc0-5.161,4.199-9.36,9.36-9.36S25.36,11.839,25.36,17z M20.36,17h-0.72c0-2.007-1.633-3.64-3.64-3.64s-3.64,1.633-3.64,3.64h-0.72\tc0-2.404,1.956-4.36,4.36-4.36S20.36,14.596,20.36,17z"
  })), children);
});

module.exports = Insurance_02;
