// @carbon/pictograms-react/lib/supply-chain--02/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const SupplyChain_02 = /*#__PURE__*/React__default["default"].forwardRef(function SupplyChain_02(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M7.36,19.652v-7.304\tc2.669-0.178,4.81-2.319,4.988-4.988h7.304c0.178,2.669,2.318,4.81,4.987,4.988V18h0.721v-5.652c2.788-0.186,5-2.514,5-5.348\tc0-2.956-2.405-5.36-5.36-5.36c-2.834,0-5.162,2.211-5.348,5h-7.304c-0.186-2.789-2.514-5-5.348-5C4.044,1.64,1.64,4.044,1.64,7\tc0,2.834,2.211,5.162,5,5.348v7.304c-2.789,0.186-5,2.514-5,5.348c0,2.955,2.404,5.36,5.36,5.36c2.834,0,5.162-2.212,5.348-5H18\tv-0.72h-5.652C12.17,21.971,10.029,19.83,7.36,19.652z M25,2.36c2.559,0,4.64,2.082,4.64,4.64s-2.081,4.64-4.64,4.64\tS20.36,9.559,20.36,7S22.441,2.36,25,2.36z M2.36,7c0-2.559,2.082-4.64,4.64-4.64S11.64,4.441,11.64,7S9.559,11.64,7,11.64\tS2.36,9.559,2.36,7z M7,29.64c-2.559,0-4.64-2.081-4.64-4.64S4.441,20.36,7,20.36s4.64,2.081,4.64,4.64S9.559,29.64,7,29.64z M31,20.64h-2c-1.425,0-2.832,0.905-3.647,2.141C25.216,20.658,23.127,18.64,21,18.64h-2c-0.199,0-0.36,0.161-0.36,0.36v2\tc0,2.2,2.16,4.36,4.36,4.36h1.64V30h0.721v-2.64H27c2.2,0,4.36-2.16,4.36-4.36v-2C31.36,20.801,31.199,20.64,31,20.64z M24.64,24.131l-2.385-2.386l-0.51,0.51l2.386,2.385H23c-1.803,0-3.64-1.837-3.64-3.64v-1.64H21c1.803,0,3.64,1.837,3.64,3.64\tV24.131z M30.64,23c0,1.803-1.837,3.64-3.64,3.64h-1.131l2.386-2.385l-0.51-0.51l-2.385,2.386V25c0-1.803,1.837-3.64,3.64-3.64h1.64\tC30.64,21.36,30.64,23,30.64,23z M7,8C6.448,8,6,7.552,6,7s0.448-1,1-1s1,0.448,1,1S7.552,8,7,8z M24,7c0-0.552,0.448-1,1-1\ts1,0.448,1,1s-0.448,1-1,1S24,7.552,24,7z M8,25c0,0.552-0.448,1-1,1s-1-0.448-1-1s0.448-1,1-1S8,24.448,8,25z"
  })), children);
});

module.exports = SupplyChain_02;
