// @carbon/pictograms-react/es/cics--performance-analyzer-for-z-os/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const CicsPerformanceAnalyzerForZOs = /*#__PURE__*/React.forwardRef(function CicsPerformanceAnalyzerForZOs(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M30,30.36H4\tc-1.301,0-2.36-1.059-2.36-2.36V2h0.72v26c0,0.904,0.736,1.64,1.64,1.64h4.64V14h0.72v15.64h6.28V23h0.72v6.64h6.28V20h0.721v9.64\tH30V30.36z M25,14.36h-4v-0.72h4c2.559,0,4.64-2.082,4.64-4.64V7h0.721v2C30.36,11.956,27.955,14.36,25,14.36z M24,10.36\tc-1.302,0-2.36-1.059-2.36-2.36S22.698,5.64,24,5.64S26.36,6.699,26.36,8S25.302,10.36,24,10.36z M24,6.36\tc-0.904,0-1.64,0.736-1.64,1.64S23.096,9.64,24,9.64S25.64,8.904,25.64,8S24.904,6.36,24,6.36z M18.36,9h-0.72V7\tc0-2.956,2.405-5.36,5.36-5.36h4v0.72h-4c-2.559,0-4.64,2.082-4.64,4.64V9z"
  })), children);
});

export { CicsPerformanceAnalyzerForZOs as default };
