// @carbon/pictograms-react/es/automated/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Automated = /*#__PURE__*/React.forwardRef(function Automated(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M28,29.36c-0.625,0-1.153-0.425-1.312-1H4v-0.72\th22.688c0.158-0.575,0.687-1,1.312-1c0.75,0,1.36,0.61,1.36,1.36S28.75,29.36,28,29.36z M28,27.36c-0.353,0-0.64,0.287-0.64,0.64\ts0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S28.353,27.36,28,27.36z M28,21.36c-0.625,0-1.153-0.425-1.312-1h-5.377\tc-0.158,0.575-0.687,1-1.312,1s-1.153-0.425-1.312-1H4v-0.72h14.688c0.158-0.575,0.687-1,1.312-1s1.153,0.425,1.312,1h5.377\tc0.158-0.575,0.687-1,1.312-1c0.75,0,1.36,0.61,1.36,1.36S28.75,21.36,28,21.36z M28,19.36c-0.353,0-0.64,0.287-0.64,0.64\ts0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S28.353,19.36,28,19.36z M20,19.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64\ts0.64-0.287,0.64-0.64S20.353,19.36,20,19.36z M28,13.36c-0.625,0-1.153-0.424-1.312-1h-5.377c-0.158,0.576-0.687,1-1.312,1\ts-1.153-0.424-1.312-1H13.31c-0.158,0.576-0.686,1-1.312,1s-1.153-0.424-1.312-1H4v-0.72h6.688c0.158-0.576,0.686-1,1.312-1\ts1.153,0.424,1.312,1h5.377c0.158-0.576,0.687-1,1.312-1s1.153,0.424,1.312,1h5.377c0.158-0.576,0.687-1,1.312-1\tc0.75,0,1.36,0.61,1.36,1.36S28.75,13.36,28,13.36z M28,11.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64\ts0.64-0.287,0.64-0.64S28.353,11.36,28,11.36z M20,11.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64\tS20.353,11.36,20,11.36z M12,11.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S12.353,11.36,12,11.36\tz M28,5.36c-0.625,0-1.153-0.424-1.312-1h-5.377c-0.158,0.576-0.687,1-1.312,1s-1.153-0.424-1.312-1H13.31\tc-0.158,0.576-0.686,1-1.312,1s-1.153-0.424-1.312-1H5.312c-0.158,0.576-0.686,1-1.312,1C3.25,5.36,2.64,4.75,2.64,4\tS3.25,2.64,4,2.64c0.625,0,1.153,0.424,1.312,1h5.377c0.158-0.576,0.686-1,1.312-1s1.153,0.424,1.312,1h5.377\tc0.158-0.576,0.687-1,1.312-1s1.153,0.424,1.312,1h5.377c0.158-0.576,0.687-1,1.312-1c0.75,0,1.36,0.61,1.36,1.36\tS28.75,5.36,28,5.36z M28,3.36c-0.353,0-0.64,0.287-0.64,0.64S27.647,4.64,28,4.64S28.64,4.353,28.64,4S28.353,3.36,28,3.36z M20,3.36c-0.353,0-0.64,0.287-0.64,0.64S19.647,4.64,20,4.64S20.64,4.353,20.64,4S20.353,3.36,20,3.36z M12,3.36\tc-0.353,0-0.64,0.287-0.64,0.64S11.647,4.64,12,4.64S12.64,4.353,12.64,4S12.353,3.36,12,3.36z M4,3.36\tC3.647,3.36,3.36,3.647,3.36,4S3.647,4.64,4,4.64S4.64,4.353,4.64,4S4.353,3.36,4,3.36z"
  })), children);
});

export { Automated as default };
