// @carbon/pictograms-react/lib/location/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Location = /*#__PURE__*/React__default["default"].forwardRef(function Location(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,31.36c-0.086,0-0.173-0.031-0.242-0.094C15.304,30.854,4.64,21.055,4.64,12\tC4.64,5.736,9.736,0.64,16,0.64S27.36,5.736,27.36,12c0,8.057-10.652,18.801-11.106,19.255C16.185,31.324,16.092,31.36,16,31.36z M16,1.36C10.133,1.36,5.36,6.133,5.36,12c0,7.98,8.992,16.933,10.629,18.495C17.601,28.819,26.64,19.103,26.64,12\tC26.64,6.133,21.867,1.36,16,1.36z M16,16.36c-2.404,0-4.36-1.956-4.36-4.36S13.596,7.64,16,7.64s4.36,1.956,4.36,4.36\tS18.404,16.36,16,16.36z M16,8.36c-2.007,0-3.64,1.633-3.64,3.64s1.633,3.64,3.64,3.64s3.64-1.633,3.64-3.64S18.007,8.36,16,8.36z"
  })), children);
});

module.exports = Location;
