// @carbon/pictograms-react/es/train/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Train = /*#__PURE__*/React.forwardRef(function Train(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M25.688,31.179l-3.423-6l0.625-0.357l3.423,6L25.688,31.179z M6.312,31.179l-0.625-0.357l3.423-6l0.625,0.357\tL6.312,31.179z M26,23.36H6c-0.199,0-0.36-0.161-0.36-0.36V9c0-3.507,2.853-6.36,6.36-6.36h8c3.507,0,6.36,2.853,6.36,6.36v14\tC26.36,23.199,26.199,23.36,26,23.36z M6.36,22.64h19.28V9c0-3.11-2.529-5.64-5.64-5.64h-8C8.89,3.36,6.36,5.89,6.36,9V22.64z M22.5,20.36c-1.025,0-1.86-0.835-1.86-1.86s0.835-1.86,1.86-1.86s1.86,0.834,1.86,1.86S23.525,20.36,22.5,20.36z M22.5,17.36\tc-0.629,0-1.14,0.511-1.14,1.14s0.511,1.14,1.14,1.14s1.14-0.511,1.14-1.14S23.129,17.36,22.5,17.36z M9.5,20.36\tc-1.025,0-1.86-0.835-1.86-1.86s0.834-1.86,1.86-1.86s1.86,0.834,1.86,1.86S10.525,20.36,9.5,20.36z M9.5,17.36\tc-0.628,0-1.14,0.511-1.14,1.14s0.512,1.14,1.14,1.14s1.14-0.511,1.14-1.14S10.128,17.36,9.5,17.36z M24,15.36H8\tc-0.199,0-0.36-0.161-0.36-0.36V9c0-2.404,1.956-4.36,4.36-4.36h8c2.404,0,4.36,1.956,4.36,4.36v6\tC24.36,15.199,24.199,15.36,24,15.36z M8.36,14.64h15.28V9c0-2.007-1.633-3.64-3.64-3.64h-8C9.993,5.36,8.36,6.993,8.36,9V14.64z M21,1.36H11V0.64h10V1.36z"
  })), children);
});

export { Train as default };
