// @carbon/pictograms-react/lib/cics--interdependency-analyzer-for-z-os/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const CicsInterdependencyAnalyzerForZOs = /*#__PURE__*/React__default["default"].forwardRef(function CicsInterdependencyAnalyzerForZOs(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M26,30.36\tc-1.302,0-2.36-1.059-2.36-2.36s1.059-2.36,2.36-2.36s2.36,1.059,2.36,2.36S27.302,30.36,26,30.36z M26,26.36\tc-0.904,0-1.64,0.735-1.64,1.64s0.735,1.64,1.64,1.64s1.64-0.735,1.64-1.64S26.904,26.36,26,26.36z M16,30.36\tc-1.301,0-2.36-1.059-2.36-2.36s1.059-2.36,2.36-2.36c1.302,0,2.36,1.059,2.36,2.36S17.302,30.36,16,30.36z M16,26.36\tc-0.904,0-1.64,0.735-1.64,1.64s0.736,1.64,1.64,1.64c0.904,0,1.64-0.735,1.64-1.64S16.904,26.36,16,26.36z M6,30.36\tc-1.301,0-2.36-1.059-2.36-2.36S4.699,25.64,6,25.64S8.36,26.698,8.36,28S7.301,30.36,6,30.36z M6,26.36\tc-0.904,0-1.64,0.735-1.64,1.64S5.096,29.64,6,29.64S7.64,28.904,7.64,28S6.904,26.36,6,26.36z M26.36,24h-0.72v-3\tc0-0.353-0.287-0.64-0.64-0.64h-8.64V24h-0.72v-3.64H7c-0.353,0-0.64,0.287-0.64,0.64v3H5.64v-3c0-0.75,0.61-1.36,1.36-1.36h8.64V16\th0.72v3.64H25c0.75,0,1.36,0.61,1.36,1.36V24z M17,14.36h-4v-0.72h4c2.559,0,4.64-2.082,4.64-4.64V7h0.721v2\tC22.36,11.956,19.955,14.36,17,14.36z M16,10.36c-1.301,0-2.36-1.059-2.36-2.36S14.699,5.64,16,5.64c1.302,0,2.36,1.059,2.36,2.36\tS17.302,10.36,16,10.36z M16,6.36c-0.904,0-1.64,0.736-1.64,1.64S15.096,9.64,16,9.64c0.904,0,1.64-0.736,1.64-1.64\tS16.904,6.36,16,6.36z M10.36,9H9.64V7c0-2.956,2.404-5.36,5.36-5.36h4v0.72h-4c-2.559,0-4.64,2.082-4.64,4.64\tC10.36,7,10.36,9,10.36,9z"
  })), children);
});

module.exports = CicsInterdependencyAnalyzerForZOs;
