// @carbon/pictograms-react/es/person--06/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Person_06 = /*#__PURE__*/React.forwardRef(function Person_06(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M18.36,30h-0.72V20c0-0.199,0.161-0.36,0.36-0.36h3c0.353,0,0.64-0.287,0.64-0.64v-9\tc0-0.353-0.287-0.64-0.64-0.64H11c-0.353,0-0.64,0.287-0.64,0.64v9c0,0.353,0.287,0.64,0.64,0.64h3c0.199,0,0.36,0.161,0.36,0.36v10\th-0.72v-9.64H11c-0.75,0-1.36-0.61-1.36-1.36v-9c0-0.75,0.61-1.36,1.36-1.36h10c0.75,0,1.36,0.61,1.36,1.36v9\tc0,0.75-0.61,1.36-1.36,1.36h-2.64V30z M16,7.36c-1.577,0-2.86-1.283-2.86-2.86S14.423,1.64,16,1.64s2.86,1.283,2.86,2.86\tS17.577,7.36,16,7.36z M16,2.36c-1.18,0-2.14,0.96-2.14,2.14S14.82,6.64,16,6.64s2.14-0.96,2.14-2.14S17.18,2.36,16,2.36z"
  })), children);
});

export { Person_06 as default };
