// @carbon/pictograms-react/es/control/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Control = /*#__PURE__*/React.forwardRef(function Control(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M23,31.36c-0.75,0-1.36-0.61-1.36-1.36\tc0-0.625,0.425-1.153,1-1.312V1h0.721v27.688c0.575,0.158,1,0.687,1,1.312C24.36,30.75,23.75,31.36,23,31.36z M23,29.36\tc-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S23.353,29.36,23,29.36z M30.36,31h-0.72v-8.688\tc-0.575-0.158-1-0.687-1-1.312s0.425-1.153,1-1.312V1h0.721v18.688c0.575,0.158,1,0.687,1,1.312s-0.425,1.153-1,1.312L30.36,31\tL30.36,31z M30,20.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S30.353,20.36,30,20.36z M16.36,31\th-0.72V17.312c-0.576-0.158-1-0.686-1-1.312s0.424-1.153,1-1.312V1h0.72v13.688c0.576,0.158,1,0.686,1,1.312\tc0,0.626-0.425,1.153-1,1.312V31z M16,15.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64\tS16.353,15.36,16,15.36z M9.36,31H8.64V3.312c-0.576-0.158-1-0.686-1-1.312c0-0.75,0.61-1.36,1.36-1.36S10.36,1.25,10.36,2\tc0,0.625-0.424,1.153-1,1.312V31z M9,1.36C8.647,1.36,8.36,1.647,8.36,2S8.647,2.64,9,2.64S9.64,2.353,9.64,2S9.353,1.36,9,1.36z M2.36,31H1.64v-8.688c-0.576-0.158-1-0.687-1-1.312s0.424-1.153,1-1.312V1h0.72v18.688c0.576,0.158,1,0.687,1,1.312\ts-0.424,1.153-1,1.312C2.36,22.312,2.36,31,2.36,31z M2,20.36c-0.353,0-0.64,0.287-0.64,0.64S1.647,21.64,2,21.64\tS2.64,21.353,2.64,21S2.353,20.36,2,20.36z"
  })), children);
});

export { Control as default };
