// @carbon/pictograms-react/lib/chart--radar/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const ChartRadar = /*#__PURE__*/React__default["default"].forwardRef(function ChartRadar(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M26,31.36H6c-0.16,0-0.3-0.105-0.345-0.259l-5-17c-0.04-0.135,0.003-0.281,0.109-0.374l15-13\tc0.136-0.117,0.336-0.117,0.472,0l15,13c0.107,0.092,0.149,0.238,0.11,0.374l-5,17C26.301,31.255,26.16,31.36,26,31.36z M6.731,30.64h18.538l-5.514-7.168L9.328,27.264L6.731,30.64z M20.379,23.103l5.476,7.117l4.619-15.707l-4.133,1.102\tc-0.013,0.039-0.033,0.076-0.06,0.11L20.379,23.103z M1.526,14.513l4.62,15.707l2.494-3.242V16.41L1.526,14.513z M16,18.591\tL10.146,26.2l9.148-3.327L16,18.591z M9.36,16.602v9.44l6.025-7.833L9.36,16.602z M16.615,18.209l3.312,4.306l5.277-6.597\tL16.615,18.209z M16.36,5.9v11.631l8.834-2.355L16.36,5.9z M9.495,15.893l6.145,1.639V6.236L9.495,15.893z M1.749,13.827\tl7.016,1.871L15.64,4.895V1.789L1.749,13.827z M16.36,4.856l9.627,10.108l4.264-1.137L16.36,1.789V4.856z"
  })), children);
});

module.exports = ChartRadar;
