// @carbon/pictograms-react/lib/subsecond/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Subsecond = /*#__PURE__*/React__default["default"].forwardRef(function Subsecond(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M21.812,6.654l-0.623-0.36l1.5-2.598l0.623,0.36\tL21.812,6.654z M8.688,27.944l0.623,0.359l1.5-2.598l-0.623-0.359L8.688,27.944z M19.971,2.57l-0.695-0.187L18.5,5.281l0.695,0.187\tL19.971,2.57z M12.029,29.431l0.695,0.186l0.776-2.898l-0.695-0.186L12.029,29.431z M15.64,30h0.72v-3h-0.72V30z M16.36,2h-0.72v3\th0.72V2z M18.499,26.718l0.776,2.898l0.695-0.186l-0.776-2.898L18.499,26.718z M13.5,5.282l-0.776-2.898L12.029,2.57l0.776,2.898\tL13.5,5.282z M21.188,25.706l1.5,2.598l0.623-0.359l-1.5-2.598L21.188,25.706z M10.812,6.293l-1.5-2.598l-0.624,0.36l1.5,2.598\tL10.812,6.293z M23.523,24.033l2.121,2.121l0.51-0.51l-2.121-2.121L23.523,24.033z M8.476,7.967L6.355,5.846L5.846,6.355\tl2.121,2.121L8.476,7.967z M25.347,21.812l2.598,1.5l0.359-0.623l-2.598-1.5L25.347,21.812z M6.654,10.188l-2.598-1.5l-0.36,0.624\tl2.598,1.5L6.654,10.188z M26.532,19.194l2.898,0.776l0.186-0.695l-2.898-0.776L26.532,19.194z M5.468,12.805L2.57,12.029\tl-0.187,0.695L5.282,13.5L5.468,12.805z M27,15.64v0.72h3v-0.72H27z M5,15.64H2v0.72h3V15.64z M26.718,13.5l2.898-0.776\tl-0.186-0.695l-2.898,0.776L26.718,13.5z M5.282,18.499l-2.898,0.776l0.187,0.695l2.898-0.776L5.282,18.499z M25.347,10.188\tl0.359,0.623l2.598-1.5l-0.359-0.623L25.347,10.188z M6.293,21.188l-2.598,1.5l0.36,0.623l2.598-1.5L6.293,21.188z M17.898,14.611\tc0.287,0.391,0.462,0.868,0.462,1.389c0,1.302-1.059,2.36-2.36,2.36s-2.36-1.059-2.36-2.36s1.059-2.36,2.36-2.36\tc0.521,0,0.998,0.175,1.389,0.461l4.454-4.454c-3.385-3.114-8.667-3.039-11.95,0.243c-3.369,3.369-3.369,8.851,0,12.219l-0.509,0.51\tc-3.65-3.649-3.65-9.588,0-13.237c3.563-3.563,9.301-3.639,12.968-0.243l5.393-5.393l0.51,0.509L17.898,14.611z M17.64,16\tc0-0.904-0.735-1.64-1.64-1.64s-1.64,0.736-1.64,1.64c0,0.904,0.736,1.64,1.64,1.64S17.64,16.904,17.64,16z M5.846,25.645\tl0.509,0.51l2.121-2.121l-0.509-0.51L5.846,25.645z"
  })), children);
});

module.exports = Subsecond;
