// @carbon/pictograms-react/es/snowflake/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Snowflake = /*#__PURE__*/React.forwardRef(function Snowflake(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M24.36,30h-0.72v-5.851l-7.28-7.28V28h-0.72V16.869\tl-7.28,7.28V30H7.64v-5.131l-4.386,4.386l-0.509-0.51L7.13,24.36H2v-0.72h5.851l7.28-7.28H9.126l-4.901,3.921l-0.45-0.562\tl4.199-3.359H3v-0.72h4.974l-4.199-3.359l0.45-0.562l4.901,3.921h6.004L7.85,8.36H2V7.64h5.131L2.746,3.254l0.509-0.509L7.64,7.131\tV2h0.72v5.851l7.28,7.28V4h0.72v11.131l7.28-7.28V2h0.721v5.131l4.385-4.385l0.51,0.509L24.869,7.64H30v0.72h-5.851l-7.28,7.28\th6.005l4.901-3.921l0.449,0.562l-4.198,3.359H29v0.72h-4.974l4.198,3.359l-0.449,0.562l-4.901-3.921h-6.005l7.28,7.28H30v0.721\th-5.131l4.386,4.385l-0.51,0.51L24.36,24.87C24.36,24.87,24.36,30,24.36,30z"
  })), children);
});

export { Snowflake as default };
