// @carbon/pictograms-react/es/developer-tools/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const DeveloperTools = /*#__PURE__*/React.forwardRef(function DeveloperTools(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M24.317,29.171l-0.635-0.342L30.591,16\tL23.683,3.171l0.635-0.342l7,13c0.057,0.107,0.057,0.235,0,0.342L24.317,29.171z M7.683,29.171l-7-13\tc-0.058-0.107-0.058-0.235,0-0.342l7-13l0.634,0.342L1.409,16l6.908,12.829L7.683,29.171z M24,23.36c-0.625,0-1.153-0.425-1.312-1H8\tv-0.72h14.688c0.158-0.575,0.687-1,1.312-1c0.75,0,1.36,0.61,1.36,1.36S24.75,23.36,24,23.36z M24,21.36\tc-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S24.353,21.36,24,21.36z M6,17.36\tc-0.75,0-1.36-0.61-1.36-1.36S5.25,14.64,6,14.64c0.625,0,1.153,0.424,1.312,1H27v0.72H7.312C7.153,16.936,6.625,17.36,6,17.36z M6,15.36c-0.353,0-0.64,0.287-0.64,0.64S5.647,16.64,6,16.64S6.64,16.353,6.64,16S6.353,15.36,6,15.36z M24,11.36\tc-0.625,0-1.153-0.424-1.312-1H8V9.64h14.688c0.158-0.576,0.687-1,1.312-1c0.75,0,1.36,0.61,1.36,1.36S24.75,11.36,24,11.36z M24,9.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S24.353,9.36,24,9.36z"
  })), children);
});

export { DeveloperTools as default };
