// @carbon/pictograms-react/lib/hyper-protect/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const HyperProtect = /*#__PURE__*/React__default["default"].forwardRef(function HyperProtect(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,31.36C8.082,31.36,1.64,24.918,1.64,17V1\tc0-0.199,0.161-0.36,0.36-0.36h28c0.199,0,0.36,0.161,0.36,0.36v16C30.36,24.918,23.918,31.36,16,31.36z M2.36,1.36V17\tc0,7.521,6.119,13.64,13.64,13.64S29.64,24.522,29.64,17V1.36h-6.28V16c0,4.059-3.302,7.36-7.36,7.36S8.64,20.058,8.64,16V1.36H2.36\tz M9.36,1.36V16c0,3.661,2.979,6.64,6.64,6.64s6.64-2.979,6.64-6.64V1.36H9.36z"
  })), children);
});

module.exports = HyperProtect;
