// @carbon/pictograms-react/lib/chart--quadrant-plot/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const ChartQuadrantPlot = /*#__PURE__*/React__default["default"].forwardRef(function ChartQuadrantPlot(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M12,21c0,0.552-0.448,1-1,1s-1-0.448-1-1\ts0.448-1,1-1S12,20.448,12,21z M21,20c-0.552,0-1,0.448-1,1s0.448,1,1,1s1-0.448,1-1S21.552,20,21,20z M28,20c-0.552,0-1,0.448-1,1\ts0.448,1,1,1s1-0.448,1-1S28.552,20,28,20z M21,27c-0.552,0-1,0.448-1,1s0.448,1,1,1s1-0.448,1-1S21.552,27,21,27z M11,10\tc-0.552,0-1,0.448-1,1s0.448,1,1,1s1-0.448,1-1S11.552,10,11,10z M27,4c-0.552,0-1,0.448-1,1s0.448,1,1,1s1-0.448,1-1S27.552,4,27,4\tz M5,10c-0.552,0-1,0.448-1,1s0.448,1,1,1s1-0.448,1-1S5.552,10,5,10z M16.36,31h-0.72V16.36H1v-0.72h14.64V1h0.72v14.64H31v0.72\tH16.36V31z M6,28.36c-1.301,0-2.36-1.059-2.36-2.36S4.699,23.64,6,23.64S8.36,24.698,8.36,26S7.301,28.36,6,28.36z M6,24.36\tc-0.904,0-1.64,0.735-1.64,1.64S5.096,27.64,6,27.64S7.64,26.904,7.64,26S6.904,24.36,6,24.36z M22,12.36\tc-1.302,0-2.36-1.059-2.36-2.36S20.698,7.64,22,7.64s2.36,1.059,2.36,2.36S23.302,12.36,22,12.36z M22,8.36\tc-0.904,0-1.64,0.736-1.64,1.64s0.735,1.64,1.64,1.64s1.64-0.736,1.64-1.64S22.904,8.36,22,8.36z M8,7.36\tC6.699,7.36,5.64,6.301,5.64,5S6.699,2.64,8,2.64S10.36,3.699,10.36,5S9.301,7.36,8,7.36z M8,3.36C7.096,3.36,6.36,4.096,6.36,5\tS7.096,6.64,8,6.64S9.64,5.904,9.64,5S8.904,3.36,8,3.36z"
  })), children);
});

module.exports = ChartQuadrantPlot;
