// @carbon/pictograms-react/lib/stairs--down/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const StairsDown = /*#__PURE__*/React__default["default"].forwardRef(function StairsDown(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31,0.64v0.72h-5.64V7c0,0.199-0.161,0.36-0.36,0.36h-5.64V13c0,0.199-0.161,0.36-0.36,0.36h-5.64V19\tc0,0.199-0.161,0.36-0.36,0.36H7.36V25c0,0.199-0.161,0.36-0.36,0.36H1.36V31H0.64v-6c0-0.199,0.161-0.36,0.36-0.36h5.64V19\tc0-0.199,0.161-0.36,0.36-0.36h5.64V13c0-0.199,0.161-0.36,0.36-0.36h5.64V7c0-0.199,0.161-0.36,0.36-0.36h5.64V1\tc0-0.199,0.161-0.36,0.36-0.36H31z M28.745,20.745L21.36,28.13V23h-0.72v6.36H27v-0.72h-5.13l7.385-7.385L28.745,20.745z"
  })), children);
});

module.exports = StairsDown;
