// @carbon/pictograms-react/lib/office/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Office = /*#__PURE__*/React__default["default"].forwardRef(function Office(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31,31.36H1c-0.199,0-0.36-0.161-0.36-0.36V1c0-0.199,0.161-0.36,0.36-0.36h18c0.199,0,0.36,0.161,0.36,0.36\tv2.64H25c0.199,0,0.36,0.161,0.36,0.36v3.64H31c0.199,0,0.36,0.161,0.36,0.36v23C31.36,31.199,31.199,31.36,31,31.36z M19.36,30.64\th11.28V8.36H19.36V30.64z M12.36,30.64h6.28V1.36H1.36v29.28h6.28V23c0-0.199,0.161-0.36,0.36-0.36h4c0.199,0,0.36,0.161,0.36,0.36\tV30.64z M8.36,30.64h3.28v-7.28H8.36V30.64z M19.36,7.64h5.279V4.36H19.36V7.64z M27.36,25h-0.72v-2h0.721L27.36,25L27.36,25z M23.36,25h-0.72v-2h0.721L23.36,25L23.36,25z M16.36,25h-0.72v-2h0.72V25z M4.36,25H3.64v-2h0.72V25z M27.36,19h-0.72v-2h0.721\tL27.36,19L27.36,19z M23.36,19h-0.72v-2h0.721L23.36,19L23.36,19z M16.36,19h-0.72v-2h0.72V19z M12.36,19h-0.72v-2h0.72V19z M8.36,19H7.64v-2h0.72V19z M4.36,19H3.64v-2h0.72V19z M27.36,13h-0.72v-2h0.721L27.36,13L27.36,13z M23.36,13h-0.72v-2h0.721\tL23.36,13L23.36,13z M16.36,13h-0.72v-2h0.72V13z M12.36,13h-0.72v-2h0.72V13z M8.36,13H7.64v-2h0.72V13z M4.36,13H3.64v-2h0.72V13z M16.36,7h-0.72V5h0.72V7z M12.36,7h-0.72V5h0.72V7z M8.36,7H7.64V5h0.72V7z M4.36,7H3.64V5h0.72V7z"
  })), children);
});

module.exports = Office;
