// @carbon/pictograms-react/es/esg-reporting--02/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const EsgReporting_02 = /*#__PURE__*/React.forwardRef(function EsgReporting_02(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M5.5,14.14c-1.025,0-1.86,0.834-1.86,1.86\ts0.834,1.86,1.86,1.86S7.36,17.025,7.36,16S6.525,14.14,5.5,14.14z M5.5,17.14c-0.628,0-1.14-0.511-1.14-1.14s0.512-1.14,1.14-1.14\tS6.64,15.372,6.64,16S6.128,17.14,5.5,17.14z M5.5,19.14c-1.025,0-1.86,0.835-1.86,1.86s0.834,1.86,1.86,1.86S7.36,22.025,7.36,21\tS6.525,19.14,5.5,19.14z M5.5,22.14c-0.628,0-1.14-0.511-1.14-1.14s0.512-1.14,1.14-1.14S6.64,20.371,6.64,21S6.128,22.14,5.5,22.14\tz M5.5,24.14c-1.025,0-1.86,0.835-1.86,1.86s0.834,1.86,1.86,1.86S7.36,27.025,7.36,26S6.525,24.14,5.5,24.14z M5.5,27.14\tc-0.628,0-1.14-0.511-1.14-1.14s0.512-1.14,1.14-1.14S6.64,25.371,6.64,26S6.128,27.14,5.5,27.14z M28.745,24.255l1.386,1.385h-7.77\tv-4.28h4.77l-1.386,1.385l0.51,0.51L28.509,21l-2.254-2.255l-0.51,0.51l1.386,1.385h-4.77v-4.28h7.771l-1.386,1.385l0.51,0.51\tL31.509,16l-2.254-2.254l-0.51,0.509l1.386,1.386H22.36V4c0-0.199-0.161-0.36-0.36-0.36H5.36V1.36h20.28V13h0.721V1\tc0-0.199-0.161-0.36-0.36-0.36H5C4.801,0.64,4.64,0.801,4.64,1v2.64H1C0.801,3.64,0.64,3.801,0.64,4v27\tc0,0.199,0.161,0.36,0.36,0.36h21c0.199,0,0.36-0.161,0.36-0.36v-4.64h7.77l-1.386,1.385l0.51,0.51L31.509,26l-2.254-2.255\tL28.745,24.255z M21.64,30.64H1.36V4.36h20.28v11.28H9v0.72h12.64v4.28H9v0.721h12.64v4.279H9v0.721h12.64V30.64z"
  })), children);
});

export { EsgReporting_02 as default };
