// @carbon/pictograms-react/lib/natural--language--classifier/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const NaturalLanguageClassifier = /*#__PURE__*/React__default["default"].forwardRef(function NaturalLanguageClassifier(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M19,31.36h-6c-0.199,0-0.36-0.161-0.36-0.36v-6c0-0.199,0.161-0.36,0.36-0.36h2.64\tv-6.28H12c-0.199,0-0.36-0.161-0.36-0.36v-4c0-0.199,0.161-0.36,0.36-0.36h8c0.199,0,0.36,0.161,0.36,0.36v4\tc0,0.199-0.161,0.36-0.36,0.36h-3.64v6.279H19c0.199,0,0.36,0.161,0.36,0.36v6C19.36,31.199,19.199,31.36,19,31.36z M13.36,30.64\th5.28v-5.28h-5.28C13.36,25.36,13.36,30.64,13.36,30.64z M12.36,17.64h7.28v-3.28h-7.28C12.36,14.36,12.36,17.64,12.36,17.64z M31,18.36h-8c-0.199,0-0.36-0.161-0.36-0.36v-4c0-0.199,0.161-0.36,0.36-0.36h3.64V7.36h-3.104c-0.129,0-0.247-0.068-0.312-0.18\ts-0.064-0.249,0-0.36l3.464-6c0.129-0.223,0.494-0.223,0.623,0l3.464,6c0.064,0.111,0.064,0.249,0,0.36s-0.183,0.18-0.312,0.18\tH27.36v6.28H31c0.199,0,0.36,0.161,0.36,0.36v4C31.36,18.199,31.199,18.36,31,18.36z M23.36,17.64h7.279v-3.28H23.36V17.64z M24.159,6.64h5.682L27,1.72L24.159,6.64z M9,18.36H1c-0.199,0-0.36-0.161-0.36-0.36v-4c0-0.199,0.161-0.36,0.36-0.36h3.64v-6.3\tc-1.684-0.18-3-1.61-3-3.341c0-1.853,1.507-3.36,3.36-3.36S8.36,2.147,8.36,4c0,1.731-1.316,3.161-3,3.341v6.3H9\tc0.199,0,0.36,0.161,0.36,0.36v4C9.36,18.199,9.199,18.36,9,18.36z M1.36,17.64h7.28v-3.28H1.36V17.64z M5,1.36\tC3.544,1.36,2.36,2.544,2.36,4S3.544,6.64,5,6.64S7.64,5.456,7.64,4S6.456,1.36,5,1.36z"
  })), children);
});

module.exports = NaturalLanguageClassifier;
