// @carbon/pictograms-react/es/lock--02/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Lock_02 = /*#__PURE__*/React.forwardRef(function Lock_02(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M17.5,21.5c0,0.704-0.487,1.29-1.14,1.452V25c0,0.199-0.161,0.36-0.36,0.36s-0.36-0.161-0.36-0.36v-2.048\tc-0.654-0.162-1.14-0.748-1.14-1.452c0-0.828,0.672-1.5,1.5-1.5C16.828,20,17.5,20.672,17.5,21.5z M26.355,13v18\tc0,0.196-0.159,0.355-0.355,0.355H6c-0.196,0-0.355-0.159-0.355-0.355V13c0-0.196,0.159-0.355,0.355-0.355h4.64V6\tc0-2.956,2.404-5.36,5.36-5.36c2.955,0,5.36,2.404,5.36,5.36v6.645H26C26.196,12.645,26.355,12.804,26.355,13z M11.359,12.645h9.28\tV6c0-2.559-2.081-4.64-4.64-4.64S11.36,3.441,11.36,6L11.359,12.645z M6.355,13.355v17.289h19.289V13.355H6.355z"
  })), children);
});

export { Lock_02 as default };
