// @carbon/pictograms-react/lib/waste--electronic/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const WasteElectronic = /*#__PURE__*/React__default["default"].forwardRef(function WasteElectronic(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M9.254,28.255l-0.509-0.51l5.385-5.385H1\tc-0.199,0-0.36-0.161-0.36-0.36V10c0-0.199,0.161-0.36,0.36-0.36h13.851l5.894-5.895l0.51,0.509L9.869,15.64H21\tc0.146,0,0.277,0.087,0.333,0.222c0.056,0.134,0.024,0.289-0.078,0.392L9.254,28.255z M3.36,21.64h11.491l5.28-5.28H9\tc-0.146,0-0.277-0.087-0.333-0.222c-0.056-0.134-0.025-0.289,0.078-0.392l5.385-5.386H3.36V21.64z M1.36,21.64h1.28V10.36H1.36\tV21.64z M29,22.36H18v-0.72h8.64V10.36H18V9.64h11c0.199,0,0.36,0.161,0.36,0.36v3.64H31c0.199,0,0.36,0.161,0.36,0.36v4\tc0,0.199-0.161,0.36-0.36,0.36h-1.64V22C29.36,22.199,29.199,22.36,29,22.36z M27.36,21.64h1.279V10.36H27.36V21.64z M29.36,17.64\th1.279v-3.28H29.36V17.64z"
  })), children);
});

module.exports = WasteElectronic;
