// @carbon/pictograms-react/lib/blockchain/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Blockchain = /*#__PURE__*/React__default["default"].forwardRef(function Blockchain(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M30,30.36h-6c-0.199,0-0.36-0.161-0.36-0.36v-2.64h-4.28V30c0,0.199-0.161,0.36-0.36,0.36h-6\tc-0.199,0-0.36-0.161-0.36-0.36v-2.64H8.36V30c0,0.199-0.161,0.36-0.36,0.36H2c-0.199,0-0.36-0.161-0.36-0.36v-6\tc0-0.199,0.161-0.36,0.36-0.36h2.64v-4.28H2c-0.199,0-0.36-0.161-0.36-0.36v-6c0-0.199,0.161-0.36,0.36-0.36h2.64V8.36H2\tC1.801,8.36,1.64,8.199,1.64,8V2c0-0.199,0.161-0.36,0.36-0.36h6c0.199,0,0.36,0.161,0.36,0.36v2.64h4.28V2\tc0-0.199,0.161-0.36,0.36-0.36h6c0.199,0,0.36,0.161,0.36,0.36v2.64h4.279V2c0-0.199,0.161-0.36,0.36-0.36h6\tc0.199,0,0.36,0.161,0.36,0.36v6c0,0.199-0.161,0.36-0.36,0.36h-2.64v4.28H30c0.199,0,0.36,0.161,0.36,0.36v6\tc0,0.199-0.161,0.36-0.36,0.36h-2.64v4.279H30c0.199,0,0.36,0.161,0.36,0.36v6C30.36,30.199,30.199,30.36,30,30.36z M24.36,29.64\th5.279v-5.28H24.36V29.64z M13.36,29.64h5.28v-5.28h-5.28C13.36,24.36,13.36,29.64,13.36,29.64z M2.36,29.64h5.28v-5.28H2.36V29.64z M19.36,26.64h4.279V24c0-0.199,0.161-0.36,0.36-0.36h2.64v-4.28H24c-0.199,0-0.36-0.161-0.36-0.36v-2.64h-4.28V19\tc0,0.199-0.161,0.36-0.36,0.36h-2.64v4.279H19c0.199,0,0.36,0.161,0.36,0.36V26.64z M8.36,26.64h4.28V24\tc0-0.199,0.161-0.36,0.36-0.36h2.64v-4.28H13c-0.199,0-0.36-0.161-0.36-0.36v-2.64H8.36V19c0,0.199-0.161,0.36-0.36,0.36H5.36v4.279\tH8c0.199,0,0.36,0.161,0.36,0.36V26.64z M27,18.64h2.64v-5.28h-5.28v5.28H27z M16,18.64h2.64v-5.28h-5.28v5.28H16z M5,18.64h2.64\tv-5.28H2.36v5.28H5z M19.36,15.64h4.279V13c0-0.199,0.161-0.36,0.36-0.36h2.64V8.36H24c-0.199,0-0.36-0.161-0.36-0.36V5.36h-4.28V8\tc0,0.199-0.161,0.36-0.36,0.36h-2.64v4.28H19c0.199,0,0.36,0.161,0.36,0.36V15.64z M8.36,15.64h4.28V13\tc0-0.199,0.161-0.36,0.36-0.36h2.64V8.36H13c-0.199,0-0.36-0.161-0.36-0.36V5.36H8.36V8c0,0.199-0.161,0.36-0.36,0.36H5.36v4.28H8\tc0.199,0,0.36,0.161,0.36,0.36V15.64z M27,7.64h2.64V2.36h-5.28v5.28C24.36,7.64,27,7.64,27,7.64z M16,7.64h2.64V2.36h-5.28v5.28\tC13.36,7.64,16,7.64,16,7.64z M5,7.64h2.64V2.36H2.36v5.28C2.36,7.64,5,7.64,5,7.64z"
  })), children);
});

module.exports = Blockchain;
