// @carbon/pictograms-react/lib/servers/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Servers = /*#__PURE__*/React__default["default"].forwardRef(function Servers(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M23,31.36H9c-0.199,0-0.36-0.161-0.36-0.36v-3.64H1c-0.199,0-0.36-0.161-0.36-0.36V5\tc0-0.199,0.161-0.36,0.36-0.36h7.64V1c0-0.199,0.161-0.36,0.36-0.36h14c0.199,0,0.36,0.161,0.36,0.36v3.64H31\tc0.199,0,0.36,0.161,0.36,0.36v22c0,0.199-0.161,0.36-0.36,0.36h-7.64V31C23.36,31.199,23.199,31.36,23,31.36z M9.36,30.64h13.28\tV1.36H9.36V30.64z M23.36,26.64h7.279V5.36H23.36v2.28H27v0.72h-3.64v2.28H27v0.72h-3.64v2.28H27v0.72h-3.64V26.64z M1.36,26.64\th7.28V14.36H5v-0.72h3.64v-2.28H5v-0.72h3.64V8.36H5V7.64h3.64V5.36H1.36V26.64z M20,10.36h-8V9.64h8V10.36z M20,7.36h-8V6.64h8\tV7.36z M20,4.36h-8V3.64h8V4.36z"
  })), children);
});

module.exports = Servers;
