// @carbon/pictograms-react/lib/sprout/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Sprout = /*#__PURE__*/React__default["default"].forwardRef(function Sprout(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M25.666,29.135C25.587,28.939,23.639,24.36,16,24.36c-7.638,0-9.587,4.579-9.666,4.774l-0.668-0.269\tc0.084-0.21,2.123-5.078,9.975-5.223V16.36H13c-4.127,0-7.36-3.233-7.36-7.36V6c0-0.199,0.161-0.36,0.36-0.36h3\tc2.965,0,5.468,1.668,6.643,4.138C15.756,5.757,18.947,2.64,23,2.64h3c0.199,0,0.36,0.161,0.36,0.36v3c0,4.127-3.233,7.36-7.36,7.36\th-2.64v10.284c7.852,0.145,9.89,5.013,9.974,5.223L25.666,29.135z M6.36,6.36V9c0,3.723,2.917,6.64,6.64,6.64h2.131l-5.385-5.386\tl0.509-0.509l5.386,5.385V13c0-3.723-2.917-6.64-6.64-6.64H6.36z M16.869,12.64H19c3.724,0,6.64-2.917,6.64-6.64V3.36H23\tc-3.724,0-6.64,2.917-6.64,6.64v2.131l5.385-5.385l0.51,0.509L16.869,12.64z"
  })), children);
});

module.exports = Sprout;
