// @carbon/pictograms-react/lib/cloud--infrastructure-management/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const CloudInfrastructureManagement = /*#__PURE__*/React__default["default"].forwardRef(function CloudInfrastructureManagement(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M29,30.36\tc-1.302,0-2.36-1.059-2.36-2.36c0-1.179,0.869-2.159,2-2.333V22.36H16.36v3.307c1.131,0.174,2,1.154,2,2.333\tc0,1.302-1.059,2.36-2.36,2.36s-2.36-1.059-2.36-2.36c0-1.179,0.869-2.159,2-2.333V22.36H3.36v3.307c1.131,0.174,2,1.154,2,2.333\tc0,1.302-1.059,2.36-2.36,2.36S0.64,29.302,0.64,28c0-1.179,0.869-2.159,2-2.333V22c0-0.199,0.161-0.36,0.36-0.36h12.64V18h0.72\tv3.64H29c0.199,0,0.36,0.161,0.36,0.36v3.667c1.131,0.174,2,1.154,2,2.333C31.36,29.302,30.302,30.36,29,30.36z M29,26.36\tc-0.904,0-1.64,0.735-1.64,1.64s0.735,1.64,1.64,1.64s1.64-0.735,1.64-1.64S29.904,26.36,29,26.36z M16,26.36\tc-0.904,0-1.64,0.735-1.64,1.64s0.736,1.64,1.64,1.64c0.904,0,1.64-0.735,1.64-1.64S16.904,26.36,16,26.36z M3,26.36\tc-0.904,0-1.64,0.735-1.64,1.64S2.096,29.64,3,29.64S4.64,28.904,4.64,28S3.904,26.36,3,26.36z M26,16.36H16\tc-0.199,0-0.36-0.161-0.36-0.36V7c0-0.199,0.161-0.36,0.36-0.36h10c0.199,0,0.36,0.161,0.36,0.36v9\tC26.36,16.199,26.199,16.36,26,16.36z M16.36,15.64h9.28v-2.28h-9.28V15.64z M16.36,12.64h9.28v-2.28h-9.28V12.64z M16.36,9.64h9.28\tV7.36h-9.28V9.64z M14.667,16.36h-4.051c-2.743,0-4.975-2.286-4.975-5.097c0-3.073,2.583-5.533,5.569-5.165\tc1.358-2.358,3.13-3.458,5.56-3.458c1.922,0,3.674,0.903,4.934,2.543l-0.57,0.438c-1.121-1.458-2.67-2.262-4.364-2.262\tc-2.213,0-3.818,1.05-5.052,3.304c-0.074,0.135-0.225,0.21-0.378,0.182c-2.604-0.454-4.98,1.738-4.98,4.418\tc0,2.414,1.909,4.377,4.255,4.377h4.051L14.667,16.36L14.667,16.36z M18.5,14.86h-1v-0.72h1V14.86z M18.5,11.86h-1v-0.72h1V11.86z M18.5,8.86h-1V8.14h1V8.86z"
  })), children);
});

module.exports = CloudInfrastructureManagement;
