// @carbon/pictograms-react/lib/wifi/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Wifi = /*#__PURE__*/React__default["default"].forwardRef(function Wifi(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,27.86c-1.854,0-3.362-1.509-3.362-3.362s1.508-3.362,3.362-3.362s3.362,1.509,3.362,3.362\tS17.854,27.86,16,27.86z M16,21.856c-1.457,0-2.642,1.185-2.642,2.642S14.543,27.14,16,27.14s2.642-1.185,2.642-2.642\tS17.457,21.856,16,21.856z M22.896,18.477c-1.929-1.616-4.379-2.505-6.899-2.505c-2.518,0-4.966,0.888-6.894,2.501L8.642,17.92\tc2.058-1.721,4.67-2.668,7.356-2.668c2.689,0,5.303,0.949,7.362,2.672L22.896,18.477z M26.827,14.531\tc-2.985-2.654-6.831-4.116-10.831-4.116c-3.996,0-7.84,1.46-10.825,4.111l-0.479-0.538c3.117-2.768,7.131-4.292,11.303-4.292\tc4.175,0,8.192,1.526,11.309,4.298L26.827,14.531z M30.757,10.594C26.719,6.896,21.478,4.86,15.998,4.86\tc-5.478,0-10.717,2.035-14.754,5.729l-0.486-0.531C4.927,6.242,10.34,4.14,15.998,4.14c5.66,0,11.075,2.103,15.246,5.922\tL30.757,10.594z"
  })), children);
});

module.exports = Wifi;
