// @carbon/pictograms-react/lib/chart--river/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const ChartRiver = /*#__PURE__*/React__default["default"].forwardRef(function ChartRiver(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31,31.36H1c-0.199,0-0.36-0.161-0.36-0.36V1h0.72v9.264l8.419-6.548\tc0.127-0.097,0.303-0.101,0.433-0.007l10.808,7.86l9.795-5.877c0.112-0.067,0.249-0.068,0.363-0.005C31.29,5.75,31.36,5.87,31.36,6\tv14c0,0.126-0.066,0.243-0.175,0.309l-10,6c-0.124,0.074-0.279,0.067-0.397-0.018L10.007,18.45L1.36,25.176v5.464H31V31.36z M10,17.64c0.074,0,0.148,0.023,0.212,0.069l10.808,7.859l9.62-5.772v-6.16l-9.454,5.673c-0.124,0.075-0.279,0.068-0.397-0.018\tL10.007,11.45L1.36,18.176v6.088l8.419-6.548C9.844,17.665,9.922,17.64,10,17.64z M10,10.64c0.074,0,0.148,0.023,0.212,0.069\tl10.808,7.859l9.62-5.772v-6.16l-9.454,5.673c-0.124,0.075-0.279,0.068-0.397-0.018L10.007,4.45L1.36,11.176v6.087l8.419-6.548\tC9.844,10.666,9.922,10.64,10,10.64z"
  })), children);
});

module.exports = ChartRiver;
