// @carbon/pictograms-react/lib/crop/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Crop = /*#__PURE__*/React__default["default"].forwardRef(function Crop(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M4.36,31H3.64v-2.64H1v-0.72h2.64V25h0.72v2.64H7v0.721H4.36V31z M26,28.36H10v-0.72h16 c0.904,0,1.64-0.735,1.64-1.64V10h0.721v16C28.36,27.302,27.302,28.36,26,28.36z M4.36,22H3.64V6c0-1.301,1.059-2.36,2.36-2.36h16 v0.72H6C5.096,4.36,4.36,5.096,4.36,6V22z M28.36,7h-0.72V4.36H25V3.64h2.64V1h0.721v2.64H31v0.72h-2.64V7z"
  })), children);
});

module.exports = Crop;
