// @carbon/pictograms-react/lib/supply--chain--optimization--02/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const SupplyChainOptimization_02 = /*#__PURE__*/React__default["default"].forwardRef(function SupplyChainOptimization_02(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M11,30.36H3v-0.72h8V30.36z M12.36,29h-0.72v-8h0.72V29z M2.36,29H1.64v-8h0.72 C2.36,21,2.36,29,2.36,29z M26.5,26.86h-3c-0.199,0-0.36-0.161-0.36-0.36v-1.14H14v-0.72h9.14V23.5c0-0.199,0.161-0.36,0.36-0.36 h1.14V14h0.721v9.14h1.14c0.199,0,0.36,0.161,0.36,0.36v3C26.86,26.699,26.699,26.86,26.5,26.86z M23.86,26.14h2.279v-2.28H23.86 V26.14z M8.5,26.86h-3c-0.199,0-0.36-0.161-0.36-0.36v-3c0-0.199,0.161-0.36,0.36-0.36h3c0.199,0,0.36,0.161,0.36,0.36v3 C8.86,26.699,8.699,26.86,8.5,26.86z M5.86,26.14h2.28v-2.28H5.86V26.14z M11,20.36H3v-0.72h8V20.36z M7.36,18H6.64v-4h0.72 C7.36,14,7.36,18,7.36,18z M29,12.36h-8v-0.72h8V12.36z M11,12.36H3v-0.72h8V12.36z M30.36,11h-0.72V3h0.721L30.36,11L30.36,11z M20.36,11h-0.72V3h0.721L20.36,11L20.36,11z M12.36,11h-0.72V3h0.72V11z M2.36,11H1.64V3h0.72C2.36,3,2.36,11,2.36,11z M26.5,8.86 h-3c-0.199,0-0.36-0.161-0.36-0.36v-3c0-0.199,0.161-0.36,0.36-0.36h3c0.199,0,0.36,0.161,0.36,0.36v3 C26.86,8.699,26.699,8.86,26.5,8.86z M23.86,8.14h2.279V5.86H23.86V8.14z M8.5,8.86h-3c-0.199,0-0.36-0.161-0.36-0.36v-3 c0-0.199,0.161-0.36,0.36-0.36h3c0.199,0,0.36,0.161,0.36,0.36v3C8.86,8.699,8.699,8.86,8.5,8.86z M5.86,8.14h2.28V5.86H5.86V8.14z M18,7.36h-4V6.64h4V7.36z M29,2.36h-8V1.64h8V2.36z M11,2.36H3V1.64h8V2.36z"
  })), children);
});

module.exports = SupplyChainOptimization_02;
