// @carbon/pictograms-react/es/faster-innovation-with-partners/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const FasterInnovationWithPartners = /*#__PURE__*/React.forwardRef(function FasterInnovationWithPartners(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M20,27.36\tc-0.047,0-0.093-0.01-0.138-0.027C19.728,27.277,19.64,27.146,19.64,27v-3.64h-5.307c-0.174,1.131-1.154,2-2.333,2\ts-2.159-0.869-2.333-2H3.312c-0.158,0.575-0.686,1-1.312,1c-0.75,0-1.36-0.61-1.36-1.36S1.25,21.64,2,21.64\tc0.625,0,1.153,0.425,1.312,1h6.356c0.174-1.131,1.154-2,2.333-2s2.159,0.869,2.333,2H20c0.199,0,0.36,0.161,0.36,0.36v3.131\tl9.771-9.771h-3.798c-0.174,1.131-1.154,2-2.333,2s-2.159-0.869-2.333-2H18.36V18c0,0.199-0.161,0.36-0.36,0.36h-4\tc-0.199,0-0.36-0.161-0.36-0.36v-1.64h-3.308c-0.174,1.131-1.154,2-2.333,2s-2.159-0.869-2.333-2H3.312\tc-0.158,0.576-0.686,1-1.312,1c-0.75,0-1.36-0.61-1.36-1.36S1.25,14.64,2,14.64c0.625,0,1.153,0.424,1.312,1h2.356\tc0.174-1.131,1.154-2,2.333-2s2.159,0.869,2.333,2h3.308V14c0-0.199,0.161-0.36,0.36-0.36h4c0.199,0,0.36,0.161,0.36,0.36v1.64\th3.307c0.174-1.131,1.154-2,2.333-2s2.159,0.869,2.333,2h3.798L20.36,5.869V9c0,0.199-0.161,0.36-0.36,0.36h-5.667\tc-0.174,1.131-1.154,2-2.333,2s-2.159-0.869-2.333-2H3.312c-0.158,0.576-0.686,1-1.312,1c-0.75,0-1.36-0.61-1.36-1.36\tS1.25,7.64,2,7.64c0.625,0,1.153,0.424,1.312,1h6.356c0.174-1.131,1.154-2,2.333-2s2.159,0.869,2.333,2h5.307V5\tc0-0.146,0.088-0.277,0.223-0.333c0.134-0.056,0.288-0.025,0.393,0.078l11,11c0.141,0.141,0.141,0.368,0,0.509l-11,11\tC20.186,27.323,20.094,27.36,20,27.36z M12,21.36c-0.904,0-1.64,0.735-1.64,1.64s0.736,1.64,1.64,1.64s1.64-0.735,1.64-1.64\tS12.904,21.36,12,21.36z M2,22.36c-0.353,0-0.64,0.287-0.64,0.64S1.647,23.64,2,23.64S2.64,23.353,2.64,23S2.353,22.36,2,22.36z M24,14.36c-0.904,0-1.64,0.736-1.64,1.64c0,0.904,0.735,1.64,1.64,1.64s1.64-0.735,1.64-1.64S24.904,14.36,24,14.36z M14.36,17.64\th3.28v-3.28h-3.28V17.64z M8,14.36c-0.904,0-1.64,0.736-1.64,1.64c0,0.904,0.736,1.64,1.64,1.64S9.64,16.904,9.64,16\tS8.904,14.36,8,14.36z M2,15.36c-0.353,0-0.64,0.287-0.64,0.64S1.647,16.64,2,16.64S2.64,16.353,2.64,16S2.353,15.36,2,15.36z M12,7.36c-0.904,0-1.64,0.736-1.64,1.64s0.736,1.64,1.64,1.64S13.64,9.904,13.64,9S12.904,7.36,12,7.36z M2,8.36\tC1.647,8.36,1.36,8.647,1.36,9S1.647,9.64,2,9.64S2.64,9.353,2.64,9S2.353,8.36,2,8.36z"
  })), children);
});

export { FasterInnovationWithPartners as default };
