// @carbon/pictograms-react/es/idea/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Idea = /*#__PURE__*/React.forwardRef(function Idea(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M19,31.36h-6v-0.72h6V31.36z M20,29.36h-8v-0.72h8V29.36z M20,27.36h-8v-0.72h8V27.36z M20,25.36h-8\tc-0.199,0-0.36-0.161-0.36-0.36v-1.453c0-1.622,0-4.073-2.041-6.195c-1.908-1.986-2.959-4.503-2.959-7.087\tc0-5.397,4.111-9.624,9.36-9.624s9.36,4.227,9.36,9.624c0,2.584-1.051,5.102-2.959,7.086c-2.041,2.123-2.041,4.574-2.041,6.196V25\tC20.36,25.199,20.199,25.36,20,25.36z M12.36,24.64h7.28v-1.093c0-1.632,0-4.363,2.242-6.695c1.778-1.851,2.758-4.19,2.758-6.588\tc0-4.993-3.795-8.904-8.64-8.904s-8.64,3.911-8.64,8.904c0,2.398,0.979,4.737,2.758,6.588c2.242,2.332,2.242,5.063,2.242,6.695\tC12.36,23.547,12.36,24.64,12.36,24.64z"
  })), children);
});

export { Idea as default };
