// @carbon/pictograms-react/es/secure--devops/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const SecureDevops = /*#__PURE__*/React.forwardRef(function SecureDevops(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M16,31.36C8.082,31.36,1.64,24.918,1.64,17V1\tc0-0.199,0.161-0.36,0.36-0.36h28c0.199,0,0.36,0.161,0.36,0.36v16C30.36,24.918,23.918,31.36,16,31.36z M2.36,1.36V17\tc0,7.521,6.119,13.64,13.64,13.64S29.64,24.522,29.64,17V1.36H2.36z M21.5,21.36c-3.231,0-5.86-2.629-5.86-5.86\tc0-2.834-2.306-5.14-5.14-5.14s-5.14,2.306-5.14,5.14s2.306,5.14,5.14,5.14c1.373,0,2.664-0.534,3.635-1.506l0.509,0.51\tc-1.106,1.106-2.578,1.717-4.144,1.717c-3.231,0-5.86-2.629-5.86-5.86s2.628-5.86,5.86-5.86s5.86,2.628,5.86,5.86\tc0,2.834,2.306,5.14,5.14,5.14s5.14-2.306,5.14-5.14s-2.306-5.14-5.14-5.14c-1.373,0-2.663,0.535-3.634,1.505l-0.51-0.509\tC18.464,10.25,19.935,9.64,21.5,9.64c3.231,0,5.86,2.628,5.86,5.86S24.731,21.36,21.5,21.36z"
  })), children);
});

export { SecureDevops as default };
