// @carbon/pictograms-react/lib/integration--03/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Integration_03 = /*#__PURE__*/React__default["default"].forwardRef(function Integration_03(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,31.36c-0.092,0-0.184-0.035-0.254-0.105\tl-15-15c-0.141-0.141-0.141-0.368,0-0.509l15-15c0.141-0.141,0.368-0.141,0.509,0l15,15c0.141,0.141,0.141,0.368,0,0.509l-15,15\tC16.184,31.325,16.092,31.36,16,31.36z M6.009,20.5L16,30.491L21.491,25L16,19.509l-2.818,2.818\tc0.113,0.198,0.178,0.429,0.178,0.673c0,0.75-0.61,1.36-1.36,1.36s-1.36-0.61-1.36-1.36s0.61-1.36,1.36-1.36\tc0.245,0,0.474,0.065,0.673,0.179l3.073-3.073c0.141-0.141,0.368-0.141,0.509,0L22,24.491L30.491,16L16,1.509L1.509,16L5.5,19.991\tL15.746,9.746c0.141-0.141,0.368-0.141,0.509,0l7.073,7.073c0.198-0.113,0.429-0.178,0.673-0.178c0.75,0,1.36,0.61,1.36,1.36\ts-0.61,1.36-1.36,1.36s-1.36-0.61-1.36-1.36c0-0.244,0.065-0.475,0.179-0.673L16,10.509L6.009,20.5z M12,22.36\tc-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S12.353,22.36,12,22.36z M24,17.36\tc-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S24.353,17.36,24,17.36z"
  })), children);
});

module.exports = Integration_03;
