// @carbon/pictograms-react/es/business--continuity--02/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const BusinessContinuity_02 = /*#__PURE__*/React.forwardRef(function BusinessContinuity_02(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M30,30.36H12\tc-0.199,0-0.36-0.161-0.36-0.36v-8.64H11c-0.199,0-0.36-0.161-0.36-0.36v-0.64H2c-0.199,0-0.36-0.161-0.36-0.36V2\tc0-0.199,0.161-0.36,0.36-0.36h18c0.199,0,0.36,0.161,0.36,0.36v8.64H21c0.199,0,0.36,0.161,0.36,0.36v0.64H30\tc0.199,0,0.36,0.161,0.36,0.36v18C30.36,30.199,30.199,30.36,30,30.36z M12.36,29.64h17.28V12.36h-8.28V13\tc0,0.199-0.161,0.36-0.36,0.36h-0.64V20c0,0.199-0.161,0.36-0.36,0.36h-6.64V21c0,0.199-0.161,0.36-0.36,0.36h-0.64\tC12.36,21.36,12.36,29.64,12.36,29.64z M11.36,20.64h1.28v-1.28h-1.28V20.64z M13.36,19.64h6.28v-6.28H19\tc-0.199,0-0.36-0.161-0.36-0.36v-0.64h-6.28v6.28H13c0.199,0,0.36,0.161,0.36,0.36V19.64z M2.36,19.64h8.28V19\tc0-0.199,0.161-0.36,0.36-0.36h0.64V12c0-0.199,0.161-0.36,0.36-0.36h6.64V11c0-0.199,0.161-0.36,0.36-0.36h0.64V2.36H2.36V19.64z M19.36,12.64h1.279v-1.28H19.36V12.64z"
  })), children);
});

export { BusinessContinuity_02 as default };
