// @carbon/pictograms-react/es/blockchain--02/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Blockchain_02 = /*#__PURE__*/React.forwardRef(function Blockchain_02(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M16,31.36C7.53,31.36,0.64,24.47,0.64,16\tS7.53,0.64,16,0.64S31.36,7.53,31.36,16S24.47,31.36,16,31.36z M16,1.36C7.927,1.36,1.36,7.927,1.36,16\tc0,8.072,6.567,14.64,14.64,14.64c8.072,0,14.64-6.567,14.64-14.64S24.072,1.36,16,1.36z M16.36,29h-0.72v-5h0.72V29z M22,23.36h-5\tv-0.72h5V23.36z M15,23.36h-5v-0.72h5V23.36z M16.36,22h-0.72v-5h0.72V22z M9.36,22H8.64v-5h0.72C9.36,17,9.36,22,9.36,22z M29,16.36h-5v-0.72h5V16.36z M22,16.36h-5v-0.72h5V16.36z M8,16.36H3v-0.72h5V16.36z M23.36,15h-0.72v-5h0.721L23.36,15L23.36,15z M16.36,15h-0.72v-5h0.72V15z M22,9.36h-5V8.64h5V9.36z M15,9.36h-5V8.64h5V9.36z M16.36,8h-0.72V3h0.72V8z"
  })), children);
});

export { Blockchain_02 as default };
