// @carbon/pictograms-react/lib/ecosystem/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Ecosystem = /*#__PURE__*/React__default["default"].forwardRef(function Ecosystem(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,31.36c-0.668,0-1.226-0.485-1.339-1.122\tc-3.338-0.615-5.85-3.496-6.013-6.887c-3.391-0.163-6.272-2.675-6.887-6.013C1.125,17.226,0.64,16.668,0.64,16\tc0-0.75,0.61-1.36,1.36-1.36S3.36,15.25,3.36,16c0,0.581-0.365,1.077-0.878,1.272c0.577,2.968,3.145,5.197,6.167,5.358\tc0.154-3.104,2.239-5.707,5.075-6.631c-2.836-0.924-4.921-3.528-5.075-6.631c-1.642,0.085-3.173,0.765-4.344,1.936l-0.509-0.509\tc1.307-1.307,3.019-2.062,4.853-2.147c0.085-1.834,0.84-3.546,2.147-4.853l0.509,0.509c-1.171,1.171-1.851,2.703-1.936,4.344\tc3.103,0.154,5.707,2.239,6.631,5.075c0.924-2.836,3.527-4.921,6.631-5.075c-0.161-3.022-2.391-5.59-5.358-6.167\tC17.077,2.995,16.581,3.36,16,3.36c-0.75,0-1.36-0.61-1.36-1.36S15.25,0.64,16,0.64c0.668,0,1.226,0.485,1.339,1.121\tc3.338,0.615,5.85,3.497,6.013,6.887c3.391,0.163,6.271,2.675,6.887,6.013c0.637,0.113,1.122,0.67,1.122,1.339\tc0,0.75-0.61,1.36-1.36,1.36s-1.36-0.61-1.36-1.36c0-0.581,0.366-1.077,0.879-1.272c-0.577-2.968-3.145-5.197-6.168-5.359\tc-0.154,3.103-2.238,5.707-5.075,6.631c2.837,0.924,4.921,3.528,5.075,6.631c1.642-0.085,3.174-0.765,4.345-1.936l0.51,0.51\tc-1.308,1.307-3.02,2.061-4.854,2.146c-0.086,1.834-0.84,3.546-2.146,4.854l-0.51-0.51c1.171-1.171,1.851-2.703,1.936-4.345\tc-3.103-0.154-5.707-2.238-6.631-5.075c-0.924,2.837-3.528,4.921-6.631,5.075c0.162,3.023,2.391,5.591,5.359,6.168\tc0.195-0.513,0.691-0.879,1.272-0.879c0.75,0,1.36,0.61,1.36,1.36S16.75,31.36,16,31.36z M16,29.36c-0.353,0-0.64,0.287-0.64,0.64\ts0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S16.353,29.36,16,29.36z M16.37,16.37c0.186,3.367,2.893,6.073,6.26,6.26\tC22.443,19.263,19.737,16.556,16.37,16.37z M15.63,16.37c-3.367,0.186-6.074,2.893-6.26,6.26\tC12.737,22.443,15.444,19.737,15.63,16.37z M30,15.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64\tS30.353,15.36,30,15.36z M2,15.36c-0.353,0-0.64,0.287-0.64,0.64S1.647,16.64,2,16.64S2.64,16.353,2.64,16S2.353,15.36,2,15.36z M22.63,9.37c-3.367,0.186-6.074,2.893-6.26,6.26C19.737,15.444,22.443,12.737,22.63,9.37z M9.37,9.37\tc0.186,3.367,2.893,6.074,6.26,6.26C15.444,12.263,12.737,9.556,9.37,9.37z M16,1.36c-0.353,0-0.64,0.287-0.64,0.64\tS15.647,2.64,16,2.64c0.353,0,0.64-0.287,0.64-0.64S16.353,1.36,16,1.36z"
  })), children);
});

module.exports = Ecosystem;
