// @carbon/pictograms-react/lib/plastics/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Plastics = /*#__PURE__*/React__default["default"].forwardRef(function Plastics(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M29,31.36H3c-0.184,0-0.337-0.139-0.358-0.32\tl-0.964-8.68H1v-0.72h4.64V1c0-0.199,0.161-0.36,0.36-0.36h4c0.199,0,0.36,0.161,0.36,0.36v6c0,0.904,0.736,1.64,1.64,1.64h6\tc0.904,0,1.64-0.736,1.64-1.64V1c0-0.199,0.161-0.36,0.36-0.36h4c0.199,0,0.36,0.161,0.36,0.36v15.64H31v0.72h-0.665l-0.976,13.665\tC29.346,31.214,29.188,31.36,29,31.36z M23.335,30.64h5.33l0.163-2.279h-5.656L23.335,30.64z M15.402,30.64h7.211l-0.948-13.28H21\tv-0.72h2.64V1.36h-3.28V7c0,1.301-1.059,2.36-2.36,2.36h-6c-1.301,0-2.36-1.059-2.36-2.36V1.36H6.36v20.28H17v0.721h-0.678\tL15.402,30.64z M3.322,30.64h11.355l0.698-6.279H2.625L3.322,30.64z M23.121,27.64h5.758l0.592-8.279H22.53L23.121,27.64z M2.544,23.64h12.911l0.142-1.279H2.402L2.544,23.64z M22.479,18.64h7.043l0.092-1.279h-7.227L22.479,18.64z M10.254,19.255\tl-0.509-0.51l6-6l0.509,0.509L10.254,19.255z M15.254,17.255l-0.509-0.509l5-5l0.51,0.509L15.254,17.255z"
  })), children);
});

module.exports = Plastics;
