// @carbon/pictograms-react/lib/extend-the-data-center/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const ExtendTheDataCenter = /*#__PURE__*/React__default["default"].forwardRef(function ExtendTheDataCenter(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31.255,0.746\tC31.188,0.678,31.096,0.64,31,0.64L1.046,0.643c-0.085-0.009-0.174,0.01-0.248,0.059C0.699,0.769,0.64,0.88,0.64,1v30\tc0,0.199,0.161,0.36,0.36,0.36h30c0.199,0,0.36-0.161,0.36-0.36V1C31.36,0.904,31.322,0.813,31.255,0.746z M1.36,20.363l9.771-0.003\tL1.36,30.131V20.363z M11.64,30.64H1.869l9.771-9.771C11.64,20.869,11.64,30.64,11.64,30.64z M30.64,30.64H12.36V20.15L25.64,6.87V9\th0.721V5.64H23v0.72h2.131l-13.28,13.28L1.36,19.643V1.363L30.64,1.36V30.64z"
  })), children);
});

module.exports = ExtendTheDataCenter;
