// @carbon/pictograms-react/lib/balloon/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Balloon = /*#__PURE__*/React__default["default"].forwardRef(function Balloon(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M19.36,31h-0.72c0-3.389-0.729-4.561-1.5-5.802\tc-0.588-0.944-1.194-1.919-1.458-3.838H14.5c-0.136,0-0.261-0.077-0.322-0.199s-0.048-0.268,0.034-0.377l1.092-1.456\tC11.019,18.931,7.64,14.899,7.64,10c0-5.161,3.75-9.36,8.36-9.36c4.609,0,8.36,4.199,8.36,9.36c0,4.899-3.379,8.931-7.665,9.328\tl1.092,1.456c0.082,0.109,0.095,0.255,0.034,0.377c-0.062,0.122-0.186,0.199-0.322,0.199h-1.091c0.249,1.7,0.78,2.556,1.342,3.458\tC18.542,26.09,19.36,27.405,19.36,31z M15.22,20.64h1.56L16,19.6L15.22,20.64z M16,1.36c-4.213,0-7.64,3.876-7.64,8.64\ts3.427,8.64,7.64,8.64s7.64-3.875,7.64-8.64S20.213,1.36,16,1.36z M10.36,10H9.64c0-4.058,2.853-7.36,6.36-7.36v0.72\tC12.89,3.36,10.36,6.339,10.36,10z"
  })), children);
});

module.exports = Balloon;
