// @carbon/pictograms-react/lib/flashing--content/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const FlashingContent = /*#__PURE__*/React__default["default"].forwardRef(function FlashingContent(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M29,3.5C29,3.776,28.776,4,28.5,4S28,3.776,28,3.5S28.224,3,28.5,3S29,3.224,29,3.5z M26.5,3\tC26.224,3,26,3.224,26,3.5S26.224,4,26.5,4S27,3.776,27,3.5S26.776,3,26.5,3z M24.5,3C24.224,3,24,3.224,24,3.5S24.224,4,24.5,4\tS25,3.776,25,3.5S24.776,3,24.5,3z M31,1.64H1C0.801,1.64,0.64,1.801,0.64,2v28c0,0.199,0.161,0.36,0.36,0.36h30\tc0.199,0,0.36-0.161,0.36-0.36V2C31.36,1.801,31.199,1.64,31,1.64z M30.64,2.36v2.28H1.36V2.36H30.64z M1.36,29.64V5.36h29.28v24.28\tC30.64,29.64,1.36,29.64,1.36,29.64z M14,20.36c-0.061,0-0.122-0.016-0.177-0.047C13.71,20.249,13.64,20.13,13.64,20v-6\tc0-0.13,0.07-0.25,0.183-0.313c0.113-0.062,0.251-0.062,0.362,0.005l5,3c0.108,0.065,0.175,0.182,0.175,0.309\tc0,0.126-0.066,0.243-0.175,0.309l-5,3C14.128,20.343,14.064,20.36,14,20.36z M14.36,14.636v4.729L18.3,17L14.36,14.636z M16,23.36\tc-3.507,0-6.36-2.854-6.36-6.36s2.853-6.36,6.36-6.36s6.36,2.853,6.36,6.36S19.507,23.36,16,23.36z M16,11.36\tc-3.11,0-5.64,2.53-5.64,5.64s2.53,5.64,5.64,5.64s5.64-2.529,5.64-5.64C21.64,13.89,19.11,11.36,16,11.36z M16.36,7h-0.72v2h0.72V7\tz M16.36,25h-0.72v2h0.72V25z M8,16.64H6v0.72h2V16.64z M26,16.64h-2v0.72h2V16.64z M10.598,11.089L9.183,9.674l-0.509,0.509\tl1.415,1.415L10.598,11.089z M23.326,23.816l-1.414-1.414l-0.51,0.51l1.414,1.414L23.326,23.816z M10.598,22.912l-0.509-0.51\tl-1.415,1.414l0.509,0.51L10.598,22.912z M23.326,10.183l-0.51-0.509l-1.414,1.415l0.51,0.509L23.326,10.183z"
  })), children);
});

module.exports = FlashingContent;
