// @carbon/pictograms-react/lib/time--plot/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const TimePlot = /*#__PURE__*/React__default["default"].forwardRef(function TimePlot(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31,31.36H1c-0.199,0-0.36-0.161-0.36-0.36V1h0.72v14.64H11c0.096,0,0.187,0.038,0.254,0.105\tl7.895,7.894H31v0.721H19c-0.096,0-0.188-0.038-0.255-0.105l-7.894-7.895H1.36v14.28H31V31.36z M24,15.36\tc-4.059,0-7.36-3.302-7.36-7.36S19.941,0.64,24,0.64S31.36,3.942,31.36,8S28.059,15.36,24,15.36z M17.37,8.36\tc0.181,3.376,2.895,6.089,6.27,6.271V13h0.721v1.63c3.375-0.181,6.089-2.895,6.27-6.271H29V7.64h1.63\tc-0.181-3.376-2.895-6.089-6.27-6.271V3h-0.72V1.37c-3.375,0.181-6.089,2.895-6.27,6.271H19v0.72h-1.63V8.36z M25.745,10.254l-2-2\tC23.678,8.187,23.64,8.096,23.64,8V4h0.721v3.851l1.895,1.895L25.745,10.254z"
  })), children);
});

module.exports = TimePlot;
