// @carbon/pictograms-react/lib/secure--hybrid--cloud/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const SecureHybridCloud = /*#__PURE__*/React__default["default"].forwardRef(function SecureHybridCloud(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M19,31.36c-3.507,0-6.36-2.854-6.36-6.36\tv-7.223l6.36-3.18l6.36,3.18V25C25.36,28.507,22.507,31.36,19,31.36z M13.36,18.223V25c0,3.11,2.53,5.64,5.64,5.64\ts5.64-2.529,5.64-5.64v-6.777L19,15.402L13.36,18.223z M11,19.64l-3.644,0.004C3.994,19.186,1.36,15.94,1.36,12.255\tc0-3.933,3.195-7.133,7.122-7.133c0.932,0,1.842,0.18,2.707,0.535c0.175,0.073,0.381-0.006,0.463-0.181\tc1.17-2.5,3.705-4.117,6.457-4.117c3.791,0,6.916,2.969,7.113,6.758c0.01,0.182,0.153,0.328,0.335,0.341\tc2.851,0.195,5.084,2.59,5.084,5.454c0,2.533-1.647,5.056-3.752,5.746l0.225,0.684c2.382-0.779,4.248-3.604,4.248-6.429\tc0-3.135-2.366-5.774-5.443-6.143c-0.367-4.023-3.739-7.13-7.81-7.13c-2.918,0-5.614,1.649-6.959,4.23\tc-0.859-0.31-1.755-0.468-2.669-0.468c-4.324,0-7.841,3.523-7.841,7.853c0,4.104,2.845,7.587,6.667,8.105H11V19.64z"
  })), children);
});

module.exports = SecureHybridCloud;
