// @carbon/pictograms-react/lib/high--speed--data-transport/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const HighSpeedDataTransport = /*#__PURE__*/React__default["default"].forwardRef(function HighSpeedDataTransport(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,27.36H5v-0.72h11\tc5.867,0,10.64-4.772,10.64-10.64h0.721C27.36,22.264,22.264,27.36,16,27.36z M2.5,21.86c-1.025,0-1.86-0.835-1.86-1.86\ts0.834-1.86,1.86-1.86c0.902,0,1.657,0.646,1.825,1.5h9.277C12.42,18.86,11.64,17.519,11.64,16c0-2.404,1.956-4.36,4.36-4.36h11.675\tc0.168-0.854,0.923-1.5,1.825-1.5c1.025,0,1.86,0.834,1.86,1.86s-0.835,1.86-1.86,1.86c-0.902,0-1.657-0.646-1.825-1.5h-9.277\tc1.182,0.781,1.963,2.121,1.963,3.64c0,2.404-1.956,4.36-4.36,4.36H4.325C4.157,21.214,3.402,21.86,2.5,21.86z M2.5,18.86\tc-0.628,0-1.14,0.511-1.14,1.14s0.512,1.14,1.14,1.14S3.64,20.629,3.64,20S3.128,18.86,2.5,18.86z M16,19.64\tc2.007,0,3.64-1.633,3.64-3.64s-1.633-3.64-3.64-3.64s-3.64,1.633-3.64,3.64C12.36,18.007,13.993,19.64,16,19.64z M29.5,10.86\tc-0.629,0-1.14,0.512-1.14,1.14s0.511,1.14,1.14,1.14s1.14-0.512,1.14-1.14S30.129,10.86,29.5,10.86z M4.64,16\tc-0.001-3.006,1.181-5.849,3.327-8.004C10.122,5.832,12.975,4.64,16,4.64h11v0.72H16c-2.833,0-5.504,1.117-7.522,3.144\tC6.466,10.523,5.359,13.186,5.36,16H4.64z"
  })), children);
});

module.exports = HighSpeedDataTransport;
