// @carbon/pictograms-react/lib/identity--trust--assessment/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const IdentityTrustAssessment = /*#__PURE__*/React__default["default"].forwardRef(function IdentityTrustAssessment(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M28.36,31h-0.72v-5.237c0-5.419-3.679-10.076-8.946-11.324\tc-0.15-0.036-0.261-0.163-0.275-0.316c-0.015-0.154,0.071-0.3,0.212-0.363c2.34-1.041,3.851-3.365,3.851-5.919\tc0-3.573-2.907-6.48-6.48-6.48c-3.574,0-6.481,2.907-6.481,6.48c0,2.555,1.512,4.878,3.851,5.919\tc0.141,0.063,0.226,0.209,0.212,0.363c-0.014,0.153-0.125,0.281-0.275,0.316c-5.27,1.248-8.949,5.905-8.949,11.324V31H3.64v-5.237\tc0-5.444,3.497-10.164,8.611-11.779c-2.123-1.296-3.452-3.618-3.452-6.144c0-3.97,3.23-7.2,7.201-7.2s7.201,3.23,7.201,7.2\tc0,2.525-1.329,4.847-3.452,6.144c5.114,1.615,8.611,6.334,8.611,11.779V31z M20.255,23.255l-0.51-0.51L15,27.491l-1.746-1.746\tl-0.509,0.51L15,28.509L20.255,23.255z"
  })), children);
});

module.exports = IdentityTrustAssessment;
