// @carbon/pictograms-react/lib/hybrid--cloud/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const HybridCloud = /*#__PURE__*/React__default["default"].forwardRef(function HybridCloud(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,31.36C7.53,31.36,0.64,24.47,0.64,16S7.53,0.64,16,0.64S31.36,7.53,31.36,16\tS24.47,31.36,16,31.36z M16.36,30.063v0.572c4.594-0.111,8.67-2.35,11.28-5.766V12h0.721v11.839\tc1.442-2.268,2.279-4.957,2.279-7.839c0-1.998-0.402-3.904-1.13-5.641L18.356,3.92l0.359-0.624l10.256,5.922\tC26.574,4.65,21.835,1.497,16.36,1.364v8.428l5.016,2.896c0.111,0.064,0.181,0.184,0.181,0.312s-0.069,0.248-0.181,0.312L11.524,19\tl4.656,2.688c0.111,0.064,0.18,0.183,0.18,0.312v7.232l8.924-5.151l0.359,0.623L16.36,30.063z M3.028,22.783\tc2.398,4.567,7.137,7.72,12.612,7.853v-8.428l-5.016-2.896c-0.111-0.064-0.18-0.183-0.18-0.312s0.068-0.247,0.18-0.312L20.477,13\tl-4.657-2.688c-0.111-0.064-0.18-0.184-0.18-0.312V2.767L6.716,7.919l-0.36-0.623l9.284-5.361V1.364\tC11.046,1.476,6.969,3.714,4.36,7.13V20H3.64V8.161C2.197,10.429,1.36,13.119,1.36,16c0,1.998,0.402,3.904,1.131,5.642l11.153,6.439\tl-0.36,0.623L3.028,22.783z"
  })), children);
});

module.exports = HybridCloud;
