// @carbon/pictograms-react/lib/lantern/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Lantern = /*#__PURE__*/React__default["default"].forwardRef(function Lantern(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M27.36,16.5c0-3.974-3.371-7.345-8-8.464V6\tc0-0.199-0.161-0.36-0.36-0.36h-2.64V1h-0.72v4.64H13c-0.199,0-0.36,0.161-0.36,0.36v2.036c-4.629,1.12-8,4.491-8,8.464\tc0,3.976,3.375,7.347,8.007,8.466C12.646,24.978,12.64,24.988,12.64,25v5c0,0.199,0.161,0.36,0.36,0.36s0.36-0.161,0.36-0.36v-2.64\th1.28V30c0,0.199,0.161,0.36,0.36,0.36s0.36-0.161,0.36-0.36v-2.64h1.28V30c0,0.199,0.161,0.36,0.36,0.36\tc0.199,0,0.36-0.161,0.36-0.36v-2.64h1.279V30c0,0.199,0.161,0.36,0.36,0.36s0.36-0.161,0.36-0.36v-5\tc0-0.012-0.006-0.023-0.007-0.034C23.985,23.847,27.36,20.476,27.36,16.5z M13.36,6.36h5.28v1.521C17.793,7.724,16.908,7.64,16,7.64\ts-1.792,0.083-2.64,0.241V6.36z M16,8.36c3.661,0,6.64,3.652,6.64,8.14s-2.979,8.14-6.64,8.14s-6.64-3.651-6.64-8.14\tS12.339,8.36,16,8.36z M5.36,16.5c0-3.477,2.864-6.452,6.882-7.616C10.086,10.431,8.64,13.266,8.64,16.5s1.446,6.068,3.602,7.616\tC8.224,22.951,5.36,19.977,5.36,16.5z M18.64,26.64h-5.28v-1.531c0.849,0.158,1.73,0.251,2.64,0.251s1.791-0.093,2.64-0.251V26.64z M19.758,24.116c2.155-1.548,3.603-4.382,3.603-7.616s-1.447-6.069-3.603-7.616c4.018,1.164,6.882,4.139,6.882,7.616\tS23.775,22.951,19.758,24.116z"
  })), children);
});

module.exports = Lantern;
