// @carbon/pictograms-react/lib/identify-and-access/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const IdentifyAndAccess = /*#__PURE__*/React__default["default"].forwardRef(function IdentifyAndAccess(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M30,30.36h-7v-0.72h6.64V14h0.721v16\tC30.36,30.199,30.199,30.36,30,30.36z M18,30.36H2c-0.199,0-0.36-0.161-0.36-0.36v-7h0.72v6.64H18V30.36z M11,23.36\tc-0.062,0-0.124-0.017-0.179-0.048C10.709,23.248,10.64,23.129,10.64,23V9c0-0.129,0.069-0.248,0.181-0.312\tc0.111-0.065,0.249-0.064,0.36,0.001l12,7c0.11,0.064,0.179,0.183,0.179,0.311s-0.068,0.247-0.179,0.311l-12,7\tC11.125,23.344,11.062,23.36,11,23.36z M11.36,9.627v12.746L22.285,16L11.36,9.627z M2.36,18H1.64V2c0-0.199,0.161-0.36,0.36-0.36h7\tv0.72H2.36V18z M30.36,9h-0.72V2.36H14V1.64h16c0.199,0,0.36,0.161,0.36,0.36V9z"
  })), children);
});

module.exports = IdentifyAndAccess;
