// @carbon/pictograms-react/es/tablet--device--check/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const TabletDeviceCheck = /*#__PURE__*/React.forwardRef(function TabletDeviceCheck(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M14,17.509l-3.254-3.254l0.509-0.509L14,16.491l6.745-6.745l0.51,0.509L14,17.509z M17,28.5\tc0-0.552-0.448-1-1-1s-1,0.448-1,1s0.448,1,1,1S17,29.052,17,28.5z M28.36,31V1c0-0.199-0.161-0.36-0.36-0.36H4\tC3.801,0.64,3.64,0.801,3.64,1v30c0,0.199,0.161,0.36,0.36,0.36h24C28.199,31.36,28.36,31.199,28.36,31z M4.36,26.36h23.28v4.279\tH4.36V26.36z M4.36,1.36h23.28v24.28H4.36V1.36z"
  })), children);
});

export { TabletDeviceCheck as default };
