// @carbon/pictograms-react/lib/real-estate/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const RealEstate = /*#__PURE__*/React__default["default"].forwardRef(function RealEstate(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M29,30.36H3c-0.199,0-0.36-0.161-0.36-0.36V2\tc0-0.199,0.161-0.36,0.36-0.36h16c0.199,0,0.36,0.161,0.36,0.36v11.64H29c0.199,0,0.36,0.161,0.36,0.36v16\tC29.36,30.199,29.199,30.36,29,30.36z M3.36,29.64h25.28V14.36h-9.28V27h-0.72V2.36H3.36V29.64z M25,26.36h-2\tc-0.199,0-0.36-0.161-0.36-0.36v-2c0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2\tC25.36,26.199,25.199,26.36,25,26.36z M23.36,25.64h1.279v-1.28H23.36V25.64z M15,26.36h-2c-0.199,0-0.36-0.161-0.36-0.36v-2\tc0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2C15.36,26.199,15.199,26.36,15,26.36z M13.36,25.64h1.28v-1.28\th-1.28V25.64z M9,26.36H7c-0.199,0-0.36-0.161-0.36-0.36v-2c0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2\tC9.36,26.199,9.199,26.36,9,26.36z M7.36,25.64h1.28v-1.28H7.36V25.64z M25,20.36h-2c-0.199,0-0.36-0.161-0.36-0.36v-2\tc0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2C25.36,20.199,25.199,20.36,25,20.36z M23.36,19.64h1.279v-1.28\tH23.36V19.64z M15,20.36h-2c-0.199,0-0.36-0.161-0.36-0.36v-2c0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2\tC15.36,20.199,15.199,20.36,15,20.36z M13.36,19.64h1.28v-1.28h-1.28V19.64z M9,20.36H7c-0.199,0-0.36-0.161-0.36-0.36v-2\tc0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2C9.36,20.199,9.199,20.36,9,20.36z M7.36,19.64h1.28v-1.28H7.36\tV19.64z M15,14.36h-2c-0.199,0-0.36-0.161-0.36-0.36v-2c0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2\tC15.36,14.199,15.199,14.36,15,14.36z M13.36,13.64h1.28v-1.28h-1.28V13.64z M9,14.36H7c-0.199,0-0.36-0.161-0.36-0.36v-2\tc0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2C9.36,14.199,9.199,14.36,9,14.36z M7.36,13.64h1.28v-1.28H7.36\tV13.64z M15,8.36h-2c-0.199,0-0.36-0.161-0.36-0.36V6c0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2\tC15.36,8.199,15.199,8.36,15,8.36z M13.36,7.64h1.28V6.36h-1.28V7.64z M9,8.36H7C6.801,8.36,6.64,8.199,6.64,8V6\tc0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2C9.36,8.199,9.199,8.36,9,8.36z M7.36,7.64h1.28V6.36H7.36V7.64z"
  })), children);
});

module.exports = RealEstate;
