// @carbon/pictograms-react/lib/chart--parallel/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const ChartParallel = /*#__PURE__*/React__default["default"].forwardRef(function ChartParallel(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31.36,31h-0.72V18.809l-9.28,8.351V31h-0.72v-3.827l-9.28-7.424V31h-0.72V19.869l-9.28,9.28V31\tH0.64V1h0.72v6.468l9.28-3.712V1h0.72v2.756l9.28,3.712V1h0.721v6.417l9.279-4.64V1h0.721L31.36,31L31.36,31z M1.36,18.173v9.958\tl9.28-9.28v-8.102L1.36,18.173z M11.36,18.827l9.28,7.424v-6.102l-9.28-9.28V18.827z M21.36,20.14v6.052l9.279-8.352V9.931\tL21.36,20.14z M11.36,9.851l9.28,9.28V8.244l-9.28-3.712V9.851z M21.36,8.223v10.846l9.28-10.208V3.583L21.36,8.223z M1.36,8.244\tv9.007l9.28-7.424V4.532L1.36,8.244z"
  })), children);
});

module.exports = ChartParallel;
