// @carbon/pictograms-react/lib/flexible/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Flexible = /*#__PURE__*/React__default["default"].forwardRef(function Flexible(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16.36,31h-0.72v-5c0-5.713,4.647-10.36,10.36-10.36\th4.131l-5.386-5.386l0.51-0.509L31.509,16l-6.254,6.255l-0.51-0.51l5.386-5.385H26c-5.315,0-9.64,4.325-9.64,9.64V31z M6.746,22.255\tL0.491,16l6.255-6.254l0.509,0.509L1.869,15.64H6c3.289,0,6.31,1.51,8.288,4.144l-0.576,0.432C11.872,17.766,9.061,16.36,6,16.36\tH1.869l5.385,5.385L6.746,22.255z M16.36,16h-0.72V1.869l-5.386,5.385L9.746,6.746L16,0.491l6.255,6.255l-0.51,0.509L16.36,1.869V16\tz"
  })), children);
});

module.exports = Flexible;
