// @carbon/pictograms-react/es/test--tubes/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const TestTubes = /*#__PURE__*/React.forwardRef(function TestTubes(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M26,31.36c-1.853,0-3.36-1.508-3.36-3.36V6.36h-3.28V28c0,1.853-1.508,3.36-3.36,3.36\ts-3.36-1.508-3.36-3.36V6.36H9.36V28c0,1.853-1.507,3.36-3.36,3.36S2.64,29.853,2.64,28V6.36H1C0.801,6.36,0.64,6.199,0.64,6\tS0.801,5.64,1,5.64h1.64V1.36H2C1.801,1.36,1.64,1.199,1.64,1S1.801,0.64,2,0.64h8c0.199,0,0.36,0.161,0.36,0.36\tS10.199,1.36,10,1.36H9.36v4.28h3.28V1.36H12c-0.199,0-0.36-0.161-0.36-0.36S11.801,0.64,12,0.64h8c0.199,0,0.36,0.161,0.36,0.36\tS20.199,1.36,20,1.36h-0.64v4.28h3.279V1.36H22c-0.199,0-0.36-0.161-0.36-0.36S21.801,0.64,22,0.64h8c0.199,0,0.36,0.161,0.36,0.36\tS30.199,1.36,30,1.36h-0.64v4.28H31c0.199,0,0.36,0.161,0.36,0.36S31.199,6.36,31,6.36h-1.64V28C29.36,29.853,27.853,31.36,26,31.36\tz M23.36,13.36V28c0,1.456,1.184,2.64,2.64,2.64s2.64-1.184,2.64-2.64V13.36H23.36z M13.36,23.36V28c0,1.456,1.185,2.64,2.64,2.64\tc1.456,0,2.64-1.184,2.64-2.64v-4.64H13.36z M3.36,17.36V28c0,1.456,1.185,2.64,2.64,2.64S8.64,29.456,8.64,28V17.36H3.36z M13.36,22.64h5.28V6.36h-5.28C13.36,6.36,13.36,22.64,13.36,22.64z M3.36,16.64h5.28V6.36H3.36V16.64z M23.36,12.64h5.279V6.36\tH23.36V12.64z M23.36,5.64h5.279V1.36H23.36V5.64z M13.36,5.64h5.28V1.36h-5.28C13.36,1.36,13.36,5.64,13.36,5.64z M3.36,5.64h5.28\tV1.36H3.36V5.64z"
  })), children);
});

export { TestTubes as default };
