// @carbon/pictograms-react/es/folder/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Folder = /*#__PURE__*/React.forwardRef(function Folder(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M30,27.36H2c-0.75,0-1.36-0.61-1.36-1.36V4c0-0.75,0.61-1.36,1.36-1.36h8.5c0.425,0,0.833,0.203,1.088,0.543\tl1.416,2.223C13.108,5.544,13.299,5.64,13.5,5.64H30c0.75,0,1.36,0.61,1.36,1.36v19C31.36,26.75,30.75,27.36,30,27.36z M2,3.36\tC1.647,3.36,1.36,3.647,1.36,4v22c0,0.353,0.287,0.64,0.64,0.64h28c0.353,0,0.64-0.287,0.64-0.64V7c0-0.353-0.287-0.64-0.64-0.64\tH13.5c-0.426,0-0.833-0.203-1.088-0.544l-1.416-2.223C10.892,3.456,10.7,3.36,10.5,3.36H2z"
  })), children);
});

export { Folder as default };
