// @carbon/pictograms-react/lib/repeat/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Repeat = /*#__PURE__*/React__default["default"].forwardRef(function Repeat(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M4.746,31.255L0.491,27l4.255-4.255l0.509,0.51L1.869,26.64H29c0.904,0,1.64-0.735,1.64-1.64v-9h0.721v9 c0,1.302-1.059,2.36-2.36,2.36H1.869l3.385,3.385L4.746,31.255z M1.36,16H0.64V7c0-1.301,1.059-2.36,2.36-2.36h27.131l-3.386-3.386 l0.51-0.509L31.509,5l-4.254,4.254l-0.51-0.509l3.386-3.386H3C2.096,5.36,1.36,6.096,1.36,7V16z"
  })), children);
});

module.exports = Repeat;
