// @carbon/pictograms-react/lib/fairness/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Fairness = /*#__PURE__*/React__default["default"].forwardRef(function Fairness(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M26,29.36H6v-0.72h6.659c0.167-1.565,1.415-2.813,2.981-2.98V11.333c-1.131-0.174-2-1.154-2-2.333\ts0.869-2.159,2-2.333V3.36H6.136l4.173,6.955c0.034,0.056,0.051,0.12,0.051,0.185c0,2.68-2.18,4.86-4.86,4.86s-4.86-2.18-4.86-4.86\tc0-0.065,0.018-0.129,0.051-0.185l4.5-7.5C5.256,2.707,5.374,2.64,5.5,2.64h21c0.126,0,0.243,0.066,0.309,0.175l4.5,7.5\tc0.033,0.056,0.052,0.12,0.052,0.185c0,2.68-2.181,4.86-4.86,4.86s-4.86-2.18-4.86-4.86c0-0.065,0.019-0.129,0.052-0.185\tl4.173-6.955H16.36v3.308c1.131,0.174,2,1.154,2,2.333s-0.869,2.159-2,2.333v14.327c1.566,0.167,2.814,1.415,2.981,2.98H26V29.36z M13.384,28.64h5.231c-0.176-1.286-1.281-2.28-2.615-2.28C14.667,26.36,13.561,27.354,13.384,28.64z M22.376,10.86\tc0.183,2.115,1.963,3.78,4.124,3.78s3.941-1.665,4.124-3.78H22.376z M1.375,10.86c0.183,2.115,1.963,3.78,4.125,3.78\ts3.941-1.665,4.125-3.78H1.375z M16,7.36c-0.904,0-1.64,0.736-1.64,1.64s0.736,1.64,1.64,1.64c0.904,0,1.64-0.736,1.64-1.64\tS16.904,7.36,16,7.36z M22.636,10.14h7.729L26.5,3.7L22.636,10.14z M1.636,10.14h7.729L5.5,3.7L1.636,10.14z"
  })), children);
});

module.exports = Fairness;
