// @carbon/pictograms-react/es/sustainability--02/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Sustainability_02 = /*#__PURE__*/React.forwardRef(function Sustainability_02(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M31,14.64h-2c-2.198,0-3.36,1.162-3.36,3.36\tv2v1c0,0.904-0.735,1.64-1.64,1.64h-7.64v-8.28H21c1.302,0,2.36-1.059,2.36-2.36v-0.64H25c2.198,0,3.36-1.162,3.36-3.36V6\tc0-0.199-0.161-0.36-0.36-0.36h-2c-2.198,0-3.36,1.162-3.36,3.36v2v1c0,0.904-0.735,1.64-1.64,1.64h-4.64V8V6.36H18\tc2.198,0,3.36-1.162,3.36-3.36V1c0-0.199-0.161-0.36-0.36-0.36h-2c-2.091,0-3.231,1.062-3.339,3.056\tC15.101,3.008,14.215,2.64,13,2.64h-2c-0.199,0-0.36,0.161-0.36,0.36v2c0,2.198,1.162,3.36,3.36,3.36h1.64v9.28H11\tc-0.904,0-1.64-0.735-1.64-1.64v-1v-2c0-2.198-1.162-3.36-3.36-3.36H4c-0.199,0-0.36,0.161-0.36,0.36v2\tc0,2.198,1.162,3.36,3.36,3.36h1.64V16c0,1.302,1.059,2.36,2.36,2.36h4.64v8.279H8c-0.904,0-1.64-0.735-1.64-1.64v-1v-2\tc0-2.198-1.162-3.36-3.36-3.36H1c-0.199,0-0.36,0.161-0.36,0.36v2c0,2.198,1.162,3.36,3.36,3.36h1.64V25\tc0,1.302,1.059,2.36,2.36,2.36h7.64V31h0.72v-7.64H24c1.302,0,2.36-1.059,2.36-2.36v-0.64H28c2.198,0,3.36-1.162,3.36-3.36v-2\tC31.36,14.801,31.199,14.64,31,14.64z M26,6.36h1.64V8c0,1.801-0.839,2.64-2.64,2.64h-1.131l2.386-2.386l-0.51-0.509L23.36,10.13V9\tC23.36,7.199,24.199,6.36,26,6.36z M4.36,12v-1.64H6c1.801,0,2.64,0.839,2.64,2.64v1.131l-2.386-2.386l-0.509,0.509l2.386,2.386H7\tC5.199,14.64,4.36,13.801,4.36,12z M1.36,21v-1.64H3c1.801,0,2.64,0.839,2.64,2.64v1.131l-2.386-2.386l-0.509,0.51L5.13,23.64H4\tC2.199,23.64,1.36,22.801,1.36,21z M19,1.36h1.64V3c0,1.801-0.839,2.64-2.64,2.64h-1.131l2.386-2.386l-0.51-0.509L16.36,5.131V4\tC16.36,2.199,17.199,1.36,19,1.36z M11.36,5V3.36H13c1.801,0,2.64,0.839,2.64,2.64v1.131l-2.386-2.386l-0.509,0.509l2.386,2.386H14\tC12.199,7.64,11.36,6.801,11.36,5z M30.64,17c0,1.801-0.839,2.64-2.64,2.64h-1.13l2.385-2.385l-0.51-0.509L26.36,19.13V18\tc0-1.801,0.839-2.64,2.64-2.64h1.64V17z"
  })), children);
});

export { Sustainability_02 as default };
