// @carbon/pictograms-react/lib/load-balancer/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const LoadBalancer = /*#__PURE__*/React__default["default"].forwardRef(function LoadBalancer(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M29,30.36c-0.75,0-1.36-0.61-1.36-1.36\tc0-0.625,0.425-1.153,1-1.312V3h0.721v24.688c0.575,0.158,1,0.687,1,1.312C30.36,29.75,29.75,30.36,29,30.36z M29,28.36\tc-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S29.353,28.36,29,28.36z M3.36,29H2.64V4.312\tc-0.576-0.158-1-0.686-1-1.312c0-0.75,0.61-1.36,1.36-1.36S4.36,2.25,4.36,3c0,0.625-0.424,1.153-1,1.312\tC3.36,4.312,3.36,29,3.36,29z M3,2.36C2.647,2.36,2.36,2.647,2.36,3S2.647,3.64,3,3.64S3.64,3.353,3.64,3S3.353,2.36,3,2.36z M24,25.36c-0.75,0-1.36-0.61-1.36-1.36c0-0.625,0.425-1.153,1-1.312V8h0.721v14.688c0.575,0.158,1,0.687,1,1.312\tC25.36,24.75,24.75,25.36,24,25.36z M24,23.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64\tS24.353,23.36,24,23.36z M8.36,24H7.64V9.312c-0.576-0.158-1-0.686-1-1.312c0-0.75,0.61-1.36,1.36-1.36S9.36,7.25,9.36,8\tc0,0.625-0.424,1.153-1,1.312V24z M8,7.36C7.647,7.36,7.36,7.647,7.36,8c0,0.353,0.287,0.64,0.64,0.64c0.353,0,0.64-0.287,0.64-0.64\tS8.353,7.36,8,7.36z M19,20.36c-0.75,0-1.36-0.61-1.36-1.36c0-0.625,0.425-1.153,1-1.312V13h0.721v4.688\tc0.575,0.158,1,0.687,1,1.312C20.36,19.75,19.75,20.36,19,20.36z M19,18.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64\ts0.64-0.287,0.64-0.64S19.353,18.36,19,18.36z M13.36,19h-0.72v-4.688c-0.576-0.158-1-0.686-1-1.312c0-0.75,0.61-1.36,1.36-1.36\ts1.36,0.61,1.36,1.36c0,0.625-0.424,1.153-1,1.312C13.36,14.312,13.36,19,13.36,19z M13,12.36c-0.353,0-0.64,0.287-0.64,0.64\ts0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S13.353,12.36,13,12.36z"
  })), children);
});

module.exports = LoadBalancer;
