// @carbon/pictograms-react/lib/telecom/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Telecom = /*#__PURE__*/React__default["default"].forwardRef(function Telecom(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M29,31.36H3v-0.72h6.633l5.013-18.708C14.038,11.505,13.64,10.798,13.64,10c0-1.301,1.059-2.36,2.36-2.36\tc1.302,0,2.36,1.059,2.36,2.36c0,0.798-0.398,1.505-1.007,1.932l5.014,18.708H29V31.36z M10.379,30.64h11.243l-0.879-3.279h-9.485\tL10.379,30.64z M11.451,26.64h9.099l-1.146-4.279h-6.806L11.451,26.64z M12.791,21.64h6.419l-1.146-4.279h-4.126L12.791,21.64z M14.13,16.64h3.74l-1.175-4.385c-0.439,0.136-0.951,0.136-1.39,0L14.13,16.64z M16,8.36c-0.904,0-1.64,0.736-1.64,1.64\ts0.736,1.64,1.64,1.64c0.904,0,1.64-0.736,1.64-1.64S16.904,8.36,16,8.36z M7.276,18.755C4.931,16.419,3.64,13.31,3.64,10\tc0-3.301,1.286-6.405,3.62-8.74l0.509,0.509C5.571,3.968,4.36,6.891,4.36,10c0,3.117,1.216,6.045,3.424,8.245L7.276,18.755z M24.74,18.74l-0.51-0.51c2.199-2.198,3.409-5.121,3.409-8.23c0-3.11-1.21-6.033-3.409-8.231l0.51-0.509\tc2.334,2.334,3.62,5.438,3.62,8.74S27.074,16.405,24.74,18.74z M9.382,16.618C7.614,14.851,6.64,12.5,6.64,10\ts0.974-4.851,2.742-6.618l0.509,0.509C8.259,5.522,7.36,7.692,7.36,10s0.899,4.478,2.531,6.109L9.382,16.618z M22.619,16.618\tl-0.51-0.509c1.632-1.632,2.53-3.802,2.53-6.109s-0.898-4.477-2.53-6.109l0.51-0.509C24.387,5.15,25.36,7.5,25.36,10\tS24.387,14.85,22.619,16.618z M20.497,14.497l-0.51-0.509c1.065-1.066,1.652-2.482,1.652-3.988s-0.587-2.922-1.652-3.988l0.51-0.509\tC21.698,6.704,22.36,8.301,22.36,10S21.698,13.296,20.497,14.497z M11.503,14.497C10.302,13.296,9.64,11.699,9.64,10\ts0.662-3.296,1.863-4.497l0.509,0.509C10.947,7.077,10.36,8.494,10.36,10s0.587,2.923,1.652,3.988L11.503,14.497z"
  })), children);
});

module.exports = Telecom;
