// @carbon/pictograms-react/lib/flag/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Flag = /*#__PURE__*/React__default["default"].forwardRef(function Flag(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M4.36,31H3.64V1c0-0.199,0.161-0.36,0.36-0.36h24\tc0.131,0,0.251,0.071,0.314,0.186c0.063,0.114,0.061,0.254-0.009,0.365L23.425,9l4.881,7.809c0.069,0.111,0.072,0.25,0.009,0.366\tC28.251,17.289,28.131,17.36,28,17.36H4.36V31z M4.36,16.64h22.991l-4.656-7.449c-0.072-0.117-0.072-0.265,0-0.382l4.656-7.449H4.36\tV16.64z"
  })), children);
});

module.exports = Flag;
