// @carbon/pictograms-react/lib/hurricane/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Hurricane = /*#__PURE__*/React__default["default"].forwardRef(function Hurricane(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M6,31.36v-0.72c9.684,0,15.862-3.374,17.898-9.622C22.235,23.626,19.316,25.36,16,25.36\tc-5.161,0-9.36-4.199-9.36-9.36C6.64,6.238,13.696,0.64,26,0.64v0.72c-9.684,0-15.862,3.375-17.898,9.623\tC9.765,8.374,12.684,6.64,16,6.64c5.161,0,9.36,4.199,9.36,9.36C25.36,25.762,18.304,31.36,6,31.36z M16,7.36\tc-4.764,0-8.64,3.876-8.64,8.64s3.876,8.64,8.64,8.64s8.64-3.876,8.64-8.64S20.764,7.36,16,7.36z M16,20.36\tc-2.404,0-4.36-1.956-4.36-4.36s1.956-4.36,4.36-4.36s4.36,1.956,4.36,4.36S18.404,20.36,16,20.36z M16,12.36\tc-2.007,0-3.64,1.633-3.64,3.64s1.633,3.64,3.64,3.64s3.64-1.633,3.64-3.64S18.007,12.36,16,12.36z"
  })), children);
});

module.exports = Hurricane;
