// @carbon/pictograms-react/lib/cics--transaction-server-for-z-os/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const CicsTransactionServerForZOs = /*#__PURE__*/React__default["default"].forwardRef(function CicsTransactionServerForZOs(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M18,28.36h-8v-0.72h8\tc5.315,0,9.64-4.324,9.64-9.64v-4h0.721v4C28.36,23.713,23.713,28.36,18,28.36z M4,24.36c-1.301,0-2.36-1.059-2.36-2.36\tS2.699,19.64,4,19.64S6.36,20.698,6.36,22S5.301,24.36,4,24.36z M4,20.36c-0.904,0-1.64,0.735-1.64,1.64S3.096,23.64,4,23.64\tS5.64,22.904,5.64,22S4.904,20.36,4,20.36z M16,20.36c-2.404,0-4.36-1.956-4.36-4.36s1.956-4.36,4.36-4.36s4.36,1.956,4.36,4.36\tS18.404,20.36,16,20.36z M16,12.36c-2.007,0-3.64,1.633-3.64,3.64s1.633,3.64,3.64,3.64s3.64-1.633,3.64-3.64S18.007,12.36,16,12.36\tz M4.36,18H3.64v-4C3.64,8.288,8.288,3.64,14,3.64h8v0.72h-8c-5.315,0-9.64,4.325-9.64,9.64C4.36,14,4.36,18,4.36,18z M28,12.36\tc-1.302,0-2.36-1.059-2.36-2.36S26.698,7.64,28,7.64s2.36,1.059,2.36,2.36S29.302,12.36,28,12.36z M28,8.36\tc-0.904,0-1.64,0.736-1.64,1.64s0.735,1.64,1.64,1.64s1.64-0.736,1.64-1.64S28.904,8.36,28,8.36z"
  })), children);
});

module.exports = CicsTransactionServerForZOs;
