// @carbon/pictograms-react/es/curve--exponential/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const CurveExponential = /*#__PURE__*/React.forwardRef(function CurveExponential(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M16.36,31h-0.72V16.36H1v-0.72h14.64v-2.026\tC11.388,14.36,6.412,14.36,1,14.36v-0.72c5.432,0,10.415,0,14.64-0.774V1h0.72v11.724c6.142-1.285,10.575-4.375,12.29-11.805\tl0.701,0.162c-1.823,7.902-6.514,11.102-12.991,12.397v2.162H31v0.72H16.36V31z"
  })), children);
});

export { CurveExponential as default };
