// @carbon/pictograms-react/lib/select--product/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const SelectProduct = /*#__PURE__*/React__default["default"].forwardRef(function SelectProduct(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M23,10.509l-2.255-2.255l0.51-0.509L23,9.491l3.745-3.745l0.51,0.509L23,10.509z M30.36,30V18\tc0-0.199-0.161-0.36-0.36-0.36H18c-0.199,0-0.36,0.161-0.36,0.36v12c0,0.199,0.161,0.36,0.36,0.36h12\tC30.199,30.36,30.36,30.199,30.36,30z M18.36,18.36h11.28v11.28H18.36V18.36z M14.36,30V18c0-0.199-0.161-0.36-0.36-0.36H2\tc-0.199,0-0.36,0.161-0.36,0.36v12c0,0.199,0.161,0.36,0.36,0.36h12C14.199,30.36,14.36,30.199,14.36,30z M2.36,18.36h11.28v11.28\tH2.36V18.36z M30.36,14V2c0-0.199-0.161-0.36-0.36-0.36H18c-0.199,0-0.36,0.161-0.36,0.36v12c0,0.199,0.161,0.36,0.36,0.36h12\tC30.199,14.36,30.36,14.199,30.36,14z M18.36,2.36h11.28v11.28H18.36V2.36z M14.36,14V2c0-0.199-0.161-0.36-0.36-0.36H2\tC1.801,1.64,1.64,1.801,1.64,2v12c0,0.199,0.161,0.36,0.36,0.36h12C14.199,14.36,14.36,14.199,14.36,14z M2.36,2.36h11.28v11.28\tH2.36V2.36z"
  })), children);
});

module.exports = SelectProduct;
