// @carbon/pictograms-react/lib/open-source/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const OpenSource = /*#__PURE__*/React__default["default"].forwardRef(function OpenSource(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M11,28.36c-0.047,0-0.094-0.01-0.139-0.028\tC5.867,26.25,2.64,21.409,2.64,16C2.64,8.633,8.633,2.64,16,2.64S29.36,8.633,29.36,16c0,5.409-3.228,10.25-8.222,12.332\tc-0.099,0.04-0.206,0.037-0.3-0.011c-0.094-0.047-0.163-0.132-0.188-0.234l-2-8c-0.035-0.14,0.018-0.288,0.134-0.375\tc1.18-0.886,1.855-2.238,1.855-3.712c0-2.559-2.081-4.64-4.64-4.64s-4.64,2.082-4.64,4.64c0,1.473,0.677,2.826,1.856,3.712\tc0.116,0.087,0.168,0.235,0.133,0.375l-2,8c-0.025,0.103-0.094,0.188-0.188,0.234C11.111,28.348,11.056,28.36,11,28.36z M19.406,20.139l1.84,7.361c4.506-2.054,7.394-6.522,7.394-11.5c0-6.97-5.67-12.64-12.64-12.64S3.36,9.03,3.36,16\tc0,4.978,2.888,9.446,7.394,11.5l1.84-7.361C11.349,19.116,10.64,17.621,10.64,16c0-2.956,2.404-5.36,5.36-5.36\tc2.955,0,5.36,2.404,5.36,5.36C21.36,17.621,20.65,19.116,19.406,20.139z"
  })), children);
});

module.exports = OpenSource;
