// @carbon/pictograms-react/es/ibm--z-and-linuxone-single-frame/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
let didWarnAboutDeprecation = false;
const IbmZAndLinuxoneSingleFrame = /*#__PURE__*/React.forwardRef(function IbmZAndLinuxoneSingleFrame(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  if (process.env.NODE_ENV !== "production") {
    if (!didWarnAboutDeprecation) {
      didWarnAboutDeprecation = true;
      console.warn(`The IbmZAndLinuxoneSingleFrame component has been deprecated and will be ` + `removed in the next major version of @carbon/icons-react.`);
    }
  }
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M21,30.36H11c-0.199,0-0.36-0.161-0.36-0.36V2c0-0.199,0.161-0.36,0.36-0.36h10c0.199,0,0.36,0.161,0.36,0.36v28 C21.36,30.199,21.199,30.36,21,30.36z M19.36,29.64h1.279V2.36H19.36V29.64z M12.869,29.64h5.771V24.3l-2.331,3.886 c-0.041,0.067-0.102,0.119-0.175,0.148L12.869,29.64z M11.36,23.36v6.108l4.394-1.758l2.61-4.351L11.36,23.36L11.36,23.36z M12.3,22.64h6.34v-5.56l-2.352,3.136c-0.028,0.037-0.063,0.069-0.103,0.093L12.3,22.64z M11.36,16.36v6.004l4.395-2.637 l2.526-3.368L11.36,16.36L11.36,16.36z M11.72,15.64h6.919V9.636l-4.394,2.637L11.72,15.64z M11.36,9.36v5.56l2.352-3.136 c0.028-0.038,0.063-0.069,0.103-0.093L17.7,9.36H11.36z M11.636,8.64h7.004V2.532l-4.394,1.757L11.636,8.64z M11.36,2.36V7.7 l2.332-3.885c0.041-0.067,0.102-0.12,0.175-0.149l3.265-1.306H11.36z"
  })), children);
});

export { IbmZAndLinuxoneSingleFrame as default };
