// @carbon/pictograms-react/es/data--management/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const DataManagement = /*#__PURE__*/React.forwardRef(function DataManagement(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M29,31.36h-2c-0.199,0-0.36-0.161-0.36-0.36v-2\tc0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2C29.36,31.199,29.199,31.36,29,31.36z M27.36,30.64h1.279v-1.28\tH27.36V30.64z M17,31.36h-2c-0.199,0-0.36-0.161-0.36-0.36v-2c0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2\tC17.36,31.199,17.199,31.36,17,31.36z M15.36,30.64h1.28v-1.28h-1.28V30.64z M5,31.36H3c-0.199,0-0.36-0.161-0.36-0.36v-2\tc0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2C5.36,31.199,5.199,31.36,5,31.36z M3.36,30.64h1.28v-1.28H3.36\tV30.64z M22.36,31h-0.72V1h0.721L22.36,31L22.36,31z M10.36,31H9.64V1h0.72C10.36,1,10.36,31,10.36,31z M17,24.36h-2\tc-0.199,0-0.36-0.161-0.36-0.36v-2c0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2\tC17.36,24.199,17.199,24.36,17,24.36z M15.36,23.64h1.28v-1.28h-1.28V23.64z M5,24.36H3c-0.199,0-0.36-0.161-0.36-0.36v-2\tc0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2C5.36,24.199,5.199,24.36,5,24.36z M3.36,23.64h1.28v-1.28H3.36\tV23.64z M29,17.36h-2c-0.199,0-0.36-0.161-0.36-0.36v-2c0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2\tC29.36,17.199,29.199,17.36,29,17.36z M27.36,16.64h1.279v-1.28H27.36V16.64z M17,17.36h-2c-0.199,0-0.36-0.161-0.36-0.36v-2\tc0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2C17.36,17.199,17.199,17.36,17,17.36z M15.36,16.64h1.28v-1.28\th-1.28V16.64z M5,17.36H3c-0.199,0-0.36-0.161-0.36-0.36v-2c0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2\tC5.36,17.199,5.199,17.36,5,17.36z M3.36,16.64h1.28v-1.28H3.36V16.64z M29,10.36h-2c-0.199,0-0.36-0.161-0.36-0.36V8\tc0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2C29.36,10.199,29.199,10.36,29,10.36z M27.36,9.64h1.279V8.36H27.36\tV9.64z M17,10.36h-2c-0.199,0-0.36-0.161-0.36-0.36V8c0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2\tC17.36,10.199,17.199,10.36,17,10.36z M15.36,9.64h1.28V8.36h-1.28V9.64z M29,3.36h-2c-0.199,0-0.36-0.161-0.36-0.36V1\tc0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2C29.36,3.199,29.199,3.36,29,3.36z M27.36,2.64h1.279V1.36H27.36\tV2.64z M5,3.36H3C2.801,3.36,2.64,3.199,2.64,3V1c0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2\tC5.36,3.199,5.199,3.36,5,3.36z M3.36,2.64h1.28V1.36H3.36V2.64z"
  })), children);
});

export { DataManagement as default };
