// @carbon/pictograms-react/es/curve--quadratic/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const CurveQuadratic = /*#__PURE__*/React.forwardRef(function CurveQuadratic(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M21.591,27.473L19.109,24h1.276l1.777,2.605\th0.041L24.047,24h1.208l-2.469,3.391L25.308,31h-1.275l-1.818-2.74h-0.041L20.208,31H19L21.591,27.473z M29.574,26.035H26.8v-0.699\tl1.255-1.1c0.424-0.373,0.619-0.648,0.619-0.997v-0.075c0-0.35-0.264-0.545-0.59-0.545c-0.413,0-0.607,0.27-0.699,0.596\tl-0.653-0.252c0.172-0.533,0.602-0.997,1.416-0.997c0.842,0,1.323,0.504,1.323,1.192c0,0.664-0.453,1.089-1.008,1.547l-0.842,0.693\th1.954L29.574,26.035L29.574,26.035z M16.36,31h-0.72V16.36H1v-0.72h14.64V8.853c-1.198-3.445-1.921-6.478-1.991-6.771l0.701-0.165\tc0.277,1.177,0.72,2.828,1.29,4.639V1h0.72v7.723c0.872,2.482,1.936,5.027,3.105,6.917h6.886c2.167-3.883,3.734-10.526,4.295-13.703\tl0.709,0.125c-0.087,0.491-1.555,8.642-4.198,13.578H31v0.72h-4.253c-1.076,1.786-2.329,3-3.747,3c-1.414,0-2.736-1.215-3.915-3\tH16.36V31z M19.938,16.36c0.976,1.39,2.013,2.28,3.062,2.28c1.047,0,2.029-0.89,2.921-2.28H19.938z M16.36,15.64h2.275\tc-0.848-1.428-1.612-3.125-2.275-4.827V15.64z"
  })), children);
});

export { CurveQuadratic as default };
