// @carbon/pictograms-react/lib/math--curve/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const MathCurve = /*#__PURE__*/React__default["default"].forwardRef(function MathCurve(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31,31.36H1c-0.199,0-0.36-0.161-0.36-0.36V1h0.72v29.64H31V31.36z M4.254,28.255l-0.509-0.51\tl5.67-5.67c-4.32-2.401-6.044,0.019-6.117,0.126l-0.598-0.4c0.021-0.033,2.183-3.149,7.241-0.252l6.123-6.123\tc-2.028-4.19-3.885-8.114-2.319-9.681c2.106-2.11,7.807,1.767,9.218,2.782l5.781-5.782l0.51,0.509l-5.669,5.669\tc1.436,0.779,2.747,1.019,3.883,0.691c1.515-0.436,2.203-1.763,2.21-1.776l0.644,0.323c-0.033,0.065-0.824,1.612-2.643,2.142\tc-1.37,0.398-2.923,0.112-4.621-0.852l-6.123,6.123c2.027,4.19,3.886,8.114,2.319,9.681c-2.11,2.103-7.809-1.767-9.218-2.782\tL4.254,28.255z M10.552,21.957c2.103,1.492,6.725,4.256,8.193,2.788c1.183-1.183-0.626-5.068-2.349-8.632L10.552,21.957z M15.461,5.838c-0.498,0-0.915,0.124-1.207,0.416c-1.182,1.183,0.627,5.069,2.349,8.633l5.845-5.844\tC20.764,7.848,17.462,5.838,15.461,5.838z"
  })), children);
});

module.exports = MathCurve;
