// @carbon/pictograms-react/es/ai--transparency/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const AiTransparency = /*#__PURE__*/React.forwardRef(function AiTransparency(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M31,31.36H7c-0.199,0-0.36-0.161-0.36-0.36v-5.64H5v-0.72h4v0.721H7.36v5.279h23.28V7.36H25\tc-0.199,0-0.36-0.161-0.36-0.36V1.36H7.36v5.28H9v0.72H5V6.64h1.64V1c0-0.199,0.161-0.36,0.36-0.36h18\tc0.096,0,0.188,0.038,0.255,0.105l6,6C31.322,6.813,31.36,6.904,31.36,7v24C31.36,31.199,31.199,31.36,31,31.36z M25.36,6.64h4.771\tL25.36,1.869V6.64z M19,25.36h-8v-0.72h7.64V7.36H11V6.64h8c0.199,0,0.36,0.161,0.36,0.36v3.64H27v0.72h-7.64v3.28H27v0.72h-7.64\tv3.28H27v0.721h-7.64v3.279H27v0.721h-7.64V25C19.36,25.199,19.199,25.36,19,25.36z M3,25.36H1c-0.199,0-0.36-0.161-0.36-0.36v-2\th0.72v1.64H3V25.36z M14.86,21h-0.72v-0.5c0-1.823-1.235-3.452-3.003-3.96c-0.142-0.041-0.245-0.165-0.259-0.312\ts0.063-0.288,0.195-0.355c0.812-0.413,1.317-1.226,1.317-2.123c0-1.318-1.072-2.39-2.39-2.39s-2.39,1.072-2.39,2.39\tc0,0.897,0.504,1.71,1.317,2.123c0.132,0.067,0.209,0.208,0.195,0.355s-0.117,0.271-0.259,0.312C7.095,17.048,5.86,18.677,5.86,20.5\tV21H5.14v-0.5c0-1.898,1.137-3.616,2.839-4.398C7.294,15.52,6.89,14.669,6.89,13.75c0-1.715,1.395-3.11,3.11-3.11\ts3.11,1.395,3.11,3.11c0,0.919-0.404,1.77-1.089,2.352c1.702,0.782,2.839,2.5,2.839,4.398V21z M1.36,21H0.64v-4h0.72V21z M1.36,15\tH0.64v-4h0.72V15z M1.36,9H0.64V7c0-0.199,0.161-0.36,0.36-0.36h2v0.72H1.36V9z"
  })), children);
});

export { AiTransparency as default };
