// @carbon/pictograms-react/lib/target--area/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const TargetArea = /*#__PURE__*/React__default["default"].forwardRef(function TargetArea(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M15,16c0-0.552,0.448-1,1-1s1,0.448,1,1s-0.448,1-1,1S15,16.553,15,16z M26,30.64H1.36V1H0.64v30\tc0,0.199,0.161,0.36,0.36,0.36h25V30.64z M31.36,1c0-0.199-0.161-0.36-0.36-0.36H6v0.72h24.64V31h0.721L31.36,1L31.36,1z M26.36,16\tc0-5.712-4.647-10.36-10.36-10.36C10.288,5.64,5.64,10.288,5.64,16c0,5.713,4.647,10.36,10.36,10.36S26.36,21.713,26.36,16z M25.64,16c0,5.315-4.324,9.64-9.64,9.64S6.36,21.315,6.36,16S10.685,6.36,16,6.36S25.64,10.685,25.64,16z M21.36,16\tc0-2.956-2.405-5.36-5.36-5.36c-2.956,0-5.36,2.404-5.36,5.36c0,2.955,2.404,5.36,5.36,5.36C18.955,21.36,21.36,18.955,21.36,16z M20.64,16c0,2.559-2.081,4.64-4.64,4.64s-4.64-2.081-4.64-4.64s2.082-4.64,4.64-4.64S20.64,13.441,20.64,16z"
  })), children);
});

module.exports = TargetArea;
