// @carbon/pictograms-react/lib/threat--management/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const ThreatManagement = /*#__PURE__*/React__default["default"].forwardRef(function ThreatManagement(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M14,16.385H8v-0.72h6V16.385z M24,15.665h-6\tv0.72h6V15.665z M16.36,8.025h-0.72v6h0.72V8.025z M16.36,18.025h-0.72v6h0.72V18.025z M27.36,20.025V6.811L16,0.615L4.64,6.811\tv13.214c0,6.264,5.096,11.36,11.36,11.36C22.264,31.386,27.36,26.289,27.36,20.025z M16,1.435l10.64,5.804v12.787\tc0,5.867-4.772,10.64-10.64,10.64c-5.867,0-10.64-4.772-10.64-10.64V7.239L16,1.435z M14.121,21.425\tc-1.624-0.581-2.939-1.897-3.521-3.521l-0.678,0.242c0.653,1.824,2.132,3.304,3.957,3.956L14.121,21.425z M22.078,18.146\tL21.4,17.904c-0.582,1.624-1.898,2.939-3.521,3.521l0.242,0.678C19.944,21.45,21.424,19.972,22.078,18.146z M14.122,10.625\tl-0.243-0.678c-1.824,0.653-3.303,2.132-3.956,3.956l0.678,0.243C11.182,12.522,12.498,11.206,14.122,10.625z M22.078,13.903\tc-0.653-1.824-2.133-3.303-3.957-3.956l-0.242,0.678c1.623,0.581,2.939,1.897,3.521,3.521L22.078,13.903z"
  })), children);
});

module.exports = ThreatManagement;
