// @carbon/pictograms-react/lib/americas/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Americas = /*#__PURE__*/React__default["default"].forwardRef(function Americas(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,31.36C7.53,31.36,0.64,24.47,0.64,16S7.53,0.64,16,0.64S31.36,7.53,31.36,16S24.47,31.36,16,31.36z M12.558,30.231c1.104,0.267,2.257,0.408,3.442,0.408c7.27,0,13.318-5.325,14.45-12.279H28c-0.126,0-0.243-0.066-0.309-0.174\tl-3.349-5.542c-0.067-0.112-0.068-0.252-0.004-0.365l3.103-5.403c-2.54-3.179-6.375-5.28-10.698-5.498l2.065,3.436\tc0.082,0.137,0.064,0.311-0.043,0.429l-5.321,5.805l6.866,11.771c0.086,0.147,0.058,0.334-0.068,0.448L12.558,30.231z M8.981,28.845\tc0.868,0.477,1.79,0.867,2.753,1.161l7.808-7.076l-6.853-11.749c-0.08-0.136-0.061-0.308,0.046-0.424l5.318-5.802l-2.16-3.595\tC10.345,1.4,5.521,4.542,3.072,9.134l1.46,2.506H10c0.134,0,0.257,0.074,0.319,0.193l2.607,5c0.078,0.15,0.041,0.334-0.089,0.443\tl-3.398,2.826l1.89,4.251c0.046,0.104,0.041,0.224-0.015,0.322L8.981,28.845z M2.689,9.906C1.836,11.763,1.36,13.827,1.36,16\tc0,5.274,2.804,9.906,6.999,12.483l2.239-4.002l-1.927-4.335c-0.065-0.147-0.025-0.32,0.099-0.423l3.384-2.815L9.782,12.36H4.325\tc-0.128,0-0.247-0.068-0.311-0.179L2.689,9.906z M28.203,17.64h2.346c0.061-0.538,0.091-1.085,0.091-1.64\tc0-3.167-1.011-6.102-2.727-8.5l-2.844,4.954L28.203,17.64z"
  })), children);
});

module.exports = Americas;
