// @carbon/pictograms-react/es/download--02/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path, _path2;
const Download_02 = /*#__PURE__*/React.forwardRef(function Download_02(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M31,31.36H1c-0.199,0-0.36-0.161-0.36-0.36V12c0-0.199,0.161-0.36,0.36-0.36h8v0.72H1.36v18.28h29.28V12.36H23v-0.72h8 c0.199,0,0.36,0.161,0.36,0.36v19C31.36,31.199,31.199,31.36,31,31.36z"
  })), _path2 || (_path2 = /*#__PURE__*/React.createElement("path", {
    d: "M16 25.509L9.746 19.255 10.254 18.745 15.64 24.131 15.64 1 16.36 1 16.36 24.131 21.745 18.745 22.255 19.255z"
  })), children);
});

export { Download_02 as default };
