// @carbon/pictograms-react/lib/ibm--rpa/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const IbmRpa = /*#__PURE__*/React__default["default"].forwardRef(function IbmRpa(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M13,10c0,0.552-0.448,1-1,1s-1-0.448-1-1s0.448-1,1-1\tS13,9.448,13,10z M20,9c-0.552,0-1,0.448-1,1s0.448,1,1,1s1-0.448,1-1S20.552,9,20,9z M28.36,31h-0.72V20.36H4.36V31H3.64V20\tc0-0.199,0.161-0.36,0.36-0.36h8.64v-2.28H8c-0.199,0-0.36-0.161-0.36-0.36v-6.64H5V9.64h2.64V5c0-0.199,0.161-0.36,0.36-0.36h3.64\tV1h0.72v3.64h7.28V1h0.721v3.64H24c0.199,0,0.36,0.161,0.36,0.36v4.64H27v0.72h-2.64V17c0,0.199-0.161,0.36-0.36,0.36h-4.64v2.279\tH28c0.199,0,0.36,0.161,0.36,0.36V31z M13.36,19.64h5.28v-2.28h-5.28C13.36,17.36,13.36,19.64,13.36,19.64z M8.36,16.64h15.28V5.36\tH8.36V16.64z"
  })), children);
});

module.exports = IbmRpa;
