// @carbon/pictograms-react/lib/coupon/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Coupon = /*#__PURE__*/React__default["default"].forwardRef(function Coupon(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31,28.36H1c-0.133,0-0.255-0.073-0.317-0.19\ts-0.056-0.259,0.018-0.369L2.567,25L0.7,22.199c-0.081-0.12-0.081-0.278,0-0.398L2.567,19L0.7,16.2\tc-0.081-0.121-0.081-0.278,0-0.399L2.567,13L0.7,10.2c-0.081-0.121-0.081-0.278,0-0.399L2.567,7L0.701,4.2\tC0.627,4.089,0.62,3.947,0.683,3.83S0.867,3.64,1,3.64h30c0.199,0,0.36,0.161,0.36,0.36v24C31.36,28.199,31.199,28.36,31,28.36z M9.36,27.64h21.28V4.36H9.36V6H8.64V4.36H1.673L3.299,6.8c0.081,0.121,0.081,0.278,0,0.399L1.433,10L3.3,12.8\tc0.081,0.121,0.081,0.278,0,0.399L1.433,16L3.3,18.801c0.081,0.12,0.081,0.278,0,0.398L1.433,22L3.3,24.801\tc0.081,0.12,0.081,0.278,0,0.398l-1.626,2.44H8.64V26h0.72C9.36,26,9.36,27.64,9.36,27.64z M9.36,24H8.64v-4h0.72\tC9.36,20,9.36,24,9.36,24z M20.36,22h-0.72v-1.64H16v-0.72h6c0.904,0,1.64-0.735,1.64-1.64s-0.735-1.64-1.64-1.64h-4\tc-1.301,0-2.36-1.059-2.36-2.36s1.059-2.36,2.36-2.36h1.64V10h0.721v1.64H24v0.72h-6c-0.904,0-1.64,0.736-1.64,1.64\ts0.736,1.64,1.64,1.64h4c1.302,0,2.36,1.059,2.36,2.36c0,1.302-1.059,2.36-2.36,2.36h-1.64V22z M9.36,18H8.64v-4h0.72\tC9.36,14,9.36,18,9.36,18z M9.36,12H8.64V8h0.72C9.36,8,9.36,12,9.36,12z"
  })), children);
});

module.exports = Coupon;
