// @carbon/pictograms-react/lib/rank/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Rank = /*#__PURE__*/React__default["default"].forwardRef(function Rank(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31,31.36H13v-0.72h18V31.36z M4,29.86c-1.025,0-1.86-0.835-1.86-1.86c0-0.902,0.646-1.657,1.5-1.825v-7.59\tC2.371,18.409,1.39,17.317,1.39,16s0.98-2.409,2.25-2.585V7.341c-1.684-0.18-3-1.61-3-3.341c0-1.853,1.507-3.36,3.36-3.36\tS7.36,2.147,7.36,4c0,1.731-1.316,3.161-3,3.341v6.074c1.266,0.18,2.25,1.281,2.25,2.585c0,1.317-0.98,2.409-2.25,2.585v7.591\tc0.852,0.171,1.5,0.931,1.5,1.824C5.86,29.025,5.025,29.86,4,29.86z M4,26.86c-0.628,0-1.14,0.511-1.14,1.14S3.372,29.14,4,29.14\tS5.14,28.629,5.14,28c0-0.613-0.5-1.124-1.114-1.14H4z M4,14.11c-1.042,0-1.89,0.848-1.89,1.89S2.958,17.89,4,17.89\tS5.89,17.042,5.89,16c0-1.018-0.829-1.865-1.85-1.89H4z M4,1.36C2.544,1.36,1.36,2.544,1.36,4S2.544,6.64,4,6.64S6.64,5.456,6.64,4\tS5.456,1.36,4,1.36z M31,28.36H13v-0.72h18V28.36z M31,25.36H13v-0.72h18V25.36z M31,19.36H13v-0.72h18V19.36z M31,16.36H13v-0.72\th18V16.36z M31,13.36H13v-0.72h18V13.36z M31,7.36H13V6.64h18V7.36z M31,4.36H13V3.64h18V4.36z M31,1.36H13V0.64h18V1.36z"
  })), children);
});

module.exports = Rank;
