// @carbon/pictograms-react/es/systems-devops--deploy/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const SystemsDevopsDeploy = /*#__PURE__*/React.forwardRef(function SystemsDevopsDeploy(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M30.509,23l-4.254,4.255l-0.51-0.51l3.386-3.385H18c-1.577,0-2.86-1.283-2.86-2.86\ts1.283-2.86,2.86-2.86h2.033v0.721H18c-1.18,0-2.14,0.96-2.14,2.14s0.96,2.14,2.14,2.14h11.131l-3.386-3.385l0.51-0.51L30.509,23z M12.36,4.532L16,2.417l11.64,6.79V15h0.721V9c0-0.128-0.068-0.247-0.179-0.311l-12-7c-0.111-0.065-0.25-0.065-0.362,0l-4,2.325\tc-0.111,0.064-0.179,0.183-0.179,0.311V13h0.72V4.532H12.36z M8.36,6H7.64v10h0.72V6z M4.36,9H3.64v14\tc0,0.128,0.068,0.246,0.179,0.311l12,7l0.362-0.621L4.36,22.793V9z"
  })), children);
});

export { SystemsDevopsDeploy as default };
