// @carbon/pictograms-react/lib/person--01/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Person_01 = /*#__PURE__*/React__default["default"].forwardRef(function Person_01(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M18.36,30h-0.72v-9c0-0.199,0.161-0.36,0.36-0.36h2c0.904,0,1.64-0.735,1.64-1.64v-5\tc0-2.559-2.081-4.64-4.64-4.64h-2c-2.559,0-4.64,2.082-4.64,4.64v5c0,0.904,0.736,1.64,1.64,1.64h2c0.199,0,0.36,0.161,0.36,0.36v9\th-0.72v-8.64H12c-1.301,0-2.36-1.059-2.36-2.36v-5c0-2.956,2.404-5.36,5.36-5.36h2c2.955,0,5.36,2.404,5.36,5.36v5\tc0,1.302-1.059,2.36-2.36,2.36h-1.64V30z M16,7.36c-1.577,0-2.86-1.283-2.86-2.86S14.423,1.64,16,1.64s2.86,1.283,2.86,2.86\tS17.577,7.36,16,7.36z M16,2.36c-1.18,0-2.14,0.96-2.14,2.14S14.82,6.64,16,6.64s2.14-0.96,2.14-2.14S17.18,2.36,16,2.36z"
  })), children);
});

module.exports = Person_01;
