// @carbon/pictograms-react/lib/watsonx--governance/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const WatsonxGovernance = /*#__PURE__*/React__default["default"].forwardRef(function WatsonxGovernance(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,31.36C7.53,31.36,0.64,24.47,0.64,16\tc0-3.432,1.161-6.788,3.273-9.479c-0.555-0.729-0.5-1.777,0.164-2.442c0.667-0.665,1.715-0.72,2.443-0.166\tC9.213,1.8,12.567,0.64,16,0.64c0.003,0,0.004,0,0.01,0C24.475,0.645,31.36,7.534,31.36,16c0,3.434-1.161,6.787-3.274,9.479\tc0.556,0.729,0.501,1.776-0.164,2.442c-0.666,0.664-1.712,0.722-2.442,0.164C22.788,30.199,19.433,31.36,16,31.36z M4.454,6.999\tC2.457,9.559,1.36,12.743,1.36,16c0,8.072,6.567,14.64,14.64,14.64c3.258,0,6.441-1.097,9.001-3.094\tc-0.417-0.712-0.32-1.645,0.29-2.255c0.611-0.609,1.541-0.707,2.256-0.29c1.997-2.56,3.093-5.743,3.093-9.001\tc0-5.135-2.657-9.661-6.669-12.275c2.388,2.171,3.89,5.301,3.89,8.775c0,6.54-5.32,11.86-11.86,11.86v-0.72\tc6.143,0,11.14-4.997,11.14-11.14c0-4.01-2.13-7.533-5.318-9.495C23.386,4.525,24.36,6.651,24.36,9c0,4.61-3.751,8.36-8.36,8.36\tv-0.72c4.213,0,7.64-3.427,7.64-7.64c0-2.895-1.618-5.419-3.998-6.715C20.399,3.143,20.86,4.269,20.86,5.5\tc0,2.68-2.181,4.86-4.86,4.86V9.64c2.283,0,4.14-1.857,4.14-4.14S18.283,1.36,16,1.36c-3.258,0-6.441,1.096-9.001,3.093\tc0.417,0.712,0.321,1.645-0.29,2.255C6.099,7.319,5.166,7.415,4.454,6.999z M26.606,25.467c-0.292,0-0.584,0.111-0.806,0.334\tc-0.444,0.444-0.444,1.167,0,1.611c0.443,0.445,1.168,0.445,1.611,0c0.444-0.444,0.444-1.167,0-1.611\tC27.19,25.578,26.898,25.467,26.606,25.467z M5.393,4.254c-0.292,0-0.584,0.111-0.806,0.333c-0.444,0.444-0.444,1.167,0,1.612\tc0.444,0.444,1.167,0.443,1.612,0c0.444-0.445,0.444-1.168,0-1.613C5.977,4.365,5.685,4.254,5.393,4.254z M13,25.36h-2\tc-0.199,0-0.36-0.161-0.36-0.36v-2c0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2\tC13.36,25.199,13.199,25.36,13,25.36z M11.36,24.64h1.28v-1.28h-1.28V24.64z M13,18.36h-2c-0.199,0-0.36-0.161-0.36-0.36v-2\tc0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2C13.36,18.199,13.199,18.36,13,18.36z M11.36,17.64h1.28v-1.28\th-1.28V17.64z M13,11.36h-2c-0.199,0-0.36-0.161-0.36-0.36V9c0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2\tC13.36,11.199,13.199,11.36,13,11.36z M11.36,10.64h1.28V9.36h-1.28V10.64z"
  })), children);
});

module.exports = WatsonxGovernance;
