// @carbon/pictograms-react/es/controls-framework/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const ControlsFramework = /*#__PURE__*/React.forwardRef(function ControlsFramework(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M31,31.36H1c-0.199,0-0.36-0.161-0.36-0.36V1\tc0-0.199,0.161-0.36,0.36-0.36h30c0.199,0,0.36,0.161,0.36,0.36v30C31.36,31.199,31.199,31.36,31,31.36z M1.36,30.64h29.28V1.36\tH1.36V30.64z M29,29.36H3c-0.199,0-0.36-0.161-0.36-0.36V3c0-0.199,0.161-0.36,0.36-0.36h26c0.199,0,0.36,0.161,0.36,0.36v26\tC29.36,29.199,29.199,29.36,29,29.36z M3.36,28.64h25.28V3.36H3.36V28.64z M15,23.36c-0.625,0-1.153-0.425-1.312-1H8v-0.72h5.688\tc0.158-0.575,0.686-1,1.312-1s1.153,0.425,1.312,1H24v0.721h-7.688C16.153,22.936,15.625,23.36,15,23.36z M15,21.36\tc-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S15.353,21.36,15,21.36z M11,19.36\tc-0.625,0-1.153-0.425-1.312-1H8v-0.72h1.688c0.158-0.575,0.686-1,1.312-1s1.153,0.424,1.312,1H24v0.721H12.312\tC12.153,18.936,11.625,19.36,11,19.36z M11,17.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64\tS11.353,17.36,11,17.36z M21,15.36c-0.625,0-1.153-0.424-1.312-1H8v-0.72h11.688c0.158-0.576,0.687-1,1.312-1s1.153,0.424,1.312,1\tH24v0.72h-1.688C22.153,14.936,21.625,15.36,21,15.36z M21,13.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64\ts0.64-0.287,0.64-0.64S21.353,13.36,21,13.36z M18,11.36c-0.625,0-1.153-0.424-1.312-1H8V9.64h8.688c0.158-0.576,0.687-1,1.312-1\ts1.153,0.424,1.312,1H24v0.72h-4.688C19.153,10.936,18.625,11.36,18,11.36z M18,9.36c-0.353,0-0.64,0.287-0.64,0.64\ts0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S18.353,9.36,18,9.36z"
  })), children);
});

export { ControlsFramework as default };
