// @carbon/pictograms-react/es/hazy/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Hazy = /*#__PURE__*/React.forwardRef(function Hazy(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M17,27.36h-5v-0.72h5V27.36z M22,23.36H12v-0.72h10V23.36z M8,23.36H3v-0.72h5V23.36z M31,19.36H1v-0.72h30\tV19.36z M16,16.36v-0.72c2.559,0,4.64-2.082,4.64-4.64S18.559,6.36,16,6.36S11.36,8.441,11.36,11h-0.72\tc0-2.956,2.404-5.36,5.36-5.36c2.955,0,5.36,2.404,5.36,5.36S18.955,16.36,16,16.36z M28.984,15.36h-5v-0.72h5V15.36z M11,15.36H3\tv-0.72h8V15.36z M8,11.36H1v-0.72h7V11.36z"
  })), children);
});

export { Hazy as default };
