// @carbon/pictograms-react/lib/person--09/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Person_09 = /*#__PURE__*/React__default["default"].forwardRef(function Person_09(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M18.36,30h-0.72V20c0-0.199,0.161-0.36,0.36-0.36h3c0.353,0,0.64-0.287,0.64-0.64v-2\tc0-0.353-0.287-0.64-0.64-0.64H11c-0.353,0-0.64,0.287-0.64,0.64v2c0,0.353,0.287,0.64,0.64,0.64h3c0.199,0,0.36,0.161,0.36,0.36v10\th-0.72v-9.64H11c-0.75,0-1.36-0.61-1.36-1.36v-2c0-0.75,0.61-1.36,1.36-1.36h10c0.75,0,1.36,0.61,1.36,1.36v2\tc0,0.75-0.61,1.36-1.36,1.36h-2.64V30z M16,14.36c-1.577,0-2.86-1.283-2.86-2.86S14.423,8.64,16,8.64s2.86,1.283,2.86,2.86\tS17.577,14.36,16,14.36z M16,9.36c-1.18,0-2.14,0.96-2.14,2.14s0.96,2.14,2.14,2.14s2.14-0.96,2.14-2.14S17.18,9.36,16,9.36z"
  })), children);
});

module.exports = Person_09;
