// @carbon/pictograms-react/lib/cics--configuration-manager-for-z-os/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const CicsConfigurationManagerForZOs = /*#__PURE__*/React__default["default"].forwardRef(function CicsConfigurationManagerForZOs(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M25,30.36h-4v-0.72h4\tc2.559,0,4.64-2.081,4.64-4.64v-2h0.721v2C30.36,27.955,27.955,30.36,25,30.36z M16,30.36h-2.36c-1.121,0-2.095-0.798-2.314-1.897\tl-0.469-2.346c-0.357-0.183-0.708-0.385-1.045-0.604L7.545,26.28c-1.042,0.354-2.251-0.104-2.8-1.056l-2.36-4.088\tc-0.561-0.971-0.356-2.213,0.486-2.953l1.788-1.571C4.646,16.409,4.64,16.205,4.64,16c0-0.204,0.013-0.404,0.026-0.604L2.87,13.817\tc-0.842-0.74-1.046-1.982-0.486-2.953l2.36-4.088c0.55-0.95,1.76-1.409,2.8-1.055l2.253,0.762c0.312-0.202,0.662-0.399,1.06-0.6\tl0.469-2.346c0.22-1.099,1.193-1.897,2.314-1.897h4.721c1.121,0,2.095,0.798,2.314,1.897l0.469,2.346\tc0.357,0.182,0.708,0.384,1.045,0.604l2.268-0.767c1.044-0.354,2.252,0.105,2.8,1.055l2.36,4.088l-0.623,0.36l-2.36-4.088\tc-0.382-0.661-1.221-0.978-1.946-0.734l-2.435,0.824c-0.106,0.035-0.224,0.021-0.316-0.042c-0.405-0.273-0.832-0.52-1.269-0.733\tc-0.101-0.049-0.173-0.143-0.194-0.252L19.97,3.679c-0.155-0.764-0.83-1.319-1.61-1.319h-4.72c-0.779,0-1.456,0.555-1.608,1.319\tl-0.503,2.519c-0.022,0.11-0.094,0.203-0.195,0.252c-0.502,0.246-0.921,0.484-1.283,0.729C9.957,7.242,9.84,7.258,9.733,7.221\tl-2.42-0.819C6.589,6.158,5.749,6.475,5.367,7.136l-2.36,4.088c-0.39,0.674-0.248,1.538,0.338,2.052l1.931,1.697\tc0.084,0.074,0.129,0.183,0.122,0.295l-0.009,0.12C5.375,15.591,5.36,15.793,5.36,16c0,0.249,0.01,0.496,0.027,0.741\tc0.007,0.112-0.038,0.221-0.122,0.295l-1.92,1.688c-0.585,0.515-0.728,1.378-0.338,2.053l2.36,4.088\tc0.381,0.661,1.222,0.979,1.946,0.733l2.434-0.823c0.108-0.036,0.223-0.02,0.316,0.042c0.406,0.273,0.833,0.521,1.27,0.734\tc0.101,0.05,0.172,0.143,0.194,0.253l0.503,2.519c0.153,0.764,0.829,1.317,1.608,1.317H16V30.36z M24,26.36\tc-1.302,0-2.36-1.059-2.36-2.36s1.059-2.36,2.36-2.36s2.36,1.059,2.36,2.36S25.302,26.36,24,26.36z M24,22.36\tc-0.904,0-1.64,0.735-1.64,1.64s0.735,1.64,1.64,1.64s1.64-0.735,1.64-1.64S24.904,22.36,24,22.36z M18.36,25h-0.72v-2\tc0-2.955,2.405-5.36,5.36-5.36h4v0.721h-4c-2.559,0-4.64,2.081-4.64,4.64V25z M16,21.36c-2.956,0-5.36-2.405-5.36-5.36\tc0-2.956,2.404-5.36,5.36-5.36c2.955,0,5.36,2.404,5.36,5.36h-0.72c0-2.559-2.081-4.64-4.64-4.64s-4.64,2.082-4.64,4.64\ts2.082,4.64,4.64,4.64V21.36z"
  })), children);
});

module.exports = CicsConfigurationManagerForZOs;
