// @carbon/pictograms-react/lib/add--document/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const AddDocument = /*#__PURE__*/React__default["default"].forwardRef(function AddDocument(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31,31.36H7c-0.199,0-0.36-0.161-0.36-0.36v-6.979c-3.464-1.021-6-4.23-6-8.021s2.536-7,6-8.021V1\tc0-0.199,0.161-0.36,0.36-0.36h18c0.096,0,0.188,0.038,0.255,0.105l6,6C31.322,6.813,31.36,6.904,31.36,7v24\tC31.36,31.199,31.199,31.36,31,31.36z M7.36,30.64h23.28V7.36H25c-0.199,0-0.36-0.161-0.36-0.36V1.36H7.36v6.441\tC7.89,7.696,8.439,7.64,9,7.64c4.61,0,8.36,3.75,8.36,8.36c0,4.609-3.75,8.36-8.36,8.36c-0.561,0-1.11-0.056-1.64-0.162\tC7.36,24.198,7.36,30.64,7.36,30.64z M9,8.36c-4.213,0-7.64,3.427-7.64,7.64S4.787,23.64,9,23.64s7.64-3.427,7.64-7.64\tS13.213,8.36,9,8.36z M25.36,6.64h4.771L25.36,1.869V6.64z M9.36,20H8.64v-3.64H5v-0.72h3.64V12h0.72v3.64H13v0.72H9.36V20z"
  })), children);
});

module.exports = AddDocument;
