// @carbon/pictograms-react/lib/dublin--castle/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const DublinCastle = /*#__PURE__*/React__default["default"].forwardRef(function DublinCastle(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M26.64,25v4.36h2.721V25c0-0.75-0.61-1.36-1.36-1.36S26.64,24.25,26.64,25z M28.64,25v3.64h-1.28V25\tc0-0.353,0.287-0.64,0.64-0.64S28.64,24.647,28.64,25z M4,23.64c-0.75,0-1.36,0.61-1.36,1.36v4.36h2.72V25\tC5.36,24.25,4.75,23.64,4,23.64z M4.64,28.64H3.36V25c0-0.353,0.287-0.64,0.64-0.64S4.64,24.647,4.64,25V28.64z M19.36,11.64h-6.72\tv4H12v0.72h8v-0.72h-0.64V11.64z M18.64,15.64h-1.28v-3.28h1.279L18.64,15.64L18.64,15.64z M16.64,15.64h-1.28v-3.28h1.28V15.64z M13.36,12.36h1.28v3.28h-1.28V12.36z M8.64,29.36h2.72v-2.72H8.64V29.36z M9.36,27.36h1.28v1.279H9.36V27.36z M12.64,29.36h2.72\tv-2.72h-2.72V29.36z M13.36,27.36h1.28v1.279h-1.28V27.36z M16.64,29.36h2.72v-2.72h-2.72V29.36z M17.36,27.36h1.279v1.279H17.36\tV27.36z M20.64,26.64v2.721h2.721V26.64H20.64z M22.64,28.64h-1.28v-1.28h1.279L22.64,28.64L22.64,28.64z M31,20.36\tc0.199,0,0.36-0.161,0.36-0.36v-2h-0.72v1.64h-1.28V18h-0.72v1.64h-1.28V18h-0.72v1.64h-1.28V7.149l2-2V0.64h-2.72v2h-1.28v-2h-2.72\tv2h-1.28v-2h-2.72v2h-1.28v-2h-2.72v2h-1.28v-2H8.64v2H7.36v-2H4.64v4.509l2,2V19.64H5.36V18H4.64v1.64H3.36V18H2.64v1.64H1.36V18\tH0.64v2c0,0.199,0.161,0.36,0.36,0.36h5.64v1.279h-6v0.721h6v8.279H1v0.721h30v-0.72h-5.64v-8.28h6v-0.72h-6v-1.28H31z M5.36,4.851\tV1.36h1.28v2h2.72v-2h1.28v2h2.72v-2h1.28v2h2.72v-2h1.279v2h2.721v-2h1.279v2h2.721v-2h1.279v3.491L24.851,6.64h-1.49V5\tc0-0.199-0.161-0.36-0.36-0.36S22.64,4.801,22.64,5v1.64h-1.28V5c0-0.199-0.161-0.36-0.36-0.36S20.64,4.801,20.64,5v1.64h-1.28V5\tc0-0.199-0.161-0.36-0.36-0.36S18.64,4.801,18.64,5v1.64h-1.28V5c0-0.199-0.161-0.36-0.36-0.36c-0.199,0-0.36,0.161-0.36,0.36v1.64\th-1.28V5c0-0.199-0.161-0.36-0.36-0.36S14.64,4.801,14.64,5v1.64h-1.28V5c0-0.199-0.161-0.36-0.36-0.36S12.64,4.801,12.64,5v1.64\th-1.28V5c0-0.199-0.161-0.36-0.36-0.36S10.64,4.801,10.64,5v1.64H9.36V5c0-0.199-0.161-0.36-0.36-0.36S8.64,4.801,8.64,5v1.64H7.149\tL5.36,4.851z M7.36,30.64v-5.28h17.28v5.279L7.36,30.64L7.36,30.64z M24.64,22.36v2.279h-1.28V23c0-0.199-0.161-0.36-0.36-0.36\ts-0.36,0.161-0.36,0.36v1.64h-1.28V23c0-0.199-0.161-0.36-0.36-0.36s-0.36,0.161-0.36,0.36v1.64h-1.28V23\tc0-0.199-0.161-0.36-0.36-0.36s-0.36,0.161-0.36,0.36v1.64h-1.28V23c0-0.199-0.161-0.36-0.36-0.36c-0.199,0-0.36,0.161-0.36,0.36\tv1.64h-1.28V23c0-0.199-0.161-0.36-0.36-0.36s-0.36,0.161-0.36,0.36v1.64h-1.28V23c0-0.199-0.161-0.36-0.36-0.36\ts-0.36,0.161-0.36,0.36v1.64h-1.28V23c0-0.199-0.161-0.36-0.36-0.36s-0.36,0.161-0.36,0.36v1.64H9.36V23\tc0-0.199-0.161-0.36-0.36-0.36S8.64,22.801,8.64,23v1.64H7.36v-2.28v-0.72V7.36h17.28v14.28C24.64,21.64,24.64,22.36,24.64,22.36z"
  })), children);
});

module.exports = DublinCastle;
