// @carbon/pictograms-react/lib/door--handle/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const DoorHandle = /*#__PURE__*/React__default["default"].forwardRef(function DoorHandle(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M22.64,17h0.721v14c0,0.199-0.161,0.36-0.36,0.36H1c-0.199,0-0.36-0.161-0.36-0.36V1\tc0-0.199,0.161-0.36,0.36-0.36h22c0.199,0,0.36,0.161,0.36,0.36v6h-0.72V1.36H1.36v29.28h21.28C22.64,30.64,22.64,17,22.64,17z M12.36,25.958v-2.031C12.733,23.782,13,23.425,13,23c0-0.552-0.448-1-1-1s-1,0.448-1,1c0,0.425,0.267,0.782,0.64,0.927v2.031H12.36\tz M31.36,12c0,1.025-0.835,1.86-1.86,1.86H17.027c-0.758,2.042-2.726,3.5-5.027,3.5c-2.956,0-5.36-2.405-5.36-5.36\tS9.044,6.64,12,6.64c2.302,0,4.27,1.458,5.027,3.5H29.5C30.525,10.14,31.36,10.975,31.36,12z M16.64,12\tc0-2.559-2.082-4.64-4.64-4.64S7.36,9.441,7.36,12s2.082,4.64,4.64,4.64S16.64,14.559,16.64,12z M30.64,12\tc0-0.628-0.511-1.14-1.14-1.14H17.238c0.08,0.368,0.122,0.749,0.122,1.14s-0.042,0.772-0.122,1.14H29.5\tC30.129,13.14,30.64,12.628,30.64,12z"
  })), children);
});

module.exports = DoorHandle;
