// @carbon/pictograms-react/lib/connect--applications/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const ConnectApplications = /*#__PURE__*/React__default["default"].forwardRef(function ConnectApplications(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M30,29.36H20\tc-0.199,0-0.36-0.161-0.36-0.36v-9c0-0.199,0.161-0.36,0.36-0.36h4.64v-3.28H7.36v3.28H12c0.199,0,0.36,0.161,0.36,0.36v9\tc0,0.199-0.161,0.36-0.36,0.36H2c-0.199,0-0.36-0.161-0.36-0.36v-9c0-0.199,0.161-0.36,0.36-0.36h4.64V16\tc0-0.199,0.161-0.36,0.36-0.36h8.64v-3.28H11c-0.199,0-0.36-0.161-0.36-0.36V3c0-0.199,0.161-0.36,0.36-0.36h10\tc0.199,0,0.36,0.161,0.36,0.36v9c0,0.199-0.161,0.36-0.36,0.36h-4.64v3.28H25c0.199,0,0.36,0.161,0.36,0.36v3.64H30\tc0.199,0,0.36,0.161,0.36,0.36v9C30.36,29.199,30.199,29.36,30,29.36z M20.36,28.64h9.279v-8.28H20.36V28.64z M2.36,28.64h9.28\tv-8.28H2.36V28.64z M11.36,11.64h9.28V3.36h-9.28V11.64z M24,22.36h-2v-0.72h2V22.36z M6,22.36H4v-0.72h2V22.36z M15,5.36h-2V4.64h2\tV5.36z"
  })), children);
});

module.exports = ConnectApplications;
