// @carbon/pictograms-react/lib/balanced/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Balanced = /*#__PURE__*/React__default["default"].forwardRef(function Balanced(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,30.36c-7.393,0-13.606-5.698-14.284-13.03\tC1.102,17.199,0.64,16.653,0.64,16s0.462-1.199,1.076-1.33C2.394,7.338,8.607,1.64,16,1.64c7.394,0,13.607,5.698,14.284,13.03\tc0.613,0.131,1.076,0.677,1.076,1.33s-0.463,1.199-1.076,1.33C29.607,24.662,23.394,30.36,16,30.36z M2.436,17.288\tC3.09,24.24,8.986,29.64,16,29.64s12.91-5.399,13.564-12.352c-0.425-0.144-0.757-0.493-0.876-0.928h-4.377\tc-0.158,0.576-0.687,1-1.312,1s-1.153-0.425-1.312-1H17.31c-0.158,0.576-0.686,1-1.312,1s-1.153-0.425-1.312-1h-4.377\tc-0.114,0.417-0.423,0.754-0.822,0.909c0.599,3.085,3.339,5.371,6.51,5.371c1.773,0,3.441-0.69,4.695-1.945l0.51,0.51\tc-1.388,1.39-3.236,2.155-5.202,2.155c-3.537,0-6.592-2.567-7.229-6.02c-0.522-0.089-0.944-0.477-1.082-0.981H3.312\tC3.192,16.795,2.86,17.145,2.436,17.288z M30,15.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64\tS30.353,15.36,30,15.36z M23,15.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S23.353,15.36,23,15.36\tz M16,15.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S16.353,15.36,16,15.36z M9,15.36\tc-0.353,0-0.64,0.287-0.64,0.64S8.647,16.64,9,16.64S9.64,16.353,9.64,16S9.353,15.36,9,15.36z M2,15.36\tc-0.353,0-0.64,0.287-0.64,0.64S1.647,16.64,2,16.64S2.64,16.353,2.64,16S2.353,15.36,2,15.36z M24.312,15.64h4.377\tc0.119-0.436,0.451-0.785,0.876-0.929C28.91,7.759,23.014,2.36,16,2.36S3.09,7.759,2.436,14.711\tc0.425,0.144,0.756,0.493,0.876,0.929h4.377c0.158-0.576,0.686-1,1.312-1s1.153,0.424,1.312,1h4.377c0.158-0.576,0.686-1,1.312-1\tc0.626,0,1.153,0.424,1.312,1h4.377c0.114-0.417,0.423-0.754,0.822-0.909c-0.6-3.085-3.34-5.371-6.511-5.371\tc-1.773,0-3.441,0.691-4.695,1.945l-0.509-0.509C12.186,9.406,14.034,8.64,16,8.64c3.537,0,6.592,2.566,7.229,6.019\tC23.752,14.749,24.173,15.136,24.312,15.64z"
  })), children);
});

module.exports = Balanced;
