// @carbon/pictograms-react/es/encryption/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Encryption = /*#__PURE__*/React.forwardRef(function Encryption(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    fill: "none",
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeMiterlimit: "10",
    strokeWidth: ".72",
    d: "M10,21c0-3.314,2.686-6,6-6s6,2.686,6,6s-2.686,6-6,6S10,24.314,10,21z M16,11c-5.523,0-10,4.477-10,10s4.477,10,10,10\ts10-4.477,10-10S21.523,11,16,11z M11,17.682V6c0-2.761,2.239-5,5-5s5,2.239,5,5v11.682 M16,22v2 M16,20c-0.552,0-1,0.448-1,1\ts0.448,1,1,1s1-0.448,1-1S16.552,20,16,20z"
  })), children);
});

export { Encryption as default };
