// @carbon/pictograms-react/es/metal-can/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const MetalCan = /*#__PURE__*/React.forwardRef(function MetalCan(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M28,7.36V6.64H14.648l11.504-5.313l-0.303-0.653\tL12.932,6.64H12v0.72h0.64v4.28h-2.28V11c0-0.75-0.61-1.36-1.36-1.36S7.64,10.25,7.64,11v0.64H5v0.72h0.501l-0.842,2.526\tC4.646,14.923,4.64,14.961,4.64,15v14c0,0.056,0.013,0.111,0.038,0.161l1,2C5.739,31.283,5.864,31.36,6,31.36h9h13v-0.72h-0.64V7.36\tH28z M8.36,11c0-0.353,0.287-0.64,0.64-0.64S9.64,10.647,9.64,11v0.64H8.36V11z M8,12.36h2h4.741l0.899,2.699V28.64H5.36V15.36H13\tv-0.72H5.499l0.76-2.28H8z M14.777,30.64H6.223l-0.64-1.279h9.835L14.777,30.64z M26.64,9.64H22v0.72h4.64v2.28H22v0.72h4.64v2.28\tH22v0.72h4.64v2.28H22v0.721h4.64v2.279H22v0.721h4.64v2.279H22v0.721h4.64v2.279H22v0.721h4.64v2.279H15.583l0.739-1.479\tc0.025-0.05,0.038-0.105,0.038-0.161V15c0-0.039-0.006-0.077-0.019-0.114l-0.842-2.526H16v-0.72h-2.64V7.36h13.28V9.64z"
  })), children);
});

export { MetalCan as default };
