// @carbon/pictograms-react/lib/extensible/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Extensible = /*#__PURE__*/React__default["default"].forwardRef(function Extensible(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M3,31.36H1c-0.199,0-0.36-0.161-0.36-0.36v-2\tc0-0.199,0.161-0.36,0.36-0.36h0.64V2c0-0.199,0.161-0.36,0.36-0.36h26.64V1c0-0.199,0.161-0.36,0.36-0.36h2\tc0.199,0,0.36,0.161,0.36,0.36v2c0,0.199-0.161,0.36-0.36,0.36h-0.64V30c0,0.199-0.161,0.36-0.36,0.36H3.36V31\tC3.36,31.199,3.199,31.36,3,31.36z M1.36,30.64h1.28v-1.28H1.36V30.64z M22.36,29.64h7.279V3.36H29c-0.199,0-0.36-0.161-0.36-0.36\tV2.36H2.36v7.28h18.28V9c0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2c0,0.199-0.161,0.36-0.36,0.36h-0.64\tC22.36,11.36,22.36,29.64,22.36,29.64z M14.36,29.64h7.28V11.36H21c-0.199,0-0.36-0.161-0.36-0.36v-0.64H2.36v7.28h10.28V17\tc0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2c0,0.199-0.161,0.36-0.36,0.36h-0.64\tC14.36,19.36,14.36,29.64,14.36,29.64z M3.36,29.64h10.28V19.36H13c-0.199,0-0.36-0.161-0.36-0.36v-0.64H2.36v10.28H3\tc0.199,0,0.36,0.161,0.36,0.36C3.36,29,3.36,29.64,3.36,29.64z M13.36,18.64h1.28v-1.28h-1.28V18.64z M21.36,10.64h1.279V9.36H21.36\tV10.64z M29.36,2.64h1.279V1.36H29.36V2.64z"
  })), children);
});

module.exports = Extensible;
