// @carbon/pictograms-react/lib/first--aid/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const FirstAid = /*#__PURE__*/React__default["default"].forwardRef(function FirstAid(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M19,25.36h-6c-0.199,0-0.36-0.161-0.36-0.36v-5.64H7c-0.199,0-0.36-0.161-0.36-0.36v-6\tc0-0.199,0.161-0.36,0.36-0.36h5.64V7c0-0.199,0.161-0.36,0.36-0.36h6c0.199,0,0.36,0.161,0.36,0.36v5.64H25\tc0.199,0,0.36,0.161,0.36,0.36v6c0,0.199-0.161,0.36-0.36,0.36h-5.64V25C19.36,25.199,19.199,25.36,19,25.36z M13.36,24.64h5.28V19\tc0-0.199,0.161-0.36,0.36-0.36h5.64v-5.28H19c-0.199,0-0.36-0.161-0.36-0.36V7.36h-5.28V13c0,0.199-0.161,0.36-0.36,0.36H7.36v5.28\tH13c0.199,0,0.36,0.161,0.36,0.36V24.64z M16,31.36C7.53,31.36,0.64,24.47,0.64,16S7.53,0.64,16,0.64S31.36,7.53,31.36,16\tS24.47,31.36,16,31.36z M16,1.36C7.927,1.36,1.36,7.927,1.36,16c0,8.072,6.567,14.64,14.64,14.64c8.072,0,14.64-6.567,14.64-14.64\tS24.072,1.36,16,1.36z"
  })), children);
});

module.exports = FirstAid;
