// @carbon/pictograms-react/es/unlock--01/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Unlock_01 = /*#__PURE__*/React.forwardRef(function Unlock_01(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M17.5,20.5c0,0.704-0.486,1.29-1.14,1.452V24c0,0.199-0.161,0.36-0.36,0.36s-0.36-0.161-0.36-0.36\tv-2.048c-0.654-0.162-1.14-0.748-1.14-1.452c0-0.828,0.672-1.5,1.5-1.5C16.828,19,17.5,19.672,17.5,20.5z M26.36,21\tc0,5.713-4.647,10.36-10.36,10.36c-5.712,0-10.36-4.647-10.36-10.36c0-3.749,2.009-7.03,5-8.849V6c0-2.956,2.404-5.36,5.36-5.36\tc2.955,0,5.36,2.404,5.36,5.36h-0.72c0-2.559-2.081-4.64-4.64-4.64S11.36,3.441,11.36,6v5.751c1.398-0.704,2.971-1.11,4.64-1.11\tC21.713,10.64,26.36,15.288,26.36,21z M25.64,21c0-5.315-4.324-9.64-9.64-9.64S6.36,15.685,6.36,21s4.325,9.64,9.64,9.64\tS25.64,26.315,25.64,21z"
  })), children);
});

export { Unlock_01 as default };
