// @carbon/pictograms-react/es/ibm--automation-platform/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const IbmAutomationPlatform = /*#__PURE__*/React.forwardRef(function IbmAutomationPlatform(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M22,31.36c-1.853,0-3.36-1.508-3.36-3.36v-9h0.721v9c0,1.456,1.184,2.64,2.64,2.64\ts2.64-1.184,2.64-2.64v-5h0.721v5C25.36,29.853,23.853,31.36,22,31.36z M10,31.36c-1.853,0-3.36-1.508-3.36-3.36v-9h0.72v9\tc0,1.456,1.185,2.64,2.64,2.64s2.64-1.184,2.64-2.64V4c0-1.853,1.507-3.36,3.36-3.36S19.36,2.147,19.36,4v9h-0.72V4\tc0-1.456-1.184-2.64-2.64-2.64S13.36,2.545,13.36,4v24C13.36,29.853,11.853,31.36,10,31.36z M27.255,20.255l-0.51-0.51l3.386-3.385\tH16v-0.72h14.131l-3.386-3.386l0.51-0.509L31.509,16L27.255,20.255z M10,16.36H1v-0.72h9V16.36z M7.36,13H6.64V4\tc0-1.456-1.185-2.64-2.64-2.64S1.36,2.544,1.36,4v9H0.64V4c0-1.853,1.507-3.36,3.36-3.36S7.36,2.147,7.36,4V13z"
  })), children);
});

export { IbmAutomationPlatform as default };
