// @carbon/pictograms-react/es/whistle/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Whistle = /*#__PURE__*/React.forwardRef(function Whistle(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M11,30.36C5.288,30.36,0.64,25.713,0.64,20\tC0.64,14.288,5.288,9.64,11,9.64h8c0.199,0,0.36,0.161,0.36,0.36v1.64h3.279V10c0-0.199,0.161-0.36,0.36-0.36h8\tc0.199,0,0.36,0.161,0.36,0.36v6c0,0.171-0.121,0.319-0.29,0.353l-9.71,1.942V20C21.36,25.713,16.712,30.36,11,30.36z M11,10.36\tc-5.315,0-9.64,4.325-9.64,9.64s4.325,9.64,9.64,9.64s9.64-4.324,9.64-9.64v-2c0-0.172,0.121-0.319,0.29-0.353l9.71-1.942V10.36\th-7.28V12c0,0.199-0.161,0.36-0.36,0.36h-4c-0.199,0-0.36-0.161-0.36-0.36v-1.64H11z M11,26.36c-3.507,0-6.36-2.854-6.36-6.36\ts2.853-6.36,6.36-6.36s6.36,2.853,6.36,6.36S14.507,26.36,11,26.36z M11,14.36c-3.11,0-5.64,2.53-5.64,5.64s2.53,5.64,5.64,5.64\ts5.64-2.529,5.64-5.64C16.64,16.89,14.11,14.36,11,14.36z M25,7.36c-0.092,0-0.185-0.035-0.255-0.105\tc-0.141-0.141-0.141-0.368,0-0.509l4-4c0.141-0.141,0.369-0.141,0.51,0s0.141,0.368,0,0.509l-4,4C25.185,7.325,25.092,7.36,25,7.36z M17,7.36c-0.092,0-0.184-0.035-0.254-0.105l-4-4c-0.141-0.141-0.141-0.368,0-0.509s0.368-0.141,0.509,0l4,4\tc0.141,0.141,0.141,0.368,0,0.509C17.185,7.325,17.092,7.36,17,7.36z M21,7.36c-0.199,0-0.36-0.161-0.36-0.36V1\tc0-0.199,0.161-0.36,0.36-0.36S21.36,0.801,21.36,1v6C21.36,7.199,21.199,7.36,21,7.36z"
  })), children);
});

export { Whistle as default };
