// @carbon/pictograms-react/lib/enterprise--design--thinking--02/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const EnterpriseDesignThinking_02 = /*#__PURE__*/React__default["default"].forwardRef(function EnterpriseDesignThinking_02(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31,31.36H1v-0.72h30V31.36z M31,27.36H16v-0.72h15V27.36z M7.409,22.36\tc-1.764,0-3.558-0.64-4.84-1.908c-1.244-1.23-1.929-2.808-1.929-4.442s0.685-3.212,1.928-4.443c2.502-2.474,6.938-2.572,9.306-0.203\tl8.794,8.793c2.084,2.086,6.021,1.972,8.257-0.239c1.105-1.094,1.715-2.486,1.715-3.92s-0.609-2.826-1.715-3.921\tc-2.233-2.21-6.188-2.306-8.296-0.199l-2.375,2.375l-0.51-0.509l2.375-2.375c2.371-2.372,6.81-2.279,9.312,0.196\tc1.244,1.231,1.929,2.805,1.929,4.433s-0.685,3.202-1.929,4.432c-2.501,2.475-6.921,2.589-9.272,0.237l-8.793-8.794\tC9.261,9.77,5.31,9.868,3.075,12.079C1.969,13.174,1.36,14.57,1.36,16.01s0.609,2.836,1.715,3.93\tc2.229,2.203,6.179,2.292,8.289,0.184l1.764-1.764H10v-0.72h4.36V22h-0.72v-3.135l-1.768,1.769\tC10.719,21.787,9.077,22.359,7.409,22.36z M16,5.36H1V4.64h15V5.36z M31,1.36H1V0.64h30V1.36z"
  })), children);
});

module.exports = EnterpriseDesignThinking_02;
