// @carbon/pictograms-react/lib/financial--services/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const FinancialServices = /*#__PURE__*/React__default["default"].forwardRef(function FinancialServices(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31,31.36H1c-0.199,0-0.36-0.161-0.36-0.36\tV1h0.72v14.64h4.308c0.174-1.131,1.154-2,2.333-2c1.301,0,2.36,1.059,2.36,2.36c0,0.521-0.169,1.003-0.457,1.395l4.703,4.702\tc0.609-0.448,1.505-0.578,2.224-0.306l6.714-11.935C22.994,9.423,22.64,8.752,22.64,8c0-1.301,1.059-2.36,2.36-2.36\tc1.179,0,2.159,0.869,2.333,2H31v0.72h-3.667c-0.174,1.131-1.154,2-2.333,2c-0.292,0-0.571-0.053-0.829-0.15l-6.714,11.935\tc0.55,0.433,0.903,1.104,0.903,1.855c0,1.302-1.059,2.36-2.36,2.36s-2.36-1.059-2.36-2.36c0-0.521,0.169-1.003,0.457-1.395\tl-4.703-4.702C9.003,18.19,8.521,18.36,8,18.36c-1.179,0-2.159-0.869-2.333-2H1.36v14.28H31V31.36z M16,22.36\tc-0.904,0-1.64,0.735-1.64,1.64s0.736,1.64,1.64,1.64c0.904,0,1.64-0.735,1.64-1.64S16.904,22.36,16,22.36z M8,14.36\tc-0.904,0-1.64,0.736-1.64,1.64c0,0.904,0.736,1.64,1.64,1.64S9.64,16.904,9.64,16S8.904,14.36,8,14.36z M25,6.36\tc-0.904,0-1.64,0.736-1.64,1.64S24.096,9.64,25,9.64S26.64,8.904,26.64,8S25.904,6.36,25,6.36z"
  })), children);
});

module.exports = FinancialServices;
