// @carbon/pictograms-react/es/minimize/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Minimize = /*#__PURE__*/React.forwardRef(function Minimize(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M30.745,31.255l-9.385-9.386V30h-0.72v-9.36H30v0.721h-8.131l9.386,9.385L30.745,31.255z M1.254,31.255 l-0.509-0.51l9.385-9.385H2v-0.72h9.36V30h-0.72v-8.131L1.254,31.255z M30,11.36h-9.36V2h0.721v8.131l9.385-9.385l0.51,0.509 l-9.386,9.386H30V11.36z M11.36,11.36H2v-0.72h8.131L0.746,1.254l0.509-0.509l9.386,9.385V2h0.72v9.36H11.36z"
  })), children);
});

export { Minimize as default };
