// @carbon/pictograms-react/lib/integration--02/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Integration_02 = /*#__PURE__*/React__default["default"].forwardRef(function Integration_02(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,31.36C7.53,31.36,0.64,24.47,0.64,16\tS7.53,0.64,16,0.64S31.36,7.53,31.36,16c0,3.607-1.28,7.1-3.61,9.871c0.137,0.212,0.217,0.465,0.217,0.735\tc0,0.75-0.61,1.36-1.36,1.36c-0.271,0-0.523-0.08-0.736-0.217C23.101,30.08,19.607,31.36,16,31.36z M16,1.36\tC7.927,1.36,1.36,7.927,1.36,16c0,8.072,6.567,14.64,14.64,14.64c1.314,0,2.613-0.178,3.862-0.522\tc-0.319,0.027-0.641,0.042-0.963,0.042c-6.208,0-11.259-5.052-11.259-11.26S12.691,7.641,18.899,7.641s11.26,5.051,11.26,11.259\tc0,0.322-0.015,0.644-0.042,0.964c0.345-1.251,0.522-2.549,0.522-3.863C30.64,7.927,24.072,1.36,16,1.36z M18.899,8.36\tc-5.812,0-10.54,4.728-10.54,10.54s4.728,10.539,10.54,10.539c1.077,0,2.143-0.166,3.158-0.485\tc-0.086,0.003-0.173,0.005-0.259,0.005c-3.947,0-7.159-3.212-7.159-7.159s3.211-7.159,7.159-7.159s7.159,3.211,7.159,7.159\tc0,0.086-0.002,0.172-0.005,0.257c0.319-1.015,0.485-2.079,0.485-3.156C29.438,13.088,24.711,8.36,18.899,8.36z M21.799,15.36\tc-3.551,0-6.439,2.888-6.439,6.439s2.888,6.438,6.439,6.438c1.277,0,2.502-0.373,3.563-1.083c-0.074-0.168-0.116-0.353-0.116-0.548\tc0-0.75,0.61-1.36,1.36-1.36c0.195,0,0.38,0.042,0.548,0.116c0.71-1.062,1.083-2.286,1.083-3.563\tC28.237,18.248,25.35,15.36,21.799,15.36z M26.606,25.967c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64\tS26.959,25.967,26.606,25.967z"
  })), children);
});

module.exports = Integration_02;
