// @carbon/pictograms-react/lib/curve--exponential/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const CurveExponential = /*#__PURE__*/React__default["default"].forwardRef(function CurveExponential(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16.36,31h-0.72V16.36H1v-0.72h14.64v-2.026\tC11.388,14.36,6.412,14.36,1,14.36v-0.72c5.432,0,10.415,0,14.64-0.774V1h0.72v11.724c6.142-1.285,10.575-4.375,12.29-11.805\tl0.701,0.162c-1.823,7.902-6.514,11.102-12.991,12.397v2.162H31v0.72H16.36V31z"
  })), children);
});

module.exports = CurveExponential;
