// @carbon/pictograms-react/lib/curve--power/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const CurvePower = /*#__PURE__*/React__default["default"].forwardRef(function CurvePower(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M21.591,27.473L19.109,24h1.276l1.777,2.605h0.041\tL24.047,24h1.208l-2.469,3.391L25.308,31h-1.275l-1.818-2.74h-0.041L20.208,31H19L21.591,27.473z M28.704,23h0.688l-1.203,3.525\tc-0.149,0.43-0.361,0.613-0.854,0.613h-0.441v-0.584h0.538l0.138-0.419L26.498,23h0.727l0.522,1.582l0.195,0.756h0.035l0.206-0.756\tL28.704,23z M16.36,31h-0.72V16.36H1v-0.72h14.64V8.853c-1.198-3.445-1.921-6.478-1.991-6.771l0.701-0.165\tc0.277,1.177,0.72,2.828,1.29,4.639V1h0.72v7.723c0.872,2.482,1.936,5.027,3.105,6.917h6.886c2.167-3.883,3.734-10.526,4.295-13.703\tl0.709,0.125c-0.087,0.491-1.555,8.642-4.198,13.578H31v0.72h-4.253c-1.076,1.786-2.329,3-3.747,3c-1.414,0-2.736-1.215-3.915-3\tH16.36V31z M19.938,16.36c0.976,1.39,2.013,2.28,3.062,2.28c1.047,0,2.029-0.89,2.921-2.28H19.938z M16.36,15.64h2.275\tc-0.848-1.428-1.612-3.125-2.275-4.827V15.64z"
  })), children);
});

module.exports = CurvePower;
