// @carbon/pictograms-react/lib/choose-how-to-get-started/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const ChooseHowToGetStarted = /*#__PURE__*/React__default["default"].forwardRef(function ChooseHowToGetStarted(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M30,30.36H2\tc-0.199,0-0.36-0.161-0.36-0.36V2c0-0.199,0.161-0.36,0.36-0.36h26.64V1c0-0.199,0.161-0.36,0.36-0.36h2\tc0.199,0,0.36,0.161,0.36,0.36v2c0,0.199-0.161,0.36-0.36,0.36h-0.64V30C30.36,30.199,30.199,30.36,30,30.36z M2.36,29.64h27.28\tV3.36H29c-0.199,0-0.36-0.161-0.36-0.36V2.36H2.36V29.64z M29.36,2.64h1.279V1.36H29.36V2.64z M25,25.36H7\tc-0.199,0-0.36-0.161-0.36-0.36V7c0-0.199,0.161-0.36,0.36-0.36h16.64V6c0-0.199,0.161-0.36,0.36-0.36h2\tc0.199,0,0.36,0.161,0.36,0.36v2c0,0.199-0.161,0.36-0.36,0.36h-0.64V25C25.36,25.199,25.199,25.36,25,25.36z M7.36,24.64h17.28\tV8.36H24c-0.199,0-0.36-0.161-0.36-0.36V7.36H7.36V24.64z M24.36,7.64h1.279V6.36H24.36V7.64z M20,20.36h-8\tc-0.199,0-0.36-0.161-0.36-0.36v-8c0-0.199,0.161-0.36,0.36-0.36h6.64V11c0-0.199,0.161-0.36,0.36-0.36h2\tc0.199,0,0.36,0.161,0.36,0.36v2c0,0.199-0.161,0.36-0.36,0.36h-0.64V20C20.36,20.199,20.199,20.36,20,20.36z M12.36,19.64h7.28\tv-6.28H19c-0.199,0-0.36-0.161-0.36-0.36v-0.64h-6.28C12.36,12.36,12.36,19.64,12.36,19.64z M19.36,12.64h1.279v-1.28H19.36V12.64z"
  })), children);
});

module.exports = ChooseHowToGetStarted;
