// @carbon/pictograms-react/es/question/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Question = /*#__PURE__*/React.forwardRef(function Question(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M23.668,29.312l-0.359-0.623c4.522-2.61,7.331-7.473,7.331-12.688c0-8.073-6.567-14.64-14.64-14.64\tC7.927,1.36,1.36,7.927,1.36,16c0,5.216,2.81,10.078,7.332,12.688l-0.36,0.623C3.587,26.573,0.64,21.473,0.64,16\tC0.64,7.53,7.53,0.64,16,0.64S31.36,7.53,31.36,16C31.36,21.473,28.413,26.573,23.668,29.312z M16.355,23.995h-0.72\tc0-3.844,1.823-5.201,3.585-6.515c1.599-1.19,3.108-2.315,3.108-5.41c0-3.308-2.657-5.709-6.319-5.709\tc-3.485,0-6.32,2.849-6.32,6.35H8.97c0-3.898,3.158-7.07,7.04-7.07c4.079,0,7.04,2.704,7.04,6.429c0,3.456-1.729,4.743-3.398,5.987\tC17.956,19.32,16.355,20.513,16.355,23.995z M16,28.5c-0.552,0-1,0.448-1,1s0.448,1,1,1s1-0.448,1-1S16.552,28.5,16,28.5z"
  })), children);
});

export { Question as default };
