// @carbon/pictograms-react/lib/cloud--sap/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const CloudSap = /*#__PURE__*/React__default["default"].forwardRef(function CloudSap(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,30.36H2c-0.199,0-0.36-0.161-0.36-0.36V16h0.72v13.64h13.491L29.13,16.377H19v-0.72h11\tc0.146,0,0.277,0.087,0.333,0.222c0.056,0.135,0.024,0.29-0.078,0.393l-14,13.983C16.187,30.322,16.095,30.36,16,30.36z M10,18.36\tc-4.61,0-8.36-3.75-8.36-8.36S5.39,1.64,10,1.64s8.36,3.75,8.36,8.36S14.61,18.36,10,18.36z M9.514,17.625\tC9.675,17.635,9.837,17.64,10,17.64c2.231,0,4.242-0.961,5.64-2.491V9.294h0.72v4.935c0.808-1.212,1.28-2.666,1.28-4.229\tc0-0.804-0.124-1.579-0.355-2.307l-5.075-2.93l0.36-0.624l4.276,2.469c-1.016-2.042-2.919-3.568-5.201-4.07L6.57,5.468L6.21,4.844\tl4.277-2.469C10.326,2.365,10.163,2.36,10,2.36c-2.231,0-4.242,0.961-5.64,2.492v5.854H3.64V5.771C2.832,6.983,2.36,8.437,2.36,10\tc0,0.804,0.125,1.579,0.356,2.307l5.075,2.93l-0.36,0.624l-4.276-2.469c1.016,2.042,2.919,3.568,5.202,4.07l5.074-2.93l0.36,0.623\tL9.514,17.625z"
  })), children);
});

module.exports = CloudSap;
