// @carbon/pictograms-react/lib/petri--culture/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const PetriCulture = /*#__PURE__*/React__default["default"].forwardRef(function PetriCulture(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16.5,16.001c0,0.276-0.224,0.5-0.5,0.5s-0.5-0.224-0.5-0.5s0.224-0.5,0.5-0.5\tC16.276,15.501,16.5,15.725,16.5,16.001z M20.5,5.5c-0.552,0-1,0.448-1,1s0.448,1,1,1s1-0.448,1-1S21.052,5.5,20.5,5.5z M8.5,20.5\tc-0.552,0-1,0.448-1,1s0.448,1,1,1s1-0.448,1-1S9.052,20.5,8.5,20.5z M12.501,7c-0.276,0-0.5,0.224-0.5,0.5s0.224,0.5,0.5,0.5\ts0.5-0.224,0.5-0.5S12.777,7,12.501,7z M23,20.5c-0.276,0-0.5,0.224-0.5,0.5s0.224,0.5,0.5,0.5s0.5-0.224,0.5-0.5\tS23.276,20.5,23,20.5z M16,31.36C7.53,31.36,0.64,24.47,0.64,16S7.53,0.64,16,0.64S31.36,7.53,31.36,16S24.47,31.36,16,31.36z M16,1.36C7.927,1.36,1.36,7.927,1.36,16c0,8.072,6.567,14.64,14.64,14.64c8.072,0,14.64-6.567,14.64-14.64S24.072,1.36,16,1.36z M16.5,27.36c-1.577,0-2.86-1.283-2.86-2.86s1.283-2.86,2.86-2.86s2.86,1.283,2.86,2.86S18.077,27.36,16.5,27.36z M16.5,22.36\tc-1.18,0-2.14,0.96-2.14,2.14s0.96,2.14,2.14,2.14s2.14-0.96,2.14-2.14S17.68,22.36,16.5,22.36z M6,19.36\tc-1.301,0-2.36-1.059-2.36-2.36S4.699,14.64,6,14.64S8.36,15.699,8.36,17C8.36,18.302,7.301,19.36,6,19.36z M6,15.36\tc-0.904,0-1.64,0.736-1.64,1.64S5.096,18.64,6,18.64S7.64,17.904,7.64,17S6.904,15.36,6,15.36z M25,15.36\tc-1.577,0-2.86-1.283-2.86-2.86S23.423,9.64,25,9.64s2.86,1.283,2.86,2.86S26.577,15.36,25,15.36z M25,10.36\tc-1.18,0-2.14,0.96-2.14,2.14s0.96,2.14,2.14,2.14s2.14-0.96,2.14-2.14S26.18,10.36,25,10.36z M12.5,10.36\tc-1.577,0-2.86-1.283-2.86-2.86s1.283-2.86,2.86-2.86s2.86,1.283,2.86,2.86S14.077,10.36,12.5,10.36z M12.5,5.36\tc-1.18,0-2.14,0.96-2.14,2.14s0.96,2.14,2.14,2.14s2.14-0.96,2.14-2.14S13.68,5.36,12.5,5.36z"
  })), children);
});

module.exports = PetriCulture;
