// @carbon/pictograms-react/lib/mass-data-migration/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const MassDataMigration = /*#__PURE__*/React__default["default"].forwardRef(function MassDataMigration(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M30,31.36c-0.625,0-1.153-0.425-1.312-1H2\tv-0.72h26.688c0.158-0.575,0.687-1,1.312-1c0.75,0,1.36,0.61,1.36,1.36S30.75,31.36,30,31.36z M30,29.36\tc-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S30.353,29.36,30,29.36z M23,24.36\tc-0.625,0-1.153-0.425-1.312-1H2v-0.72h19.688c0.158-0.575,0.687-1,1.312-1s1.153,0.425,1.312,1H30v0.721h-5.688\tC24.153,23.936,23.625,24.36,23,24.36z M23,22.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64\tS23.353,22.36,23,22.36z M16,17.36c-0.625,0-1.153-0.425-1.312-1H2v-0.72h12.688c0.158-0.576,0.686-1,1.312-1\tc0.626,0,1.153,0.424,1.312,1H30v0.72H17.312C17.153,16.936,16.625,17.36,16,17.36z M16,15.36c-0.353,0-0.64,0.287-0.64,0.64\ts0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S16.353,15.36,16,15.36z M9,10.36c-0.625,0-1.153-0.424-1.312-1H2V8.64h5.688\tc0.158-0.576,0.686-1,1.312-1s1.153,0.424,1.312,1H30v0.72H10.312C10.153,9.936,9.625,10.36,9,10.36z M9,8.36\tC8.647,8.36,8.36,8.647,8.36,9S8.647,9.64,9,9.64S9.64,9.353,9.64,9S9.353,8.36,9,8.36z M2,3.36C1.25,3.36,0.64,2.75,0.64,2\tS1.25,0.64,2,0.64c0.625,0,1.153,0.424,1.312,1H30v0.72H3.312C3.153,2.936,2.625,3.36,2,3.36z M2,1.36C1.647,1.36,1.36,1.647,1.36,2\tS1.647,2.64,2,2.64S2.64,2.353,2.64,2S2.353,1.36,2,1.36z"
  })), children);
});

module.exports = MassDataMigration;
