// @carbon/pictograms-react/es/secure--data/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const SecureData = /*#__PURE__*/React.forwardRef(function SecureData(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M16,31.36C8.082,31.36,1.64,24.918,1.64,17V1c0-0.199,0.161-0.36,0.36-0.36h28\tc0.199,0,0.36,0.161,0.36,0.36v16C30.36,24.918,23.918,31.36,16,31.36z M16.36,26.361v4.274c3.757-0.099,7.14-1.723,9.552-4.274\tH16.36z M2.36,1.36V17c0,7.4,5.925,13.444,13.28,13.636V1.36H2.36z M16.36,25.641h10.186c1.033-1.259,1.847-2.705,2.379-4.279H16.36\tV25.641z M16.36,20.641h12.787c0.321-1.159,0.493-2.38,0.493-3.641v-0.64H16.36V20.641z M16.36,15.64h13.28v-4.28H16.36V15.64z M16.36,10.64h13.28V6.36H16.36V10.64z M16.36,5.64h13.28V1.36H16.36V5.64z"
  })), children);
});

export { SecureData as default };
