// @carbon/pictograms-react/lib/file--transfer/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const FileTransfer = /*#__PURE__*/React__default["default"].forwardRef(function FileTransfer(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31.36,31h-0.72v-5.64H22c-0.199,0-0.36-0.161-0.36-0.36v-8.64H1.36V31H0.64V16\tc0-0.199,0.161-0.36,0.36-0.36h21c0.096,0,0.188,0.038,0.255,0.105l9,9c0.067,0.067,0.105,0.159,0.105,0.255\tC31.36,25,31.36,31,31.36,31z M22.36,24.64h7.771l-7.771-7.771V24.64z M31,13.36H1c-0.199,0-0.36-0.161-0.36-0.36V1h0.72v11.64\th29.28V1h0.721v12C31.36,13.199,31.199,13.36,31,13.36z"
  })), children);
});

module.exports = FileTransfer;
