// @carbon/pictograms-react/lib/condition--builder/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const ConditionBuilder = /*#__PURE__*/React__default["default"].forwardRef(function ConditionBuilder(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M30,30.36H2c-0.199,0-0.36-0.161-0.36-0.36V2c0-0.199,0.161-0.36,0.36-0.36h28\tc0.199,0,0.36,0.161,0.36,0.36v28C30.36,30.199,30.199,30.36,30,30.36z M2.36,29.64h27.28V2.36H2.36V29.64z M8.155,20.896h2.976\tv-6.048H8.155v-1.104h4.256v7.152h2.784V22h-7.04V20.896z M11.771,11.92c-0.374,0-0.635-0.077-0.784-0.232\tc-0.149-0.154-0.224-0.349-0.224-0.584v-0.256c0-0.235,0.074-0.429,0.224-0.584s0.411-0.232,0.784-0.232s0.634,0.077,0.784,0.232\tc0.149,0.155,0.224,0.349,0.224,0.584v0.256c0,0.235-0.075,0.43-0.224,0.584C12.406,11.843,12.145,11.92,11.771,11.92z M17.313,20.896h2.784v-6.048h-2.944v-1.104h2.944v-1.696c0-0.554,0.146-1.008,0.44-1.36s0.755-0.528,1.384-0.528h2.768v1.104\th-3.312v2.48h3.312v1.104h-3.312v6.048h2.944V22h-7.008C17.313,22,17.313,20.896,17.313,20.896z"
  })), children);
});

module.exports = ConditionBuilder;
