// @carbon/pictograms-react/es/migration/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Migration = /*#__PURE__*/React.forwardRef(function Migration(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M2,3.36c0.625,0,1.146-0.425,1.305-1h25.389\tc0.159,0.575,0.681,1,1.306,1c0.75,0,1.36-0.61,1.36-1.36S30.75,0.64,30,0.64c-0.625,0-1.147,0.425-1.306,1H3.305\tc-0.159-0.575-0.681-1-1.305-1C1.25,0.64,0.64,1.25,0.64,2S1.25,3.36,2,3.36z M30,1.36c0.353,0,0.64,0.287,0.64,0.64\tS30.353,2.64,30,2.64S29.36,2.353,29.36,2S29.647,1.36,30,1.36z M2,1.36c0.353,0,0.64,0.287,0.64,0.64S2.353,2.64,2,2.64\tS1.36,2.353,1.36,2S1.647,1.36,2,1.36z M30,7.64c-0.625,0-1.147,0.425-1.306,1H3.305c-0.159-0.575-0.681-1-1.305-1\tC1.25,7.64,0.64,8.25,0.64,9S1.25,10.36,2,10.36c0.625,0,1.146-0.425,1.305-1h25.389c0.159,0.575,0.681,1,1.306,1\tc0.75,0,1.36-0.61,1.36-1.36S30.75,7.64,30,7.64z M2,9.64C1.647,9.64,1.36,9.353,1.36,9S1.647,8.36,2,8.36S2.64,8.647,2.64,9\tS2.353,9.64,2,9.64z M30,9.64c-0.353,0-0.64-0.287-0.64-0.64S29.647,8.36,30,8.36S30.64,8.647,30.64,9S30.353,9.64,30,9.64z M30,21.64c-0.624,0-1.147,0.425-1.306,1H3.305c-0.159-0.575-0.681-1-1.305-1c-0.75,0-1.36,0.61-1.36,1.36S1.25,24.36,2,24.36\tc0.624,0,1.146-0.425,1.305-1h25.389c0.159,0.575,0.681,1,1.306,1c0.75,0,1.36-0.61,1.36-1.36S30.75,21.64,30,21.64z M2,23.64\tc-0.353,0-0.64-0.287-0.64-0.64S1.647,22.36,2,22.36S2.64,22.647,2.64,23S2.353,23.64,2,23.64z M30,23.64\tc-0.353,0-0.64-0.287-0.64-0.64s0.287-0.64,0.64-0.64s0.64,0.287,0.64,0.64S30.353,23.64,30,23.64z M30,28.64\tc-0.624,0-1.147,0.425-1.306,1H3.305c-0.159-0.575-0.681-1-1.305-1c-0.75,0-1.36,0.61-1.36,1.36S1.25,31.36,2,31.36\tc0.624,0,1.146-0.425,1.305-1h25.389c0.159,0.575,0.681,1,1.306,1c0.75,0,1.36-0.61,1.36-1.36S30.75,28.64,30,28.64z M2,30.64\tc-0.353,0-0.64-0.287-0.64-0.64S1.647,29.36,2,29.36S2.64,29.647,2.64,30S2.353,30.64,2,30.64z M30,30.64\tc-0.353,0-0.64-0.287-0.64-0.64s0.287-0.64,0.64-0.64s0.64,0.287,0.64,0.64S30.353,30.64,30,30.64z M30,14.64\tc-0.625,0-1.147,0.425-1.306,1H3.305c-0.159-0.575-0.681-1-1.305-1c-0.75,0-1.36,0.61-1.36,1.36S1.25,17.36,2,17.36\tc0.625,0,1.147-0.426,1.305-1h25.389c0.159,0.575,0.681,1,1.306,1c0.75,0,1.36-0.61,1.36-1.36S30.75,14.64,30,14.64z M2,16.64\tc-0.353,0-0.64-0.287-0.64-0.64S1.647,15.36,2,15.36S2.64,15.647,2.64,16C2.64,16.353,2.353,16.64,2,16.64z M30,16.64\tc-0.353,0-0.64-0.287-0.64-0.64s0.287-0.64,0.64-0.64s0.64,0.287,0.64,0.64C30.64,16.353,30.353,16.64,30,16.64z"
  })), children);
});

export { Migration as default };
