// @carbon/pictograms-react/lib/resources/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Resources = /*#__PURE__*/React__default["default"].forwardRef(function Resources(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,31.36c-0.059,0-0.117-0.015-0.171-0.043l-13-7 C2.713,24.254,2.64,24.133,2.64,24V8c0-0.132,0.073-0.254,0.189-0.317l13-7c0.107-0.058,0.234-0.058,0.342,0l13,7 C29.287,7.746,29.36,7.868,29.36,8v16c0,0.133-0.073,0.254-0.189,0.317l-13,7C16.117,31.346,16.059,31.36,16,31.36z M3.36,23.785 L16,30.591l12.64-6.806V8.215L16,1.409L3.36,8.215V23.785z M22.36,24H21.64v-4c0-2.333-1.471-4.453-3.66-5.275 c-0.128-0.048-0.218-0.165-0.231-0.301s0.052-0.269,0.168-0.341c1.079-0.673,1.724-1.825,1.724-3.082c0-2.007-1.633-3.64-3.64-3.64 c-2.007,0-3.64,1.633-3.64,3.64c0,1.257,0.645,2.409,1.724,3.082c0.116,0.073,0.181,0.205,0.168,0.341 c-0.014,0.136-0.104,0.253-0.232,0.301c-2.189,0.822-3.66,2.942-3.66,5.275v4H9.64v-4c0-2.415,1.397-4.627,3.535-5.692 C12.204,13.485,11.64,12.289,11.64,11c0-2.404,1.956-4.36,4.36-4.36s4.36,1.956,4.36,4.36c0,1.289-0.564,2.485-1.535,3.308 c2.138,1.064,3.535,3.277,3.535,5.692V24z"
  })), children);
});

module.exports = Resources;
