// @carbon/pictograms-react/lib/time--lapse/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const TimeLapse = /*#__PURE__*/React__default["default"].forwardRef(function TimeLapse(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M9.252,20.312l-0.36-0.623l6.749-3.896V5h0.72v11.208L9.252,20.312z M16,30.64\tC7.927,30.64,1.36,24.072,1.36,16C1.36,7.927,7.927,1.36,16,1.36c8.072,0,14.64,6.567,14.64,14.64c0,4.402-1.915,8.466-5.279,11.259\tV22H24.64v6.36H31v-0.72h-4.976c3.403-2.924,5.336-7.11,5.336-11.64c0-8.47-6.89-15.36-15.36-15.36S0.64,7.53,0.64,16\tS7.53,31.36,16,31.36V30.64z"
  })), children);
});

module.exports = TimeLapse;
