// @carbon/pictograms-react/lib/advanced--threats/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const AdvancedThreats = /*#__PURE__*/React__default["default"].forwardRef(function AdvancedThreats(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M3.746,27.745L5.491,26l-1.745-1.745l0.509-0.51L6,25.491l1.746-1.746l0.509,0.51L6.509,26\tl1.745,1.745l-0.509,0.51L6,26.509l-1.746,1.746L3.746,27.745z M31.36,1v20c0,0.199-0.161,0.36-0.36,0.36H11.36V31\tc0,0.199-0.161,0.36-0.36,0.36H1c-0.199,0-0.36-0.161-0.36-0.36V21c0-0.199,0.161-0.36,0.36-0.36h9.64V1\tc0-0.199,0.161-0.36,0.36-0.36h20C31.199,0.64,31.36,0.801,31.36,1z M10.64,21.36H1.36v9.279h9.28\tC10.64,30.639,10.64,21.36,10.64,21.36z M30.64,1.36H11.36v19.28h19.28V1.36z M17.255,15.254L21,11.509l3.745,3.745l0.51-0.509\tL21.509,11l3.746-3.746l-0.51-0.509L21,10.491l-3.745-3.745l-0.509,0.509L20.491,11l-3.745,3.746L17.255,15.254z"
  })), children);
});

module.exports = AdvancedThreats;
