// @carbon/pictograms-react/lib/no--smoking/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const NoSmoking = /*#__PURE__*/React__default["default"].forwardRef(function NoSmoking(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M27.36,20.36h-0.72v-4.72h0.721L27.36,20.36L27.36,20.36z M31.36,16c0-8.47-6.89-15.36-15.36-15.36\tS0.64,7.53,0.64,16S7.53,31.36,16,31.36S31.36,24.47,31.36,16z M15.131,15.64H5c-0.199,0-0.36,0.161-0.36,0.36v4\tc0,0.199,0.161,0.36,0.36,0.36h14.851l6.239,6.238C23.462,29.102,19.907,30.64,16,30.64C7.927,30.64,1.36,24.072,1.36,16\tc0-3.907,1.539-7.461,4.042-10.089L15.131,15.64z M25,20.36c0.199,0,0.36-0.161,0.36-0.36v-4c0-0.199-0.161-0.36-0.36-0.36h-8.851\tL5.911,5.401C8.539,2.898,12.093,1.36,16,1.36c8.072,0,14.64,6.567,14.64,14.64c0,3.907-1.538,7.462-4.041,10.09l-5.729-5.729\tL25,20.36L25,20.36z M16.869,16.36h7.771v3.28h-4.49L16.869,16.36z M5.36,16.36h10.491l3.28,3.28H5.36V16.36z M27.36,13\tc0-2.404-1.956-4.36-4.36-4.36h-5c-0.904,0-1.64-0.736-1.64-1.64V6h-0.72v1c0,1.301,1.059,2.36,2.36,2.36h5\tc2.007,0,3.64,1.633,3.64,3.64v1h0.721L27.36,13L27.36,13z M25.36,13c0-1.301-1.059-2.36-2.36-2.36h-5\tc-2.007,0-3.64-1.633-3.64-3.64V6h-0.72v1c0,2.404,1.956,4.36,4.36,4.36h5c0.904,0,1.64,0.736,1.64,1.64v1h0.721L25.36,13L25.36,13z"
  })), children);
});

module.exports = NoSmoking;
