// @carbon/pictograms-react/es/drag-and-drop-interface/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const DragAndDropInterface = /*#__PURE__*/React.forwardRef(function DragAndDropInterface(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M22.213,13.066\tc-1.606,0-3.114,0.419-4.427,1.148c0.73-1.313,1.148-2.821,1.148-4.427c0-5.044-4.104-9.147-9.147-9.147S0.64,4.743,0.64,9.787\ts4.103,9.147,9.147,9.147c1.606,0,3.114-0.419,4.427-1.148c-0.729,1.313-1.148,2.821-1.148,4.427c0,5.044,4.103,9.147,9.147,9.147\ts9.147-4.104,9.147-9.147S27.257,13.066,22.213,13.066z M1.36,9.787c0-4.646,3.781-8.427,8.427-8.427s8.427,3.781,8.427,8.427\tc0,2.194-0.85,4.188-2.229,5.689l-4.808-4.808C11.29,10.47,11.36,10.244,11.36,10c0-0.75-0.61-1.36-1.36-1.36S8.64,9.25,8.64,10\ts0.61,1.36,1.36,1.36c0.244,0,0.47-0.07,0.668-0.183l4.808,4.808c-1.501,1.379-3.495,2.229-5.689,2.229\tC5.141,18.214,1.36,14.434,1.36,9.787z M10,10.64c-0.353,0-0.64-0.287-0.64-0.64S9.647,9.36,10,9.36s0.64,0.287,0.64,0.64\tS10.353,10.64,10,10.64z M22.213,30.64c-4.646,0-8.427-3.78-8.427-8.427c0-2.194,0.85-4.188,2.229-5.689l7.116,7.116H21v0.721h3.36\tV21h-0.72v2.13l-7.116-7.116c1.501-1.379,3.495-2.229,5.689-2.229c4.646,0,8.427,3.781,8.427,8.427S26.859,30.64,22.213,30.64z"
  })), children);
});

export { DragAndDropInterface as default };
