// @carbon/pictograms-react/es/chart--stepper/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const ChartStepper = /*#__PURE__*/React.forwardRef(function ChartStepper(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M31,31.36H1c-0.199,0-0.36-0.161-0.36-0.36V1h0.72v22.64h9.28V16c0-0.199,0.161-0.36,0.36-0.36h9.64\tV8c0-0.199,0.161-0.36,0.36-0.36h10v0.72h-9.64V16c0,0.199-0.161,0.36-0.36,0.36h-9.64V24c0,0.199-0.161,0.36-0.36,0.36H1.36v6.279\tH31V31.36z"
  })), children);
});

export { ChartStepper as default };
