// @carbon/pictograms-react/es/application--security/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const ApplicationSecurity = /*#__PURE__*/React.forwardRef(function ApplicationSecurity(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    d: "M29,2.5C29,2.776,28.776,3,28.5,3S28,2.776,28,2.5S28.224,2,28.5,2S29,2.224,29,2.5z M27,2.5C27,2.776,26.776,3,26.5,3\tS26,2.776,26,2.5S26.224,2,26.5,2S27,2.224,27,2.5z M25,2.5C25,2.776,24.776,3,24.5,3S24,2.776,24,2.5S24.224,2,24.5,2\tS25,2.224,25,2.5z M15,23.36H1c-0.199,0-0.36-0.161-0.36-0.36V1c0-0.199,0.161-0.36,0.36-0.36h30c0.199,0,0.36,0.161,0.36,0.36v22\th-0.72V4.36H1.36v18.28H15V23.36z M1.36,3.64h29.28V1.36H1.36V3.64z M10,14.36H4v-0.72h6V14.36z M10,11.36H4v-0.72h6V11.36z M10,8.36H4V7.64h6V8.36z M23,31.36c-3.507,0-6.36-2.854-6.36-6.36v-7.223l6.36-3.18l6.36,3.18V25\tC29.36,28.507,26.507,31.36,23,31.36z M17.36,18.223V25c0,3.11,2.529,5.64,5.64,5.64s5.64-2.529,5.64-5.64v-6.777L23,15.402\tL17.36,18.223z",
    clipRule: "evenodd"
  })), children);
});

export { ApplicationSecurity as default };
