// @carbon/pictograms-react/lib/nyc--brooklyn/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const NycBrooklyn = /*#__PURE__*/React__default["default"].forwardRef(function NycBrooklyn(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M7.64,16h0.72v4H7.64V16z M7.64,14h0.72v-4H7.64V14z M8.36,4H7.64v4h0.72V4z M9.64,20h0.72v-4H9.64\tV20z M9.64,14h0.72v-4H9.64V14z M10.36,4H9.64v4h0.72C10.36,8,10.36,4,10.36,4z M5.64,20h0.72v-4H5.64V20z M5.64,14h0.72v-4H5.64V14\tz M6.36,4H5.64v4h0.72C6.36,8,6.36,4,6.36,4z M3.64,20h0.72v-4H3.64V20z M3.64,14h0.72v-4H3.64V14z M4.36,4H3.64v4h0.72V4z M1.64,20\th0.72v-4H1.64V20z M1.64,14h0.72v-4H1.64V14z M1.64,8h0.72V4H1.64V8z M31,13.64v0.72H20.36V31h-0.72v-6.64H4.36V31H3.64v-7.36h8\tV1.36H1V0.64h11.36v7h4v16h3.28v-10h4v-3.28H23c-0.199,0-0.36-0.161-0.36-0.36V4c0-0.121,0.061-0.233,0.161-0.299l3-2\tc0.119-0.081,0.279-0.081,0.398,0l3,2C29.3,3.767,29.36,3.879,29.36,4v6c0,0.199-0.161,0.36-0.36,0.36h-0.64v3.28\tC28.36,13.64,31,13.64,31,13.64z M12.36,23.64h3.28V8.36h-3.28V23.64z M23.36,9.64h5.279V4.193L26,2.433l-2.64,1.76V9.64z M24.36,13.64h3.279v-3.28H24.36V13.64z M27.64,24h0.721v-3H27.64V24z M23.64,24h0.721v-3H23.64V24z M27.64,29h0.721v-3H27.64V29z M23.64,29h0.721v-3H23.64V29z M23.64,19h0.721v-3H23.64V19z M27.64,19h0.721v-3H27.64V19z M7.64,31h0.72v-2.64h3.28V31h0.72v-2.64\th3.28V31h0.72v-3.36H7.64V31z"
  })), children);
});

module.exports = NycBrooklyn;
