// @carbon/pictograms-react/es/deployment/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Deployment = /*#__PURE__*/React.forwardRef(function Deployment(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M30.36,30.36H22v-0.72h7.131L16.673,17.182\tc-0.397,0.227-0.948,0.227-1.346,0L2.869,29.64H10v0.721H1.64V22h0.72v7.131l12.458-12.458C14.705,16.474,14.64,16.245,14.64,16\ts0.065-0.474,0.178-0.673L2.36,2.869V10H1.64V1.64H10v0.72H2.869l12.458,12.458c0.397-0.227,0.948-0.227,1.346,0L29.131,2.36H22\tV1.64h8.36V10h-0.72V2.869L17.182,15.327c0.113,0.199,0.179,0.428,0.179,0.673c0,0.245-0.065,0.474-0.179,0.673L29.64,29.131V22\th0.721L30.36,30.36L30.36,30.36z M16,15.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64\tS16.353,15.36,16,15.36z"
  })), children);
});

export { Deployment as default };
