// @carbon/pictograms-react/lib/cloud/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Cloud = /*#__PURE__*/React__default["default"].forwardRef(function Cloud(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M24,27.36H10c-5.161,0-9.36-4.199-9.36-9.36\tS4.839,8.64,10,8.64c0.483,0,0.984,0.042,1.493,0.125C13.092,6.776,15.452,5.64,18,5.64c4.278,0,7.804,3.152,8.298,7.372\tC29.29,13.996,31.36,16.834,31.36,20C31.36,24.059,28.059,27.36,24,27.36z M10,9.36c-4.764,0-8.64,3.876-8.64,8.64\ts3.876,8.64,8.64,8.64h14c3.661,0,6.64-2.979,6.64-6.64c0-2.929-1.964-5.548-4.776-6.369c-0.143-0.042-0.245-0.166-0.258-0.314\tC25.253,9.351,21.983,6.36,18,6.36c-2.395,0-4.608,1.099-6.071,3.016c-0.083,0.108-0.219,0.16-0.352,0.136\tC11.03,9.41,10.514,9.36,10,9.36z"
  })), children);
});

module.exports = Cloud;
