// @carbon/pictograms-react/lib/wheat/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Wheat = /*#__PURE__*/React__default["default"].forwardRef(function Wheat(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M9.36,31H8.64v-2.652c-2.789-0.186-5-2.514-5-5.348V3h0.72v2.931l1.78,1.028V2h0.72v5.375l1.78,1.028V1h0.72\tv7.402l1.78-1.028V2h0.72v4.959l1.78-1.028V3h0.72v20c0,2.834-2.211,5.162-5,5.348C9.36,28.348,9.36,31,9.36,31z M9.36,24.208v3.418\tc2.391-0.185,4.28-2.188,4.28-4.626v-1.264L9.36,24.208z M4.36,21.736V23c0,2.438,1.889,4.441,4.28,4.626v-3.418L4.36,21.736z M9.36,19.287v4.089l4.28-2.471v-4.089L9.36,19.287z M4.36,20.905l4.28,2.471v-4.089l-4.28-2.471V20.905z M9.36,14.208v4.248\tl4.28-2.471v-4.248L9.36,14.208z M4.36,15.985l4.28,2.471v-4.248l-4.28-2.471V15.985z M9.36,9.234v4.143l4.28-2.471V6.763\tL9.36,9.234z M4.36,10.905l4.28,2.471V9.234L4.36,6.763V10.905z M23.36,31h-0.72v-2.652c-2.788-0.186-5-2.514-5-5.348V3h0.721v2.931\tl1.779,1.028V2h0.721v5.375l1.779,1.028V1h0.721v7.402l1.779-1.028V2h0.721v4.959l1.779-1.028V3h0.721v20\tc0,2.834-2.212,5.162-5,5.348L23.36,31L23.36,31z M23.36,24.208v3.418c2.391-0.185,4.279-2.188,4.279-4.626v-1.264L23.36,24.208z M18.36,21.736V23c0,2.438,1.889,4.441,4.279,4.626v-3.418L18.36,21.736z M23.36,19.287v4.089l4.279-2.471v-4.089L23.36,19.287z M18.36,20.905l4.279,2.471v-4.089l-4.279-2.471V20.905z M23.36,14.208v4.248l4.279-2.471v-4.248L23.36,14.208z M18.36,15.985\tl4.279,2.471v-4.248l-4.279-2.471V15.985z M23.36,9.234v4.143l4.279-2.471V6.763L23.36,9.234z M18.36,10.905l4.279,2.471V9.234\tL18.36,6.763V10.905z"
  })), children);
});

module.exports = Wheat;
