// @carbon/pictograms-react/es/stairs/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Stairs = /*#__PURE__*/React.forwardRef(function Stairs(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M1.36,31H0.64v-6c0-0.199,0.161-0.36,0.36-0.36h5.64V19c0-0.199,0.161-0.36,0.36-0.36h5.64V13\tc0-0.199,0.161-0.36,0.36-0.36h5.64V7c0-0.199,0.161-0.36,0.36-0.36h5.64V1c0-0.199,0.161-0.36,0.36-0.36h6v0.72h-5.64V7\tc0,0.199-0.161,0.36-0.36,0.36h-5.64V13c0,0.199-0.161,0.36-0.36,0.36h-5.64V19c0,0.199-0.161,0.36-0.36,0.36H7.36V25\tc0,0.199-0.161,0.36-0.36,0.36H1.36V31z"
  })), children);
});

export { Stairs as default };
