// @carbon/pictograms-react/lib/network--of-devices/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const NetworkOfDevices = /*#__PURE__*/React__default["default"].forwardRef(function NetworkOfDevices(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M3,31.36H1c-0.199,0-0.36-0.161-0.36-0.36\tv-2c0-0.199,0.161-0.36,0.36-0.36h0.64V2c0-0.199,0.161-0.36,0.36-0.36h26.64V1c0-0.199,0.161-0.36,0.36-0.36h2\tc0.199,0,0.36,0.161,0.36,0.36v2c0,0.199-0.161,0.36-0.36,0.36h-0.64V30c0,0.199-0.161,0.36-0.36,0.36H3.36V31\tC3.36,31.199,3.199,31.36,3,31.36z M1.36,30.64h1.28v-1.28H1.36V30.64z M3.36,29.64h26.28v-6.28H9.36V24\tc0,0.199-0.161,0.36-0.36,0.36H7c-0.199,0-0.36-0.161-0.36-0.36v-0.64H2.36v5.279H3c0.199,0,0.36,0.161,0.36,0.36\tC3.36,28.999,3.36,29.64,3.36,29.64z M7.36,23.64h1.28v-1.28H7.36V23.64z M9.36,22.64h20.28v-6.28H17.36V17\tc0,0.199-0.161,0.36-0.36,0.36h-2c-0.199,0-0.36-0.161-0.36-0.36v-0.64H2.36v6.28h4.28V22c0-0.199,0.161-0.36,0.36-0.36h2\tc0.199,0,0.36,0.161,0.36,0.36V22.64z M15.36,16.64h1.28v-1.28h-1.28V16.64z M17.36,15.64h12.28V9.36h-5.28V10\tc0,0.199-0.161,0.36-0.36,0.36h-2c-0.199,0-0.36-0.161-0.36-0.36V9.36H2.36v6.28h12.28V15c0-0.199,0.161-0.36,0.36-0.36h2\tc0.199,0,0.36,0.161,0.36,0.36V15.64z M22.36,9.64h1.279V8.36H22.36V9.64z M24.36,8.64h5.279V3.36H29c-0.199,0-0.36-0.161-0.36-0.36\tV2.36H2.36v6.28h19.28V8c0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36C24.36,8,24.36,8.64,24.36,8.64z M29.36,2.64\th1.279V1.36H29.36V2.64z"
  })), children);
});

module.exports = NetworkOfDevices;
