// @carbon/pictograms-react/lib/washington-dc--monument/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const WashingtonDcMonument = /*#__PURE__*/React__default["default"].forwardRef(function WashingtonDcMonument(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M28.36,29.64v-5.28h2v-2.72h-2.72V22v2.36v5.279h-3.28V24.36h2v-2.72h-2.72V22v2.36v5.279\th-3.28V24.36h2v-2.72h-2.72V22v2.36v5.279h-1.28V5c0-0.056-0.014-0.111-0.038-0.161l-2-4c-0.121-0.244-0.522-0.244-0.644,0l-2,4\tC13.653,4.889,13.64,4.944,13.64,5v24.64h-3.28v-5.28h2v-2.72H9.64V22v2.36v5.279H6.36V24.36h2v-2.72H5.64V22v2.36v5.279H2.36V24.36\th2v-2.72H1.64V22v2.36v5.279H1v0.721h30v-0.72H28.36z M29.64,23.64h-1.28v-1.28h1.279L29.64,23.64L29.64,23.64z M25.64,23.64h-1.28\tv-1.28h1.279L25.64,23.64L25.64,23.64z M21.64,23.64h-1.28v-1.28h1.279L21.64,23.64L21.64,23.64z M11.64,23.64h-1.28v-1.28h1.28\tV23.64z M7.64,23.64H6.36v-1.28h1.28C7.64,22.36,7.64,23.64,7.64,23.64z M3.64,23.64H2.36v-1.28h1.28\tC3.64,22.36,3.64,23.64,3.64,23.64z M14.36,5.085L16,1.805l1.64,3.28V29.64h-3.28V5.085z"
  })), children);
});

module.exports = WashingtonDcMonument;
