// @carbon/pictograms-react/lib/access-management/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const AccessManagement = /*#__PURE__*/React__default["default"].forwardRef(function AccessManagement(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,31.36C7.53,31.36,0.64,24.47,0.64,16\tS7.53,0.64,16,0.64S31.36,7.53,31.36,16S24.47,31.36,16,31.36z M16,1.36C7.927,1.36,1.36,7.927,1.36,16\tc0,8.072,6.567,14.64,14.64,14.64c8.072,0,14.64-6.567,14.64-14.64S24.072,1.36,16,1.36z M28,19.36h-2\tc-0.199,0-0.36-0.161-0.36-0.36c0-0.353-0.287-0.64-0.64-0.64s-0.64,0.287-0.64,0.64c0,0.199-0.161,0.36-0.36,0.36h-2\tc-0.199,0-0.36-0.161-0.36-0.36v-2.64H11.143c-0.573,1.741-2.214,3-4.143,3c-2.404,0-4.36-1.956-4.36-4.36S4.596,10.64,7,10.64\tc1.929,0,3.57,1.26,4.143,3H28c0.199,0,0.36,0.161,0.36,0.36v5C28.36,19.199,28.199,19.36,28,19.36z M26.312,18.64h1.328v-4.28\tH11.313c0.031,0.209,0.047,0.423,0.047,0.64s-0.016,0.431-0.047,0.64H22c0.199,0,0.36,0.161,0.36,0.36v2.64h1.328\tc0.158-0.575,0.687-1,1.312-1S26.153,18.064,26.312,18.64z M7,11.36c-2.007,0-3.64,1.633-3.64,3.64S4.993,18.64,7,18.64\ts3.64-1.633,3.64-3.64S9.007,11.36,7,11.36z"
  })), children);
});

module.exports = AccessManagement;
