// @carbon/pictograms-react/lib/data--backup/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const DataBackup = /*#__PURE__*/React__default["default"].forwardRef(function DataBackup(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M13.5,7.36H7V6.64h6.5V7.36z M13.5,15.64H7v0.72h6.5V15.64z M7,25.36h6.5v-0.72H7V25.36z M30.36,7v9\tc0,0.199-0.161,0.36-0.36,0.36h-3.64V19c0,0.199-0.161,0.36-0.36,0.36H6c-0.199,0-0.36-0.161-0.36-0.36v-2.64H2.36v8.28h3.28V22\tc0-0.199,0.161-0.36,0.36-0.36h20c0.199,0,0.36,0.161,0.36,0.36v6c0,0.199-0.161,0.36-0.36,0.36H6c-0.199,0-0.36-0.161-0.36-0.36\tv-2.64H2c-0.199,0-0.36-0.161-0.36-0.36v-9c0-0.199,0.161-0.36,0.36-0.36h3.64V13c0-0.199,0.161-0.36,0.36-0.36h20\tc0.199,0,0.36,0.161,0.36,0.36v2.64h3.279V7.36H26.36V10c0,0.199-0.161,0.36-0.36,0.36H6c-0.199,0-0.36-0.161-0.36-0.36V4\tc0-0.199,0.161-0.36,0.36-0.36h20c0.199,0,0.36,0.161,0.36,0.36v2.64H30C30.199,6.64,30.36,6.801,30.36,7z M25.64,4.36H6.36v5.28\th19.28C25.64,9.64,25.64,4.36,25.64,4.36z M6.36,27.64h19.28v-5.28H6.36V27.64z M25.64,13.36H6.36v5.28h19.28\tC25.64,18.64,25.64,13.36,25.64,13.36z M24,7c0-0.276-0.224-0.5-0.5-0.5S23,6.724,23,7s0.224,0.5,0.5,0.5S24,7.276,24,7z M24,16\tc0-0.276-0.224-0.5-0.5-0.5S23,15.724,23,16c0,0.276,0.224,0.5,0.5,0.5S24,16.276,24,16z M24,25c0-0.276-0.224-0.5-0.5-0.5\tS23,24.724,23,25s0.224,0.5,0.5,0.5S24,25.276,24,25z"
  })), children);
});

module.exports = DataBackup;
