// @carbon/pictograms-react/lib/augmented--reality/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const AugmentedReality = /*#__PURE__*/React__default["default"].forwardRef(function AugmentedReality(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M22,31.36H4c-0.199,0-0.36-0.161-0.36-0.36V1c0-0.199,0.161-0.36,0.36-0.36h18\tc0.199,0,0.36,0.161,0.36,0.36v2.791l6.818,3.896C29.291,7.751,29.36,7.871,29.36,8v8c0,0.129-0.069,0.249-0.182,0.312l-6.818,3.896\tV31C22.36,31.199,22.199,31.36,22,31.36z M4.36,30.64h17.28v-6.28H4.36V30.64z M4.36,23.64h17.28v-3.431l-6.818-3.896\tC14.709,16.249,14.64,16.129,14.64,16V8c0-0.129,0.069-0.249,0.181-0.312l6.818-3.896V1.36H4.36V23.64z M22.36,12.209v7.171\tl6.279-3.589V8.62L22.36,12.209z M15.36,15.791l6.28,3.589v-7.171L15.36,8.62V15.791z M15.726,8L22,11.585L28.274,8L22,4.415\tL15.726,8z M14,27.5c0-0.552-0.448-1-1-1s-1,0.448-1,1s0.448,1,1,1S14,28.052,14,27.5z"
  })), children);
});

module.exports = AugmentedReality;
