// @carbon/pictograms-react/lib/identify-and-resolve-issues/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const IdentifyAndResolveIssues = /*#__PURE__*/React__default["default"].forwardRef(function IdentifyAndResolveIssues(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M30,1.64H2\tC1.801,1.64,1.64,1.801,1.64,2v28c0,0.199,0.161,0.36,0.36,0.36h28c0.199,0,0.36-0.161,0.36-0.36V2C30.36,1.801,30.199,1.64,30,1.64\tz M29.64,29.64H2.36V2.36h27.28C29.64,2.36,29.64,29.64,29.64,29.64z M8,17.36c0.244,0,0.47-0.07,0.669-0.183L14,22.509\tL27.255,9.254l-0.51-0.509L14,21.491l-4.823-4.823C9.29,16.47,9.36,16.244,9.36,16c0-0.75-0.61-1.36-1.36-1.36S6.64,15.25,6.64,16\tS7.25,17.36,8,17.36z M8,15.36c0.353,0,0.64,0.287,0.64,0.64S8.353,16.64,8,16.64S7.36,16.353,7.36,16S7.647,15.36,8,15.36z"
  })), children);
});

module.exports = IdentifyAndResolveIssues;
