// @carbon/pictograms-react/lib/list--bullet/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const ListBullet = /*#__PURE__*/React__default["default"].forwardRef(function ListBullet(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M4,31.36c-1.853,0-3.36-1.508-3.36-3.36S2.147,24.64,4,24.64S7.36,26.147,7.36,28S5.853,31.36,4,31.36\tz M4,25.36c-1.456,0-2.64,1.184-2.64,2.64S2.544,30.64,4,30.64S6.64,29.456,6.64,28S5.456,25.36,4,25.36z M31,28.36H11v-0.72h20\tV28.36z M4,19.36c-1.853,0-3.36-1.508-3.36-3.36S2.147,12.64,4,12.64S7.36,14.147,7.36,16S5.853,19.36,4,19.36z M4,13.36\tc-1.456,0-2.64,1.185-2.64,2.64c0,1.456,1.185,2.64,2.64,2.64S6.64,17.456,6.64,16S5.456,13.36,4,13.36z M31,16.36H11v-0.72h20\tV16.36z M4,7.36C2.147,7.36,0.64,5.853,0.64,4S2.147,0.64,4,0.64S7.36,2.147,7.36,4S5.853,7.36,4,7.36z M4,1.36\tC2.544,1.36,1.36,2.544,1.36,4S2.544,6.64,4,6.64S6.64,5.456,6.64,4S5.456,1.36,4,1.36z M31,4.36H11V3.64h20V4.36z"
  })), children);
});

module.exports = ListBullet;
