// @carbon/pictograms-react/lib/assets/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Assets = /*#__PURE__*/React__default["default"].forwardRef(function Assets(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31,0.64H10C9.801,0.64,9.64,0.801,9.64,1v2.64H1\tC0.801,3.64,0.64,3.801,0.64,4v27c0,0.199,0.161,0.36,0.36,0.36h21c0.199,0,0.36-0.161,0.36-0.36v-2.64H31\tc0.199,0,0.36-0.161,0.36-0.36V1C31.36,0.801,31.199,0.64,31,0.64z M1.36,30.64V4.36h14.28V10c0,0.199,0.161,0.36,0.36,0.36h5.64\tv20.28H1.36z M16.36,4.869l4.771,4.771H16.36V4.869z M30.64,27.64h-8.28V10c0-0.096-0.038-0.187-0.105-0.254l-6-6\tC16.187,3.678,16.096,3.64,16,3.64h-5.64V1.36h20.28C30.64,1.36,30.64,27.64,30.64,27.64z M4,15.64h15v0.72H4V15.64z M4,18.64h15\tv0.721H4V18.64z M4,21.64h15v0.721H4V21.64z M4,24.64h15v0.721H4V24.64z"
  })), children);
});

module.exports = Assets;
