// @carbon/pictograms-react/lib/document--security/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const DocumentSecurity = /*#__PURE__*/React__default["default"].forwardRef(function DocumentSecurity(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M24.5,31.36c-3.782,0-6.86-3.078-6.86-6.86V17c0-0.199,0.161-0.36,0.36-0.36h13\tc0.199,0,0.36,0.161,0.36,0.36v7.5C31.36,28.282,28.282,31.36,24.5,31.36z M18.36,17.36v7.14c0,3.386,2.754,6.14,6.14,6.14\ts6.14-2.754,6.14-6.14v-7.14H18.36z M16,31.36H1c-0.199,0-0.36-0.161-0.36-0.36V1c0-0.199,0.161-0.36,0.36-0.36h18\tc0.096,0,0.188,0.038,0.255,0.105l6,6C25.322,6.813,25.36,6.904,25.36,7v7h-0.72V7.36H19c-0.199,0-0.36-0.161-0.36-0.36V1.36H1.36\tv29.28H16V31.36z M19.36,6.64h4.771L19.36,1.869V6.64z M15,27.36H5v-0.72h10V27.36z M15,23.36H5v-0.72h10V23.36z M15,19.36H5v-0.72\th10V19.36z M5,15.36v-0.72h10v0.72H5z M21,11.36H5v-0.72h16V11.36z"
  })), children);
});

module.exports = DocumentSecurity;
