// @carbon/pictograms-react/lib/watsonx--code-assistant-for-z/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const WatsonxCodeAssistantForZ = /*#__PURE__*/React__default["default"].forwardRef(function WatsonxCodeAssistantForZ(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M27.255,30.255l-0.51-0.51\tL30.491,26l-3.746-3.745l0.51-0.51L31.509,26L27.255,30.255z M18.745,30.255L14.491,26l4.254-4.255l0.51,0.51L15.509,26l3.746,3.745\tL18.745,30.255z M21.334,31.134l-0.668-0.268l4-10l0.668,0.268L21.334,31.134z M13,30.36H1c-0.199,0-0.36-0.161-0.36-0.36V2\tc0-0.199,0.161-0.36,0.36-0.36h12c0.199,0,0.36,0.161,0.36,0.36v28C13.36,30.199,13.199,30.36,13,30.36z M1.36,29.64h10.56\tL1.36,21.72V29.64z M1.6,21l11.04,8.28V12.72L1.6,21z M1.36,2.769V20.28l11.06-8.295L1.36,2.769z M1.994,2.36l10.646,8.872V2.36\tH1.994z M29.36,19h-0.72V2.36H17.36V19h-0.72V2c0-0.199,0.161-0.36,0.36-0.36h12c0.199,0,0.36,0.161,0.36,0.36\tC29.36,2,29.36,19,29.36,19z"
  })), children);
});

module.exports = WatsonxCodeAssistantForZ;
