// @carbon/pictograms-react/lib/mobile--add/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const MobileAdd = /*#__PURE__*/React__default["default"].forwardRef(function MobileAdd(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M20,28.5c-0.552,0-1-0.448-1-1s0.448-1,1-1s1,0.448,1,1S20.552,28.5,20,28.5z M29.36,31V1\tc0-0.199-0.161-0.36-0.36-0.36H11c-0.199,0-0.36,0.161-0.36,0.36v2.647c-4.443,0.189-8,3.863-8,8.353s3.557,8.163,8,8.353V31\tc0,0.199,0.161,0.36,0.36,0.36h18C29.199,31.36,29.36,31.199,29.36,31z M11.36,24.36h17.28v6.279H11.36V24.36z M11.36,20.353\tc4.443-0.189,8-3.863,8-8.353s-3.557-8.163-8-8.353V1.36h17.28v22.28H11.36V20.353z M18.64,12c0,4.213-3.427,7.64-7.64,7.64\tS3.36,16.213,3.36,12S6.787,4.36,11,4.36S18.64,7.787,18.64,12z M11.36,12.359H15v-0.72h-3.64V8h-0.72v3.64H7v0.72h3.64V16h0.72\tC11.36,16,11.36,12.359,11.36,12.359z"
  })), children);
});

module.exports = MobileAdd;
