// @carbon/pictograms-react/lib/mortar-and-pestle/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const MortarAndPestle = /*#__PURE__*/React__default["default"].forwardRef(function MortarAndPestle(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M15.982,30.36C7.522,30.36,0.64,23.47,0.64,15c0-0.199,0.161-0.36,0.36-0.36h5.647L2.049,6.678\tc-0.447-0.774-0.565-1.68-0.333-2.549s0.788-1.594,1.562-2.041c1.604-0.926,3.664-0.375,4.59,1.229l6.536,11.323H31\tc0.199,0,0.36,0.161,0.36,0.36C31.36,23.47,24.462,30.36,15.982,30.36z M1.364,15.36c0.191,7.907,6.675,14.28,14.618,14.28\tc7.962,0,14.461-6.373,14.654-14.28H1.364z M7.479,14.64h6.093L7.245,3.678C6.517,2.417,4.899,1.983,3.639,2.711\tc-0.608,0.351-1.044,0.92-1.227,1.604c-0.183,0.683-0.09,1.395,0.26,2.002L7.479,14.64z"
  })), children);
});

module.exports = MortarAndPestle;
