// @carbon/pictograms-react/es/trophy/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Trophy = /*#__PURE__*/React.forwardRef(function Trophy(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M23,31.36H9v-0.72h14V31.36z M21,29.36H11v-0.72h4.64\tv-6.287c-4.324-0.185-7.808-3.669-7.993-7.993H7c-2.404,0-4.36-1.956-4.36-4.36V5c0-0.199,0.161-0.36,0.36-0.36h4.64V2\tc0-0.199,0.161-0.36,0.36-0.36h16c0.199,0,0.36,0.161,0.36,0.36v2.64H29c0.199,0,0.36,0.161,0.36,0.36v5\tc0,2.404-1.956,4.36-4.36,4.36h-0.647c-0.185,4.324-3.669,7.808-7.993,7.993v6.287H21V29.36z M8.36,2.36V14\tc0,4.213,3.427,7.64,7.64,7.64s7.64-3.427,7.64-7.64V2.36H8.36z M24.36,13.64H25c2.007,0,3.64-1.633,3.64-3.64V5.36h-4.28V13.64z M3.36,5.36V10c0,2.007,1.633,3.64,3.64,3.64h0.64V5.36H3.36z"
  })), children);
});

export { Trophy as default };
