// @carbon/pictograms-react/es/unauthorized--user--access/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const UnauthorizedUserAccess = /*#__PURE__*/React.forwardRef(function UnauthorizedUserAccess(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M28.36,25.763V31h-0.72v-5.237c0-5.419-3.679-10.076-8.946-11.324\tc-0.15-0.036-0.261-0.163-0.275-0.316c-0.015-0.154,0.071-0.3,0.212-0.363c2.34-1.041,3.851-3.365,3.851-5.919\tc0-3.573-2.907-6.48-6.48-6.48c-3.574,0-6.481,2.907-6.481,6.48c0,2.555,1.512,4.878,3.851,5.919\tc0.141,0.063,0.226,0.209,0.212,0.363c-0.014,0.153-0.125,0.281-0.275,0.316c-5.27,1.248-8.949,5.905-8.949,11.324V31H3.64v-5.237\tc0-5.444,3.497-10.164,8.611-11.779c-2.123-1.296-3.452-3.618-3.452-6.144c0-3.97,3.23-7.2,7.201-7.2s7.201,3.23,7.201,7.2\tc0,2.525-1.329,4.847-3.452,6.144C24.863,15.599,28.36,20.318,28.36,25.763z M18.745,21.745L16,24.49l-2.746-2.745l-0.509,0.51\tL15.49,25l-2.745,2.745l0.509,0.51L16,25.51l2.745,2.745l0.51-0.51L16.51,25l2.745-2.745L18.745,21.745z"
  })), children);
});

export { UnauthorizedUserAccess as default };
