// @carbon/pictograms-react/lib/process--automation/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const ProcessAutomation = /*#__PURE__*/React__default["default"].forwardRef(function ProcessAutomation(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M28,31.36c-0.625,0-1.153-0.425-1.312-1\th-9.377c-0.158,0.575-0.686,1-1.312,1s-1.153-0.425-1.312-1H4.5c-2.128,0-3.86-1.732-3.86-3.86s1.731-3.86,3.86-3.86h10.188\tc0.158-0.575,0.686-1,1.312-1c0.626,0,1.153,0.425,1.312,1H27.5c1.731,0,3.14-1.408,3.14-3.14s-1.408-3.14-3.14-3.14H17.312\tc-0.158,0.576-0.686,1-1.312,1s-1.153-0.425-1.312-1H4.5c-2.128,0-3.86-1.731-3.86-3.86S2.372,8.64,4.5,8.64h10.188\tc0.158-0.576,0.686-1,1.312-1c0.626,0,1.153,0.424,1.312,1H27.5c1.731,0,3.14-1.409,3.14-3.14s-1.408-3.14-3.14-3.14H17.312\tc-0.158,0.576-0.686,1-1.312,1s-1.153-0.424-1.312-1H1V1.64h13.688c0.158-0.576,0.686-1,1.312-1c0.626,0,1.153,0.424,1.312,1H27.5\tc2.128,0,3.86,1.731,3.86,3.86s-1.732,3.86-3.86,3.86H17.312c-0.158,0.576-0.686,1-1.312,1s-1.153-0.424-1.312-1H4.5\tc-1.731,0-3.14,1.409-3.14,3.14s1.409,3.14,3.14,3.14h10.188c0.158-0.576,0.686-1,1.312-1c0.626,0,1.153,0.424,1.312,1H27.5\tc2.128,0,3.86,1.732,3.86,3.86s-1.732,3.86-3.86,3.86H17.312c-0.158,0.575-0.686,1-1.312,1s-1.153-0.425-1.312-1H4.5\tc-1.731,0-3.14,1.408-3.14,3.14s1.409,3.14,3.14,3.14h10.188c0.158-0.575,0.686-1,1.312-1c0.626,0,1.153,0.425,1.312,1h9.377\tc0.158-0.575,0.687-1,1.312-1c0.75,0,1.36,0.61,1.36,1.36S28.75,31.36,28,31.36z M28,29.36c-0.353,0-0.64,0.287-0.64,0.64\ts0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S28.353,29.36,28,29.36z M16,29.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64\ts0.64-0.287,0.64-0.64S16.353,29.36,16,29.36z M16,22.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64\tS16.353,22.36,16,22.36z M16,15.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S16.353,15.36,16,15.36\tz M16,8.36c-0.353,0-0.64,0.287-0.64,0.64S15.647,9.64,16,9.64c0.353,0,0.64-0.287,0.64-0.64S16.353,8.36,16,8.36z M16,1.36\tc-0.353,0-0.64,0.287-0.64,0.64S15.647,2.64,16,2.64c0.353,0,0.64-0.287,0.64-0.64S16.353,1.36,16,1.36z"
  })), children);
});

module.exports = ProcessAutomation;
