// @carbon/pictograms-react/lib/trash/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path, _path2;
const Trash = /*#__PURE__*/React__default["default"].forwardRef(function Trash(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M26,31.36H6c-0.199,0-0.36-0.161-0.36-0.36V5.36H2V4.64h8.64V1c0-0.199,0.161-0.36,0.36-0.36h10 c0.199,0,0.36,0.161,0.36,0.36v3.64H30v0.72h-3.64V31C26.36,31.199,26.199,31.36,26,31.36z M6.36,30.64h19.28V5.36H6.36V30.64z M11.36,4.64h9.28V1.36h-9.28V4.64z"
  })), _path2 || (_path2 = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M21,27.36c-0.199,0-0.36-0.161-0.36-0.36V9c0-0.199,0.161-0.36,0.36-0.36S21.36,8.801,21.36,9v18 C21.36,27.199,21.199,27.36,21,27.36z M16,27.36c-0.199,0-0.36-0.161-0.36-0.36V9c0-0.199,0.161-0.36,0.36-0.36 S16.36,8.801,16.36,9v18C16.36,27.199,16.199,27.36,16,27.36z M11,27.36c-0.199,0-0.36-0.161-0.36-0.36V9 c0-0.199,0.161-0.36,0.36-0.36S11.36,8.801,11.36,9v18C11.36,27.199,11.199,27.36,11,27.36z"
  })), children);
});

module.exports = Trash;
