// @carbon/pictograms-react/lib/taipei--bubble-tea/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const TaipeiBubbleTea = /*#__PURE__*/React__default["default"].forwardRef(function TaipeiBubbleTea(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M22,30.36H10c-0.186,0-0.341-0.142-0.358-0.326L7.673,9.36H6V8.64h12.334l3.335-7.782\tl0.662,0.283L19.117,8.64H26v0.72h-1.673l-1.969,20.674C22.341,30.219,22.186,30.36,22,30.36z M10.327,29.64h11.346l1.359-14.28\tH8.967L10.327,29.64z M16.546,14.64h6.556l0.503-5.28h-4.796L16.546,14.64z M8.899,14.64h6.864l2.263-5.28h-9.63L8.899,14.64z M19,28.36c-0.75,0-1.36-0.61-1.36-1.36s0.61-1.36,1.36-1.36s1.36,0.61,1.36,1.36S19.75,28.36,19,28.36z M19,26.36\tc-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S19.353,26.36,19,26.36z M13,28.36\tc-0.75,0-1.36-0.61-1.36-1.36s0.61-1.36,1.36-1.36s1.36,0.61,1.36,1.36S13.75,28.36,13,28.36z M13,26.36\tc-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S13.353,26.36,13,26.36z M16,25.36\tc-0.75,0-1.36-0.61-1.36-1.36s0.61-1.36,1.36-1.36s1.36,0.61,1.36,1.36S16.75,25.36,16,25.36z M16,23.36\tc-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S16.353,23.36,16,23.36z M13,22.36\tc-0.75,0-1.36-0.61-1.36-1.36s0.61-1.36,1.36-1.36s1.36,0.61,1.36,1.36S13.75,22.36,13,22.36z M13,20.36\tc-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S13.353,20.36,13,20.36z M19,20.36\tc-0.75,0-1.36-0.61-1.36-1.36s0.61-1.36,1.36-1.36s1.36,0.61,1.36,1.36S19.75,20.36,19,20.36z M19,18.36\tc-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S19.353,18.36,19,18.36z"
  })), children);
});

module.exports = TaipeiBubbleTea;
