// @carbon/pictograms-react/lib/government--01/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Government_01 = /*#__PURE__*/React__default["default"].forwardRef(function Government_01(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31,31.36H1c-0.199,0-0.36-0.161-0.36-0.36v-3\tc0-0.199,0.161-0.36,0.36-0.36h1.64V25c0-0.199,0.161-0.36,0.36-0.36h1.64V22c0-0.199,0.161-0.36,0.36-0.36h0.64v-5.28H5\tc-0.199,0-0.36-0.161-0.36-0.36v-2.64H1c-0.153,0-0.289-0.096-0.34-0.241c-0.05-0.144-0.004-0.305,0.115-0.4l15-12\tc0.132-0.104,0.318-0.104,0.45,0l15,12c0.12,0.096,0.166,0.256,0.115,0.4c-0.051,0.145-0.187,0.241-0.34,0.241h-3.64V16\tc0,0.199-0.161,0.36-0.36,0.36h-0.64v5.28H27c0.199,0,0.36,0.161,0.36,0.36v2.64H29c0.199,0,0.36,0.161,0.36,0.36v2.64H31\tc0.199,0,0.36,0.161,0.36,0.36v3C31.36,31.199,31.199,31.36,31,31.36z M1.36,30.64h29.28v-2.28H1.36V30.64z M3.36,27.64h25.28v-2.28\tH3.36V27.64z M21.36,24.64h5.279v-2.28H21.36V24.64z M19.36,24.64h1.279V22c0-0.199,0.161-0.36,0.36-0.36h0.64v-5.28H21\tc-0.199,0-0.36-0.161-0.36-0.36v-2.64h-1.28V16c0,0.199-0.161,0.36-0.36,0.36h-0.64v5.28H19c0.199,0,0.36,0.161,0.36,0.36V24.64z M13.36,24.64h5.28v-2.28h-5.28C13.36,22.36,13.36,24.64,13.36,24.64z M11.36,24.64h1.28V22c0-0.199,0.161-0.36,0.36-0.36h0.64\tv-5.28H13c-0.199,0-0.36-0.161-0.36-0.36v-2.64h-1.28V16c0,0.199-0.161,0.36-0.36,0.36h-0.64v5.28H11c0.199,0,0.36,0.161,0.36,0.36\tV24.64z M5.36,24.64h5.28v-2.28H5.36V24.64z M22.36,21.64h3.279v-5.28H22.36V21.64z M14.36,21.64h3.28v-5.28h-3.28V21.64z M6.36,21.64h3.28v-5.28H6.36V21.64z M21.36,15.64h5.279v-2.28H21.36V15.64z M13.36,15.64h5.28v-2.28h-5.28\tC13.36,13.36,13.36,15.64,13.36,15.64z M5.36,15.64h5.28v-2.28H5.36V15.64z M2.026,12.64h27.947L16,1.461L2.026,12.64z M16,10.86\tc-1.577,0-2.86-1.283-2.86-2.86S14.423,5.14,16,5.14S18.86,6.423,18.86,8S17.577,10.86,16,10.86z M16,5.86\tc-1.18,0-2.14,0.96-2.14,2.14s0.96,2.14,2.14,2.14S18.14,9.18,18.14,8S17.18,5.86,16,5.86z"
  })), children);
});

module.exports = Government_01;
