// @carbon/pictograms-react/es/chart--candlestick/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const ChartCandlestick = /*#__PURE__*/React.forwardRef(function ChartCandlestick(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M31,31.36H1c-0.199,0-0.36-0.161-0.36-0.36V1h0.72v29.64H31V31.36z M25.36,28h-0.72v-2.64H23\tc-0.199,0-0.36-0.161-0.36-0.36V13c0-0.199,0.161-0.36,0.36-0.36h1.64V10h0.721v2.64H27c0.199,0,0.36,0.161,0.36,0.36v12\tc0,0.199-0.161,0.36-0.36,0.36h-1.64V28z M23.36,24.64h3.279V13.36H23.36V24.64z M7.36,25H6.64v-5.64H5\tc-0.199,0-0.36-0.161-0.36-0.36v-8c0-0.199,0.161-0.36,0.36-0.36h1.64V5h0.72v5.64H9c0.199,0,0.36,0.161,0.36,0.36v8\tc0,0.199-0.161,0.36-0.36,0.36H7.36V25z M5.36,18.64h3.28v-7.28H5.36V18.64z M16.36,21h-0.72v-5.64H14\tc-0.199,0-0.36-0.161-0.36-0.36V9c0-0.199,0.161-0.36,0.36-0.36h1.64V3h0.72v5.64H18c0.199,0,0.36,0.161,0.36,0.36v6\tc0,0.199-0.161,0.36-0.36,0.36h-1.64V21z M14.36,14.64h3.28V9.36h-3.28V14.64z"
  })), children);
});

export { ChartCandlestick as default };
