// @carbon/pictograms-react/lib/golf--hole/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const GolfHole = /*#__PURE__*/React__default["default"].forwardRef(function GolfHole(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M11,30.36c-1.916,0-3.36-0.8-3.36-1.86\tc0-0.993,1.266-1.757,3-1.851V18.36H1v-0.72h9.64V2h0.72v0.64H21c0.199,0,0.36,0.161,0.36,0.36v7c0,0.199-0.161,0.36-0.36,0.36\th-9.64v16.29c1.734,0.094,3,0.857,3,1.851C14.36,29.561,12.916,30.36,11,30.36z M11.36,27.371v2.258\tc1.327-0.086,2.28-0.635,2.28-1.129S12.687,27.457,11.36,27.371z M10.64,27.371c-1.327,0.086-2.28,0.635-2.28,1.129\ts0.953,1.043,2.28,1.129V27.371z M11.36,9.64h9.28V3.36h-9.28V9.64z M23,24.36c-0.75,0-1.36-0.61-1.36-1.36s0.61-1.36,1.36-1.36\ts1.36,0.61,1.36,1.36S23.75,24.36,23,24.36z M23,22.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64\tS23.353,22.36,23,22.36z M31,18.36H13v-0.72h18V18.36z"
  })), children);
});

module.exports = GolfHole;
