// @carbon/pictograms-react/es/resilience/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Resilience = /*#__PURE__*/React.forwardRef(function Resilience(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M31,31.36H1v-0.72h30V31.36z M16,28.36\tc-5.037,0-10.681-0.753-15.099-2.015l0.197-0.691C5.456,26.897,11.026,27.64,16,27.64s10.544-0.742,14.901-1.985l0.197,0.691\tC26.681,27.607,21.036,28.36,16,28.36z M16,25.36c-6.223,0-11.769-1.484-15.216-4.072l0.433-0.576C4.542,23.208,9.93,24.64,16,24.64\ts11.459-1.432,14.784-3.928l0.432,0.576C27.769,23.876,22.223,25.36,16,25.36z M16,22.509l-6.254-6.254l0.509-0.509l5.386,5.385V1\th0.72v20.131l5.385-5.385l0.51,0.509L16,22.509z"
  })), children);
});

export { Resilience as default };
