// @carbon/pictograms-react/lib/racetrack/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Racetrack = /*#__PURE__*/React__default["default"].forwardRef(function Racetrack(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M21,26.36H11C5.288,26.36,0.64,21.713,0.64,16C0.64,10.288,5.288,5.64,11,5.64h10\tc5.713,0,10.36,4.647,10.36,10.36S26.713,26.36,21,26.36z M11,6.36c-5.315,0-9.64,4.325-9.64,9.64s4.325,9.64,9.64,9.64h10\tc5.315,0,9.64-4.324,9.64-9.64S26.315,6.36,21,6.36h-6.64v2.28H21c4.059,0,7.36,3.302,7.36,7.36c0,4.059-3.302,7.36-7.36,7.36H11\tc-4.058,0-7.36-3.302-7.36-7.36S6.942,8.64,11,8.64h0.64V6.36H11z M11,9.36c-3.661,0-6.64,2.979-6.64,6.64s2.979,6.64,6.64,6.64h10\tc3.661,0,6.64-2.979,6.64-6.64S24.661,9.36,21,9.36h-6.64v2.28H21c2.404,0,4.36,1.956,4.36,4.36s-1.956,4.36-4.36,4.36H11\tc-2.404,0-4.36-1.956-4.36-4.36s1.956-4.36,4.36-4.36h0.64V9.36H11z M11,12.36c-2.007,0-3.64,1.633-3.64,3.64s1.633,3.64,3.64,3.64\th10c2.007,0,3.64-1.633,3.64-3.64s-1.633-3.64-3.64-3.64C21,12.36,11,12.36,11,12.36z M12.36,11.64h1.28V9.36h-1.28V11.64z M12.36,8.64h1.28V6.36h-1.28V8.64z"
  })), children);
});

module.exports = Racetrack;
