// @carbon/pictograms-react/es/sharing-data/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const SharingData = /*#__PURE__*/React.forwardRef(function SharingData(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M2.746,31.255L0.491,29l2.255-2.255l0.509,0.51\tL1.869,28.64h26.819c0.158-0.575,0.687-1,1.312-1c0.75,0,1.36,0.61,1.36,1.36s-0.61,1.36-1.36,1.36c-0.625,0-1.153-0.425-1.312-1\tH1.869l1.385,1.385L2.746,31.255z M30,28.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64\tS30.353,28.36,30,28.36z M29.255,23.255l-0.51-0.51l1.386-1.385H3.312c-0.158,0.575-0.686,1-1.312,1c-0.75,0-1.36-0.61-1.36-1.36\tS1.25,19.64,2,19.64c0.625,0,1.153,0.425,1.312,1h26.819l-1.386-1.385l0.51-0.51L31.509,21L29.255,23.255z M2,20.36\tc-0.353,0-0.64,0.287-0.64,0.64S1.647,21.64,2,21.64S2.64,21.353,2.64,21S2.353,20.36,2,20.36z M2.746,13.254L0.491,11l2.255-2.254\tl0.509,0.509L1.869,10.64h26.819c0.158-0.576,0.687-1,1.312-1c0.75,0,1.36,0.61,1.36,1.36s-0.61,1.36-1.36,1.36\tc-0.625,0-1.153-0.424-1.312-1H1.869l1.385,1.386L2.746,13.254z M30,10.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64\ts0.64-0.287,0.64-0.64S30.353,10.36,30,10.36z M29.255,5.254l-0.51-0.509l1.386-1.386H3.312c-0.158,0.576-0.686,1-1.312,1\tC1.25,4.36,0.64,3.75,0.64,3S1.25,1.64,2,1.64c0.625,0,1.153,0.424,1.312,1h26.819l-1.386-1.386l0.51-0.509L31.509,3L29.255,5.254z M2,2.36C1.647,2.36,1.36,2.647,1.36,3S1.647,3.64,2,3.64S2.64,3.353,2.64,3S2.353,2.36,2,2.36z"
  })), children);
});

export { SharingData as default };
