// @carbon/pictograms-react/es/micro/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Micro = /*#__PURE__*/React.forwardRef(function Micro(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M30,30.36H2c-0.199,0-0.36-0.161-0.36-0.36V2\tc0-0.199,0.161-0.36,0.36-0.36h28c0.199,0,0.36,0.161,0.36,0.36v28C30.36,30.199,30.199,30.36,30,30.36z M22.36,29.64h7.279V2.36\tH2.36v7.28H22c0.199,0,0.36,0.161,0.36,0.36V29.64z M12.36,29.64h9.28V10.36H2.36v9.28H12c0.199,0,0.36,0.161,0.36,0.36V29.64z M2.36,29.64h9.28v-9.28H2.36V29.64z M8,26.36H6c-0.199,0-0.36-0.161-0.36-0.36v-2c0-0.199,0.161-0.36,0.36-0.36h2\tc0.199,0,0.36,0.161,0.36,0.36v2C8.36,26.199,8.199,26.36,8,26.36z M6.36,25.64h1.28v-1.28H6.36V25.64z"
  })), children);
});

export { Micro as default };
