// @carbon/pictograms-react/es/person--03/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Person_03 = /*#__PURE__*/React.forwardRef(function Person_03(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M18.36,30h-0.72v-9c0-0.151,0.096-0.287,0.238-0.339c1.136-0.41,1.762-1.285,1.762-2.462V13\tc0-2.007-1.633-3.64-3.64-3.64s-3.64,1.633-3.64,3.64v5.21c0,1.17,0.626,2.041,1.762,2.451c0.143,0.052,0.237,0.188,0.237,0.339v9\th-0.72v-8.756c-1.275-0.551-2-1.644-2-3.034V13c0-2.404,1.956-4.36,4.36-4.36s4.36,1.956,4.36,4.36v5.199\tc0,1.396-0.726,2.494-2,3.045V30H18.36z M16,7.36c-1.577,0-2.86-1.283-2.86-2.86S14.423,1.64,16,1.64s2.86,1.283,2.86,2.86\tS17.577,7.36,16,7.36z M16,2.36c-1.18,0-2.14,0.96-2.14,2.14S14.82,6.64,16,6.64s2.14-0.96,2.14-2.14S17.18,2.36,16,2.36z"
  })), children);
});

export { Person_03 as default };
