// @carbon/pictograms-react/lib/football/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Football = /*#__PURE__*/React__default["default"].forwardRef(function Football(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M7.316,31.36c-1.918,0-3.843-0.232-5.721-0.69\tc-0.13-0.031-0.232-0.134-0.264-0.265c-0.458-1.878-0.69-3.803-0.69-5.721C0.64,11.426,11.426,0.64,24.685,0.64\tc1.921,0,3.845,0.232,5.72,0.69c0.131,0.032,0.232,0.134,0.265,0.264c0.459,1.876,0.691,3.801,0.691,5.721\tC31.36,20.573,20.574,31.36,7.316,31.36z M2.395,30.115c1.622,0.349,3.274,0.524,4.921,0.524c12.861,0,23.324-10.463,23.324-23.324\tc0-1.648-0.177-3.3-0.525-4.92l-0.859,0.859l-0.51-0.509l0.86-0.86c-1.619-0.349-3.271-0.525-4.921-0.525\tC11.823,1.36,1.36,11.823,1.36,24.685c0,1.646,0.177,3.299,0.525,4.921l0.86-0.86l0.509,0.51L2.395,30.115z M4.254,28.255\tl-0.509-0.51l1-1l0.509,0.51L4.254,28.255z M6.254,26.255l-0.509-0.51l1-1l0.509,0.51L6.254,26.255z M8.254,24.255l-0.509-0.51l1-1\tl0.509,0.51L8.254,24.255z M12.746,23.255l-4-4l0.509-0.51L11,20.491L12.491,19l-1.745-1.745l0.509-0.509L13,18.491L14.491,17\tl-1.745-1.746l0.509-0.509L15,16.491L16.491,15l-1.745-1.746l0.509-0.509L17,14.491L18.491,13l-1.746-1.746l0.509-0.509L19,12.491\tL20.491,11l-1.746-1.746l0.51-0.509l4,4l-0.51,0.509L21,11.509L19.509,13l1.746,1.746l-0.51,0.509L19,13.509L17.509,15l1.746,1.746\tl-0.51,0.509L17,15.509L15.509,17l1.746,1.745l-0.509,0.51L15,17.509L13.509,19l1.745,1.745l-0.509,0.51L13,19.509L11.509,21\tl1.745,1.745L12.746,23.255z M23.255,9.254l-0.51-0.509l1-1l0.51,0.509L23.255,9.254z M25.255,7.254l-0.51-0.509l1-1l0.51,0.509\tL25.255,7.254z M27.255,5.254l-0.51-0.509l1-1l0.51,0.509L27.255,5.254z"
  })), children);
});

module.exports = Football;
