// @carbon/pictograms-react/es/new-revenue-streams/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const NewRevenueStreams = /*#__PURE__*/React.forwardRef(function NewRevenueStreams(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M20,0.64h4.36V5h-0.72V1.87L1.254,24.255\tl-0.509-0.51L23.131,1.36H20V0.64z M5,6.36h2c0.353,0,0.64,0.287,0.64,0.64S7.353,7.64,7,7.64H4v0.72h1.64V9.5h0.72V8.36H7\tc0.75,0,1.36-0.61,1.36-1.36S7.75,5.64,7,5.64H5C4.647,5.64,4.36,5.353,4.36,5S4.647,4.36,5,4.36h3V3.64H6.36V2.5H5.64v1.14H5\tC4.25,3.64,3.64,4.25,3.64,5S4.25,6.36,5,6.36z M22.36,14v17c0,0.199-0.161,0.36-0.36,0.36h-5c-0.199,0-0.36-0.161-0.36-0.36V14\tc0-0.199,0.161-0.36,0.36-0.36h5C22.199,13.64,22.36,13.801,22.36,14z M21.64,22.36h-4.28v8.279h4.279L21.64,22.36L21.64,22.36z M21.64,17.36h-4.28v4.279h4.279L21.64,17.36L21.64,17.36z M21.64,14.36h-4.28v2.28h4.279L21.64,14.36L21.64,14.36z M0.64,6\tc0-2.956,2.404-5.36,5.36-5.36S11.36,3.044,11.36,6S8.956,11.36,6,11.36S0.64,8.956,0.64,6z M1.36,6c0,2.559,2.082,4.64,4.64,4.64\tS10.64,8.559,10.64,6S8.559,1.36,6,1.36S1.36,3.441,1.36,6z M31.36,6v25c0,0.199-0.161,0.36-0.36,0.36h-5\tc-0.199,0-0.36-0.161-0.36-0.36V6c0-0.199,0.161-0.36,0.36-0.36h5C31.199,5.64,31.36,5.801,31.36,6z M30.64,17.36h-4.28v13.28h4.279\tL30.64,17.36L30.64,17.36z M30.64,10.36h-4.28v6.28h4.279L30.64,10.36L30.64,10.36z M30.64,6.36h-4.28v3.28h4.279L30.64,6.36\tL30.64,6.36z M13.36,23v8c0,0.199-0.161,0.36-0.36,0.36H8c-0.199,0-0.36-0.161-0.36-0.36v-8c0-0.199,0.161-0.36,0.36-0.36h5\tC13.199,22.64,13.36,22.801,13.36,23z M12.64,25.36H8.36v5.279h4.28C12.64,30.639,12.64,25.36,12.64,25.36z M12.64,23.36H8.36v1.279\th4.28C12.64,24.639,12.64,23.36,12.64,23.36z"
  })), children);
});

export { NewRevenueStreams as default };
