// @carbon/pictograms-react/lib/workday/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Workday = /*#__PURE__*/React__default["default"].forwardRef(function Workday(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M7,12.36c-0.013,0-0.026,0-0.04-0.002c-0.198-0.022-0.34-0.2-0.318-0.397C7.167,7.217,11.189,3.64,16,3.64\ts8.834,3.577,9.357,8.32c0.022,0.198-0.12,0.375-0.318,0.397c-0.19,0.023-0.376-0.121-0.396-0.318C24.158,7.662,20.443,4.36,16,4.36\ts-8.158,3.302-8.642,7.68C7.337,12.224,7.182,12.36,7,12.36z M24.623,16l-3.662,11.903L16.348,16h-0.695L11.04,27.903L7.377,16\tH6.624l4,13h0.763L16,17.095L20.614,29h0.763l4-13H24.623z"
  })), children);
});

module.exports = Workday;
