// @carbon/pictograms-react/lib/planning--analytics/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const PlanningAnalytics = /*#__PURE__*/React__default["default"].forwardRef(function PlanningAnalytics(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M4,18.36c-1.301,0-2.36-1.059-2.36-2.36S2.699,13.64,4,13.64S6.36,14.699,6.36,16\tC6.36,17.302,5.301,18.36,4,18.36z M4,14.36c-0.904,0-1.64,0.736-1.64,1.64c0,0.904,0.736,1.64,1.64,1.64S5.64,16.904,5.64,16\tS4.904,14.36,4,14.36z M7,12.36c-1.301,0-2.36-1.059-2.36-2.36S5.699,7.64,7,7.64S9.36,8.699,9.36,10S8.301,12.36,7,12.36z M7,8.36\tc-0.904,0-1.64,0.736-1.64,1.64S6.096,11.64,7,11.64S8.64,10.904,8.64,10S7.904,8.36,7,8.36z M7,24.36\tc-1.301,0-2.36-1.059-2.36-2.36S5.699,19.64,7,19.64S9.36,20.698,9.36,22S8.301,24.36,7,24.36z M7,20.36\tc-0.904,0-1.64,0.735-1.64,1.64S6.096,23.64,7,23.64S8.64,22.904,8.64,22S7.904,20.36,7,20.36z M20.36,10\tc0-0.199-0.161-0.36-0.36-0.36h-8c-0.199,0-0.36,0.161-0.36,0.36s0.161,0.36,0.36,0.36h8C20.199,10.36,20.36,10.199,20.36,10z M20.36,22c0-0.199-0.161-0.36-0.36-0.36h-8c-0.199,0-0.36,0.161-0.36,0.36s0.161,0.36,0.36,0.36h8\tC20.199,22.36,20.36,22.199,20.36,22z M17.36,16c0-0.199-0.161-0.36-0.36-0.36H9c-0.199,0-0.36,0.161-0.36,0.36S8.801,16.36,9,16.36\th8C17.199,16.36,17.36,16.199,17.36,16z M24.237,27.648l5.958-11c0.22-0.406,0.22-0.89,0-1.295l-5.958-11\tc-0.238-0.439-0.696-0.712-1.195-0.712H11v0.72h12.042c0.234,0,0.45,0.128,0.562,0.335l5.958,11c0.104,0.19,0.104,0.418,0,0.609\tl-5.958,11c-0.112,0.207-0.328,0.335-0.562,0.335H11v0.721h12.042C23.541,28.36,23.999,28.087,24.237,27.648z"
  })), children);
});

module.exports = PlanningAnalytics;
