// @carbon/pictograms-react/lib/dialogue/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path, _path2;
const Dialogue = /*#__PURE__*/React__default["default"].forwardRef(function Dialogue(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16.322,31.161l-0.644-0.322l4-8C19.74,22.717,19.864,22.64,20,22.64h7c2.007,0,3.64-1.633,3.64-3.64V5 c0-2.007-1.633-3.64-3.64-3.64H5C2.993,1.36,1.36,2.993,1.36,5v14c0,2.007,1.633,3.64,3.64,3.64h11v0.721H5 c-2.404,0-4.36-1.956-4.36-4.36V5c0-2.404,1.956-4.36,4.36-4.36h22c2.404,0,4.36,1.956,4.36,4.36v14c0,2.404-1.956,4.36-4.36,4.36 h-6.777L16.322,31.161z"
  })), _path2 || (_path2 = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M17.5,12.5c0,0.828-0.672,1.5-1.5,1.5c-0.828,0-1.5-0.672-1.5-1.5S15.172,11,16,11C16.828,11,17.5,11.672,17.5,12.5z M23.5,11c-0.828,0-1.5,0.672-1.5,1.5s0.672,1.5,1.5,1.5s1.5-0.672,1.5-1.5S24.328,11,23.5,11z M8.5,11C7.672,11,7,11.672,7,12.5 S7.672,14,8.5,14s1.5-0.672,1.5-1.5S9.328,11,8.5,11z"
  })), children);
});

module.exports = Dialogue;
