// @carbon/pictograms-react/es/high--performance/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const HighPerformance = /*#__PURE__*/React.forwardRef(function HighPerformance(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M31,0.64H1C0.801,0.64,0.64,0.801,0.64,1v30\tc0,0.199,0.161,0.36,0.36,0.36h30c0.199,0,0.36-0.161,0.36-0.36V1C31.36,0.801,31.199,0.64,31,0.64z M30.64,30.64H6.891\tc10.334-2.576,18.058-11.775,18.434-22.806l1.421,1.421l0.51-0.509L25,6.491l-2.255,2.255l0.51,0.509l1.362-1.362\tC24.15,20.396,13.932,30.449,1.36,30.638V1.36h29.28C30.64,1.36,30.64,30.64,30.64,30.64z"
  })), children);
});

export { HighPerformance as default };
