// @carbon/pictograms-react/es/fragile/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Fragile = /*#__PURE__*/React.forwardRef(function Fragile(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M21,31.36H11v-0.72h4.64V17.35c-3.34-0.187-6-2.963-6-6.35V1c0-0.199,0.161-0.36,0.36-0.36h10\tc0.146,0,0.277,0.087,0.333,0.222c0.056,0.134,0.024,0.289-0.078,0.392L17.509,4l1.746,1.746c0.141,0.141,0.141,0.368,0,0.509l-3,3\tl-0.509-0.509L18.491,6l-1.746-1.746c-0.141-0.141-0.141-0.368,0-0.509l2.385-2.386h-8.77V11c0,3.11,2.53,5.64,5.64,5.64\ts5.64-2.53,5.64-5.64V3h0.721v8c0,3.386-2.66,6.163-6,6.35v13.29H21V31.36z"
  })), children);
});

export { Fragile as default };
