// @carbon/pictograms-react/lib/resource-health/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const ResourceHealth = /*#__PURE__*/React__default["default"].forwardRef(function ResourceHealth(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M22,31.36h-2c-0.199,0-0.36-0.161-0.36-0.36\tv-0.64h-7.28V31c0,0.199-0.161,0.36-0.36,0.36h-2c-0.199,0-0.36-0.161-0.36-0.36v-2c0-0.199,0.161-0.36,0.36-0.36h0.64v-7.28H3.36\tV22c0,0.199-0.161,0.36-0.36,0.36H1c-0.199,0-0.36-0.161-0.36-0.36v-2c0-0.199,0.161-0.36,0.36-0.36h0.64v-7.28H1\tc-0.199,0-0.36-0.161-0.36-0.36v-2c0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v0.64h7.28V3.36H10\tC9.801,3.36,9.64,3.199,9.64,3V1c0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v0.64h7.28V1\tc0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2c0,0.199-0.161,0.36-0.36,0.36h-0.64v7.28h7.279V10\tc0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2c0,0.199-0.161,0.36-0.36,0.36h-0.64v7.28H31\tc0.199,0,0.36,0.161,0.36,0.36v2c0,0.199-0.161,0.36-0.36,0.36h-2c-0.199,0-0.36-0.161-0.36-0.36v-0.64h-7.28v7.279H22\tc0.199,0,0.36,0.161,0.36,0.36v2C22.36,31.199,22.199,31.36,22,31.36z M20.36,30.64h1.279v-1.28H20.36V30.64z M10.36,30.64h1.28\tv-1.28h-1.28V30.64z M12.36,29.64h7.28V29c0-0.199,0.161-0.36,0.36-0.36h0.64v-7.28h-9.28v7.279H12c0.199,0,0.36,0.161,0.36,0.36\tV29.64z M29.36,21.64h1.279v-1.28H29.36V21.64z M1.36,21.64h1.28v-1.28H1.36V21.64z M21.36,20.64h7.279V20\tc0-0.199,0.161-0.36,0.36-0.36h0.64v-7.28H29c-0.199,0-0.36-0.161-0.36-0.36v-0.64h-7.28V20.64z M11.36,20.64h9.28v-9.28h-9.28\tV20.64z M3.36,20.64h7.28v-9.28H3.36V12c0,0.199-0.161,0.36-0.36,0.36H2.36v7.28H3c0.199,0,0.36,0.161,0.36,0.36\tC3.36,20,3.36,20.64,3.36,20.64z M29.36,11.64h1.279v-1.28H29.36V11.64z M1.36,11.64h1.28v-1.28H1.36V11.64z M11.36,10.64h9.28V3.36\tH20c-0.199,0-0.36-0.161-0.36-0.36V2.36h-7.28V3c0,0.199-0.161,0.36-0.36,0.36h-0.64C11.36,3.36,11.36,10.64,11.36,10.64z M20.36,2.64h1.279V1.36H20.36V2.64z M10.36,2.64h1.28V1.36h-1.28V2.64z"
  })), children);
});

module.exports = ResourceHealth;
