// @carbon/pictograms-react/lib/secure--search/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const SecureSearch = /*#__PURE__*/React__default["default"].forwardRef(function SecureSearch(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M11,21.509l-4.254-4.254l0.509-0.509L11,20.491l7.745-7.746l0.51,0.509L11,21.509z M23.623,22.313\tl7.202,4.001l0.35-0.629l-7.201-4.001c0.886-1.702,1.387-3.637,1.387-5.685c0-6.815-5.545-12.36-12.36-12.36S0.64,9.185,0.64,16\tS6.185,28.36,13,28.36C17.511,28.36,21.464,25.932,23.623,22.313z M24.64,16c0,6.418-5.222,11.64-11.64,11.64S1.36,22.418,1.36,16\tS6.582,4.36,13,4.36S24.64,9.582,24.64,16z"
  })), children);
});

module.exports = SecureSearch;
