// @carbon/pictograms-react/lib/forecasting/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Forecasting = /*#__PURE__*/React__default["default"].forwardRef(function Forecasting(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,0.64C7.53,0.64,0.64,7.53,0.64,16\tS7.53,31.36,16,31.36S31.36,24.47,31.36,16S24.47,0.64,16,0.64z M16,30.64c-5.443,0-10.202-2.985-12.725-7.405L11,15.509\tl5.746,5.746c0.141,0.141,0.368,0.141,0.509,0l9.385-9.385V14h0.721v-3.36H24v0.72h2.131L17,20.491l-5.746-5.746\tc-0.141-0.141-0.368-0.141-0.509,0l-7.826,7.826C1.922,20.595,1.36,18.361,1.36,16C1.36,7.927,7.927,1.36,16,1.36\tc8.072,0,14.64,6.567,14.64,14.64C30.64,24.072,24.072,30.64,16,30.64z"
  })), children);
});

module.exports = Forecasting;
