// @carbon/pictograms-react/lib/ibm--z-and-linuxone-multi-frame/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
let didWarnAboutDeprecation = false;
const IbmZAndLinuxoneMultiFrame = /*#__PURE__*/React__default["default"].forwardRef(function IbmZAndLinuxoneMultiFrame(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  if (process.env.NODE_ENV !== "production") {
    if (!didWarnAboutDeprecation) {
      didWarnAboutDeprecation = true;
      console.warn(`The IbmZAndLinuxoneMultiFrame component has been deprecated and will be ` + `removed in the next major version of @carbon/icons-react.`);
    }
  }
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31,30.36H1c-0.199,0-0.36-0.161-0.36-0.36V2c0-0.199,0.161-0.36,0.36-0.36h30\tc0.199,0,0.36,0.161,0.36,0.36v28C31.36,30.199,31.199,30.36,31,30.36z M29.36,29.64h1.279V2.36H29.36V29.64z M22.869,29.64h5.771\tV24.3l-2.331,3.886c-0.04,0.067-0.102,0.119-0.175,0.148L22.869,29.64z M19.36,29.64h1.279V2.36H19.36V29.64z M12.869,29.64h5.771\tV24.3l-2.331,3.886c-0.041,0.067-0.102,0.119-0.175,0.148L12.869,29.64z M9.36,29.64h1.28V2.36H9.36V29.64z M2.869,29.64H8.64V24.3\tl-2.332,3.886c-0.041,0.067-0.102,0.119-0.175,0.148L2.869,29.64z M21.36,23.36v6.108l4.394-1.758l2.61-4.351L21.36,23.36\tL21.36,23.36z M11.36,23.36v6.108l4.394-1.758l2.61-4.351L11.36,23.36L11.36,23.36z M1.36,23.36v6.108l4.394-1.758l2.61-4.351\tL1.36,23.36L1.36,23.36z M22.3,22.64h6.34v-5.56l-2.352,3.136c-0.028,0.037-0.062,0.069-0.103,0.093L22.3,22.64z M12.3,22.64h6.34\tv-5.56l-2.352,3.136c-0.028,0.037-0.063,0.069-0.103,0.093L12.3,22.64z M2.3,22.64h6.34v-5.56l-2.352,3.136\tc-0.028,0.037-0.063,0.069-0.103,0.093L2.3,22.64z M21.36,16.36v6.004l4.394-2.637l2.526-3.368L21.36,16.36L21.36,16.36z M11.36,16.36v6.004l4.395-2.637l2.526-3.368L11.36,16.36L11.36,16.36z M1.36,16.36v6.004l4.395-2.637L8.28,16.36H1.36z M21.72,15.64h6.92V9.636l-4.394,2.637L21.72,15.64z M11.72,15.64h6.919V9.636l-4.394,2.637L11.72,15.64z M1.72,15.64h6.92V9.636\tl-4.395,2.637L1.72,15.64z M21.36,9.36v5.56l2.352-3.136c0.028-0.038,0.062-0.069,0.103-0.093L27.7,9.36H21.36z M11.36,9.36v5.56\tl2.352-3.136c0.028-0.038,0.063-0.069,0.103-0.093L17.7,9.36H11.36z M1.36,9.36v5.56l2.352-3.136\tc0.028-0.038,0.063-0.069,0.103-0.093L7.7,9.36H1.36z M21.636,8.64h7.004V2.532l-4.394,1.757L21.636,8.64z M11.636,8.64h7.004V2.532\tl-4.394,1.757L11.636,8.64z M1.636,8.64H8.64V2.532L4.246,4.289L1.636,8.64z M21.36,2.36V7.7l2.331-3.885\tc0.04-0.067,0.102-0.12,0.175-0.149l3.265-1.306H21.36z M11.36,2.36V7.7l2.332-3.885c0.041-0.067,0.102-0.12,0.175-0.149\tl3.265-1.306H11.36z M1.36,2.36V7.7l2.332-3.885c0.041-0.067,0.102-0.12,0.175-0.149L7.131,2.36H1.36z"
  })), children);
});

module.exports = IbmZAndLinuxoneMultiFrame;
