// @carbon/pictograms-react/es/reliability/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Reliability = /*#__PURE__*/React.forwardRef(function Reliability(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M16,30.36C8.082,30.36,1.64,23.918,1.64,16\tS8.082,1.64,16,1.64S30.36,8.082,30.36,16S23.918,30.36,16,30.36z M16,2.36C8.479,2.36,2.36,8.479,2.36,16S8.479,29.64,16,29.64\tS29.64,23.522,29.64,16C29.64,8.479,23.521,2.36,16,2.36z M14.014,22.522l-4.268-4.268l0.509-0.51l3.732,3.732l8.746-9.718\tl0.535,0.481L14.014,22.522z"
  })), children);
});

export { Reliability as default };
