// @carbon/pictograms-react/lib/ibm--z/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
let didWarnAboutDeprecation = false;
const IbmZ = /*#__PURE__*/React__default["default"].forwardRef(function IbmZ(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  if (process.env.NODE_ENV !== "production") {
    if (!didWarnAboutDeprecation) {
      didWarnAboutDeprecation = true;
      console.warn(`The IbmZ component has been deprecated and will be ` + `removed in the next major version of @carbon/icons-react.`);
    }
  }
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M28,31.36H4c-0.199,0-0.36-0.161-0.36-0.36v-6c0-0.09,0.034-0.177,0.095-0.243L19.682,7.36H4\tC3.801,7.36,3.64,7.199,3.64,7V1c0-0.199,0.161-0.36,0.36-0.36h24c0.199,0,0.36,0.161,0.36,0.36v6c0,0.09-0.034,0.177-0.095,0.243\tL12.318,24.64H28c0.199,0,0.36,0.161,0.36,0.36v6C28.36,31.199,28.199,31.36,28,31.36z M9.083,30.64h16.875L11.68,25.449\tL9.083,30.64z M4.36,30.64h3.42l-3.42-4.56V30.64z M13.543,25.36l14.096,5.126V25.36H13.543z M4.467,25.023l3.971,5.295l10.59-21.18\tL4.467,25.023z M23.562,1.682l-10.59,21.18L27.533,6.977L23.562,1.682z M4.36,6.64h14.097L4.36,1.514V6.64z M6.043,1.36\tl14.278,5.192l2.597-5.192C22.918,1.36,6.043,1.36,6.043,1.36z M24.22,1.36l3.42,4.56V1.36H24.22z"
  })), children);
});

module.exports = IbmZ;
