// @carbon/pictograms-react/lib/person--02/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Person_02 = /*#__PURE__*/React__default["default"].forwardRef(function Person_02(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M18.36,30h-0.72v-8.313c0-0.156,0.101-0.295,0.25-0.343c2.542-0.821,4.25-3.17,4.25-5.844\tc0-3.386-2.754-6.14-6.14-6.14s-6.14,2.754-6.14,6.14c0,2.674,1.708,5.022,4.251,5.844c0.148,0.048,0.249,0.187,0.249,0.343V30\th-0.72v-8.057c-2.703-0.987-4.5-3.544-4.5-6.443c0-3.783,3.077-6.86,6.86-6.86c3.782,0,6.86,3.077,6.86,6.86\tc0,2.899-1.798,5.456-4.5,6.443V30z M16,7.36c-1.577,0-2.86-1.283-2.86-2.86S14.423,1.64,16,1.64s2.86,1.283,2.86,2.86\tS17.577,7.36,16,7.36z M16,2.36c-1.18,0-2.14,0.96-2.14,2.14S14.82,6.64,16,6.64s2.14-0.96,2.14-2.14S17.181,2.36,16,2.36z"
  })), children);
});

module.exports = Person_02;
