// @carbon/pictograms-react/lib/troubleshooting/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Troubleshooting = /*#__PURE__*/React__default["default"].forwardRef(function Troubleshooting(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,31.36C7.53,31.36,0.64,24.47,0.64,16\tS7.53,0.64,16,0.64S31.36,7.53,31.36,16S24.47,31.36,16,31.36z M16,1.36C7.927,1.36,1.36,7.927,1.36,16\tc0,8.072,6.567,14.64,14.64,14.64c8.072,0,14.64-6.567,14.64-14.64S24.072,1.36,16,1.36z M16,24.36c-3.42,0-6.46-2.051-7.747-5.226\tl0.667-0.27c1.176,2.9,3.955,4.774,7.08,4.774c4.213,0,7.64-3.427,7.64-7.64h0.721C24.36,20.609,20.609,24.36,16,24.36z M16.36,22\th-0.72v-4.64H15c-0.199,0-0.36-0.161-0.36-0.36v-0.64H10v-0.72h4.64V15c0-0.199,0.161-0.36,0.36-0.36h0.64V10h0.72v4.64H17\tc0.199,0,0.36,0.161,0.36,0.36v0.64H22v0.72h-4.64V17c0,0.199-0.161,0.36-0.36,0.36h-0.64V22z M15.36,16.64h1.28v-1.28h-1.28V16.64z M8.36,16H7.64c0-4.61,3.75-8.36,8.36-8.36c3.42,0,6.461,2.051,7.747,5.225l-0.668,0.271C21.904,10.234,19.125,8.36,16,8.36\tC11.787,8.36,8.36,11.787,8.36,16z"
  })), children);
});

module.exports = Troubleshooting;
