// @carbon/pictograms-react/es/cognos--analytics/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const CognosAnalytics = /*#__PURE__*/React.forwardRef(function CognosAnalytics(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M14,6.36c-1.301,0-2.36-1.059-2.36-2.36S12.699,1.64,14,1.64S16.36,2.699,16.36,4\tS15.301,6.36,14,6.36z M14,2.36c-0.904,0-1.64,0.736-1.64,1.64S13.096,5.64,14,5.64S15.64,4.904,15.64,4S14.904,2.36,14,2.36z M10,12.36c-1.301,0-2.36-1.059-2.36-2.36S8.699,7.64,10,7.64s2.36,1.059,2.36,2.36S11.301,12.36,10,12.36z M10,8.36\tc-0.904,0-1.64,0.736-1.64,1.64s0.736,1.64,1.64,1.64s1.64-0.736,1.64-1.64S10.904,8.36,10,8.36z M6,18.36\tc-1.301,0-2.36-1.059-2.36-2.36S4.699,13.64,6,13.64S8.36,14.699,8.36,16C8.36,17.302,7.301,18.36,6,18.36z M6,14.36\tc-0.904,0-1.64,0.736-1.64,1.64c0,0.904,0.736,1.64,1.64,1.64S7.64,16.904,7.64,16S6.904,14.36,6,14.36z M10,24.36\tc-1.301,0-2.36-1.059-2.36-2.36s1.059-2.36,2.36-2.36s2.36,1.059,2.36,2.36S11.301,24.36,10,24.36z M10,20.36\tc-0.904,0-1.64,0.735-1.64,1.64s0.736,1.64,1.64,1.64s1.64-0.735,1.64-1.64S10.904,20.36,10,20.36z M14,30.36\tc-1.301,0-2.36-1.059-2.36-2.36s1.059-2.36,2.36-2.36s2.36,1.059,2.36,2.36S15.301,30.36,14,30.36z M14,26.36\tc-0.904,0-1.64,0.735-1.64,1.64s0.736,1.64,1.64,1.64s1.64-0.735,1.64-1.64S14.904,26.36,14,26.36z M28.36,4\tc0-0.199-0.161-0.36-0.36-0.36H18c-0.199,0-0.36,0.161-0.36,0.36c0,0.199,0.161,0.36,0.36,0.36h10C28.199,4.36,28.36,4.199,28.36,4z M23.36,10c0-0.199-0.161-0.36-0.36-0.36h-9c-0.199,0-0.36,0.161-0.36,0.36s0.161,0.36,0.36,0.36h9\tC23.199,10.36,23.36,10.199,23.36,10z M19.36,16c0-0.199-0.161-0.36-0.36-0.36h-9c-0.199,0-0.36,0.161-0.36,0.36\ts0.161,0.36,0.36,0.36h9C19.199,16.36,19.36,16.199,19.36,16z M23.36,22c0-0.199-0.161-0.36-0.36-0.36h-9\tc-0.199,0-0.36,0.161-0.36,0.36s0.161,0.36,0.36,0.36h9C23.199,22.36,23.36,22.199,23.36,22z M28.36,28\tc0-0.199-0.161-0.36-0.36-0.36H18c-0.199,0-0.36,0.161-0.36,0.36s0.161,0.36,0.36,0.36h10C28.199,28.36,28.36,28.199,28.36,28z"
  })), children);
});

export { CognosAnalytics as default };
