// @carbon/pictograms-react/lib/curve--inverse/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const CurveInverse = /*#__PURE__*/React__default["default"].forwardRef(function CurveInverse(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M20.591,27.473L18.109,24h1.276l1.777,2.605\th0.041L23.047,24h1.208l-2.469,3.391L24.308,31h-1.275l-1.818-2.74h-0.041L19.208,31H18L20.591,27.473z M25.186,24.618V23.89h1.639\tv0.728H25.186z M27.674,26v-0.613h1.1v-2.843h-0.052l-0.871,1.032l-0.47-0.395L28.373,22h1.14v3.387h0.917V26H27.674z M16.36,31\th-0.72V16.36H1v-0.72h14.64V1h0.72v14.64H31v0.72H16.36V31z M14.36,31h-0.72c0-11.813-0.827-12.64-12.64-12.64v-0.72\tC13.236,17.64,14.36,18.764,14.36,31z M31,14.36c-12.236,0-13.36-1.124-13.36-13.36h0.721c0,11.813,0.826,12.64,12.64,12.64\tL31,14.36L31,14.36z"
  })), children);
});

module.exports = CurveInverse;
