// @carbon/pictograms-react/lib/power--on/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const PowerOn = /*#__PURE__*/React__default["default"].forwardRef(function PowerOn(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,31.36C7.53,31.36,0.64,24.47,0.64,16\tS7.53,0.64,16,0.64S31.36,7.53,31.36,16S24.47,31.36,16,31.36z M16,1.36C7.927,1.36,1.36,7.927,1.36,16\tc0,8.072,6.567,14.64,14.64,14.64c8.072,0,14.64-6.567,14.64-14.64S24.072,1.36,16,1.36z M16,24.36c-4.61,0-8.36-3.751-8.36-8.36\tc0-3.368,2.004-6.391,5.106-7.703l0.28,0.663C10.192,10.159,8.36,12.922,8.36,16c0,4.213,3.427,7.64,7.64,7.64s7.64-3.427,7.64-7.64\tc0-3.078-1.831-5.841-4.666-7.04l0.281-0.663C22.356,9.609,24.36,12.633,24.36,16C24.36,20.609,20.609,24.36,16,24.36z M16.36,16\th-0.72V5h0.72V16z"
  })), children);
});

module.exports = PowerOn;
