// @carbon/pictograms-react/lib/quantum-computing/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const QuantumComputing = /*#__PURE__*/React__default["default"].forwardRef(function QuantumComputing(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M21,29.36h-2.8\tc-2.69,0-4.112-1.335-4.112-3.86v-1.195C9.667,23.489,7.14,19.753,7.14,14c0-6.487,3.312-10.36,8.86-10.36\tc5.631,0,8.86,3.776,8.86,10.36c0,5.616-2.364,9.348-6.5,10.292v1.348H21c0.199,0,0.36,0.161,0.36,0.36v3\tC21.36,29.199,21.199,29.36,21,29.36z M16,4.36c-5.173,0-8.14,3.514-8.14,9.64c0,5.49,2.421,9.006,6.642,9.645\tc0.176,0.026,0.306,0.178,0.306,0.355v1.5c0,2.143,1.078,3.14,3.393,3.14h2.439v-2.28H18c-0.199,0-0.36-0.161-0.36-0.36v-2\tc0-0.173,0.123-0.32,0.292-0.354c4.004-0.772,6.208-4.198,6.208-9.646C24.14,7.874,21.173,4.36,16,4.36z M16,20.86\tc-2.952,0-4.86-2.301-4.86-5.86v-2c0-3.56,1.908-5.86,4.86-5.86s4.86,2.3,4.86,5.86v2C20.86,18.56,18.952,20.86,16,20.86z M16,7.86\tc-2.554,0-4.14,1.97-4.14,5.14v2c0,3.171,1.586,5.14,4.14,5.14s4.14-1.969,4.14-5.14v-2C20.14,9.83,18.554,7.86,16,7.86z"
  })), children);
});

module.exports = QuantumComputing;
