// @carbon/pictograms-react/es/locked--network--02/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const LockedNetwork_02 = /*#__PURE__*/React.forwardRef(function LockedNetwork_02(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M17,11c0,0.425-0.267,0.782-0.64,0.927V13c0,0.199-0.161,0.36-0.36,0.36s-0.36-0.161-0.36-0.36\tv-1.073C15.267,11.782,15,11.425,15,11c0-0.552,0.448-1,1-1S17,10.448,17,11z M31.36,29c0,1.302-1.059,2.36-2.36,2.36\ts-2.36-1.059-2.36-2.36c0-1.179,0.869-2.159,2-2.333V23.36H16.36v3.316c1.13,0.175,2,1.145,2,2.324c0,1.302-1.059,2.36-2.36,2.36\ts-2.36-1.059-2.36-2.36c0-1.178,0.87-2.149,2-2.324V23.36H3.36v3.316c1.13,0.175,2,1.146,2,2.324c0,1.302-1.059,2.36-2.36,2.36\tS0.64,30.302,0.64,29c0-1.178,0.87-2.149,2-2.324V23c0-0.199,0.161-0.36,0.36-0.36h12.64V19h0.72v3.64H29\tc0.199,0,0.36,0.161,0.36,0.36v3.667C30.491,26.841,31.36,27.821,31.36,29z M3,27.36c-0.904,0-1.64,0.735-1.64,1.64\tS2.096,30.64,3,30.64S4.64,29.904,4.64,29S3.904,27.36,3,27.36z M16,27.36c-0.904,0-1.64,0.735-1.64,1.64s0.736,1.64,1.64,1.64\tc0.904,0,1.64-0.735,1.64-1.64S16.904,27.36,16,27.36z M30.64,29c0-0.904-0.735-1.64-1.64-1.64s-1.64,0.735-1.64,1.64\ts0.735,1.64,1.64,1.64S30.64,29.904,30.64,29z M10.64,17V7c0-0.199,0.161-0.36,0.36-0.36h2.14V3.5c0-1.577,1.283-2.86,2.86-2.86\ts2.86,1.283,2.86,2.86v3.14H21c0.199,0,0.36,0.161,0.36,0.36v10c0,0.199-0.161,0.36-0.36,0.36H11\tC10.801,17.36,10.64,17.199,10.64,17z M13.86,6.64h4.28V3.5c0-1.18-0.96-2.14-2.14-2.14s-2.14,0.96-2.14,2.14\tC13.86,3.5,13.86,6.64,13.86,6.64z M11.36,16.64h9.28V7.36h-9.28V16.64z"
  })), children);
});

export { LockedNetwork_02 as default };
