// @carbon/pictograms-react/lib/maximize/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Maximize = /*#__PURE__*/React__default["default"].forwardRef(function Maximize(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31.36,31.36H22v-0.72h8.131l-9.386-9.385l0.51-0.51l9.385,9.386V22h0.721v9.36H31.36z M10,31.36H0.64V22 h0.72v8.131l9.386-9.386l0.509,0.51L1.869,30.64H10V31.36z M21.255,11.254l-0.51-0.509l9.386-9.386H22V0.64h9.36V10h-0.72V1.869 L21.255,11.254z M10.746,11.254L1.36,1.869V10H0.64V0.64H10v0.72H1.869l9.385,9.386L10.746,11.254z"
  })), children);
});

module.exports = Maximize;
