// @carbon/pictograms-react/es/pill--bottle--01/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const PillBottle_01 = /*#__PURE__*/React.forwardRef(function PillBottle_01(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M24,31.36H8c-0.199,0-0.36-0.161-0.36-0.36V7.36H6C5.801,7.36,5.64,7.199,5.64,7V1\tc0-0.199,0.161-0.36,0.36-0.36h20c0.199,0,0.36,0.161,0.36,0.36v6c0,0.199-0.161,0.36-0.36,0.36h-1.64V31\tC24.36,31.199,24.199,31.36,24,31.36z M8.36,30.64h15.28v-4.28H8.36V30.64z M8.36,25.64h15.28V12.36H8.36V25.64z M8.36,11.64h15.28\tV7.36H8.36V11.64z M24,6.64h1.64V1.36h-1.28V5h-0.72V1.36h-3.28V5h-0.72V1.36h-3.28V5h-0.72V1.36h-3.28V5h-0.72V1.36H8.36V5H7.64\tV1.36H6.36v5.28C6.36,6.64,24,6.64,24,6.64z M21,20.36H11v-0.72h10V20.36z M21,18.36H11v-0.72h10V18.36z M16,15.36h-5v-0.72h5V15.36\tz"
  })), children);
});

export { PillBottle_01 as default };
