// @carbon/pictograms-react/lib/movement--of--goods--02/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const MovementOfGoods_02 = /*#__PURE__*/React__default["default"].forwardRef(function MovementOfGoods_02(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M28.255,12.746l-0.51,0.509l2.386,2.386H20.36V14c0-0.199-0.161-0.36-0.36-0.36h-4\tc-0.199,0-0.36,0.161-0.36,0.36v1.64h-5.28V14c0-0.199-0.161-0.36-0.36-0.36H6c-0.199,0-0.36,0.161-0.36,0.36v1.64H1v0.72h4.64V18\tc0,0.199,0.161,0.36,0.36,0.36h4c0.199,0,0.36-0.161,0.36-0.36v-1.64h5.28V18c0,0.199,0.161,0.36,0.36,0.36h4\tc0.199,0,0.36-0.161,0.36-0.36v-1.64h9.771l-2.386,2.385l0.51,0.51L31.509,16L28.255,12.746z M9.64,17.64H6.36v-3.28h3.28V17.64z M19.64,17.64h-3.28v-3.28h3.28V17.64z M21.64,11c0,0.199,0.161,0.36,0.36,0.36h2c0.199,0,0.36-0.161,0.36-0.36V9\tc0-0.199-0.161-0.36-0.36-0.36h-0.676c-0.189-2.785-2.492-5-5.324-5h-2.64V3c0-0.199-0.161-0.36-0.36-0.36h-2\tc-0.199,0-0.36,0.161-0.36,0.36v0.64H9v0.72h3.64V5c0,0.199,0.161,0.36,0.36,0.36h2c0.199,0,0.36-0.161,0.36-0.36V4.36H18\tc2.435,0,4.416,1.892,4.603,4.28H22c-0.199,0-0.36,0.161-0.36,0.36v0.64H5v0.72h16.64V11z M14.64,4.64h-1.28V3.36h1.28V4.64z M22.36,9.36h1.279v1.28H22.36V9.36z M24,20.64h-2c-0.199,0-0.36,0.161-0.36,0.36v0.64H5v0.721h16.64V23\tc0,0.199,0.161,0.36,0.36,0.36h0.603c-0.187,2.388-2.168,4.28-4.603,4.28h-2.64V27c0-0.199-0.161-0.36-0.36-0.36h-2\tc-0.199,0-0.36,0.161-0.36,0.36v0.64H9v0.721h3.64V29c0,0.199,0.161,0.36,0.36,0.36h2c0.199,0,0.36-0.161,0.36-0.36v-0.64H18\tc2.832,0,5.135-2.215,5.324-5H24c0.199,0,0.36-0.161,0.36-0.36v-2C24.36,20.801,24.199,20.64,24,20.64z M14.64,28.64h-1.28v-1.28\th1.28V28.64z M23.64,22.64h-1.28v-1.28h1.279L23.64,22.64L23.64,22.64z"
  })), children);
});

module.exports = MovementOfGoods_02;
