// @carbon/pictograms-react/lib/relationship--extraction/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const RelationshipExtraction = /*#__PURE__*/React__default["default"].forwardRef(function RelationshipExtraction(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31,28.36H12c-0.199,0-0.36-0.161-0.36-0.36v-4c0-0.199,0.161-0.36,0.36-0.36h19\tc0.199,0,0.36,0.161,0.36,0.36v4C31.36,28.199,31.199,28.36,31,28.36z M12.36,27.64h18.28v-3.28H12.36V27.64z M8,28.36H1\tc-0.199,0-0.36-0.161-0.36-0.36v-4c0-0.199,0.161-0.36,0.36-0.36h7c0.199,0,0.36,0.161,0.36,0.36v4C8.36,28.199,8.199,28.36,8,28.36\tz M1.36,27.64h6.28v-3.28H1.36V27.64z M31,18.36h-7c-0.199,0-0.36-0.161-0.36-0.36v-4c0-0.199,0.161-0.36,0.36-0.36h7\tc0.199,0,0.36,0.161,0.36,0.36v4C31.36,18.199,31.199,18.36,31,18.36z M24.36,17.64h6.279v-3.28H24.36V17.64z M20,18.36H1\tc-0.199,0-0.36-0.161-0.36-0.36v-4c0-0.199,0.161-0.36,0.36-0.36h19c0.199,0,0.36,0.161,0.36,0.36v4\tC20.36,18.199,20.199,18.36,20,18.36z M1.36,17.64h18.28v-3.28H1.36V17.64z M31,8.36H12c-0.199,0-0.36-0.161-0.36-0.36V4\tc0-0.199,0.161-0.36,0.36-0.36h19c0.199,0,0.36,0.161,0.36,0.36v4C31.36,8.199,31.199,8.36,31,8.36z M12.36,7.64h18.28V4.36H12.36\tV7.64z M8,8.36H1C0.801,8.36,0.64,8.199,0.64,8V4c0-0.199,0.161-0.36,0.36-0.36h7c0.199,0,0.36,0.161,0.36,0.36v4\tC8.36,8.199,8.199,8.36,8,8.36z M1.36,7.64h6.28V4.36H1.36V7.64z"
  })), children);
});

module.exports = RelationshipExtraction;
