// @carbon/pictograms-react/lib/person--05/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Person_05 = /*#__PURE__*/React__default["default"].forwardRef(function Person_05(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M18.36,30h-0.72v-9c0-0.048,0.01-0.095,0.028-0.139l3.845-9.23c0.215-0.515,0.161-1.077-0.148-1.541\tC21.055,9.626,20.558,9.36,20,9.36h-8c-0.558,0-1.055,0.266-1.364,0.73c-0.31,0.464-0.364,1.026-0.149,1.542l3.845,9.23\tc0.018,0.044,0.027,0.091,0.027,0.139v9h-0.72v-8.928l-3.817-9.164c-0.305-0.73-0.225-1.56,0.214-2.218\tC10.476,9.033,11.209,8.64,12,8.64h8c0.791,0,1.524,0.393,1.963,1.051s0.519,1.487,0.214,2.218l-3.816,9.164L18.36,30L18.36,30z M16,7.36c-1.577,0-2.86-1.283-2.86-2.86S14.423,1.64,16,1.64s2.86,1.283,2.86,2.86S17.577,7.36,16,7.36z M16,2.36\tc-1.18,0-2.14,0.96-2.14,2.14S14.82,6.64,16,6.64s2.14-0.96,2.14-2.14S17.18,2.36,16,2.36z"
  })), children);
});

module.exports = Person_05;
