// @carbon/pictograms-react/lib/db2/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Db2 = /*#__PURE__*/React__default["default"].forwardRef(function Db2(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M30,28.36H2c-0.199,0-0.36-0.161-0.36-0.36V4\tc0-0.199,0.161-0.36,0.36-0.36h28c0.199,0,0.36,0.161,0.36,0.36v24C30.36,28.199,30.199,28.36,30,28.36z M16.36,27.64h13.28v-5.28\tH16.36V27.64z M2.36,27.64h13.28v-5.28H2.36V27.64z M16.36,21.64h13.28v-5.28H16.36V21.64z M2.36,21.64h13.28v-5.28H2.36V21.64z M16.36,15.64h13.28v-5.28H16.36V15.64z M2.36,15.64h13.28v-5.28H2.36V15.64z M16.36,9.64h13.28V4.36H16.36V9.64z M2.36,9.64h13.28\tV4.36H2.36V9.64z M27,26.36c-0.75,0-1.36-0.61-1.36-1.36s0.61-1.36,1.36-1.36s1.36,0.61,1.36,1.36S27.75,26.36,27,26.36z M27,24.36\tc-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S27.353,24.36,27,24.36z M13,26.36\tc-0.75,0-1.36-0.61-1.36-1.36s0.61-1.36,1.36-1.36s1.36,0.61,1.36,1.36S13.75,26.36,13,26.36z M13,24.36\tc-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S13.353,24.36,13,24.36z M27,20.36\tc-0.75,0-1.36-0.61-1.36-1.36s0.61-1.36,1.36-1.36s1.36,0.61,1.36,1.36S27.75,20.36,27,20.36z M27,18.36\tc-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S27.353,18.36,27,18.36z M13,20.36\tc-0.75,0-1.36-0.61-1.36-1.36s0.61-1.36,1.36-1.36s1.36,0.61,1.36,1.36S13.75,20.36,13,20.36z M13,18.36\tc-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S13.353,18.36,13,18.36z M27,14.36\tc-0.75,0-1.36-0.61-1.36-1.36s0.61-1.36,1.36-1.36s1.36,0.61,1.36,1.36S27.75,14.36,27,14.36z M27,12.36\tc-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S27.353,12.36,27,12.36z M13,14.36\tc-0.75,0-1.36-0.61-1.36-1.36s0.61-1.36,1.36-1.36s1.36,0.61,1.36,1.36S13.75,14.36,13,14.36z M13,12.36\tc-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S13.353,12.36,13,12.36z M27,8.36\tc-0.75,0-1.36-0.61-1.36-1.36S26.25,5.64,27,5.64S28.36,6.25,28.36,7S27.75,8.36,27,8.36z M27,6.36c-0.353,0-0.64,0.287-0.64,0.64\tS26.647,7.64,27,7.64S27.64,7.353,27.64,7S27.353,6.36,27,6.36z M13,8.36c-0.75,0-1.36-0.61-1.36-1.36S12.25,5.64,13,5.64\tS14.36,6.25,14.36,7S13.75,8.36,13,8.36z M13,6.36c-0.353,0-0.64,0.287-0.64,0.64S12.647,7.64,13,7.64S13.64,7.353,13.64,7\tS13.353,6.36,13,6.36z"
  })), children);
});

module.exports = Db2;
