// @carbon/pictograms-react/lib/flower--01/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Flower_01 = /*#__PURE__*/React__default["default"].forwardRef(function Flower_01(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31,31.36H1v-0.72h14.64v-9.371c-6.919-2.125-7-8.207-7-8.269c0-0.199,0.161-0.36,0.36-0.36\tc0.059,0,4.799,0.062,6.64,4.406v-5.698c-2.789-0.186-5-2.514-5-5.348V1c0-0.199,0.161-0.36,0.36-0.36\tc0.043,0,2.935,0.033,4.803,1.997C16.188,2.091,17.538,0.64,21,0.64c0.199,0,0.36,0.161,0.36,0.36v5c0,2.834-2.212,5.162-5,5.348\tv10.698c1.841-4.344,6.581-4.406,6.64-4.406c0.199,0,0.36,0.161,0.36,0.36c0,0.212-0.081,5.169-7,8.235v4.404H31V31.36z M22.608,18.382c-1.396,0.13-5.97,1.017-6.236,7.058C21.562,23.022,22.455,19.539,22.608,18.382z M9.383,13.381\tc0.127,1.284,0.951,5.4,6.248,7.129C15.396,14.394,10.774,13.509,9.383,13.381z M11.36,1.378V6c0,2.559,2.082,4.64,4.64,4.64\tS20.64,8.559,20.64,6V1.366c-3.396,0.108-4.282,1.727-4.319,1.798l-0.124-0.063C16.886,4.018,17.36,5.275,17.36,7h-0.72\tC16.64,2.202,12.614,1.485,11.36,1.378z"
  })), children);
});

module.exports = Flower_01;
