// @carbon/pictograms-react/es/baby--bottle/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const BabyBottle = /*#__PURE__*/React.forwardRef(function BabyBottle(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M20,31.36h-8c-1.301,0-2.36-1.059-2.36-2.36V15.817c0-1.26,0.369-2.48,1.068-3.528l0.619-0.929H11\tc-0.199,0-0.36-0.161-0.36-0.36V8c0-0.199,0.161-0.36,0.36-0.36h0.655c0.144-1.733,1.32-3.222,2.985-3.768V2\tc0-0.75,0.61-1.36,1.36-1.36c0.75,0,1.36,0.61,1.36,1.36v1.872c1.665,0.546,2.841,2.035,2.984,3.768H21\tc0.199,0,0.36,0.161,0.36,0.36v3c0,0.199-0.161,0.36-0.36,0.36h-0.327l0.619,0.929c0.698,1.048,1.068,2.269,1.068,3.528V29\tC22.36,30.302,21.302,31.36,20,31.36z M10.36,27.36V29c0,0.904,0.736,1.64,1.64,1.64h8c0.904,0,1.64-0.735,1.64-1.64V15.817\tc0-1.117-0.327-2.199-0.947-3.129l-0.885-1.328h-7.615l-0.886,1.328c-0.62,0.93-0.947,2.012-0.947,3.129v1.823H13v0.721h-2.64v2.279\tH13v0.721h-2.64v2.279H13v0.721h-2.64v2.279H13v0.721L10.36,27.36L10.36,27.36z M11.36,10.64h9.28V8.36h-9.28V10.64z M12.378,7.64\th7.244c-0.149-1.497-1.223-2.765-2.712-3.15c-0.159-0.041-0.27-0.185-0.27-0.349V2c0-0.353-0.287-0.64-0.64-0.64\tS15.36,1.647,15.36,2v2.142c0,0.164-0.111,0.308-0.27,0.349C13.601,4.875,12.527,6.143,12.378,7.64z"
  })), children);
});

export { BabyBottle as default };
