// @carbon/pictograms-react/es/ibm--z16--plus-cloud/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const IbmZ16PlusCloud = /*#__PURE__*/React.forwardRef(function IbmZ16PlusCloud(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M15,3.64H7C6.801,3.64,6.64,3.801,6.64,4\tv24c0,0.199,0.161,0.36,0.36,0.36h8c0.199,0,0.36-0.161,0.36-0.36V4C15.36,3.801,15.199,3.64,15,3.64z M10.36,4.36h2.28v5.28h-2.28\tV4.36z M12.64,15.64h-1.771l1.771-1.771V15.64z M10.869,10.36h1.771v1.771L10.869,10.36z M10.36,10.869L12.491,13l-2.131,2.131\tC10.36,15.131,10.36,10.869,10.36,10.869z M12.64,16.36v5.28h-2.28v-5.28H12.64z M9.64,21.131L7.509,19l2.131-2.131\tC9.64,16.869,9.64,21.131,9.64,21.131z M9.64,9.131L7.509,7L9.64,4.869V9.131z M7.36,4.36h1.771L7.36,6.131V4.36z M7.36,7.869\tL9.131,9.64H7.36V7.869z M7.36,10.36h2.28v5.28H7.36V10.36z M7.36,16.36h1.771L7.36,18.131V16.36z M9.13,21.64H7.36v-1.771\tL9.13,21.64z M7.36,27.64v-5.28h2.28v5.279L7.36,27.64L7.36,27.64z M10.36,22.869L12.491,25l-2.131,2.131\tC10.36,27.131,10.36,22.869,10.36,22.869z M12.64,27.64h-1.77l1.771-1.771L12.64,27.64L12.64,27.64z M10.87,22.36h1.771v1.771\tL10.87,22.36z M14.64,27.64h-1.28V4.36h1.28V27.64z M26.384,25.36H16v-0.72h10.384c2.387,0,4.256-1.869,4.256-4.255\tc0-2.332-1.789-4.025-4.255-4.025c-0.199,0-0.36-0.161-0.36-0.36c0-4.655-3.564-9.64-8.87-9.64c-0.383,0-0.746,0.038-1.114,0.079\tL15.96,5.723c0.39-0.044,0.774-0.083,1.195-0.083c5.604,0,9.41,5.111,9.584,10.011c2.659,0.162,4.622,2.143,4.622,4.734\tC31.36,23.175,29.174,25.36,26.384,25.36z M5.9,25.057C2.803,24.162,0.64,21.26,0.64,18c0-3.222,2.161-6.123,5.256-7.056\tl0.208,0.689C3.311,12.476,1.36,15.093,1.36,18c0,2.941,1.949,5.559,4.74,6.365L5.9,25.057z"
  })), children);
});

export { IbmZ16PlusCloud as default };
