// @carbon/pictograms-react/lib/deploying-containers/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const DeployingContainers = /*#__PURE__*/React__default["default"].forwardRef(function DeployingContainers(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,0.591L2.64,7.785v16.43L16,31.409\tl13.36-7.194V7.785L16,0.591z M28.279,8.021l-11.355,6.988c-0.158-0.147-0.351-0.255-0.565-0.314v-6.39c0.575-0.159,1-0.681,1-1.305\tc0-0.75-0.61-1.36-1.36-1.36S14.64,6.25,14.64,7c0,0.625,0.425,1.146,1,1.305v6.389c-0.214,0.059-0.407,0.167-0.565,0.314\tL3.72,8.021L16,1.409L28.279,8.021z M16,16.64c-0.353,0-0.64-0.287-0.64-0.64s0.287-0.64,0.64-0.64s0.64,0.287,0.64,0.64\tC16.64,16.353,16.353,16.64,16,16.64z M16,7.64c-0.353,0-0.64-0.287-0.64-0.64S15.647,6.36,16,6.36c0.353,0,0.64,0.287,0.64,0.64\tS16.353,7.64,16,7.64z M3.36,8.644l11.34,6.979c-0.035,0.121-0.06,0.245-0.06,0.377c0,0.135,0.026,0.263,0.063,0.386l-5.786,3.616\tC8.675,19.779,8.354,19.64,8,19.64c-0.75,0-1.36,0.61-1.36,1.36S7.25,22.36,8,22.36S9.36,21.75,9.36,21\tc0-0.135-0.026-0.263-0.062-0.386l5.785-3.617c0.157,0.144,0.346,0.25,0.557,0.308v13.091L3.36,23.785V8.644z M8.64,21\tc0,0.353-0.287,0.64-0.64,0.64S7.36,21.353,7.36,21S7.647,20.36,8,20.36C8.353,20.36,8.64,20.647,8.64,21z M16.36,30.397V17.306\tc0.211-0.058,0.401-0.164,0.557-0.308l5.785,3.616C22.666,20.737,22.64,20.865,22.64,21c0,0.75,0.61,1.36,1.36,1.36\ts1.36-0.61,1.36-1.36s-0.61-1.36-1.36-1.36c-0.354,0-0.675,0.14-0.917,0.363l-5.785-3.615c0.037-0.123,0.063-0.252,0.063-0.387\tc0-0.132-0.025-0.257-0.06-0.377l11.339-6.98v15.141L16.36,30.397z M23.36,21c0-0.353,0.287-0.64,0.64-0.64s0.64,0.287,0.64,0.64\ts-0.287,0.64-0.64,0.64S23.36,21.353,23.36,21z"
  })), children);
});

module.exports = DeployingContainers;
