// @carbon/pictograms-react/es/under--utilized--security/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const UnderUtilizedSecurity = /*#__PURE__*/React.forwardRef(function UnderUtilizedSecurity(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M28.811,26.812l-4.33-2.5l0.359-0.623l4.33,2.5L28.811,26.812z M31,19.36h-5v-0.72h5\tV19.36z M16,19.36L1,19.355v-0.721l15,0.005V19.36z M24.84,14.312l-0.359-0.624l4.33-2.5l0.359,0.624L24.84,14.312z M7.16,14.312\tl-4.33-2.5l0.36-0.624l4.33,2.5L7.16,14.312z M21.312,10.52l-0.623-0.36l2.5-4.33l0.623,0.36L21.312,10.52z M10.688,10.52l-2.5-4.33\tl0.624-0.36l2.5,4.33L10.688,10.52z M16.36,9h-0.72V4h0.72V9z"
  })), children);
});

export { UnderUtilizedSecurity as default };
