// @carbon/pictograms-react/es/data--set/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const DataSet = /*#__PURE__*/React.forwardRef(function DataSet(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M28,31.36H4c-0.199,0-0.36-0.161-0.36-0.36V1c0-0.199,0.161-0.36,0.36-0.36h18\tc0.096,0,0.188,0.038,0.255,0.105l6,6C28.322,6.813,28.36,6.904,28.36,7v24C28.36,31.199,28.199,31.36,28,31.36z M4.36,30.64h23.28\tV7.36H22c-0.199,0-0.36-0.161-0.36-0.36V1.36H4.36V30.64z M22.36,6.64h4.771L22.36,1.869V6.64z M25,27.36H7\tc-0.199,0-0.36-0.161-0.36-0.36V11c0-0.199,0.161-0.36,0.36-0.36h18c0.199,0,0.36,0.161,0.36,0.36v16\tC25.36,27.199,25.199,27.36,25,27.36z M19.36,26.64h5.279v-3.28H19.36V26.64z M13.36,26.64h5.28v-3.28h-5.28\tC13.36,23.36,13.36,26.64,13.36,26.64z M7.36,26.64h5.28v-3.28H7.36V26.64z M19.36,22.64h5.279v-3.28H19.36V22.64z M13.36,22.64\th5.28v-3.28h-5.28C13.36,19.36,13.36,22.64,13.36,22.64z M7.36,22.64h5.28v-3.28H7.36V22.64z M19.36,18.64h5.279v-3.28H19.36V18.64z M13.36,18.64h5.28v-3.28h-5.28C13.36,15.36,13.36,18.64,13.36,18.64z M7.36,18.64h5.28v-3.28H7.36V18.64z M19.36,14.64h5.279v-3.28\tH19.36V14.64z M13.36,14.64h5.28v-3.28h-5.28C13.36,11.36,13.36,14.64,13.36,14.64z M7.36,14.64h5.28v-3.28H7.36V14.64z"
  })), children);
});

export { DataSet as default };
