// @carbon/pictograms-react/es/slack/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Slack = /*#__PURE__*/React.forwardRef(function Slack(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M15.36,7.36H12c-1.853,0-3.36-1.507-3.36-3.36S10.147,0.64,12,0.64S15.36,2.147,15.36,4V7.36z M12,1.36\tc-1.456,0-2.64,1.185-2.64,2.64S10.544,6.64,12,6.64h2.64V4C14.64,2.544,13.456,1.36,12,1.36z M12,15.36H4\tc-1.853,0-3.36-1.507-3.36-3.36S2.147,8.64,4,8.64h8c1.853,0,3.36,1.507,3.36,3.36S13.853,15.36,12,15.36z M4,9.36\tc-1.456,0-2.64,1.185-2.64,2.64S2.544,14.64,4,14.64h8c1.456,0,2.64-1.185,2.64-2.64S13.456,9.36,12,9.36H4z M28,15.36h-3.36V12\tc0-1.853,1.508-3.36,3.36-3.36s3.36,1.507,3.36,3.36S29.853,15.36,28,15.36z M25.36,14.64H28c1.456,0,2.64-1.185,2.64-2.64\tS29.456,9.36,28,9.36s-2.64,1.185-2.64,2.64V14.64z M20,15.36c-1.853,0-3.36-1.507-3.36-3.36V4c0-1.853,1.507-3.36,3.36-3.36\tS23.36,2.147,23.36,4v8C23.36,13.853,21.853,15.36,20,15.36z M20,1.36c-1.456,0-2.64,1.185-2.64,2.64v8\tc0,1.456,1.184,2.64,2.64,2.64s2.64-1.185,2.64-2.64V4C22.64,2.544,21.456,1.36,20,1.36z M20,31.36c-1.853,0-3.36-1.508-3.36-3.36\tv-3.36H20c1.853,0,3.36,1.508,3.36,3.36S21.853,31.36,20,31.36z M17.36,25.36V28c0,1.456,1.184,2.64,2.64,2.64s2.64-1.184,2.64-2.64\ts-1.184-2.64-2.64-2.64H17.36z M28,23.36h-8c-1.853,0-3.36-1.508-3.36-3.36s1.507-3.36,3.36-3.36h8c1.853,0,3.36,1.507,3.36,3.36\tS29.853,23.36,28,23.36z M20,17.36c-1.456,0-2.64,1.184-2.64,2.64s1.184,2.64,2.64,2.64h8c1.456,0,2.64-1.184,2.64-2.64\ts-1.184-2.64-2.64-2.64H20z M4,23.36c-1.853,0-3.36-1.508-3.36-3.36S2.147,16.64,4,16.64h3.36V20C7.36,21.853,5.853,23.36,4,23.36z M4,17.36c-1.456,0-2.64,1.184-2.64,2.64S2.544,22.64,4,22.64S6.64,21.456,6.64,20v-2.64H4z M12,31.36\tc-1.853,0-3.36-1.508-3.36-3.36v-8c0-1.853,1.507-3.36,3.36-3.36s3.36,1.507,3.36,3.36v8C15.36,29.853,13.853,31.36,12,31.36z M12,17.36c-1.456,0-2.64,1.184-2.64,2.64v8c0,1.456,1.185,2.64,2.64,2.64s2.64-1.184,2.64-2.64v-8\tC14.64,18.544,13.456,17.36,12,17.36z"
  })), children);
});

export { Slack as default };
