// @carbon/pictograms-react/lib/visibility/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Visibility = /*#__PURE__*/React__default["default"].forwardRef(function Visibility(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,25.36c-0.658,0-1.208-0.47-1.333-1.091\tc-7.495-0.782-13.675-7.732-13.938-8.032c-0.119-0.136-0.119-0.339,0-0.475c0.263-0.3,6.443-7.25,13.938-8.033\tC14.792,7.109,15.342,6.64,16,6.64c0.658,0,1.207,0.469,1.333,1.09c7.494,0.782,13.675,7.733,13.938,8.033\tc0.119,0.136,0.119,0.339,0,0.475c-0.263,0.3-6.443,7.25-13.938,8.032C17.207,24.891,16.658,25.36,16,25.36z M16,23.36\tc-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S16.353,23.36,16,23.36z M16.36,22.688\tc0.425,0.116,0.768,0.436,0.917,0.846c3.65-0.614,6.363-3.797,6.363-7.534c0-3.738-2.713-6.92-6.362-7.534\tc-0.149,0.41-0.492,0.729-0.917,0.845v3.348c1.684,0.18,3,1.61,3,3.341s-1.316,3.161-3,3.341L16.36,22.688L16.36,22.688z M14.722,8.466C11.073,9.08,8.36,12.262,8.36,16c0,3.737,2.713,6.92,6.362,7.534c0.15-0.41,0.493-0.729,0.918-0.846V19.34\tc-1.684-0.18-3-1.609-3-3.341c0-1.731,1.316-3.161,3-3.341V9.312C15.215,9.195,14.873,8.876,14.722,8.466z M21.189,9.446\tC23.13,10.988,24.36,13.368,24.36,16s-1.229,5.013-3.171,6.554c4.696-1.856,8.403-5.579,9.321-6.554\tC29.593,15.025,25.886,11.303,21.189,9.446z M1.489,16c0.918,0.975,4.625,4.697,9.321,6.554C8.87,21.013,7.64,18.632,7.64,16\ts1.229-5.013,3.17-6.554C6.115,11.303,2.407,15.025,1.489,16z M16,13.36c-1.456,0-2.64,1.185-2.64,2.64\tc0,1.456,1.185,2.64,2.64,2.64c1.456,0,2.64-1.184,2.64-2.64S17.456,13.36,16,13.36z M16,7.36c-0.353,0-0.64,0.287-0.64,0.64\tS15.647,8.64,16,8.64c0.353,0,0.64-0.287,0.64-0.64S16.353,7.36,16,7.36z M30.745,10.254C30.677,10.186,23.754,3.36,16,3.36\tc-7.761,0-14.676,6.826-14.746,6.895L0.746,9.746C1.036,9.455,7.943,2.64,16,2.64s14.965,6.815,15.255,7.105L30.745,10.254z"
  })), children);
});

module.exports = Visibility;
