// @carbon/pictograms-react/es/virtualization--02/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Virtualization_02 = /*#__PURE__*/React.forwardRef(function Virtualization_02(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M30,28.36H9\tc-0.135,0-0.253-0.075-0.314-0.185L1.692,16.187C1.659,16.132,1.64,16.068,1.64,16V4c0-0.199,0.161-0.36,0.36-0.36h21\tc0.135,0,0.253,0.074,0.314,0.185l6.993,11.989c0.033,0.054,0.053,0.118,0.053,0.187v12C30.36,28.199,30.199,28.36,30,28.36z M9.36,27.64h20.28V16.36H16.624l3.069,5.315c0.099-0.022,0.202-0.035,0.308-0.035c0.75,0,1.36,0.61,1.36,1.36s-0.61,1.36-1.36,1.36\ts-1.36-0.61-1.36-1.36c0-0.385,0.161-0.733,0.42-0.981l-3.268-5.659H9.36V27.64z M2.627,16.36L8.64,26.668V16.36H2.627z M20,22.36\tc-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S20.353,22.36,20,22.36z M23.36,15.64h6.013L23.36,5.332\tV15.64z M16.208,15.64h6.432V4.36H2.36v11.28h13.017l-3.069-5.315C12.209,10.348,12.106,10.36,12,10.36c-0.75,0-1.36-0.61-1.36-1.36\tS11.25,7.64,12,7.64S13.36,8.25,13.36,9c0,0.385-0.161,0.733-0.419,0.981L16.208,15.64z M12,8.36c-0.353,0-0.64,0.287-0.64,0.64\tS11.647,9.64,12,9.64S12.64,9.353,12.64,9S12.353,8.36,12,8.36z"
  })), children);
});

export { Virtualization_02 as default };
