// @carbon/pictograms-react/lib/red-hat--applications/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const RedHatApplications = /*#__PURE__*/React__default["default"].forwardRef(function RedHatApplications(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M28,28.36H12\tc-0.199,0-0.36-0.161-0.36-0.36v-7.64H4c-0.199,0-0.36-0.161-0.36-0.36V5c0-0.199,0.161-0.36,0.36-0.36h16\tc0.199,0,0.36,0.161,0.36,0.36v7.64H28c0.199,0,0.36,0.161,0.36,0.36v15C28.36,28.199,28.199,28.36,28,28.36z M12.36,27.64h15.28\tV13.36H12.36V27.64z M4.36,19.64h7.28V13c0-0.199,0.161-0.36,0.36-0.36h7.639V5.36H4.36V19.64z M19,16.36h-4v-0.72h4V16.36z M11,8.36H7V7.64h4V8.36z"
  })), children);
});

module.exports = RedHatApplications;
