// @carbon/pictograms-react/lib/data--privacy/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const DataPrivacy = /*#__PURE__*/React__default["default"].forwardRef(function DataPrivacy(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,31.36C9.185,31.36,3.64,25.815,3.64,19V6c0-0.199,0.161-0.36,0.36-0.36c4.354,0,8.635-1.784,11.746-4.895 c0.141-0.141,0.368-0.141,0.509,0C19.365,3.856,23.646,5.64,28,5.64c0.199,0,0.36,0.161,0.36,0.36v13 C28.36,25.815,22.815,31.36,16,31.36z M4.36,6.356V19c0,6.418,5.222,11.64,11.64,11.64S27.64,25.418,27.64,19V6.356 C23.339,6.263,19.134,4.511,16,1.504C12.866,4.511,8.661,6.263,4.36,6.356z M21.36,22h-0.72v-1.5c0-2.088-1.379-3.864-3.433-4.419 c-0.146-0.04-0.25-0.165-0.265-0.314c-0.014-0.15,0.066-0.292,0.201-0.358c0.909-0.439,1.496-1.385,1.496-2.409 c0-1.456-1.184-2.64-2.64-2.64s-2.64,1.185-2.64,2.64c0,1.023,0.587,1.969,1.497,2.409c0.135,0.065,0.216,0.208,0.202,0.358 c-0.014,0.149-0.12,0.274-0.265,0.314c-2.054,0.555-3.434,2.332-3.434,4.419V22h-0.72v-1.5c0-2.163,1.275-4.036,3.243-4.866 C13.111,14.999,12.64,14.03,12.64,13c0-1.853,1.507-3.36,3.36-3.36s3.36,1.507,3.36,3.36c0,1.03-0.471,1.999-1.243,2.634 c1.968,0.83,3.243,2.703,3.243,4.866V22z"
  })), children);
});

module.exports = DataPrivacy;
