// @carbon/pictograms-react/lib/managing--contractual--flow/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const ManagingContractualFlow = /*#__PURE__*/React__default["default"].forwardRef(function ManagingContractualFlow(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,31.36c-4.529,0-8.716-1.933-11.64-5.338V30H3.64v-5.36H9v0.721H4.743\tC7.536,28.725,11.598,30.64,16,30.64c8.072,0,14.64-6.567,14.64-14.64h0.721C31.36,24.47,24.47,31.36,16,31.36z M1.36,16H0.64\tC0.64,7.53,7.53,0.64,16,0.64c4.529,0,8.716,1.933,11.64,5.337V2h0.721v5.36H23V6.64h4.257C24.464,3.275,20.401,1.36,16,1.36\tC7.927,1.36,1.36,7.927,1.36,16z M22.255,11.746l-4-4C18.188,7.678,18.096,7.64,18,7.64h-8C9.801,7.64,9.64,7.801,9.64,8v15\tc0,0.199,0.161,0.36,0.36,0.36h12c0.199,0,0.36-0.161,0.36-0.36V12C22.36,11.904,22.322,11.813,22.255,11.746z M18.36,8.87\tl2.771,2.771H18.36V8.87z M10.36,22.64V8.36h7.28V12c0,0.199,0.161,0.36,0.36,0.36h3.64v10.28H10.36z M20,16.64h-8v0.72h8V16.64z M20,18.64h-8v0.721h8V18.64z M20,14.64h-8v0.72h8V14.64z"
  })), children);
});

module.exports = ManagingContractualFlow;
