// @carbon/pictograms-react/es/small-components-making-a-larger-whole/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const SmallComponentsMakingALargerWhole = /*#__PURE__*/React.forwardRef(function SmallComponentsMakingALargerWhole(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M16,31.36\tc-0.062,0-0.125-0.017-0.181-0.05l-12-7C3.708,24.246,3.64,24.128,3.64,24V9c0-0.129,0.069-0.248,0.181-0.312\tc0.111-0.065,0.249-0.064,0.36,0.001l5.459,3.184V5c0-0.129,0.069-0.248,0.181-0.312c0.111-0.065,0.249-0.064,0.36,0.001\tl5.459,3.184V1c0-0.129,0.069-0.248,0.181-0.312c0.111-0.065,0.248-0.064,0.36,0.001l12,7C28.292,7.753,28.36,7.872,28.36,8v15\tc0,0.129-0.069,0.248-0.181,0.312c-0.112,0.062-0.249,0.064-0.361-0.002l-5.458-3.184V27c0,0.129-0.069,0.248-0.181,0.312\tc-0.112,0.063-0.249,0.063-0.361-0.002l-5.458-3.184V31c0,0.129-0.069,0.248-0.181,0.312C16.124,31.344,16.062,31.36,16,31.36z M4.36,23.793l11.28,6.58v-6.666l-5.821-3.396C9.708,20.246,9.64,20.128,9.64,20v-7.293l-5.28-3.08V23.793z M16.36,23.293l5.28,3.08\tv-6.666l-5.28-3.08V23.293z M10.36,19.793l5.28,3.08v-6.667l-5.28-3.08C10.36,13.126,10.36,19.793,10.36,19.793z M22.36,19.293\tl5.279,3.08V8.207l-11.28-6.58v6.667l5.822,3.396c0.11,0.064,0.179,0.183,0.179,0.311C22.36,12.001,22.36,19.293,22.36,19.293z M16.36,15.793l5.28,3.08v-6.667l-5.28-3.08V15.793z M10.36,12.293l5.28,3.08V8.707l-5.28-3.08\tC10.36,5.627,10.36,12.293,10.36,12.293z"
  })), children);
});

export { SmallComponentsMakingALargerWhole as default };
