// @carbon/pictograms-react/lib/mobile--chat/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const MobileChat = /*#__PURE__*/React__default["default"].forwardRef(function MobileChat(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M11,28.5c-0.552,0-1-0.448-1-1s0.448-1,1-1s1,0.448,1,1S11.553,28.5,11,28.5z M20.36,31V16.36H28\tc1.302,0,2.36-1.059,2.36-2.36V6c0-1.301-1.059-2.36-2.36-2.36h-7.64V1c0-0.199-0.161-0.36-0.36-0.36H2\tC1.801,0.64,1.64,0.801,1.64,1v30c0,0.199,0.161,0.36,0.36,0.36h18C20.199,31.36,20.36,31.199,20.36,31z M2.36,24.36h17.28v6.279\tH2.36V24.36z M2.36,1.36h17.28v2.28H17c-1.301,0-2.36,1.059-2.36,2.36v14c0,0.146,0.087,0.277,0.222,0.333\tc0.134,0.053,0.29,0.024,0.392-0.078l3.895-3.895h0.49v7.28H2.36V1.36z M28,4.36c0.904,0,1.64,0.736,1.64,1.64v8\tc0,0.904-0.735,1.64-1.64,1.64h-9c-0.096,0-0.188,0.038-0.255,0.105L15.36,19.13V6c0-0.904,0.736-1.64,1.64-1.64H28z M27,11.64h-9\tv0.72h9V11.64z M27,7.64h-9v0.72h9V7.64z"
  })), children);
});

module.exports = MobileChat;
