// @carbon/pictograms-react/lib/cloud--native--02/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const CloudNative_02 = /*#__PURE__*/React__default["default"].forwardRef(function CloudNative_02(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M27,28.36H15c-0.199,0-0.36-0.161-0.36-0.36\tV16c0-0.199,0.161-0.36,0.36-0.36h12c0.199,0,0.36,0.161,0.36,0.36v5.47c1.873-0.885,3.279-3.217,3.279-5.557\tc0-2.863-2.233-5.259-5.084-5.454c-0.182-0.013-0.325-0.159-0.335-0.341c-0.197-3.79-3.322-6.758-7.113-6.758\tc-2.752,0-5.286,1.616-6.457,4.117c-0.082,0.175-0.288,0.254-0.463,0.181C10.323,7.303,9.412,7.123,8.48,7.123\tc-3.927,0-7.122,3.2-7.122,7.133c0,3.685,2.634,6.93,5.996,7.388L12,21.64v0.721H7.307C3.485,21.843,0.64,18.36,0.64,14.256\tc0-4.33,3.518-7.853,7.841-7.853c0.914,0,1.809,0.157,2.667,0.468c1.346-2.581,4.042-4.23,6.959-4.23c4.07,0,7.442,3.107,7.81,7.13\tc3.077,0.368,5.443,3.008,5.443,6.143c0,2.726-1.738,5.452-4,6.34V28C27.36,28.199,27.199,28.36,27,28.36z M15.36,27.64h11.28V16.36\tH15.36V27.64z M20,19.36h-3v-0.72h3V19.36z"
  })), children);
});

module.exports = CloudNative_02;
