// @carbon/pictograms-react/lib/liquids/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Liquids = /*#__PURE__*/React__default["default"].forwardRef(function Liquids(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M5.36,30H4.64v-4h0.72C5.36,26,5.36,30,5.36,30z M7.36,28H6.64v-8h0.72C7.36,20,7.36,28,7.36,28z M3.36,28H2.64v-4h0.72C3.36,24,3.36,28,3.36,28z M5.36,24H4.64v-3.901\tc0-1.031,0.66-1.935,1.644-2.248l0.607-0.193l0.219,0.686l-0.607,0.193C5.82,18.755,5.36,19.383,5.36,20.099\tC5.36,20.099,5.36,24,5.36,24z M3.36,22H2.64v-1.799c0-1.901,1.216-3.569,3.026-4.151l0.73-0.234l0.22,0.686l-0.73,0.234\tc-1.511,0.485-2.526,1.877-2.526,3.465C3.36,20.201,3.36,22,3.36,22z M8.886,20.093L5.003,5.597L5.699,5.41l0.17,0.634l20.208-4.396\tc0.188-0.04,0.374,0.073,0.424,0.258l2.847,10.625c0.05,0.186-0.055,0.377-0.238,0.436l-19.7,6.298l0.172,0.642L8.886,20.093z M6.055,6.741L9.222,18.57l2.176-0.695L8.286,6.255L6.055,6.741z M8.99,6.103l3.095,11.552l11.71-3.743L21.003,3.489L8.99,6.103z M21.707,3.335l2.774,10.356l4.082-1.305l-2.669-9.962L21.707,3.335z"
  })), children);
});

module.exports = Liquids;
