// @carbon/pictograms-react/es/ibm--z16/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const IbmZ16 = /*#__PURE__*/React.forwardRef(function IbmZ16(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M16.003,31.36H31c0.199,0,0.36-0.161,0.36-0.36V1\tc0-0.199-0.161-0.36-0.36-0.36H16.003c-0.002,0-0.004,0-0.006,0H1C0.801,0.64,0.64,0.801,0.64,1v14.999c0,0.001,0,0.002,0,0.003V31\tc0,0.199,0.161,0.36,0.36,0.36h14.997c0.001,0,0.002,0,0.003,0C16.001,31.36,16.002,31.36,16.003,31.36z M16.36,16.36h14.28v14.28\tH16.36V16.36z M1.36,16.869L15.131,30.64H1.36V16.869z M15.64,16.36v13.771L1.869,16.36H15.64z M16.36,1.36h14.28v14.28H16.36V1.36z M15.64,1.869V15.64H1.869L15.64,1.869z M15.131,1.36L1.36,15.131V1.36H15.131z"
  })), children);
});

export { IbmZ16 as default };
