// @carbon/pictograms-react/lib/ibm--z16--single-frame/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const IbmZ16SingleFrame = /*#__PURE__*/React__default["default"].forwardRef(function IbmZ16SingleFrame(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M15,28.36c-0.001,0-0.002,0-0.003,0H12\tc-0.199,0-0.36-0.161-0.36-0.36v-8.999c0-0.001,0-0.001,0-0.002V7.001c0-0.001,0-0.002,0-0.003V4c0-0.199,0.161-0.36,0.36-0.36\th2.997c0.002,0,0.004,0,0.006,0H20c0.199,0,0.36,0.161,0.36,0.36v24c0,0.199-0.161,0.36-0.36,0.36h-4.997\tC15.002,28.36,15.001,28.36,15,28.36z M18.36,27.64h1.279V4.36H18.36v8.639c0,0.001,0,0.002,0,0.003V25c0,0.001,0,0.001,0,0.002\tV27.64z M15.869,27.64h1.771v-1.771L15.869,27.64z M12.36,27.64h2.28v-5.28h-2.28V27.64z M15.36,22.869v4.262L17.491,25\tL15.36,22.869z M15.869,22.36l1.771,1.771V22.36H15.869z M15.36,21.64h2.28v-5.28h-2.28V21.64z M12.36,21.64h1.771l-1.771-1.771\tV21.64z M12.509,19l2.131,2.131v-4.262L12.509,19z M12.36,16.36v1.771l1.771-1.771H12.36z M15.869,15.64h1.771v-1.771L15.869,15.64z M12.36,15.64h2.28v-5.28h-2.28V15.64z M15.36,10.869v4.262L17.491,13L15.36,10.869z M15.869,10.36l1.771,1.771V10.36H15.869z M15.36,9.64h2.28V4.36h-2.28V9.64z M12.36,9.64h1.771L12.36,7.869V9.64z M12.509,7l2.131,2.131V4.869L12.509,7z M12.36,4.36v1.771\tl1.771-1.771H12.36z"
  })), children);
});

module.exports = IbmZ16SingleFrame;
