// @carbon/pictograms-react/es/add--device/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const AddDevice = /*#__PURE__*/React.forwardRef(function AddDevice(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M31,30.36h-8c-0.199,0-0.36-0.161-0.36-0.36v-2.64H2v-0.72h20.64v-1.28H5c-0.75,0-1.36-0.61-1.36-1.36\tV13.926c-1.819-1.341-3-3.499-3-5.926c0-4.058,3.302-7.36,7.36-7.36c3.938,0,7.163,3.108,7.351,7H27c0.75,0,1.36,0.61,1.36,1.36\tv6.64H31c0.199,0,0.36,0.161,0.36,0.36v14C31.36,30.199,31.199,30.36,31,30.36z M23.36,29.64h7.279v-2.28H23.36V29.64z M23.36,26.64\th7.279V16.36H23.36V26.64z M4.36,14.396V24c0,0.353,0.287,0.64,0.64,0.64h17.64V16c0-0.199,0.161-0.36,0.36-0.36h4.64V9\tc0-0.353-0.287-0.64-0.64-0.64H15.351c-0.188,3.892-3.414,7-7.351,7C6.677,15.36,5.434,15.009,4.36,14.396z M8,1.36\tC4.339,1.36,1.36,4.339,1.36,8S4.339,14.64,8,14.64s6.64-2.979,6.64-6.64S11.661,1.36,8,1.36z M8.36,12H7.64V8.36H4V7.64h3.64V4\th0.72v3.64H12v0.72H8.36V12z"
  })), children);
});

export { AddDevice as default };
