// @carbon/pictograms-react/es/home--front/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const HomeFront = /*#__PURE__*/React.forwardRef(function HomeFront(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M31,31.36H1v-0.72h1.64V19h0.72v11.64h9.28V19c0-0.199,0.161-0.36,0.36-0.36h6\tc0.199,0,0.36,0.161,0.36,0.36v11.64h9.279V19h0.721v11.64H31V31.36z M13.36,30.64h5.28V19.36h-5.28\tC13.36,19.36,13.36,30.64,13.36,30.64z M13,16.363v-0.72l6-0.003v0.72L13,16.363z M30.745,16.254L16,1.509L1.254,16.254\tl-0.509-0.509l15-15c0.141-0.141,0.368-0.141,0.509,0L28.64,13.131V9h0.721v4.851l1.895,1.895L30.745,16.254z"
  })), children);
});

export { HomeFront as default };
