// @carbon/pictograms-react/es/storage--systems/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const StorageSystems = /*#__PURE__*/React.forwardRef(function StorageSystems(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M30,28.36H2c-0.199,0-0.36-0.161-0.36-0.36v-6\th0.72v5.64h27.28V22h0.721v6C30.36,28.199,30.199,28.36,30,28.36z M26,25.36c-0.75,0-1.36-0.61-1.36-1.36s0.61-1.36,1.36-1.36\ts1.36,0.61,1.36,1.36S26.75,25.36,26,25.36z M26,23.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64\tS26.353,23.36,26,23.36z M22,24.36H5v-0.72h17V24.36z M30,20.36H2c-0.199,0-0.36-0.161-0.36-0.36v-6h0.72v5.64h27.28V14h0.721v6\tC30.36,20.199,30.199,20.36,30,20.36z M26,17.36c-0.75,0-1.36-0.61-1.36-1.36s0.61-1.36,1.36-1.36s1.36,0.61,1.36,1.36\tS26.75,17.36,26,17.36z M26,15.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S26.353,15.36,26,15.36z M22,16.36H5v-0.72h17V16.36z M30,12.36H2c-0.199,0-0.36-0.161-0.36-0.36V4c0-0.199,0.161-0.36,0.36-0.36h28\tc0.199,0,0.36,0.161,0.36,0.36v8C30.36,12.199,30.199,12.36,30,12.36z M2.36,11.64h27.28V4.36H2.36V11.64z"
  })), children);
});

export { StorageSystems as default };
