// @carbon/pictograms-react/lib/tennis--ball/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const TennisBall = /*#__PURE__*/React__default["default"].forwardRef(function TennisBall(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M22,0.64c-5.161,0-9.36,4.199-9.36,9.36\ts4.199,9.36,9.36,9.36s9.36-4.199,9.36-9.36S27.161,0.64,22,0.64z M30.619,9.601c-4.369-0.416-7.805-3.852-8.221-8.221\tC26.844,1.584,30.415,5.156,30.619,9.601z M13.38,10.399c4.369,0.416,7.805,3.852,8.221,8.221\tC17.156,18.415,13.584,14.844,13.38,10.399z M22.324,18.623c-0.419-4.771-4.175-8.528-8.948-8.947c0.168-4.505,3.795-8.131,8.3-8.3\tc0.419,4.771,4.175,8.529,8.947,8.948C30.455,14.829,26.829,18.455,22.324,18.623z M2.254,20.255l-0.509-0.51l8-8l0.509,0.509\tL2.254,20.255z M19.745,21.745l0.51,0.51l-4,4l-0.509-0.51L19.745,21.745z M12.746,18.745l0.509,0.51l-12,12l-0.509-0.51\tL12.746,18.745z"
  })), children);
});

module.exports = TennisBall;
