// @carbon/pictograms-react/es/upload--01/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path, _path2;
const Upload_01 = /*#__PURE__*/React.forwardRef(function Upload_01(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M31.36,9h-0.72V1.36H1.36V9H0.64V1c0-0.199,0.161-0.36,0.36-0.36h30c0.199,0,0.36,0.161,0.36,0.36V9z"
  })), _path2 || (_path2 = /*#__PURE__*/React.createElement("path", {
    d: "M16.36 31L15.64 31 15.64 9.869 10.254 15.254 9.746 14.746 16 8.491 22.255 14.746 21.745 15.254 16.36 9.869z"
  })), children);
});

export { Upload_01 as default };
