// @carbon/pictograms-react/es/q-q-plot/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const QQPlot = /*#__PURE__*/React.forwardRef(function QQPlot(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M11,12c0,0.552-0.447,1-1,1c-0.553,0-1-0.448-1-1s0.447-1,1-1S11,11.448,11,12 M15,8c0-0.552-0.447-1-1-1\tc-0.553,0-1,0.448-1,1s0.447,1,1,1S15,8.552,15,8 M21,7c0-0.552-0.447-1-1-1s-1,0.448-1,1s0.447,1,1,1S21,7.552,21,7 M10,18\tc0-0.552-0.447-1-1-1c-0.553,0-1,0.448-1,1s0.447,1,1,1S10,18.552,10,18 M13,25c0-0.552-0.447-1-1-1c-0.553,0-1,0.448-1,1\ts0.447,1,1,1S13,25.552,13,25 M28,10c0-0.552-0.447-1-1-1s-1,0.448-1,1s0.447,1,1,1S28,10.552,28,10 M31,31.36H1\tc-0.199,0-0.36-0.161-0.36-0.36V1h0.72v29.64H31V31.36z M4.254,28.255l-0.509-0.51l25-25l0.51,0.509L4.254,28.255z"
  })), children);
});

export { QQPlot as default };
