// @carbon/pictograms-react/es/systems-devops--release/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const SystemsDevopsRelease = /*#__PURE__*/React.forwardRef(function SystemsDevopsRelease(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M23.86,25h-0.72v-8.14l-3.449,3.397l-0.506-0.514l4.315-4.249l4.314,4.249l-0.506,0.514\tL23.86,16.86V25z M16.181,29.689L4.36,22.793V9H3.64v14c0,0.128,0.068,0.246,0.179,0.311l12,7L16.181,29.689z M28.36,28v-5h-0.72\tv4.64h-8.28V23h-0.72v5c0,0.199,0.161,0.36,0.36,0.36h9C28.199,28.36,28.36,28.199,28.36,28z M8.36,6H7.64v10h0.72V6z M28.36,9\tc0-0.128-0.068-0.247-0.179-0.311l-12-7c-0.111-0.065-0.25-0.065-0.362,0l-4,2.325c-0.111,0.064-0.179,0.183-0.179,0.311V13h0.72\tV4.532L16,2.417l11.64,6.79V14h0.721L28.36,9L28.36,9z"
  })), children);
});

export { SystemsDevopsRelease as default };
