// @carbon/pictograms-react/lib/supply--chain--optimization--01/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const SupplyChainOptimization_01 = /*#__PURE__*/React__default["default"].forwardRef(function SupplyChainOptimization_01(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M10,31.36H2v-0.72h8V31.36z M11.36,30h-0.72v-8h0.72V30z M1.36,30H0.64v-8h0.72 V30z M20,21.36h-8v-0.72h8V21.36z M10,21.36H2v-0.72h8V21.36z M21.36,20h-0.72v-8h0.721L21.36,20L21.36,20z M11.36,20h-0.72v-8 h0.72V20z M15,18.36c-0.092,0-0.184-0.035-0.254-0.105l-2-2l0.509-0.509L15,17.491l3.745-3.746l0.51,0.509l-4,4 C15.184,18.325,15.092,18.36,15,18.36z M30,11.36h-8v-0.72h8V11.36z M20,11.36h-8v-0.72h8V11.36z M31.36,10h-0.72V2h0.721L31.36,10 L31.36,10z M21.36,10h-0.72V2h0.721L21.36,10L21.36,10z M30,1.36h-8V0.64h8V1.36z"
  })), children);
});

module.exports = SupplyChainOptimization_01;
