// @carbon/pictograms-react/lib/scale--02/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Scale_02 = /*#__PURE__*/React__default["default"].forwardRef(function Scale_02(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M10,31.36H1c-0.199,0-0.36-0.161-0.36-0.36v-9\tc0-0.154,0.097-0.286,0.234-0.337l9.99-3.996l4.788-16.758C15.692,0.754,15.833,0.64,16,0.64h15c0.199,0,0.36,0.161,0.36,0.36v15\tc0,0.167-0.114,0.308-0.269,0.348l-16.759,4.789l-3.996,9.989C10.286,31.263,10.154,31.36,10,31.36z M1.36,30.64h8.28v-8.28H1.36\tV30.64z M10.36,22.271v6.859l3.098-7.744L10.36,22.271z M2.869,21.64h6.859l0.885-3.098L2.869,21.64z M11.459,18.204l-0.935,3.271\tl3.271-0.935l1.558-3.895L11.459,18.204z M16.244,16.36l-1.572,3.931L28.43,16.36H16.244z M15.64,3.57l-3.931,13.759l3.931-1.573\tV3.57z M16.36,15.64h14.28V1.36H16.36V15.64z"
  })), children);
});

module.exports = Scale_02;
