// @carbon/pictograms-react/lib/dining/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Dining = /*#__PURE__*/React__default["default"].forwardRef(function Dining(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M23,31.36h-3c-0.199,0-0.36-0.161-0.36-0.36V1c0-0.199,0.161-0.36,0.36-0.36h2c1.853,0,3.36,1.507,3.36,3.36\tv12c0,0.199-0.161,0.36-0.36,0.36h-1.64V31C23.36,31.199,23.199,31.36,23,31.36z M20.36,30.64h2.279V16\tc0-0.199,0.161-0.36,0.36-0.36h1.64V4c0-1.456-1.184-2.64-2.64-2.64h-1.64L20.36,30.64L20.36,30.64z M12,31.36H9\tc-0.199,0-0.36-0.161-0.36-0.36V13.36H8c-1.301,0-2.36-1.059-2.36-2.36V1h0.72v10c0,0.904,0.736,1.64,1.64,1.64h1\tc0.199,0,0.36,0.161,0.36,0.36v17.64h2.28V13c0-0.199,0.161-0.36,0.36-0.36h1c0.904,0,1.64-0.736,1.64-1.64V1h0.72v10\tc0,1.301-1.059,2.36-2.36,2.36h-0.64V31C12.36,31.199,12.199,31.36,12,31.36z M12.36,9h-0.72V1h0.72V9z M9.36,9H8.64V1h0.72\tC9.36,1,9.36,9,9.36,9z"
  })), children);
});

module.exports = Dining;
