// @carbon/pictograms-react/lib/golf--bag/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const GolfBag = /*#__PURE__*/React__default["default"].forwardRef(function GolfBag(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M18,31.36h-6c-1.301,0-2.36-1.059-2.36-2.36V12\tc0-0.199,0.161-0.36,0.36-0.36h1.64V9.36H9C8.25,9.36,7.64,8.75,7.64,8V7c0-0.421,0.19-0.812,0.522-1.071\tc0.333-0.26,0.757-0.35,1.167-0.248l2,0.5c0.606,0.152,1.03,0.694,1.03,1.319v4.14h2.28V5.36H12c-0.75,0-1.36-0.61-1.36-1.36V3\tc0-0.421,0.19-0.812,0.522-1.071c0.333-0.26,0.756-0.35,1.167-0.248l2,0.5c0.606,0.152,1.03,0.694,1.03,1.319v8.14h2.28V2.5\tc0-0.625,0.424-1.168,1.031-1.319l2-0.5c0.411-0.104,0.835-0.011,1.166,0.247C22.169,1.188,22.36,1.579,22.36,2v1\tc0,0.75-0.61,1.36-1.36,1.36h-2.64v7.28H20c0.199,0,0.36,0.161,0.36,0.36v2.64H22c1.302,0,2.36,1.059,2.36,2.36v8\tc0,1.302-1.059,2.36-2.36,2.36h-1.64V29C20.36,30.302,19.302,31.36,18,31.36z M10.36,27.36V29c0,0.904,0.736,1.64,1.64,1.64h6\tc0.904,0,1.64-0.735,1.64-1.64v-1.64H10.36z M20.36,26.64H22c0.904,0,1.64-0.735,1.64-1.64v-8c0-0.904-0.735-1.64-1.64-1.64h-1.64\tV26.64z M10.36,26.64h9.28V15.36h-9.28V26.64z M10.36,14.64h9.28v-2.28h-9.28V14.64z M8.999,6.36c-0.141,0-0.278,0.046-0.393,0.136\tC8.45,6.618,8.36,6.802,8.36,7v1c0,0.353,0.287,0.64,0.64,0.64h2.64V7.5c0-0.294-0.2-0.549-0.485-0.621l-2-0.5\tC9.104,6.367,9.051,6.36,8.999,6.36z M11.999,2.36c-0.141,0-0.278,0.046-0.393,0.136C11.45,2.618,11.36,2.802,11.36,3v1\tc0,0.353,0.287,0.64,0.64,0.64h2.64V3.5c0-0.294-0.2-0.549-0.485-0.621l-2-0.5C12.104,2.367,12.051,2.36,11.999,2.36z M18.36,3.64\tH21c0.353,0,0.64-0.287,0.64-0.64V2c0-0.198-0.089-0.382-0.246-0.504c-0.155-0.123-0.355-0.163-0.549-0.117l-2,0.5\tC18.56,1.951,18.36,2.206,18.36,2.5V3.64z"
  })), children);
});

module.exports = GolfBag;
