// @carbon/pictograms-react/es/rich--text--format/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const RichTextFormat = /*#__PURE__*/React.forwardRef(function RichTextFormat(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M31,31.36H7c-0.199,0-0.36-0.161-0.36-0.36v-7.64H1c-0.199,0-0.36-0.161-0.36-0.36V8\tc0-0.199,0.161-0.36,0.36-0.36h5.64V1c0-0.199,0.161-0.36,0.36-0.36h18c0.096,0,0.188,0.038,0.255,0.105l6,6\tC31.322,6.813,31.36,6.904,31.36,7v24C31.36,31.199,31.199,31.36,31,31.36z M7.36,30.64h23.28V7.36H25\tc-0.199,0-0.36-0.161-0.36-0.36V1.36H7.36v6.28H16c0.199,0,0.36,0.161,0.36,0.36v2.641H27v0.72H16.36v3.28H27v0.72H16.36v3.279H27\tv0.721H16.36v3.279H27v0.721H7.36V30.64z M1.36,22.64h14.28v-2.49L12,16.51l-1.746,1.745c-0.141,0.141-0.368,0.141-0.509,0L5,13.51\tl-3.64,3.64C1.36,17.15,1.36,22.64,1.36,22.64z M12,15.641c0.092,0,0.184,0.035,0.254,0.105l3.386,3.385V8.36H1.36v7.771\tl3.386-3.385c0.141-0.141,0.368-0.141,0.509,0L10,17.491l1.746-1.745C11.816,15.676,11.908,15.641,12,15.641z M25.36,6.64h4.771\tL25.36,1.869V6.64z M23,27.36H11v-0.72h12V27.36z M10.5,13.36c-1.025,0-1.86-0.834-1.86-1.86s0.834-1.86,1.86-1.86\ts1.86,0.834,1.86,1.86S11.525,13.36,10.5,13.36z M10.5,10.36c-0.628,0-1.14,0.512-1.14,1.14s0.512,1.14,1.14,1.14\ts1.14-0.512,1.14-1.14S11.128,10.36,10.5,10.36z"
  })), children);
});

export { RichTextFormat as default };
