// @carbon/pictograms-react/lib/football--goal-post/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const FootballGoalPost = /*#__PURE__*/React__default["default"].forwardRef(function FootballGoalPost(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31,27.64H18.36V23\tc0-0.199-0.161-0.36-0.36-0.36h-1.64v-3.28H24c0.199,0,0.36-0.161,0.36-0.36V1h-0.72v17.64H8.36V1H7.64v18\tc0,0.199,0.161,0.36,0.36,0.36h7.64v3.279H14c-0.199,0-0.36,0.161-0.36,0.36v4.64H1v0.721h12.64V31c0,0.199,0.161,0.36,0.36,0.36h4\tc0.199,0,0.36-0.161,0.36-0.36v-2.64H31V27.64z M17.64,30.64h-3.28v-7.28h3.28V30.64z"
  })), children);
});

module.exports = FootballGoalPost;
