// @carbon/pictograms-react/lib/curve--logarithmic/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const CurveLogarithmic = /*#__PURE__*/React__default["default"].forwardRef(function CurveLogarithmic(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16.36,31h-0.72V16.36H1v-0.72h14.64V1h0.72\tv14.64h2.202c1.325-4.728,4.558-8,12.438-8v0.72c-7.321,0-10.402,2.897-11.698,7.28H31v0.72H19.107\tC18.36,19.413,18.36,23.103,18.36,27h-0.72c0-3.789,0-7.499,0.737-10.64H16.36V31z"
  })), children);
});

module.exports = CurveLogarithmic;
