// @carbon/pictograms-react/es/integration/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Integration = /*#__PURE__*/React.forwardRef(function Integration(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M23.5,29.36h-15c-0.129,0-0.248-0.069-0.312-0.181l-7.5-13.01c-0.064-0.111-0.064-0.249,0-0.36\tl7.5-12.99C8.252,2.708,8.372,2.64,8.5,2.64h15c0.129,0,0.247,0.068,0.312,0.18l7.5,12.99c0.064,0.111,0.064,0.249,0,0.36\tl-7.5,13.01C23.748,29.291,23.629,29.36,23.5,29.36z M9.124,28.64h13.753L16,16.72L9.124,28.64z M16.618,16.35L23.5,28.279\tl6.877-11.929C30.377,16.35,16.618,16.35,16.618,16.35z M1.623,16.35L8.5,28.279l6.882-11.929C15.382,16.35,1.623,16.35,1.623,16.35\tz M16.629,15.63h13.747L23.5,3.72L16.629,15.63z M1.624,15.63h13.748L8.5,3.72L1.624,15.63z M9.124,3.36L16,15.28l6.877-11.92\tC22.877,3.36,9.124,3.36,9.124,3.36z"
  })), children);
});

export { Integration as default };
