// @carbon/pictograms-react/es/green-it--02/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const GreenIt_02 = /*#__PURE__*/React.forwardRef(function GreenIt_02(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M16,0.64C7.53,0.64,0.64,7.53,0.64,16\tS7.53,31.36,16,31.36S31.36,24.47,31.36,16S24.47,0.64,16,0.64z M1.378,16.36H7.65c0.034,2.646,0.432,5.125,1.108,7.28H3.535\tC2.224,21.509,1.443,19.024,1.378,16.36z M8.369,16.36h7.271v7.28H9.495C8.811,21.509,8.403,19.024,8.369,16.36z M15.64,24.36v6.244\tc-2.441-0.219-4.586-2.632-5.903-6.244H15.64z M16.36,24.36h5.903c-1.317,3.612-3.461,6.025-5.903,6.244V24.36z M16.36,23.64v-7.28\th7.27c-0.034,2.665-0.441,5.149-1.126,7.28H16.36z M24.35,16.36h6.271c-0.065,2.665-0.846,5.149-2.157,7.28h-5.222\tC23.919,21.485,24.317,19.006,24.35,16.36z M16,1.36c7.951,0,14.428,6.375,14.621,14.28H16.36V13v-1.64H18\tc2.2,0,4.36-2.16,4.36-4.36V5c0-0.199-0.161-0.36-0.36-0.36h-2c-2.142,0-4.229,2.048-4.339,4.186C14.849,7.571,13.44,6.64,12,6.64\th-2C9.801,6.64,9.64,6.801,9.64,7v2c0,2.2,2.16,4.36,4.36,4.36h1.64v2.28H1.378C1.572,7.735,8.049,1.36,16,1.36z M18.745,7.746\tl-2.385,2.386V9c0-1.803,1.837-3.64,3.64-3.64h1.64V7c0,1.803-1.837,3.64-3.64,3.64h-1.131l2.386-2.386L18.745,7.746z M15.64,12.131\tl-2.386-2.386l-0.509,0.509l2.386,2.386H14c-1.803,0-3.64-1.837-3.64-3.64V7.36H12c1.803,0,3.64,1.837,3.64,3.64V12.131z M3.999,24.36h4.997c0.957,2.698,2.363,4.803,4.022,5.972C9.301,29.56,6.102,27.37,3.999,24.36z M18.982,30.333\tc1.659-1.169,3.064-3.274,4.022-5.972h4.997C25.897,27.37,22.699,29.56,18.982,30.333z"
  })), children);
});

export { GreenIt_02 as default };
