// @carbon/pictograms-react/es/select--range/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const SelectRange = /*#__PURE__*/React.forwardRef(function SelectRange(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M21,27.36H11v-0.72h10V27.36z M28.36,26h-0.72v-3c0-0.199,0.161-0.36,0.36-0.36h3v0.721h-2.64V26z M26,23.36H4c-0.199,0-0.36-0.161-0.36-0.36V8c0-0.199,0.161-0.36,0.36-0.36h24c0.199,0,0.36,0.161,0.36,0.36v13h-0.72V8.36H4.36\tv14.28H26V23.36z M21,18.36H11v-0.72h10V18.36z M21,13.36H11v-0.72h10V13.36z M21,4.36H11V3.64h10V4.36z"
  })), children);
});

export { SelectRange as default };
