// @carbon/pictograms-react/lib/back-up--and-restore/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const BackUpAndRestore = /*#__PURE__*/React__default["default"].forwardRef(function BackUpAndRestore(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31.36,16c0,8.47-6.891,15.36-15.36,15.36\tS0.64,24.47,0.64,16c0-7.276,5.168-13.606,12.288-15.052l0.144,0.706C6.286,3.032,1.36,9.065,1.36,16\tc0,8.072,6.567,14.64,14.64,14.64c8.072,0,14.64-6.567,14.64-14.64c0-6.935-4.925-12.968-11.711-14.346l0.143-0.706\tC26.192,2.394,31.36,8.724,31.36,16z M11.254,18.745l-0.509,0.51L16,24.509l5.255-5.254l-0.51-0.51l-4.385,4.386V1h-0.72v22.131\tL11.254,18.745z"
  })), children);
});

module.exports = BackUpAndRestore;
