// @carbon/pictograms-react/lib/scan-code/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const ScanCode = /*#__PURE__*/React__default["default"].forwardRef(function ScanCode(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31,31.36H21v-0.72h9.64V21h0.721v10\tC31.36,31.199,31.199,31.36,31,31.36z M11,31.36H1c-0.199,0-0.36-0.161-0.36-0.36V21h0.72v9.64H11V31.36z M25,25.36H7\tc-0.199,0-0.36-0.161-0.36-0.36v-7h0.72v6.64h17.28V18h0.721v7C25.36,25.199,25.199,25.36,25,25.36z M31,16.36H1v-0.72h30V16.36z M24.64,14.001L24.614,7.36H7.36V14H6.64V7c0-0.199,0.161-0.36,0.36-0.36h17.973c0.198,0,0.359,0.16,0.36,0.358l0.027,7\tL24.64,14.001z M31.36,11h-0.72V1.36H21V0.64h10c0.199,0,0.36,0.161,0.36,0.36V11z M1.36,11H0.64V1c0-0.199,0.161-0.36,0.36-0.36h10\tv0.72H1.36V11z"
  })), children);
});

module.exports = ScanCode;
