// @carbon/pictograms-react/lib/compost--bin/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const CompostBin = /*#__PURE__*/React__default["default"].forwardRef(function CompostBin(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M26,31.36H6c-0.199,0-0.36-0.161-0.36-0.36V5.36H2V4.64h8.64V1c0-0.199,0.161-0.36,0.36-0.36h10\tc0.199,0,0.36,0.161,0.36,0.36v3.64H30v0.72h-3.64V31C26.36,31.199,26.199,31.36,26,31.36z M6.36,30.64h19.28V5.36H6.36V30.64z M11.36,4.64h9.28V1.36h-9.28V4.64z M22,25.36H10v-0.72h5.64v-4.28H14c-2.404,0-4.36-1.956-4.36-4.36v-2\tc0-0.199,0.161-0.36,0.36-0.36h2c1.529,0,2.877,0.792,3.656,1.986c0.19-2.23,2.066-3.986,4.344-3.986h2\tc0.199,0,0.36,0.161,0.36,0.36v2c0,2.404-1.956,4.36-4.36,4.36h-1.64v6.279H22V25.36z M10.36,14.36V16c0,2.007,1.633,3.64,3.64,3.64\th1.131l-2.385-2.385l0.509-0.509l2.386,2.385V18c0-2.007-1.633-3.64-3.64-3.64H10.36z M16.869,17.64H18\tc2.007,0,3.64-1.632,3.64-3.64v-1.64H20c-2.007,0-3.64,1.633-3.64,3.64v1.131l2.385-2.385l0.51,0.509L16.869,17.64z"
  })), children);
});

module.exports = CompostBin;
