// @carbon/pictograms-react/es/proven-technology/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const ProvenTechnology = /*#__PURE__*/React.forwardRef(function ProvenTechnology(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M16,31.36C7.53,31.36,0.64,24.47,0.64,16\tS7.53,0.64,16,0.64S31.36,7.53,31.36,16S24.47,31.36,16,31.36z M16,1.36C7.927,1.36,1.36,7.927,1.36,16\tc0,8.072,6.567,14.64,14.64,14.64c8.072,0,14.64-6.567,14.64-14.64S24.072,1.36,16,1.36z M9,26.36c-0.076,0-0.151-0.024-0.215-0.071\tc-0.125-0.094-0.176-0.258-0.125-0.406l2.663-7.75l-6.537-4.844c-0.125-0.092-0.176-0.254-0.127-0.402S4.845,12.64,5,12.64h8.113\tl2.545-7.752c0.097-0.296,0.587-0.296,0.684,0l2.545,7.752H27c0.155,0,0.293,0.1,0.342,0.247s-0.003,0.31-0.128,0.402l-6.537,4.844\tl2.664,7.75c0.051,0.148,0,0.312-0.126,0.406c-0.123,0.093-0.296,0.096-0.424,0.004L16,21.442l-6.791,4.851\tC9.146,26.338,9.073,26.36,9,26.36z M6.091,13.36l5.872,4.351c0.126,0.093,0.177,0.258,0.126,0.406l-2.383,6.937l6.084-4.347\tc0.126-0.09,0.293-0.09,0.419,0l6.084,4.347l-2.384-6.937c-0.051-0.148,0-0.313,0.127-0.406l5.872-4.351h-7.283\tc-0.155,0-0.294-0.1-0.342-0.248L16,6.154l-2.285,6.958c-0.048,0.148-0.186,0.248-0.342,0.248H6.091z"
  })), children);
});

export { ProvenTechnology as default };
