// @carbon/pictograms-react/lib/dedicated--instance/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const DedicatedInstance = /*#__PURE__*/React__default["default"].forwardRef(function DedicatedInstance(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M30,31.36H2c-0.199,0-0.36-0.161-0.36-0.36\tv-8c0-0.199,0.161-0.36,0.36-0.36h28c0.199,0,0.36,0.161,0.36,0.36v2.688c0.575,0.158,1,0.687,1,1.312s-0.425,1.153-1,1.312V31\tC30.36,31.199,30.199,31.36,30,31.36z M2.36,30.64h27.28v-2.328c-0.461-0.127-0.824-0.49-0.951-0.951H6V26.64h22.688\tc0.127-0.461,0.49-0.824,0.951-0.951V23.36H2.36V30.64z M30,26.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64\ts0.64-0.287,0.64-0.64S30.353,26.36,30,26.36z M30,20.36H2v-0.72h27.64v-7.28H2v-0.72h28c0.199,0,0.36,0.161,0.36,0.36v8\tC30.36,20.199,30.199,20.36,30,20.36z M2,17.36c-0.75,0-1.36-0.61-1.36-1.36S1.25,14.64,2,14.64c0.625,0,1.153,0.424,1.312,1H26\tv0.72H3.312C3.153,16.936,2.625,17.36,2,17.36z M2,15.36c-0.353,0-0.64,0.287-0.64,0.64S1.647,16.64,2,16.64S2.64,16.353,2.64,16\tS2.353,15.36,2,15.36z M30,9.36H2C1.801,9.36,1.64,9.199,1.64,9V1c0-0.199,0.161-0.36,0.36-0.36h28c0.199,0,0.36,0.161,0.36,0.36\tv2.688c0.575,0.158,1,0.686,1,1.312s-0.425,1.153-1,1.312V9C30.36,9.199,30.199,9.36,30,9.36z M2.36,8.64h27.28V6.312\tc-0.461-0.126-0.824-0.491-0.951-0.952H6V4.64h22.688c0.127-0.461,0.49-0.825,0.951-0.952V1.36H2.36V8.64z M30,4.36\tc-0.353,0-0.64,0.287-0.64,0.64S29.647,5.64,30,5.64S30.64,5.353,30.64,5S30.353,4.36,30,4.36z"
  })), children);
});

module.exports = DedicatedInstance;
