// @carbon/pictograms-react/lib/badge/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Badge = /*#__PURE__*/React__default["default"].forwardRef(function Badge(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M15,19.509l-3.254-3.254l0.509-0.509L15,18.491l5.745-5.746l0.51,0.509L15,19.509z M23.314,28.36H31\tc0.119,0,0.231-0.06,0.298-0.158c0.067-0.1,0.081-0.225,0.036-0.336L30.388,25.5l0.946-2.366c0.045-0.111,0.031-0.236-0.036-0.336\tC31.231,22.699,31.119,22.64,31,22.64h-2.265C29.8,20.601,30.36,18.317,30.36,16c0-7.918-6.442-14.36-14.36-14.36\tS1.64,8.082,1.64,16c0,2.317,0.56,4.601,1.625,6.64H1c-0.12,0-0.231,0.06-0.298,0.158c-0.067,0.1-0.08,0.225-0.036,0.336L1.612,25.5\tl-0.947,2.366c-0.044,0.111-0.031,0.236,0.036,0.336C0.769,28.301,0.88,28.36,1,28.36h7.686c2.208,1.309,4.734,2,7.314,2\tS21.106,29.669,23.314,28.36z M21.775,28.36c-3.598,1.68-7.953,1.68-11.55,0H21.775z M2.334,25.634c0.034-0.086,0.034-0.182,0-0.268\tL1.532,23.36h28.937l-0.803,2.006c-0.034,0.086-0.034,0.182,0,0.268l0.803,2.006H1.532L2.334,25.634z M27.36,16\tc0-6.264-5.097-11.36-11.36-11.36C9.736,4.64,4.64,9.736,4.64,16c0,2.385,0.756,4.715,2.142,6.64h-2.7\tC2.955,20.62,2.36,18.328,2.36,16C2.36,8.479,8.479,2.36,16,2.36S29.64,8.479,29.64,16c0,2.328-0.595,4.62-1.723,6.64h-2.699\tC26.604,20.716,27.36,18.385,27.36,16z M5.36,16c0-5.867,4.773-10.64,10.64-10.64S26.64,10.133,26.64,16\tc0,2.411-0.824,4.762-2.325,6.64H7.686C6.185,20.762,5.36,18.41,5.36,16z"
  })), children);
});

module.exports = Badge;
