// @carbon/pictograms-react/es/bare-metal/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const BareMetal = /*#__PURE__*/React.forwardRef(function BareMetal(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M31,31.36H1c-0.199,0-0.36-0.161-0.36-0.36V1\tc0-0.199,0.161-0.36,0.36-0.36h10c0.199,0,0.36,0.161,0.36,0.36v9.64H21c0.199,0,0.36,0.161,0.36,0.36v9.64H31\tc0.199,0,0.36,0.161,0.36,0.36v10C31.36,31.199,31.199,31.36,31,31.36z M21.36,30.64h9.279v-9.28H21.36V30.64z M11.36,30.64h9.28\tV11.36h-9.28V30.64z M1.36,30.64h9.28V1.36H1.36V30.64z M27,27.36h-2c-0.199,0-0.36-0.161-0.36-0.36v-2\tc0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2C27.36,27.199,27.199,27.36,27,27.36z M25.36,26.64h1.279v-1.28\tH25.36V26.64z M17,27.36h-2c-0.199,0-0.36-0.161-0.36-0.36v-2c0-0.199,0.161-0.36,0.36-0.36h0.64v-7.28H15\tc-0.199,0-0.36-0.161-0.36-0.36v-2c0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2c0,0.199-0.161,0.36-0.36,0.36\th-0.64v7.279H17c0.199,0,0.36,0.161,0.36,0.36v2C17.36,27.199,17.199,27.36,17,27.36z M15.36,26.64h1.28v-1.28h-1.28V26.64z M15.36,16.64h1.28v-1.28h-1.28V16.64z M7,27.36H5c-0.199,0-0.36-0.161-0.36-0.36v-2c0-0.199,0.161-0.36,0.36-0.36h0.64v-7.28H5\tc-0.199,0-0.36-0.161-0.36-0.36v-2c0-0.199,0.161-0.36,0.36-0.36h0.64V7.36H5C4.801,7.36,4.64,7.199,4.64,7V5\tc0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2c0,0.199-0.161,0.36-0.36,0.36H6.36v7.28H7\tc0.199,0,0.36,0.161,0.36,0.36v2c0,0.199-0.161,0.36-0.36,0.36H6.36v7.279H7c0.199,0,0.36,0.161,0.36,0.36v2\tC7.36,27.199,7.199,27.36,7,27.36z M5.36,26.64h1.28v-1.28H5.36V26.64z M5.36,16.64h1.28v-1.28H5.36V16.64z M5.36,6.64h1.28V5.36\tH5.36V6.64z"
  })), children);
});

export { BareMetal as default };
