// @carbon/pictograms-react/lib/e-commerce/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const ECommerce = /*#__PURE__*/React__default["default"].forwardRef(function ECommerce(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M25,30.36H7c-0.17,0-0.316-0.119-0.352-0.285l-3-14\tc-0.023-0.106,0.004-0.217,0.072-0.302C3.789,15.689,3.891,15.64,4,15.64h11.64V6h0.72v9.64H28c0.108,0,0.212,0.049,0.28,0.133\tc0.068,0.085,0.095,0.196,0.071,0.302l-3,14C25.316,30.241,25.17,30.36,25,30.36z M7.291,29.64h17.418l2.846-13.28H16.36v6.329\tc0.576,0.158,1,0.687,1,1.312c0,0.75-0.61,1.36-1.36,1.36s-1.36-0.61-1.36-1.36c0-0.625,0.424-1.153,1-1.312V16.36H4.445\tL7.291,29.64z M16,23.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S16.353,23.36,16,23.36z M23.36,14h-0.72V5c0-0.353-0.287-0.64-0.64-0.64H10C9.647,4.36,9.36,4.647,9.36,5v9H8.64V5c0-0.75,0.61-1.36,1.36-1.36h12\tc0.75,0,1.36,0.61,1.36,1.36V14z"
  })), children);
});

module.exports = ECommerce;
