// @carbon/pictograms-react/lib/fault-tolerant/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const FaultTolerant = /*#__PURE__*/React__default["default"].forwardRef(function FaultTolerant(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,31.515c-0.062,0-0.125-0.017-0.18-0.049\tl-1-0.577c-0.111-0.064-0.18-0.183-0.18-0.312v-0.953L3.818,23.311C3.708,23.246,3.64,23.128,3.64,23V10.363L2.82,9.889\tC2.708,9.825,2.64,9.706,2.64,9.577V8.423c0-0.128,0.068-0.248,0.18-0.312l1-0.578c0.111-0.064,0.249-0.064,0.36,0l0.813,0.47\tl10.825-6.314c0.111-0.065,0.251-0.065,0.362,0l10.826,6.314l0.813-0.47c0.111-0.064,0.248-0.064,0.359,0l1,0.578\tc0.111,0.064,0.181,0.183,0.181,0.312v1.154c0,0.128-0.069,0.248-0.181,0.312l-0.819,0.474V23c0,0.128-0.068,0.246-0.179,0.311\tl-10.82,6.313v0.953c0,0.129-0.069,0.247-0.181,0.312l-1,0.577C16.125,31.498,16.062,31.515,16,31.515z M15.36,30.369L16,30.738\tl0.64-0.369v-0.738L16,29.262l-0.64,0.369V30.369z M16,28.485c0.062,0,0.125,0.017,0.18,0.049l0.814,0.47l4.646-2.711v-6.586\tl-4.646-2.71l-0.813,0.469c-0.111,0.064-0.249,0.064-0.36,0l-0.813-0.469l-4.646,2.71v6.586l4.646,2.711l0.814-0.47\tC15.875,28.502,15.938,28.485,16,28.485z M17.36,16.376l4.821,2.813c0.11,0.064,0.179,0.183,0.179,0.311v6.373l5.279-3.08v-12.43\tL26.82,9.889c-0.111-0.064-0.181-0.183-0.181-0.312V8.623L22.36,6.126V12.5c0,0.128-0.068,0.247-0.179,0.311l-4.821,2.812V16.376z M4.36,22.793l5.28,3.08V19.5c0-0.128,0.068-0.246,0.179-0.311l4.821-2.813v-0.753l-4.821-2.812C9.708,12.747,9.64,12.628,9.64,12.5\tV6.126L5.36,8.624v0.954c0,0.128-0.068,0.248-0.18,0.312l-0.82,0.474V22.793z M15.36,16.369l0.64,0.37l0.64-0.37v-0.738L16,15.261\tl-0.64,0.37V16.369z M16,14.485c0.062,0,0.125,0.016,0.18,0.048l0.813,0.47l4.646-2.71V5.707L16,2.417l-5.64,3.29v6.587l4.646,2.71\tl0.813-0.47C15.875,14.501,15.938,14.485,16,14.485z M27.36,9.369L28,9.739l0.64-0.37V8.631L28,8.261l-0.64,0.37V9.369z M3.36,9.369\tL4,9.739l0.64-0.37V8.631L4,8.261l-0.64,0.37C3.36,8.631,3.36,9.369,3.36,9.369z"
  })), children);
});

module.exports = FaultTolerant;
