// @carbon/pictograms-react/es/custom-reports/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const CustomReports = /*#__PURE__*/React.forwardRef(function CustomReports(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M30,30.36H2c-0.75,0-1.36-0.61-1.36-1.36V7\tc0-0.75,0.61-1.36,1.36-1.36h8.5c0.426,0,0.833,0.203,1.088,0.544l1.416,2.223C13.108,8.544,13.299,8.64,13.5,8.64H30\tc0.75,0,1.36,0.61,1.36,1.36v19C31.36,29.75,30.75,30.36,30,30.36z M24.36,29.64H30c0.353,0,0.64-0.287,0.64-0.64V10\tc0-0.353-0.287-0.64-0.64-0.64H13.5c-0.426,0-0.833-0.203-1.088-0.544l-1.416-2.223C10.892,6.456,10.701,6.36,10.5,6.36H2\tC1.647,6.36,1.36,6.647,1.36,7v22c0,0.353,0.287,0.64,0.64,0.64h5.64V13.312c-0.576-0.158-1-0.686-1-1.312\tc0-0.75,0.61-1.36,1.36-1.36S9.36,11.25,9.36,12c0,0.625-0.424,1.153-1,1.312V29.64h7.28V17.312c-0.576-0.158-1-0.686-1-1.312\tc0-0.75,0.61-1.36,1.36-1.36c0.75,0,1.36,0.61,1.36,1.36c0,0.625-0.425,1.153-1,1.312V29.64h7.28v-8.328\tc-0.575-0.158-1-0.687-1-1.312c0-0.75,0.61-1.36,1.36-1.36s1.36,0.61,1.36,1.36c0,0.625-0.425,1.153-1,1.312V29.64z M24,19.36\tc-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S24.353,19.36,24,19.36z M16,15.36\tc-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S16.353,15.36,16,15.36z M8,11.36\tc-0.353,0-0.64,0.287-0.64,0.64S7.647,12.64,8,12.64c0.353,0,0.64-0.287,0.64-0.64S8.353,11.36,8,11.36z"
  })), children);
});

export { CustomReports as default };
