// @carbon/pictograms-react/es/link/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Link = /*#__PURE__*/React.forwardRef(function Link(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M8,30.36c-1.699,0-3.296-0.662-4.497-1.863\tC2.301,27.296,1.64,25.698,1.64,24s0.662-3.296,1.863-4.497l6.243-6.243l0.509,0.509l-6.243,6.244C2.947,21.077,2.36,22.493,2.36,24\ts0.587,2.923,1.652,3.987C5.077,29.053,6.494,29.64,8,29.64s2.923-0.587,3.988-1.652l6.242-6.242l0.51,0.51l-6.243,6.242\tC11.296,29.698,9.699,30.36,8,30.36z M10.254,22.255l-0.509-0.51l12-12l0.51,0.509L10.254,22.255z M22.255,18.74l-0.51-0.51\tl6.242-6.242C29.053,10.923,29.64,9.506,29.64,8s-0.587-2.923-1.652-3.988C26.923,2.947,25.507,2.36,24,2.36\ts-2.923,0.587-3.987,1.652l-6.244,6.243L13.26,9.746l6.243-6.243C20.704,2.301,22.302,1.64,24,1.64s3.296,0.662,4.497,1.863\tS30.36,6.301,30.36,8s-0.662,3.296-1.863,4.497L22.255,18.74z"
  })), children);
});

export { Link as default };
