// @carbon/pictograms-react/lib/ai--ready/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const AiReady = /*#__PURE__*/React__default["default"].forwardRef(function AiReady(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,31.36c-6.108,0-11.636-3.618-14.082-9.216\tl0.66-0.289C4.91,27.192,10.178,30.64,16,30.64c8.072,0,14.64-6.567,14.64-14.64S24.073,1.36,16,1.36C7.927,1.36,1.36,7.927,1.36,16\tc0,1.456,1.185,2.64,2.64,2.64S6.64,17.456,6.64,16c0-5.161,4.199-9.36,9.36-9.36s9.36,4.199,9.36,9.36\tc0,1.853-1.508,3.36-3.36,3.36s-3.36-1.508-3.36-3.36c0-1.456-1.184-2.64-2.64-2.64c-1.352,0-2.47,1.021-2.622,2.333\tc0.566,0.164,0.982,0.688,0.982,1.307c0,0.75-0.61,1.36-1.36,1.36s-1.36-0.61-1.36-1.36c0-0.631,0.431-1.163,1.015-1.315\tc0.16-1.706,1.599-3.044,3.345-3.044c1.853,0,3.36,1.507,3.36,3.36c0,1.456,1.184,2.64,2.64,2.64s2.64-1.184,2.64-2.64\tc0-4.764-3.876-8.64-8.64-8.64s-8.64,3.876-8.64,8.64c0,1.853-1.507,3.36-3.36,3.36S0.64,17.853,0.64,16\tC0.64,7.53,7.53,0.64,16,0.64S31.36,7.53,31.36,16S24.47,31.36,16,31.36z M13,16.36c-0.353,0-0.64,0.287-0.64,0.64\tc0,0.353,0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64C13.64,16.647,13.353,16.36,13,16.36z"
  })), children);
});

module.exports = AiReady;
