// @carbon/pictograms-react/es/security/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Security = /*#__PURE__*/React.forwardRef(function Security(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M16,31.36C9.842,31.36,4.64,26.158,4.64,20V6.786L16,0.59l11.36,6.196V20\tC27.36,26.158,22.158,31.36,16,31.36z M5.36,7.214V20c0,5.768,4.873,10.64,10.64,10.64S26.64,25.768,26.64,20V7.214L16,1.41\tL5.36,7.214z"
  })), children);
});

export { Security as default };
