// @carbon/pictograms-react/es/security--visibility/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const SecurityVisibility = /*#__PURE__*/React.forwardRef(function SecurityVisibility(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M16,31.36C8.082,31.36,1.64,24.918,1.64,17\tV1c0-0.199,0.161-0.36,0.36-0.36h28c0.199,0,0.36,0.161,0.36,0.36v16C30.36,24.918,23.918,31.36,16,31.36z M2.36,1.36V17\tc0,7.521,6.119,13.64,13.64,13.64S29.64,24.522,29.64,17V1.36H2.36z M16,22.36c-3.507,0-6.36-2.854-6.36-6.36S12.493,9.64,16,9.64\tc3.023,0,5.594,2.093,6.219,5.018c0.646,0.104,1.142,0.667,1.142,1.342c0,0.75-0.61,1.36-1.36,1.36s-1.36-0.61-1.36-1.36\tc0-0.573,0.356-1.064,0.858-1.264C20.917,12.181,18.656,10.36,16,10.36c-3.11,0-5.64,2.53-5.64,5.64s2.53,5.64,5.64,5.64\tc1.507,0,2.923-0.587,3.987-1.652l0.51,0.51C19.296,21.698,17.699,22.36,16,22.36z M22,15.36c-0.353,0-0.64,0.287-0.64,0.64\ts0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64S22.353,15.36,22,15.36z M16,17.36c-0.75,0-1.36-0.61-1.36-1.36s0.61-1.36,1.36-1.36\ts1.36,0.61,1.36,1.36S16.75,17.36,16,17.36z M16,15.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64s0.64-0.287,0.64-0.64\tS16.353,15.36,16,15.36z"
  })), children);
});

export { SecurityVisibility as default };
