// @carbon/pictograms-react/lib/small-to-medium-business-smb/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const SmallToMediumBusinessSmb = /*#__PURE__*/React__default["default"].forwardRef(function SmallToMediumBusinessSmb(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M14,31.36H2\tc-0.199,0-0.36-0.161-0.36-0.36V11c0-0.199,0.161-0.36,0.36-0.36h12c0.199,0,0.36,0.161,0.36,0.36v20\tC14.36,31.199,14.199,31.36,14,31.36z M2.36,30.64h11.28V11.36H2.36V30.64z M30,31.36H18c-0.199,0-0.36-0.161-0.36-0.36V1\tc0-0.199,0.161-0.36,0.36-0.36h12c0.199,0,0.36,0.161,0.36,0.36v30C30.36,31.199,30.199,31.36,30,31.36z M18.36,30.64h11.28V1.36\tH18.36V30.64z M6.36,14H5.64v2h0.72C6.36,16,6.36,14,6.36,14z M10.36,14H9.64v2h0.72C10.36,16,10.36,14,10.36,14z M6.36,19H5.64v2\th0.72C6.36,21,6.36,19,6.36,19z M10.36,19H9.64v2h0.72C10.36,21,10.36,19,10.36,19z M6.36,24H5.64v2h0.72C6.36,26,6.36,24,6.36,24z M10.36,24H9.64v2h0.72C10.36,26,10.36,24,10.36,24z M22.36,14h-0.72v2h0.721L22.36,14L22.36,14z M22.36,9h-0.72v2h0.721L22.36,9\tL22.36,9z M26.36,14h-0.72v2h0.721L26.36,14L26.36,14z M26.36,9h-0.72v2h0.721L26.36,9L26.36,9z M22.36,4h-0.72v2h0.721L22.36,4\tL22.36,4z M26.36,4h-0.72v2h0.721L26.36,4L26.36,4z M22.36,19h-0.72v2h0.721L22.36,19L22.36,19z M26.36,19h-0.72v2h0.721L26.36,19\tL26.36,19z M22.36,24h-0.72v2h0.721L22.36,24L22.36,24z M26.36,24h-0.72v2h0.721L26.36,24L26.36,24z"
  })), children);
});

module.exports = SmallToMediumBusinessSmb;
