// @carbon/pictograms-react/es/retail/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Retail = /*#__PURE__*/React.forwardRef(function Retail(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M31.36,29.36H0.64V10.64h10V7c0-2.956,2.404-5.36,5.36-5.36c2.955,0,5.36,2.404,5.36,5.36v3.64h10V29.36z M1.673,28.64h28.654L24.567,20l5.76-8.64H1.673L7.433,20L1.673,28.64z M25.433,20l5.207,7.811V12.189L25.433,20z M1.36,12.189\tv15.622L6.567,20L1.36,12.189z M11.36,10.64h9.28V7c0-2.559-2.081-4.64-4.64-4.64S11.36,4.441,11.36,7V10.64z"
  })), children);
});

export { Retail as default };
