// @carbon/pictograms-react/lib/chart--high-low/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const ChartHighLow = /*#__PURE__*/React__default["default"].forwardRef(function ChartHighLow(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31,31.36H1c-0.199,0-0.36-0.161-0.36-0.36V1h0.72v29.64H31V31.36z M24.36,26h-0.72v-2.64H21v-0.72\th2.64V7h0.721v1.64H27v0.72h-2.64V26z M16.36,26h-0.72v-6.64H13v-0.72h2.64V7h0.72v5.64H19v0.72h-2.64V26z M8.36,26H7.64v-4.64H5\tv-0.72h2.64V7h0.72v3.64H11v0.72H8.36V26z"
  })), children);
});

module.exports = ChartHighLow;
