// @carbon/pictograms-react/es/dev-and-test/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const DevAndTest = /*#__PURE__*/React.forwardRef(function DevAndTest(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M25.255,11.254l-0.51-0.509L13,22.491l-3.823-3.823\tC9.29,18.47,9.36,18.244,9.36,18c0-0.75-0.61-1.36-1.36-1.36S6.64,17.25,6.64,18S7.25,19.36,8,19.36c0.244,0,0.47-0.07,0.668-0.183\tL13,23.509L25.255,11.254z M8,18.64c-0.353,0-0.64-0.287-0.64-0.64S7.647,17.36,8,17.36c0.353,0,0.64,0.287,0.64,0.64\tS8.353,18.64,8,18.64z M1.409,16l6.908,12.829l-0.634,0.342l-7-13c-0.058-0.107-0.058-0.235,0-0.342l7-13l0.634,0.342L1.409,16z M31.317,16.171l-7,13l-0.635-0.342L30.591,16L23.683,3.171l0.635-0.342l7,13C31.374,15.936,31.374,16.064,31.317,16.171z"
  })), children);
});

export { DevAndTest as default };
