// @carbon/pictograms-react/es/qr-code/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const QrCode = /*#__PURE__*/React.forwardRef(function QrCode(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M13,31.36H1c-0.199,0-0.36-0.161-0.36-0.36V19c0-0.199,0.161-0.36,0.36-0.36h12\tc0.199,0,0.36,0.161,0.36,0.36v12C13.36,31.199,13.199,31.36,13,31.36z M1.36,30.64h11.28V19.36H1.36V30.64z M10,28.36H4\tc-0.199,0-0.36-0.161-0.36-0.36v-6c0-0.199,0.161-0.36,0.36-0.36h6c0.199,0,0.36,0.161,0.36,0.36v6\tC10.36,28.199,10.199,28.36,10,28.36z M4.36,27.64h5.28v-5.28H4.36V27.64z M31,13.36H19c-0.199,0-0.36-0.161-0.36-0.36V1\tc0-0.199,0.161-0.36,0.36-0.36h12c0.199,0,0.36,0.161,0.36,0.36v12C31.36,13.199,31.199,13.36,31,13.36z M19.36,12.64h11.28V1.36\tH19.36V12.64z M13,13.36H1c-0.199,0-0.36-0.161-0.36-0.36V1c0-0.199,0.161-0.36,0.36-0.36h12c0.199,0,0.36,0.161,0.36,0.36v12\tC13.36,13.199,13.199,13.36,13,13.36z M1.36,12.64h11.28V1.36H1.36V12.64z M28,10.36h-6c-0.199,0-0.36-0.161-0.36-0.36V4\tc0-0.199,0.161-0.36,0.36-0.36h6c0.199,0,0.36,0.161,0.36,0.36v6C28.36,10.199,28.199,10.36,28,10.36z M22.36,9.64h5.279V4.36H22.36\tV9.64z M10,10.36H4c-0.199,0-0.36-0.161-0.36-0.36V4c0-0.199,0.161-0.36,0.36-0.36h6c0.199,0,0.36,0.161,0.36,0.36v6\tC10.36,10.199,10.199,10.36,10,10.36z M4.36,9.64h5.28V4.36H4.36V9.64z M23,31h-2v-2h2V31z M29,19h-2v2h2V19z M31,21h-2v2h2V21z M31,27h-2v2h2V27z M29,23h-2v4h2V23z M23,19h-4v2h4V19z M27,29v-2h-2v4h4v-2H27z M21,23h-2v2h2V23z M21,27h-2v2h2V27z M27,21h-4v4\th-2v2h4v-4h2V21z"
  })), children);
});

export { QrCode as default };
