// @carbon/pictograms-react/es/strategy--direct/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const StrategyDirect = /*#__PURE__*/React.forwardRef(function StrategyDirect(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M27,31.36H5c-0.199,0-0.36-0.161-0.36-0.36v-6c0-0.199,0.161-0.36,0.36-0.36h2.662L8.617,9.36H7\tC6.801,9.36,6.64,9.199,6.64,9V1c0-0.199,0.161-0.36,0.36-0.36h4c0.199,0,0.36,0.161,0.36,0.36v1.64h2.28V1\tc0-0.199,0.161-0.36,0.36-0.36h4c0.199,0,0.36,0.161,0.36,0.36v1.64h2.279V1c0-0.199,0.161-0.36,0.36-0.36h4\tc0.199,0,0.36,0.161,0.36,0.36v8c0,0.199-0.161,0.36-0.36,0.36h-1.617l0.955,15.28H27c0.199,0,0.36,0.161,0.36,0.36v6\tC27.36,31.199,27.199,31.36,27,31.36z M5.36,30.64h21.28v-5.28H5.36V30.64z M8.383,24.64h15.234l-0.267-4.28H21v-0.72h2.305\tL23.1,16.36H16v-0.72h7.055l-0.205-3.28H19v-0.72h3.805l-0.143-2.28H9.338L8.571,21.64H12v0.721H8.526L8.383,24.64z M7.36,8.64\th17.28V1.36h-3.28V3c0,0.199-0.161,0.36-0.36,0.36h-3c-0.199,0-0.36-0.161-0.36-0.36V1.36h-3.28V3c0,0.199-0.161,0.36-0.36,0.36h-3\tc-0.199,0-0.36-0.161-0.36-0.36V1.36H7.36V8.64z"
  })), children);
});

export { StrategyDirect as default };
