// @carbon/pictograms-react/es/cics--vsam-recovery-for-z-os/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const CicsVsamRecoveryForZOs = /*#__PURE__*/React.forwardRef(function CicsVsamRecoveryForZOs(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M11,30.36\tc-1.301,0-2.36-1.059-2.36-2.36c0-0.268,0.045-0.525,0.127-0.766C4.928,24.759,2.64,20.569,2.64,16C2.64,8.633,8.633,2.64,16,2.64\tS29.36,8.633,29.36,16S23.367,29.36,16,29.36v-0.72c6.97,0,12.64-5.67,12.64-12.64S22.97,3.36,16,3.36S3.36,9.03,3.36,16\tc0,4.302,2.144,8.249,5.745,10.596c0.431-0.58,1.12-0.956,1.896-0.956c1.301,0,2.36,1.059,2.36,2.36S12.301,30.36,11,30.36z M11,26.36c-0.904,0-1.64,0.735-1.64,1.64s0.736,1.64,1.64,1.64s1.64-0.735,1.64-1.64S11.904,26.36,11,26.36z M17,22.36h-4v-0.72h4\tc2.559,0,4.64-2.081,4.64-4.64v-2h0.721v2C22.36,19.955,19.955,22.36,17,22.36z M16,18.36c-1.301,0-2.36-1.059-2.36-2.36\ts1.059-2.36,2.36-2.36c1.302,0,2.36,1.059,2.36,2.36C18.36,17.302,17.302,18.36,16,18.36z M16,14.36c-0.904,0-1.64,0.736-1.64,1.64\tc0,0.904,0.736,1.64,1.64,1.64c0.904,0,1.64-0.735,1.64-1.64S16.904,14.36,16,14.36z M10.36,17H9.64v-2\tc0-2.956,2.404-5.36,5.36-5.36h4v0.72h-4c-2.559,0-4.64,2.082-4.64,4.64C10.36,15,10.36,17,10.36,17z"
  })), children);
});

export { CicsVsamRecoveryForZOs as default };
