// @carbon/pictograms-react/lib/tennis--net/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const TennisNet = /*#__PURE__*/React__default["default"].forwardRef(function TennisNet(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31,2.64H11.323c-0.175-1.13-1.145-2-2.323-2\tC7.699,0.64,6.64,1.699,6.64,3v9.64H5c-0.199,0-0.36,0.161-0.36,0.36v3.64H3c-0.199,0-0.36,0.161-0.36,0.36v7.64H1v0.721h2\tc0.199,0,0.36-0.161,0.36-0.36v-7.64h1.28V21c0,0.199,0.161,0.36,0.36,0.36h1.64V31h0.72V3c0-0.904,0.736-1.64,1.64-1.64\tS10.64,2.096,10.64,3v28h0.72V9.36h1.28V31h0.72v-1.64h3.28V31h0.72v-1.64h3.279V31h0.721v-1.64h3.279V31h0.721v-1.64h3.279V31\th0.721v-1.64H31v-0.72h-1.64v-3.28H31v-0.72h-1.64v-3.28H31v-0.72h-1.64v-3.28H31v-0.72h-1.64v-3.28H31v-0.72h-1.64V9.36H31V8.64\tH11.36V3.36H31V2.64z M5.36,20.64v-7.28h1.28v7.28H5.36z M13.36,13.36h3.28v3.28h-3.28V13.36z M17.36,16.64v-3.28h3.279v3.28H17.36z M20.64,17.36v3.279h-3.28V17.36H20.64z M13.36,17.36h3.28v3.279h-3.28V17.36z M13.36,21.36h3.28v3.279h-3.28V21.36z M17.36,21.36\th3.279v3.279H17.36V21.36z M21.36,21.36h3.279v3.279H21.36V21.36z M21.36,20.64v-3.28h3.279v3.279L21.36,20.64L21.36,20.64z M21.36,16.64v-3.28h3.279v3.28H21.36z M21.36,12.64V9.36h3.279v3.28C24.639,12.64,21.36,12.64,21.36,12.64z M20.64,12.64h-3.28\tV9.36h3.279L20.64,12.64L20.64,12.64z M16.64,12.64h-3.28V9.36h3.28V12.64z M13.36,28.64v-3.28h3.28v3.279L13.36,28.64L13.36,28.64z M17.36,28.64v-3.28h3.279v3.279L17.36,28.64L17.36,28.64z M21.36,28.64v-3.28h3.279v3.279L21.36,28.64L21.36,28.64z M28.64,28.64\th-3.28v-3.28h3.279L28.64,28.64L28.64,28.64z M28.64,24.64h-3.28v-3.28h3.279L28.64,24.64L28.64,24.64z M28.64,20.64h-3.28v-3.28\th3.279L28.64,20.64L28.64,20.64z M28.64,16.64h-3.28v-3.28h3.279L28.64,16.64L28.64,16.64z M28.64,12.64h-3.28V9.36h3.279\tL28.64,12.64L28.64,12.64z M12.64,6c0,0.75,0.61,1.36,1.36,1.36S15.36,6.75,15.36,6S14.75,4.64,14,4.64S12.64,5.25,12.64,6z M14.64,6c0,0.353-0.287,0.64-0.64,0.64S13.36,6.353,13.36,6S13.647,5.36,14,5.36S14.64,5.647,14.64,6z M16.64,6\tc0,0.75,0.61,1.36,1.36,1.36S19.36,6.75,19.36,6S18.75,4.64,18,4.64S16.64,5.25,16.64,6z M18.64,6c0,0.353-0.287,0.64-0.64,0.64\tS17.36,6.353,17.36,6S17.647,5.36,18,5.36S18.64,5.647,18.64,6z"
  })), children);
});

module.exports = TennisNet;
