// @carbon/pictograms-react/es/football--goal-post/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const FootballGoalPost = /*#__PURE__*/React.forwardRef(function FootballGoalPost(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M31,27.64H18.36V23\tc0-0.199-0.161-0.36-0.36-0.36h-1.64v-3.28H24c0.199,0,0.36-0.161,0.36-0.36V1h-0.72v17.64H8.36V1H7.64v18\tc0,0.199,0.161,0.36,0.36,0.36h7.64v3.279H14c-0.199,0-0.36,0.161-0.36,0.36v4.64H1v0.721h12.64V31c0,0.199,0.161,0.36,0.36,0.36h4\tc0.199,0,0.36-0.161,0.36-0.36v-2.64H31V27.64z M17.64,30.64h-3.28v-7.28h3.28V30.64z"
  })), children);
});

export { FootballGoalPost as default };
