// @carbon/pictograms-react/es/hyper-protect--containers/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const HyperProtectContainers = /*#__PURE__*/React.forwardRef(function HyperProtectContainers(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M16,31.36\tC8.082,31.36,1.64,24.918,1.64,17V1c0-0.199,0.161-0.36,0.36-0.36h28c0.199,0,0.36,0.161,0.36,0.36v16\tC30.36,24.918,23.918,31.36,16,31.36z M2.36,1.36V17c0,7.521,6.119,13.64,13.64,13.64S29.64,24.522,29.64,17V1.36H2.36z M16,22.36\tc-0.055,0-0.11-0.013-0.161-0.038l-6-3C9.717,19.261,9.64,19.137,9.64,19v-8c0-0.136,0.077-0.261,0.199-0.322l6-3\tc0.102-0.051,0.221-0.051,0.322,0l6,3c0.122,0.061,0.199,0.186,0.199,0.322v8c0,0.137-0.077,0.261-0.199,0.322l-6,3\tC16.11,22.348,16.055,22.36,16,22.36z M10.36,18.777l5.64,2.82l5.64-2.82v-7.195l-5.479,2.739c-0.102,0.051-0.221,0.051-0.322,0\tl-5.479-2.739V18.777z M10.805,11L16,13.598L21.195,11L16,8.402L10.805,11z"
  })), children);
});

export { HyperProtectContainers as default };
