// @carbon/pictograms-react/es/visual--insights/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const VisualInsights = /*#__PURE__*/React.forwardRef(function VisualInsights(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M31,31.36h-6c-0.199,0-0.36-0.161-0.36-0.36v-2.64H21c-0.199,0-0.36-0.161-0.36-0.36V16.36h-3.28\tV24c0,0.199-0.161,0.36-0.36,0.36H1c-0.199,0-0.36-0.161-0.36-0.36v-4.999c0-0.001,0-0.001,0-0.002V8c0-0.199,0.161-0.36,0.36-0.36\th16c0.199,0,0.36,0.161,0.36,0.36v7.64h3.279V4c0-0.199,0.161-0.36,0.36-0.36h5.06l1.629-2.82C27.753,0.708,27.871,0.64,28,0.64l0,0\tc0.129,0,0.247,0.068,0.312,0.18l3.464,6c0.064,0.111,0.064,0.249,0,0.36s-0.183,0.18-0.312,0.18h-6.929\tc-0.129,0-0.247-0.068-0.312-0.18c-0.064-0.112-0.064-0.249,0-0.36l1.42-2.46H21.36v11.28h3.299c0.18-1.684,1.609-3,3.341-3\tc1.853,0,3.36,1.507,3.36,3.36s-1.508,3.36-3.36,3.36c-1.731,0-3.161-1.316-3.341-3H21.36v11.28h3.279V25\tc0-0.199,0.161-0.36,0.36-0.36h6c0.199,0,0.36,0.161,0.36,0.36v6C31.36,31.199,31.199,31.36,31,31.36z M25.36,30.64h5.279v-5.28\tH25.36V30.64z M1.36,23.64h15.28v-2.49L13,17.509l-1.746,1.746c-0.141,0.141-0.368,0.141-0.509,0L6,14.509l-4.64,4.64\tC1.36,19.149,1.36,23.64,1.36,23.64z M13,16.64c0.092,0,0.184,0.035,0.254,0.105l3.386,3.385V8.36H1.36v9.771l4.386-4.385\tc0.141-0.141,0.368-0.141,0.509,0L11,18.491l1.746-1.746C12.816,16.675,12.908,16.64,13,16.64z M28,13.36\tc-1.456,0-2.64,1.185-2.64,2.64c0,1.456,1.184,2.64,2.64,2.64s2.64-1.184,2.64-2.64S29.456,13.36,28,13.36z M25.159,6.64h5.682\tL28,1.72l-1.417,2.455c-0.002,0.004-0.005,0.007-0.007,0.011L25.159,6.64z M11.5,14.359c-1.025,0-1.86-0.834-1.86-1.86\ts0.834-1.86,1.86-1.86s1.86,0.834,1.86,1.86S12.525,14.359,11.5,14.359z M11.5,11.359c-0.628,0-1.14,0.512-1.14,1.14\ts0.512,1.14,1.14,1.14s1.14-0.512,1.14-1.14S12.128,11.359,11.5,11.359z"
  })), children);
});

export { VisualInsights as default };
