// @carbon/pictograms-react/lib/podcast/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Podcast = /*#__PURE__*/React__default["default"].forwardRef(function Podcast(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16.36,31h-0.72V21h0.72V31z M26.861,26.861l-0.51-0.51c2.766-2.764,4.288-6.44,4.288-10.352\tc0-8.073-6.567-14.64-14.64-14.64C7.927,1.36,1.36,7.927,1.36,16c0,3.911,1.523,7.588,4.288,10.352l-0.509,0.51\tC2.238,23.961,0.64,20.104,0.64,16C0.64,7.53,7.53,0.64,16,0.64S31.36,7.53,31.36,16C31.36,20.104,29.763,23.961,26.861,26.861z M24.033,24.033l-0.51-0.51c2.01-2.01,3.116-4.683,3.116-7.523c0-5.867-4.772-10.64-10.64-10.64C10.133,5.36,5.36,10.133,5.36,16\tc0,2.841,1.107,5.514,3.116,7.523l-0.509,0.51c-2.146-2.146-3.327-5-3.327-8.033C4.64,9.736,9.736,4.64,16,4.64\tS27.36,9.736,27.36,16C27.36,19.033,26.179,21.887,24.033,24.033z M16,19.36c-1.853,0-3.36-1.508-3.36-3.36s1.507-3.36,3.36-3.36\ts3.36,1.507,3.36,3.36S17.853,19.36,16,19.36z M16,13.36c-1.456,0-2.64,1.185-2.64,2.64c0,1.456,1.185,2.64,2.64,2.64\tc1.456,0,2.64-1.184,2.64-2.64S17.456,13.36,16,13.36z"
  })), children);
});

module.exports = Podcast;
