// @carbon/pictograms-react/lib/gpu-computing/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const GpuComputing = /*#__PURE__*/React__default["default"].forwardRef(function GpuComputing(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M30,30.36H2c-0.199,0-0.36-0.161-0.36-0.36V2\tc0-0.199,0.161-0.36,0.36-0.36h28c0.199,0,0.36,0.161,0.36,0.36v28C30.36,30.199,30.199,30.36,30,30.36z M2.36,29.64h27.28V2.36\tH2.36V29.64z M12,27.36h-2c-0.199,0-0.36-0.161-0.36-0.36v-0.64H7.36V27c0,0.199-0.161,0.36-0.36,0.36H5\tc-0.199,0-0.36-0.161-0.36-0.36v-2c0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v0.64h2.28V25\tc0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v0.64H27v0.721H12.36V27C12.36,27.199,12.199,27.36,12,27.36z M10.36,26.64h1.28v-1.28h-1.28V26.64z M5.36,26.64h1.28v-1.28H5.36V26.64z M27,22.36h-2c-0.199,0-0.36-0.161-0.36-0.36v-0.64H12.36\tV22c0,0.199-0.161,0.36-0.36,0.36h-2c-0.199,0-0.36-0.161-0.36-0.36v-0.64H5v-0.72h4.64V20c0-0.199,0.161-0.36,0.36-0.36h2\tc0.199,0,0.36,0.161,0.36,0.36v0.64h12.28V20c0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2\tC27.36,22.199,27.199,22.36,27,22.36z M25.36,21.64h1.279v-1.28H25.36V21.64z M10.36,21.64h1.28v-1.28h-1.28V21.64z M27,17.36h-2\tc-0.199,0-0.36-0.161-0.36-0.36v-0.64h-7.28V17c0,0.199-0.161,0.36-0.36,0.36h-2c-0.199,0-0.36-0.161-0.36-0.36v-0.64H5v-0.72h9.64\tV15c0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v0.64h7.279V15c0-0.199,0.161-0.36,0.36-0.36h2\tc0.199,0,0.36,0.161,0.36,0.36v2C27.36,17.199,27.199,17.36,27,17.36z M25.36,16.64h1.279v-1.28H25.36V16.64z M15.36,16.64h1.28\tv-1.28h-1.28V16.64z M17,12.36h-2c-0.199,0-0.36-0.161-0.36-0.36v-0.64H7.36V12c0,0.199-0.161,0.36-0.36,0.36H5\tc-0.199,0-0.36-0.161-0.36-0.36v-2c0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v0.64h7.28V10\tc0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v0.64H27v0.72h-9.64V12C17.36,12.199,17.199,12.36,17,12.36z M15.36,11.64h1.28v-1.28h-1.28V11.64z M5.36,11.64h1.28v-1.28H5.36V11.64z M27,7.36h-2c-0.199,0-0.36-0.161-0.36-0.36V6.36H7.36V7\tc0,0.199-0.161,0.36-0.36,0.36H5C4.801,7.36,4.64,7.199,4.64,7V5c0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v0.64\th17.28V5c0-0.199,0.161-0.36,0.36-0.36h2c0.199,0,0.36,0.161,0.36,0.36v2C27.36,7.199,27.199,7.36,27,7.36z M25.36,6.64h1.279V5.36\tH25.36V6.64z M5.36,6.64h1.28V5.36H5.36V6.64z"
  })), children);
});

module.exports = GpuComputing;
