// @carbon/pictograms-react/lib/tablet--device/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const TabletDevice = /*#__PURE__*/React__default["default"].forwardRef(function TabletDevice(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,29.5c-0.552,0-1-0.448-1-1s0.448-1,1-1s1,0.448,1,1S16.552,29.5,16,29.5z M28.36,31V1\tc0-0.199-0.161-0.36-0.36-0.36H4C3.801,0.64,3.64,0.801,3.64,1v30c0,0.199,0.161,0.36,0.36,0.36h24\tC28.199,31.36,28.36,31.199,28.36,31z M4.36,26.36h23.28v4.279H4.36V26.36z M4.36,1.36h23.28v24.28H4.36V1.36z"
  })), children);
});

module.exports = TabletDevice;
