// @carbon/pictograms-react/lib/data--transfer/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const DataTransfer = /*#__PURE__*/React__default["default"].forwardRef(function DataTransfer(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,29.36c-7.083,0-12.897-5.542-13.333-12.518\tl-1.412,1.412l-0.509-0.51L3,15.491l2.254,2.254l-0.509,0.51l-1.354-1.354C3.855,23.452,9.333,28.64,16,28.64\tc4.73,0,9.028-2.607,11.216-6.806l0.639,0.332C25.542,26.604,21,29.36,16,29.36z M16,23.36c-4.058,0-7.36-3.302-7.36-7.36\tS11.942,8.64,16,8.64c4.059,0,7.36,3.302,7.36,7.36C23.36,20.059,20.059,23.36,16,23.36z M16,9.36c-3.661,0-6.64,2.979-6.64,6.64\ts2.979,6.64,6.64,6.64s6.64-2.979,6.64-6.64S19.661,9.36,16,9.36z M29,16.509l-2.255-2.255l0.51-0.509l1.354,1.354\tC28.145,8.548,22.667,3.36,16,3.36c-4.73,0-9.028,2.608-11.216,6.807L4.145,9.833C6.458,5.396,11,2.64,16,2.64\tc7.083,0,12.897,5.542,13.334,12.517l1.411-1.412l0.51,0.509L29,16.509z"
  })), children);
});

module.exports = DataTransfer;
