// @carbon/pictograms-react/lib/leader/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Leader = /*#__PURE__*/React__default["default"].forwardRef(function Leader(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,31.36c0,0-0.001,0-0.002,0\tC7.53,31.359,0.641,24.471,0.64,16.002c0-0.001,0-0.003,0-0.004C0.642,7.529,7.532,0.64,16,0.64c4.211,0,8.031,1.703,10.81,4.457\tc0.018,0.012,0.034,0.026,0.051,0.042c0.016,0.016,0.029,0.033,0.042,0.051C29.656,7.968,31.36,11.789,31.36,16\tc0,8.469-6.89,15.358-15.358,15.36C16.001,31.36,16.001,31.36,16,31.36z M1.367,16.463c0.24,7.706,6.464,13.93,14.17,14.17\tl-2.834-11.336L1.367,16.463z M26.726,6.045L16.543,30.63C24.365,30.344,30.64,23.891,30.64,16\tC30.64,12.159,29.153,8.659,26.726,6.045z M2.146,15.915l10.942,2.735c0.129,0.032,0.229,0.134,0.262,0.263l2.736,10.941\tL25.94,6.059L2.146,15.915z M16,1.36c-7.891,0-14.343,6.275-14.63,14.097L25.954,5.274C23.34,2.846,19.841,1.36,16,1.36z"
  })), children);
});

module.exports = Leader;
