// @carbon/pictograms-react/lib/get--ahead--of--risk--and--compliance/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const GetAheadOfRiskAndCompliance = /*#__PURE__*/React__default["default"].forwardRef(function GetAheadOfRiskAndCompliance(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M25,31.36H7v-0.72h5.657c0.157-1.615,1.367-2.825,2.983-2.982V12.333\tc-1.131-0.174-2-1.154-2-2.333s0.869-2.159,2-2.333V4.334L6.876,6.838c2.012,0.595,3.484,2.46,3.484,4.662\tc0,2.68-2.18,4.86-4.86,4.86s-4.86-2.18-4.86-4.86c0-2.479,1.866-4.531,4.267-4.824L4.901,6.654l20.999-6\tc0.149-0.042,0.321,0.021,0.406,0.155l5,8c0.069,0.111,0.072,0.251,0.009,0.365C31.251,9.289,31.131,9.36,31,9.36H21\tc-0.131,0-0.251-0.071-0.314-0.186c-0.063-0.114-0.061-0.254,0.009-0.365l4.504-7.206L16.36,4.129v3.539\tc1.131,0.174,2,1.154,2,2.333s-0.869,2.159-2,2.333v15.325c1.616,0.157,2.826,1.367,2.983,2.982H25V31.36z M13.382,30.64h5.236\tc-0.166-1.323-1.235-2.28-2.618-2.28C14.618,28.36,13.547,29.317,13.382,30.64z M5.5,7.36c-2.283,0-4.14,1.857-4.14,4.14\ts1.857,4.14,4.14,4.14s4.14-1.857,4.14-4.14S7.783,7.36,5.5,7.36z M16,8.36c-0.904,0-1.64,0.736-1.64,1.64s0.736,1.64,1.64,1.64\tc0.904,0,1.64-0.736,1.64-1.64S16.904,8.36,16,8.36z M21.649,8.64h8.701L26,1.679L21.649,8.64z"
  })), children);
});

module.exports = GetAheadOfRiskAndCompliance;
