// @carbon/pictograms-react/es/data--storage/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const DataStorage = /*#__PURE__*/React.forwardRef(function DataStorage(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M31,31.336H1c-0.186,0-0.336-0.15-0.336-0.336v-5h0.672v4.664h29.328V26h0.672v5\tC31.336,31.186,31.186,31.336,31,31.336z M31,23.336H1c-0.186,0-0.336-0.15-0.336-0.336v-5h0.672v4.664h29.328V18h0.672v5\tC31.336,23.186,31.186,23.336,31,23.336z M31,15.336H1c-0.186,0-0.336-0.15-0.336-0.336v-5h0.672v4.664h29.328V10h0.672v5\tC31.336,15.186,31.186,15.336,31,15.336z M31,7.336H1c-0.186,0-0.336-0.15-0.336-0.336V2h0.672v4.664h29.328V2h0.672v5\tC31.336,7.186,31.186,7.336,31,7.336z"
  })), children);
});

export { DataStorage as default };
