// @carbon/pictograms-react/es/audio-data/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const AudioData = /*#__PURE__*/React.forwardRef(function AudioData(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M16,31.36C7.53,31.36,0.64,24.47,0.64,16\tS7.53,0.64,16,0.64S31.36,7.53,31.36,16S24.47,31.36,16,31.36z M16,1.36C7.927,1.36,1.36,7.927,1.36,16\tc0,8.072,6.567,14.64,14.64,14.64c8.072,0,14.64-6.567,14.64-14.64S24.072,1.36,16,1.36z M23.36,23h-0.72V9h0.721L23.36,23L23.36,23\tz M11.36,23h-0.72V9h0.72V23z M17.36,21h-0.72V11h0.72V21z M8.36,20H7.64v-8h0.72V20z M26.36,19h-0.72v-6h0.721L26.36,19L26.36,19z M14.36,19h-0.72v-6h0.72V19z M20.36,17h-0.72v-2h0.721L20.36,17L20.36,17z M5.36,17H4.64v-2h0.72C5.36,15,5.36,17,5.36,17z"
  })), children);
});

export { AudioData as default };
