// @carbon/pictograms-react/lib/security--shield/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const SecurityShield = /*#__PURE__*/React__default["default"].forwardRef(function SecurityShield(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,0.59L4.64,6.786V20c0,6.158,5.202,11.36,11.36,11.36S27.36,26.158,27.36,20V6.786L16,0.59z M25.84,23.957L16,18.592v0.821l9.536,5.198c-0.699,1.407-1.694,2.651-2.909,3.636L16,24.635v0.821l5.995,3.268\tc-1.721,1.2-3.791,1.916-5.995,1.916c-5.768,0-10.64-4.872-10.64-10.64V7.214L16,1.41l10.64,5.804v5.18L16,6.592v0.82l10.64,5.801\tv5.137L16,12.549v0.82l10.64,5.802V20C26.64,21.39,26.351,22.726,25.84,23.957z"
  })), children);
});

module.exports = SecurityShield;
