// @carbon/pictograms-react/lib/scalable/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Scalable = /*#__PURE__*/React__default["default"].forwardRef(function Scalable(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31.36,0.64V10h-0.72V1.87L19.255,13.254l-0.51-0.509\tL30.131,1.36H22V0.64H31.36z M28.64,22.64H9.36V3.36H18V2.64H9C8.801,2.64,8.64,2.801,8.64,3v20c0,0.199,0.161,0.36,0.36,0.36h20\tc0.199,0,0.36-0.161,0.36-0.36v-9h-0.72V22.64z M12.64,30.64H1.36V19.36H7v-0.72H1c-0.199,0-0.36,0.161-0.36,0.36v12\tc0,0.199,0.161,0.36,0.36,0.36h12c0.199,0,0.36-0.161,0.36-0.36v-6h-0.72C12.64,25,12.64,30.64,12.64,30.64z"
  })), children);
});

module.exports = Scalable;
