// @carbon/pictograms-react/lib/chart--two-y-axis/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const ChartTwoYAxis = /*#__PURE__*/React__default["default"].forwardRef(function ChartTwoYAxis(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M19,17.64h8v0.721h-7.851l-6.895,6.895\tC12.187,25.322,12.096,25.36,12,25.36H5v-0.72h6.851l6.894-6.895C18.812,17.678,18.904,17.64,19,17.64z M5,14.36h7\tc0.096,0,0.187-0.038,0.254-0.105l6.895-6.895H27V6.64h-8c-0.096,0-0.188,0.038-0.255,0.105l-6.894,6.895H5V14.36z M31.255,2.746\tL29,0.491l-2.255,2.255l0.51,0.509L28.64,1.87v28.77H3.36V1.869l1.386,1.385l0.509-0.509L3,0.491L0.746,2.746l0.509,0.509\tL2.64,1.869V31c0,0.199,0.161,0.36,0.36,0.36h26c0.199,0,0.36-0.161,0.36-0.36V1.87l1.385,1.385L31.255,2.746z"
  })), children);
});

module.exports = ChartTwoYAxis;
