// @carbon/pictograms-react/es/ibm--ix/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const IbmIx = /*#__PURE__*/React.forwardRef(function IbmIx(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M25.5,25.86c-0.092,0-0.185-0.035-0.255-0.105l-4.495-4.496l-4.496,4.496c-0.141,0.141-0.368,0.141-0.509,0\tl-4.496-4.496l-4.496,4.496c-0.141,0.141-0.368,0.141-0.509,0l-4.75-4.75c-0.141-0.141-0.141-0.369,0-0.51L5.991,16l-4.495-4.496\tc-0.141-0.141-0.141-0.368,0-0.509l4.75-4.75c0.141-0.141,0.368-0.141,0.509,0l4.496,4.495l4.496-4.495\tc0.141-0.141,0.368-0.141,0.509,0l4.496,4.495l4.495-4.495c0.141-0.141,0.369-0.141,0.51,0l4.75,4.75\tc0.141,0.141,0.141,0.368,0,0.509L26.009,16l4.496,4.495c0.141,0.141,0.141,0.369,0,0.51l-4.75,4.75\tC25.685,25.825,25.592,25.86,25.5,25.86z M20.75,20.39c0.092,0,0.185,0.035,0.255,0.105l4.495,4.496l4.241-4.241l-4.496-4.496\tc-0.141-0.141-0.141-0.368,0-0.509l4.496-4.496L25.5,7.009l-4.495,4.495c-0.141,0.141-0.369,0.141-0.51,0L16,7.009l-4.241,4.241\tl4.495,4.496c0.07,0.07,0.105,0.162,0.105,0.254c0,0.092-0.035,0.184-0.105,0.254l-4.495,4.496L16,24.991l4.495-4.496\tC20.565,20.425,20.658,20.39,20.75,20.39z M2.259,20.75L6.5,24.991l4.241-4.241L6.5,16.509L2.259,20.75z M7.009,16l4.241,4.241\tL15.491,16l-4.241-4.241L7.009,16z M2.259,11.25L6.5,15.491l4.241-4.241L6.5,7.009L2.259,11.25z"
  })), children);
});

export { IbmIx as default };
