// @carbon/pictograms-react/lib/endpoint--protection/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const EndpointProtection = /*#__PURE__*/React__default["default"].forwardRef(function EndpointProtection(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M30.64,22.64H1.36V7c0-0.904,0.736-1.64,1.64-1.64h14V4.64H3C1.699,4.64,0.64,5.699,0.64,7v18\tc0,1.302,1.059,2.36,2.36,2.36h12.641v3.279H11v0.721h10v-0.72h-4.64v-3.28H29c1.302,0,2.36-1.059,2.36-2.36v-8h-0.72V22.64z M30.64,25c0,0.904-0.735,1.64-1.64,1.64H3c-0.904,0-1.64-0.735-1.64-1.64v-1.64h29.28C30.64,23.36,30.64,25,30.64,25z M25,1.598\tl-6.36,3.18V12c0,3.507,2.854,6.36,6.36,6.36s6.36-2.854,6.36-6.36V4.777L25,1.598z M30.64,12c0,3.11-2.529,5.64-5.64,5.64\ts-5.64-2.53-5.64-5.64V5.223L25,2.402l5.64,2.82V12z"
  })), children);
});

module.exports = EndpointProtection;
