// @carbon/pictograms-react/lib/hong-kong--cityscape/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const HongKongCityscape = /*#__PURE__*/React__default["default"].forwardRef(function HongKongCityscape(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31,31.36H1v-0.72h9.64V17c0-0.136,0.077-0.261,0.199-0.322l4.801-2.401V1h0.72v3.277\tl1.28,0.64V1h0.721v4.277l2.801,1.401C21.283,6.739,21.36,6.864,21.36,7v23.64H31V31.36z M16.36,30.64h4.28v-1.28h-4.28V30.64z M11.36,30.64h4.28v-1.28h-4.28C11.36,29.36,11.36,30.64,11.36,30.64z M16.36,28.64h4.28v-1.417l-4.28-2.141V28.64z M11.36,28.64\th4.28v-3.558l-4.28,2.141C11.36,27.223,11.36,28.64,11.36,28.64z M16.805,24.5l3.834,1.918v-3.836L16.805,24.5z M11.36,22.582v3.836\tl3.835-1.918L11.36,22.582z M16.36,20.082v3.836L20.195,22L16.36,20.082z M11.805,22l3.835,1.918v-3.836L11.805,22z M16.805,19.5\tl3.834,1.918v-3.836L16.805,19.5z M11.36,17.582v3.836l3.835-1.918L11.36,17.582z M16.36,15.083v3.835L20.195,17L16.36,15.083z M11.805,17l3.835,1.918v-3.835L11.805,17z M16.805,14.5l3.834,1.917v-3.835L16.805,14.5z M16.36,10.083v3.835L20.195,12\tL16.36,10.083z M16.805,9.5l3.834,1.917V7.583L16.805,9.5z M16.36,5.083v3.835L20.195,7L16.36,5.083z M28.36,29h-0.72V18.36h-2.28\tV27h-0.72v-1.64H23v-0.72h1.64v-1.28H23v-0.72h1.64V18c0-0.199,0.161-0.36,0.36-0.36h2.64V13c0-0.199,0.161-0.36,0.36-0.36h3v0.72\th-2.64v1.28H31v0.72h-2.64v1.28H31v0.72h-2.64v1.279H31v0.721h-2.64v4.279H31v0.721h-2.64V29z M8.36,29H7.64v-6.64H3v-0.72h2.64\tv-5.28H1v-0.72h2.64v-1.28H1v-0.72h3c0.199,0,0.36,0.161,0.36,0.36v1.64H6c0.199,0,0.36,0.161,0.36,0.36v5.64H8\tc0.199,0,0.36,0.161,0.36,0.36V29z M3,20.36H1v-0.72h2V20.36z M3,18.36H1v-0.72h2V18.36z"
  })), children);
});

module.exports = HongKongCityscape;
