// @carbon/pictograms-react/lib/college/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const College = /*#__PURE__*/React__default["default"].forwardRef(function College(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31,31.36H1c-0.199,0-0.36-0.161-0.36-0.36v-3c0-0.199,0.161-0.36,0.36-0.36h1.64V25\tc0-0.199,0.161-0.36,0.36-0.36h0.64V13.36H1c-0.153,0-0.289-0.096-0.34-0.241c-0.05-0.144-0.004-0.305,0.115-0.4l15-12\tc0.132-0.104,0.318-0.104,0.45,0l15,12c0.12,0.096,0.166,0.256,0.115,0.4c-0.051,0.145-0.187,0.241-0.34,0.241h-2.64v11.28H29\tc0.199,0,0.36,0.161,0.36,0.36v2.64H31c0.199,0,0.36,0.161,0.36,0.36v3C31.36,31.199,31.199,31.36,31,31.36z M1.36,30.64h29.28\tv-2.28H1.36V30.64z M3.36,27.64h25.28v-2.28H3.36V27.64z M22.36,24.64h5.279V13.36H22.36V24.64z M10.36,24.64h11.28V13.36H10.36\tV24.64z M4.36,24.64h5.28V13.36H4.36V24.64z M28,12.64h1.974L16,1.461L2.026,12.64H28z M25.36,23h-0.72v-8h0.721L25.36,23L25.36,23z M7.36,23H6.64v-8h0.72C7.36,15,7.36,23,7.36,23z M16,10.86c-1.577,0-2.86-1.283-2.86-2.86S14.423,5.14,16,5.14S18.86,6.423,18.86,8\tS17.577,10.86,16,10.86z M16,5.86c-1.18,0-2.14,0.96-2.14,2.14s0.96,2.14,2.14,2.14S18.14,9.18,18.14,8S17.18,5.86,16,5.86z"
  })), children);
});

module.exports = College;
