// @carbon/pictograms-react/lib/do-not--02/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const DoNot_02 = /*#__PURE__*/React__default["default"].forwardRef(function DoNot_02(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,31.36C7.53,31.36,0.64,24.47,0.64,16S7.53,0.64,16,0.64S31.36,7.53,31.36,16S24.47,31.36,16,31.36z M5.401,5.911C2.898,8.539,1.36,12.093,1.36,16c0,8.072,6.567,14.64,14.64,14.64c3.907,0,7.462-1.538,10.09-4.041L5.401,5.911z M5.911,5.401L26.599,26.09c2.503-2.628,4.041-6.183,4.041-10.09c0-8.073-6.567-14.64-14.64-14.64\tC12.093,1.36,8.539,2.898,5.911,5.401z"
  })), children);
});

module.exports = DoNot_02;
