// @carbon/pictograms-react/lib/highly-available/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const HighlyAvailable = /*#__PURE__*/React__default["default"].forwardRef(function HighlyAvailable(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,31.36C7.53,31.36,0.64,24.47,0.64,16\tc0-4.019,1.514-7.768,4.281-10.64H3V4.64h3.36V8H5.64V5.655C2.876,8.418,1.36,12.074,1.36,16c0,8.072,6.567,14.64,14.64,14.64\tc8.072,0,14.64-6.567,14.64-14.64S24.073,1.36,16,1.36c-2.741,0-5.413,0.762-7.728,2.204L7.891,2.952\tC10.32,1.439,13.125,0.64,16,0.64c8.47,0,15.36,6.89,15.36,15.36S24.47,31.36,16,31.36z M16.36,25h-0.72v-2h0.72V25z M20.188,23.974\tl-1-1.731l0.623-0.359l1,1.731L20.188,23.974z M11.812,23.974l-0.623-0.359l1-1.731l0.623,0.359L11.812,23.974z M8.386,20.812\tl-0.36-0.623l1.732-1l0.36,0.623L8.386,20.812z M23.614,20.812l-1.731-1l0.359-0.623l1.731,1L23.614,20.812z M16,18.36\tc-1.301,0-2.36-1.059-2.36-2.36s1.059-2.36,2.36-2.36c1.302,0,2.36,1.059,2.36,2.36C18.36,17.302,17.302,18.36,16,18.36z M16,14.36\tc-0.904,0-1.64,0.736-1.64,1.64c0,0.904,0.736,1.64,1.64,1.64c0.904,0,1.64-0.735,1.64-1.64S16.904,14.36,16,14.36z M25,16.36h-2\tv-0.72h2V16.36z M9,16.36H7v-0.72h2V16.36z M9.758,12.812l-1.732-1l0.36-0.624l1.732,1L9.758,12.812z M22.242,12.812l-0.359-0.623\tl1.731-1l0.359,0.623L22.242,12.812z M19.812,10.118l-0.623-0.36l1-1.732l0.623,0.36L19.812,10.118z M12.188,10.118l-1-1.732\tl0.624-0.36l1,1.732L12.188,10.118z M16.36,9h-0.72V7h0.72V9z"
  })), children);
});

module.exports = HighlyAvailable;
