// @carbon/pictograms-react/es/flexibility/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const Flexibility = /*#__PURE__*/React.forwardRef(function Flexibility(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M19.36,31h-0.72V16c0-1.301,1.059-2.36,2.36-2.36\th9.131l-2.386-2.386l0.51-0.509L31.509,14l-3.254,3.255l-0.51-0.509l2.386-2.386H21c-0.904,0-1.64,0.736-1.64,1.64V31z M16.36,31\th-0.72V1.869l-2.386,2.385l-0.509-0.509L16,0.491l3.255,3.255l-0.51,0.509L16.36,1.869V31z M13.36,31h-0.72V21\tc0-0.904-0.736-1.64-1.64-1.64H1.869l2.385,2.385l-0.509,0.51L0.491,19l3.255-3.254l0.509,0.509L1.869,18.64H11\tc1.301,0,2.36,1.059,2.36,2.36V31z"
  })), children);
});

export { Flexibility as default };
