// @carbon/pictograms-react/lib/refinery/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Refinery = /*#__PURE__*/React__default["default"].forwardRef(function Refinery(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M1.64,1c0-0.199,0.161-0.36,0.36-0.36S2.36,0.801,2.36,1c0,1.133,0.824,1.64,1.64,1.64\tc1.62,0,2.358,1.223,2.358,2.36c0,0.199-0.161,0.36-0.36,0.36S5.639,5.199,5.639,5c0-0.816-0.507-1.64-1.639-1.64\tC2.863,3.36,1.64,2.622,1.64,1z M5,1.36c1.801,0,2.64,0.839,2.64,2.64c0,0.199,0.161,0.36,0.36,0.36c0.199,0,0.36-0.161,0.36-0.36\tc0-2.198-1.162-3.36-3.36-3.36C4.801,0.64,4.64,0.801,4.64,1S4.801,1.36,5,1.36z M31.36,17v14c0,0.199-0.161,0.36-0.36,0.36H1v-0.72\th2.64v-2.28H2c-0.085,0-0.168-0.03-0.233-0.086C1.051,27.665,0.64,26.836,0.64,26s0.411-1.665,1.126-2.274\tC1.832,23.67,1.915,23.64,2,23.64h1.64V12.164C2.881,11.465,2.14,10.146,2.14,9c0-1.211,0.829-2.618,1.631-3.278\tC3.835,5.669,3.917,5.64,4,5.64h4c0.083,0,0.165,0.029,0.229,0.082C9.031,6.382,9.86,7.789,9.86,9c0,1.146-0.741,2.465-1.5,3.164\tv4.538C8.567,16.661,8.781,16.64,9,16.64h1c0.353,0,0.64-0.287,0.64-0.64V7c0-1.853,1.507-3.36,3.36-3.36h2\tc1.853,0,3.36,1.507,3.36,3.36v1.64H20c0.199,0,0.36,0.161,0.36,0.36v21.64h1.279V17c0-0.199,0.161-0.36,0.36-0.36h9\tC31.199,16.64,31.36,16.801,31.36,17z M22.36,27.64h8.279V17.36H22.36V27.64z M22.36,30.64h8.279v-2.28H22.36V30.64z M16.36,9.36\tv21.28h3.28V9.36H16.36z M15.64,9c0-0.199,0.161-0.36,0.36-0.36h0.64V7c0-0.353-0.287-0.64-0.64-0.64h-2\tc-0.353,0-0.64,0.287-0.64,0.64v9c0,1.853-1.507,3.36-3.36,3.36H9c-0.353,0-0.64,0.287-0.64,0.64v3.64H10\tc0.085,0,0.168,0.03,0.233,0.086c0.716,0.609,1.126,1.438,1.126,2.274s-0.411,1.665-1.126,2.274C10.168,28.33,10.085,28.36,10,28.36\tH8.36v2.279h7.28C15.64,30.639,15.64,9,15.64,9z M6.36,20v3.64h1.28V20c0-0.75,0.61-1.36,1.36-1.36h1c1.456,0,2.64-1.184,2.64-2.64\tV7c0-0.75,0.61-1.36,1.36-1.36h2c0.75,0,1.36,0.61,1.36,1.36v1.64h1.279V7c0-1.456-1.184-2.64-2.64-2.64h-2\tc-1.456,0-2.64,1.185-2.64,2.64v9c0,0.75-0.61,1.36-1.36,1.36H9C7.544,17.36,6.36,18.544,6.36,20z M5.64,20\tc0-1.369,0.823-2.549,2-3.072V12.36H4.36v11.28h1.281L5.64,20z M4.134,11.64h3.731C8.496,11.066,9.14,9.938,9.14,9\tS8.496,6.934,7.866,6.36H4.134C3.504,6.934,2.86,8.062,2.86,9S3.504,11.066,4.134,11.64z M9.863,27.64\tc0.489-0.456,0.777-1.061,0.777-1.64s-0.289-1.184-0.777-1.64H2.137C1.648,24.816,1.36,25.421,1.36,26s0.289,1.184,0.777,1.64H9.863\tz M4.36,30.64h3.28v-2.28H4.36V30.64z"
  })), children);
});

module.exports = Refinery;
