// @carbon/pictograms-react/lib/optimize/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Optimize = /*#__PURE__*/React__default["default"].forwardRef(function Optimize(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,10.36h1.5V9.64H16c-3.507,0-6.36,2.853-6.36,6.36\ts2.853,6.36,6.36,6.36s6.36-2.854,6.36-6.36v-4.695c0.575-0.159,1-0.681,1-1.305c0-0.75-0.61-1.36-1.36-1.36S20.64,9.25,20.64,10\tc0,0.624,0.425,1.146,1,1.305V16c0,3.11-2.529,5.64-5.64,5.64c-3.11,0-5.64-2.529-5.64-5.64C10.36,12.89,12.89,10.36,16,10.36z M21.36,10c0-0.353,0.287-0.64,0.64-0.64s0.64,0.287,0.64,0.64s-0.287,0.64-0.64,0.64S21.36,10.353,21.36,10z M24.64,6\tc0,0.624,0.425,1.146,1,1.305V16c0,5.315-4.324,9.64-9.64,9.64S6.36,21.315,6.36,16S10.685,6.36,16,6.36h6V5.64h-6\tC10.288,5.64,5.64,10.288,5.64,16c0,5.713,4.647,10.36,10.36,10.36S26.36,21.713,26.36,16V7.305c0.575-0.159,1-0.681,1-1.305\tc0-0.75-0.61-1.36-1.36-1.36S24.64,5.25,24.64,6z M26.64,6c0,0.353-0.287,0.64-0.64,0.64S25.36,6.353,25.36,6S25.647,5.36,26,5.36\tS26.64,5.647,26.64,6z M31.36,2c0-0.75-0.61-1.36-1.36-1.36S28.64,1.25,28.64,2c0,0.624,0.425,1.146,1,1.305V16\tc0,7.521-6.118,13.64-13.64,13.64C8.479,29.64,2.36,23.521,2.36,16S8.479,2.36,16,2.36h10V1.64H16C8.082,1.64,1.64,8.082,1.64,16\tS8.082,30.36,16,30.36S30.36,23.918,30.36,16V3.305C30.935,3.146,31.36,2.624,31.36,2z M30,2.64c-0.353,0-0.64-0.287-0.64-0.64\tS29.647,1.36,30,1.36S30.64,1.647,30.64,2S30.353,2.64,30,2.64z"
  })), children);
});

module.exports = Optimize;
