// @carbon/pictograms-react/lib/file--backup/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const FileBackup = /*#__PURE__*/React__default["default"].forwardRef(function FileBackup(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,31.36H1c-0.199,0-0.36-0.161-0.36-0.36V9c0-0.199,0.161-0.36,0.36-0.36h7.155V5\tc0-0.199,0.161-0.36,0.36-0.36h7.125V1c0-0.199,0.161-0.36,0.36-0.36h15c0.199,0,0.36,0.161,0.36,0.36v22\tc0,0.199-0.161,0.36-0.36,0.36h-7.124V27c0,0.199-0.161,0.36-0.36,0.36H16.36V31C16.36,31.199,16.199,31.36,16,31.36z M1.36,30.64\th14.28V9.36H1.36V30.64z M16.36,26.64h6.795V5.36H8.875v3.28H16c0.199,0,0.36,0.161,0.36,0.36V26.64z M23.876,22.64h6.764V1.36\tH16.36v3.28h7.156c0.199,0,0.36,0.161,0.36,0.36C23.876,5,23.876,22.64,23.876,22.64z"
  })), children);
});

module.exports = FileBackup;
