// @carbon/pictograms-react/lib/random--samples/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const RandomSamples = /*#__PURE__*/React__default["default"].forwardRef(function RandomSamples(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31.255,11.746l-11-11c-0.141-0.141-0.369-0.141-0.51,0l-11,11c-0.141,0.141-0.141,0.368,0,0.509\tl2.386,2.386H1c-0.199,0-0.36,0.161-0.36,0.36v16c0,0.199,0.161,0.36,0.36,0.36h16c0.199,0,0.36-0.161,0.36-0.36V20.87l2.385,2.385\tc0.07,0.07,0.163,0.105,0.255,0.105s0.185-0.035,0.255-0.105l11-11C31.396,12.114,31.396,11.886,31.255,11.746z M16.64,30.64H1.36\tV15.36h15.28V30.64z M20,22.491l-2.64-2.641V15c0-0.199-0.161-0.36-0.36-0.36h-4.851L9.509,12L20,1.509L30.491,12L20,22.491z M14,19\tc0,0.552-0.448,1-1,1s-1-0.448-1-1s0.448-1,1-1S14,18.448,14,19z M20,6c-0.552,0-1,0.448-1,1s0.448,1,1,1s1-0.448,1-1S20.552,6,20,6\tz M20,11c-0.552,0-1,0.448-1,1s0.448,1,1,1s1-0.448,1-1S20.552,11,20,11z M15,11c-0.552,0-1,0.448-1,1s0.448,1,1,1s1-0.448,1-1\tS15.552,11,15,11z M25,11c-0.552,0-1,0.448-1,1s0.448,1,1,1s1-0.448,1-1S25.552,11,25,11z M20,16c-0.552,0-1,0.448-1,1s0.448,1,1,1\ts1-0.448,1-1S20.552,16,20,16z M9,22c-0.552,0-1,0.448-1,1s0.448,1,1,1s1-0.448,1-1S9.552,22,9,22z M5,26c-0.552,0-1,0.448-1,1\ts0.448,1,1,1s1-0.448,1-1S5.552,26,5,26z"
  })), children);
});

module.exports = RandomSamples;
