// @carbon/pictograms-react/lib/hard--ice--cream/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const HardIceCream = /*#__PURE__*/React__default["default"].forwardRef(function HardIceCream(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,31.36c-0.155,0-0.292-0.1-0.341-0.246L10.71,16.268c-1.672-0.322-2.672-1.473-2.672-3.161\tc0-1.142,0.55-2.293,1.616-2.871C9.29,9.383,9.105,8.476,9.105,7.535c0-3.802,3.093-6.895,6.895-6.895s6.896,3.093,6.896,6.895\tc0,0.94-0.185,1.848-0.55,2.701c1.065,0.578,1.616,1.729,1.616,2.871c0,1.783-1.312,2.962-2.687,3.206l-4.934,14.801\tC16.292,31.261,16.155,31.36,16,31.36z M11.498,16.356L16,29.861l4.501-13.501h-8.777C11.648,16.36,11.572,16.358,11.498,16.356z M11.249,9.855c0.199,0,0.36,0.161,0.36,0.36s-0.161,0.36-0.36,0.36c-1.711,0-2.491,1.313-2.491,2.533\tc0,1.586,1.109,2.533,2.966,2.533h9.027c1.198,0,2.49-0.968,2.49-2.533c0-1.22-0.779-2.533-2.49-2.533H16.95\tc-0.199,0-0.36-0.161-0.36-0.36s0.161-0.36,0.36-0.36h3.801c0.334,0,0.643,0.041,0.925,0.116c0.332-0.769,0.499-1.587,0.499-2.436\tc0-3.405-2.77-6.175-6.175-6.175S9.825,4.131,9.825,7.535c0,0.849,0.168,1.667,0.5,2.436C10.607,9.896,10.916,9.855,11.249,9.855z"
  })), children);
});

module.exports = HardIceCream;
