// @carbon/pictograms-react/lib/touch--gesture/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const TouchGesture = /*#__PURE__*/React__default["default"].forwardRef(function TouchGesture(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31.36,2v20h-0.72V2.36H1.36v22.28H12v0.721H1c-0.199,0-0.36-0.161-0.36-0.36V2\tc0-0.199,0.161-0.36,0.36-0.36h30C31.199,1.64,31.36,1.801,31.36,2z M27.5,23.14c-0.455,0-0.867,0.171-1.191,0.443\tc-0.19-0.825-0.928-1.443-1.809-1.443c-0.52,0-0.99,0.216-1.327,0.561c-0.3-0.625-0.934-1.061-1.673-1.061\tc-0.431,0-0.824,0.154-1.14,0.401V16.5c0-1.025-0.835-1.86-1.86-1.86s-1.86,0.834-1.86,1.86v9.541\tc-0.316-0.248-0.709-0.402-1.14-0.402c-1.025,0-1.86,0.835-1.86,1.86V30h0.72v-2.5c0-0.629,0.512-1.14,1.14-1.14\ts1.14,0.511,1.14,1.14V28v2h0.72v-2v-0.5v-11c0-0.628,0.511-1.14,1.14-1.14s1.14,0.512,1.14,1.14V27c0,0.199,0.161,0.36,0.36,0.36\ts0.36-0.161,0.36-0.36v-3.5c0-0.629,0.511-1.14,1.14-1.14s1.14,0.511,1.14,1.14V27c0,0.199,0.161,0.36,0.36,0.36\ts0.36-0.161,0.36-0.36v-3c0-0.629,0.511-1.14,1.14-1.14s1.14,0.511,1.14,1.14v3c0,0.199,0.161,0.36,0.36,0.36s0.36-0.161,0.36-0.36\tv-2c0-0.629,0.511-1.14,1.14-1.14s1.14,0.511,1.14,1.14v5h0.721v-5C29.36,23.975,28.525,23.14,27.5,23.14z"
  })), children);
});

module.exports = TouchGesture;
