// @carbon/pictograms-react/lib/cardboard/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Cardboard = /*#__PURE__*/React__default["default"].forwardRef(function Cardboard(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M28,29.36H4c-0.199,0-0.36-0.161-0.36-0.36V11.065\tL0.663,3.126c-0.041-0.11-0.026-0.234,0.041-0.331S0.882,2.64,1,2.64h15c0.15,0,0.284,0.093,0.337,0.233L19,9.975l2.663-7.101\tC21.716,2.733,21.85,2.64,22,2.64h9c0.118,0,0.229,0.058,0.296,0.155s0.083,0.221,0.041,0.332l-2.977,7.939V29\tC28.36,29.199,28.199,29.36,28,29.36z M19.36,28.64h8.279V11.36H19.36V28.64z M4.36,28.64h14.28V11.36H4.36V28.64z M19.52,10.64\th8.231l2.729-7.28h-8.231L19.52,10.64z M4.25,10.64h14.23l-2.73-7.28H1.52L4.25,10.64z M14,17.36H9c-0.75,0-1.36-0.61-1.36-1.36\tc0-0.013,0-0.025,0.002-0.038C7.663,15.229,8.264,14.64,9,14.64h5c0.75,0,1.36,0.61,1.36,1.36S14.75,17.36,14,17.36z M8.36,16.027\tC8.375,16.367,8.656,16.64,9,16.64h5c0.353,0,0.64-0.287,0.64-0.64s-0.287-0.64-0.64-0.64H9c-0.352,0-0.639,0.288-0.639,0.642\tC8.361,16.01,8.361,16.019,8.36,16.027z"
  })), children);
});

module.exports = Cardboard;
