// @carbon/pictograms-react/lib/tree--diagram/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const TreeDiagram = /*#__PURE__*/React__default["default"].forwardRef(function TreeDiagram(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M19,30.36c-1.853,0-3.36-1.508-3.36-3.36c0-1.142,0.572-2.152,1.445-2.76l-2.801-5.135\tc-0.792,0.328-1.776,0.328-2.567,0L8.915,24.24c0.873,0.607,1.445,1.618,1.445,2.76c0,1.853-1.507,3.36-3.36,3.36\tS3.64,28.853,3.64,27S5.147,23.64,7,23.64c0.454,0,0.888,0.091,1.283,0.255l2.802-5.135C10.212,18.152,9.64,17.142,9.64,16\tc0-1.853,1.507-3.36,3.36-3.36c0.455,0,0.888,0.09,1.284,0.255l2.801-5.136C16.212,7.152,15.64,6.142,15.64,5\tc0-1.853,1.507-3.36,3.36-3.36S22.36,3.147,22.36,5c0,1.141-0.572,2.151-1.444,2.759l2.801,5.136\tC24.112,12.73,24.546,12.64,25,12.64c1.853,0,3.36,1.507,3.36,3.36s-1.508,3.36-3.36,3.36s-3.36-1.508-3.36-3.36\tc0-1.142,0.572-2.152,1.445-2.76l-2.801-5.135c-0.791,0.328-1.775,0.33-2.567,0l-2.801,5.136c0.872,0.607,1.444,1.618,1.444,2.759\ts-0.572,2.151-1.444,2.759l2.801,5.136C18.112,23.73,18.546,23.64,19,23.64c1.853,0,3.36,1.508,3.36,3.36S20.853,30.36,19,30.36z M19,24.36c-1.456,0-2.64,1.184-2.64,2.64s1.184,2.64,2.64,2.64s2.64-1.184,2.64-2.64S20.456,24.36,19,24.36z M7,24.36\tc-1.456,0-2.64,1.184-2.64,2.64S5.544,29.64,7,29.64S9.64,28.456,9.64,27S8.456,24.36,7,24.36z M25,13.36\tc-1.456,0-2.64,1.185-2.64,2.64c0,1.456,1.184,2.64,2.64,2.64s2.64-1.184,2.64-2.64S26.456,13.36,25,13.36z M13,13.36\tc-1.456,0-2.64,1.185-2.64,2.64c0,1.456,1.185,2.64,2.64,2.64s2.64-1.184,2.64-2.64S14.456,13.36,13,13.36z M19,2.36\tc-1.456,0-2.64,1.185-2.64,2.64S17.544,7.64,19,7.64S21.64,6.456,21.64,5S20.456,2.36,19,2.36z"
  })), children);
});

module.exports = TreeDiagram;
