// @carbon/pictograms-react/es/functions-as-a-service/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const FunctionsAsAService = /*#__PURE__*/React.forwardRef(function FunctionsAsAService(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M12.25,30.255l-0.508-0.51l1.004-1\tc0.057-0.057,0.13-0.093,0.209-0.103l7.32-0.915c0.36-0.045,0.699-0.21,0.954-0.466l6.102-6.103c0.411-0.41,0.411-1.079,0-1.491\tc-0.411-0.411-1.081-0.41-1.49,0.001l-6.237,5.07c-0.42,0.386-0.979,0.621-1.593,0.621H13.01V24.64h5.001\tc0.393,0,0.753-0.139,1.036-0.369l-0.003-0.004l0.092-0.075c0.315-0.299,0.513-0.722,0.513-1.189c0-0.354-0.287-0.642-0.64-0.642\th-7.75c-2.614,0-4.759,2.043-4.885,4.651c-0.004,0.089-0.042,0.174-0.105,0.237l-3.015,3.006l-0.509-0.51l2.918-2.909\tc0.213-2.925,2.644-5.196,5.595-5.196h7.75c0.75,0,1.36,0.61,1.36,1.36c0,0.066-0.003,0.132-0.009,0.196l4.999-4.062\tc0.663-0.666,1.79-0.667,2.48,0.024c0.692,0.692,0.691,1.818,0.001,2.51l-6.102,6.102c-0.368,0.368-0.856,0.607-1.375,0.672\tl-7.197,0.899L12.25,30.255z M25,16.36c-0.625,0-1.153-0.424-1.312-1H6v-0.72h17.688c0.158-0.576,0.687-1,1.312-1\tc0.75,0,1.36,0.61,1.36,1.36S25.75,16.36,25,16.36z M25,14.36c-0.353,0-0.64,0.287-0.64,0.64s0.287,0.64,0.64,0.64\ts0.64-0.287,0.64-0.64S25.353,14.36,25,14.36z M7,11.36c-0.75,0-1.36-0.61-1.36-1.36S6.25,8.64,7,8.64c0.625,0,1.153,0.424,1.312,1\tH26v0.72H8.312C8.153,10.936,7.625,11.36,7,11.36z M7,9.36c-0.353,0-0.64,0.287-0.64,0.64S6.647,10.64,7,10.64S7.64,10.353,7.64,10\tS7.353,9.36,7,9.36z M25,6.36c-0.625,0-1.153-0.424-1.312-1H6V4.64h17.688c0.158-0.576,0.687-1,1.312-1c0.75,0,1.36,0.61,1.36,1.36\tS25.75,6.36,25,6.36z M25,4.36c-0.353,0-0.64,0.287-0.64,0.64S24.647,5.64,25,5.64S25.64,5.353,25.64,5S25.353,4.36,25,4.36z"
  })), children);
});

export { FunctionsAsAService as default };
