// @carbon/pictograms-react/es/advanced--fraud--protection/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const AdvancedFraudProtection = /*#__PURE__*/React.forwardRef(function AdvancedFraudProtection(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M17,21.36H2.5\tc-1.061,0-1.86-0.8-1.86-1.86v-17c0-1.061,0.799-1.86,1.86-1.86h27c1.061,0,1.86,0.799,1.86,1.86V15h-0.72V2.5\tc0-0.661-0.479-1.14-1.14-1.14h-27c-0.661,0-1.14,0.479-1.14,1.14v17c0,0.66,0.479,1.14,1.14,1.14H17V21.36z M13,17.36H4v-0.72h9\tV17.36z M10,9.36H4C3.801,9.36,3.64,9.199,3.64,9V4c0-0.199,0.161-0.36,0.36-0.36h6c0.199,0,0.36,0.161,0.36,0.36v5\tC10.36,9.199,10.199,9.36,10,9.36z M8.36,8.64h1.28V6.36H8.36V8.64z M6.36,8.64h1.281V6c0-0.199,0.161-0.36,0.36-0.36h1.64V4.36\tH6.36V8.64z M4.36,8.64h1.28V7.36H4.36V8.64z M4.36,6.64h1.28V4.36H4.36V6.64z M25,31.36c-3.507,0-6.36-2.854-6.36-6.36v-7.223\tl6.36-3.18l6.36,3.18V25C31.36,28.507,28.507,31.36,25,31.36z M19.36,18.223V25c0,3.11,2.529,5.64,5.64,5.64s5.64-2.529,5.64-5.64\tv-6.777L25,15.402L19.36,18.223z"
  })), children);
});

export { AdvancedFraudProtection as default };
