// @carbon/pictograms-react/lib/video--01/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Video_01 = /*#__PURE__*/React__default["default"].forwardRef(function Video_01(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M23,27.36H1c-0.199,0-0.36-0.161-0.36-0.36V5c0-0.199,0.161-0.36,0.36-0.36h22\tc0.199,0,0.36,0.161,0.36,0.36v6.375l7.459-4.331c0.11-0.064,0.248-0.066,0.36,0c0.111,0.064,0.181,0.183,0.181,0.312v17.321\tc0,0.129-0.069,0.248-0.182,0.312c-0.11,0.063-0.249,0.063-0.36-0.002l-7.458-4.359V27C23.36,27.199,23.199,27.36,23,27.36z M1.36,26.64h21.28V20c0-0.129,0.069-0.248,0.182-0.312c0.11-0.063,0.25-0.063,0.36,0.002l7.458,4.359V7.98l-7.459,4.331\tc-0.11,0.064-0.248,0.065-0.36,0C22.709,12.248,22.64,12.129,22.64,12V5.36H1.36V26.64z"
  })), children);
});

module.exports = Video_01;
