// @carbon/pictograms-react/lib/edge/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const Edge = /*#__PURE__*/React__default["default"].forwardRef(function Edge(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M23.558,27.254h-13.5c-3.005,0-5.36-2.354-5.36-5.36\tc0-3.29,2.891-5.996,6.08-5.597c1.508-2.566,3.452-3.763,6.102-3.763c4.138,0,6.96,3.766,7.129,7.406\tc1.967,0.158,3.409,1.638,3.409,3.565C27.418,25.572,25.686,27.254,23.558,27.254z M10.102,16.974c-2.539,0-4.684,2.253-4.684,4.92\tc0,2.602,2.038,4.64,4.64,4.64h13.5c1.731,0,3.14-1.358,3.14-3.028c0-1.657-1.278-2.859-3.04-2.859c-0.199,0-0.36-0.161-0.36-0.36\tc0-3.396-2.578-7.032-6.417-7.032c-2.435,0-4.214,1.148-5.598,3.612c-0.075,0.132-0.225,0.206-0.375,0.178\tC10.639,16.997,10.368,16.974,10.102,16.974z M14.558,3.893c0,0.276-0.224,0.5-0.5,0.5s-0.5-0.224-0.5-0.5s0.224-0.5,0.5-0.5\tC14.334,3.393,14.558,3.617,14.558,3.893z M18.058,3.393c-0.276,0-0.5,0.224-0.5,0.5s0.224,0.5,0.5,0.5s0.5-0.224,0.5-0.5\tS18.334,3.393,18.058,3.393z M2.058,17.393c-0.276,0-0.5,0.224-0.5,0.5s0.224,0.5,0.5,0.5s0.5-0.224,0.5-0.5\tS2.334,17.393,2.058,17.393z M2.558,13.393c-0.276,0-0.5,0.224-0.5,0.5s0.224,0.5,0.5,0.5s0.5-0.224,0.5-0.5\tS2.834,13.393,2.558,13.393z M29.558,13.393c-0.276,0-0.5,0.224-0.5,0.5s0.224,0.5,0.5,0.5s0.5-0.224,0.5-0.5\tS29.834,13.393,29.558,13.393z M28.058,9.893c-0.276,0-0.5,0.224-0.5,0.5s0.224,0.5,0.5,0.5s0.5-0.224,0.5-0.5\tS28.334,9.893,28.058,9.893z M4.058,9.893c-0.276,0-0.5,0.224-0.5,0.5s0.224,0.5,0.5,0.5s0.5-0.224,0.5-0.5S4.334,9.893,4.058,9.893\tz M7.058,6.893c-0.276,0-0.5,0.224-0.5,0.5s0.224,0.5,0.5,0.5s0.5-0.224,0.5-0.5S7.334,6.893,7.058,6.893z M25.058,6.893\tc-0.276,0-0.5,0.224-0.5,0.5s0.224,0.5,0.5,0.5s0.5-0.224,0.5-0.5S25.334,6.893,25.058,6.893z M22.058,4.643\tc-0.276,0-0.5,0.224-0.5,0.5s0.224,0.5,0.5,0.5s0.5-0.224,0.5-0.5S22.334,4.643,22.058,4.643z M10.058,4.643\tc-0.276,0-0.5,0.224-0.5,0.5s0.224,0.5,0.5,0.5s0.5-0.224,0.5-0.5S10.334,4.643,10.058,4.643z M30.058,17.393\tc-0.276,0-0.5,0.224-0.5,0.5s0.224,0.5,0.5,0.5s0.5-0.224,0.5-0.5S30.334,17.393,30.058,17.393z"
  })), children);
});

module.exports = Edge;
