// @carbon/pictograms-react/lib/device--pairing/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const DevicePairing = /*#__PURE__*/React__default["default"].forwardRef(function DevicePairing(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M31,29.36h-9c-0.199,0-0.36-0.161-0.36-0.36v-6.64H1v-0.72h20.64v-1.28H4\tc-0.75,0-1.36-0.61-1.36-1.36V4c0-0.749,0.61-1.359,1.36-1.36h22c0.75,0,1.36,0.61,1.36,1.36v8.64H31c0.199,0,0.36,0.161,0.36,0.36\tv16C31.36,29.199,31.199,29.36,31,29.36z M22.36,28.64h8.279v-2.28H22.36V28.64z M22.36,25.64h8.279V13.36H22.36V25.64z M4,3.36\tC3.647,3.361,3.36,3.648,3.36,4v15c0,0.353,0.287,0.64,0.64,0.64h17.64V13c0-0.199,0.161-0.36,0.36-0.36h4.64V4\tc0-0.353-0.287-0.64-0.64-0.64C26,3.36,4,3.36,4,3.36z"
  })), children);
});

module.exports = DevicePairing;
