// @carbon/pictograms-react/lib/fire--extinguisher/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const FireExtinguisher = /*#__PURE__*/React__default["default"].forwardRef(function FireExtinguisher(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M21,31.36H11c-0.199,0-0.36-0.161-0.36-0.36V14c0-2.109,1.225-3.938,3-4.812V5.365\tc0-0.4,0.325-0.725,0.725-0.725h0.275V3.36h-3.491L5.254,9.254L4.746,8.746l6-6C10.813,2.678,10.904,2.64,11,2.64h3.64V1.365\tc0-0.4,0.325-0.725,0.725-0.725h1.27c0.399,0,0.726,0.325,0.726,0.725V2.64h3.279V2c0-0.176,0.128-0.326,0.301-0.355l6-1\tc0.108-0.02,0.212,0.012,0.292,0.08C27.313,0.794,27.36,0.894,27.36,1v4c0,0.106-0.047,0.206-0.128,0.275\tc-0.081,0.067-0.185,0.096-0.292,0.08l-6-1c-0.172-0.029-0.3-0.179-0.3-0.355V3.36h-3.28v1.28h0.274\tc0.399,0,0.726,0.325,0.726,0.725v3.823c1.775,0.875,3,2.703,3,4.812v17C21.36,31.199,21.199,31.36,21,31.36z M11.36,30.64h9.28\tv-6.28h-9.28V30.64z M11.36,23.64h9.28v-7.28h-9.28V23.64z M11.36,15.64h9.28V14c0-2.559-2.081-4.64-4.64-4.64\ts-4.64,2.082-4.64,4.64C11.36,14,11.36,15.64,11.36,15.64z M16,8.64c0.572,0,1.123,0.09,1.64,0.256V5.365L14.365,5.36l-0.004,3.537\tC14.877,8.73,15.429,8.64,16,8.64z M15.36,4.64h1.28V1.365L15.365,1.36L15.36,4.64z M21.36,3.695l5.279,0.88v-3.15l-5.279,0.88\tV3.695z"
  })), children);
});

module.exports = FireExtinguisher;
