// @carbon/pictograms-react/lib/cloud--guidelines/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const CloudGuidelines = /*#__PURE__*/React__default["default"].forwardRef(function CloudGuidelines(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16,31.36C7.53,31.36,0.64,24.47,0.64,16S7.53,0.64,16,0.64S31.36,7.53,31.36,16\tS24.47,31.36,16,31.36z M15.385,30.627c0.204,0.009,0.409,0.013,0.615,0.013c4.741,0,8.963-2.265,11.64-5.77V12.008h0.721v11.831\tc1.442-2.268,2.279-4.957,2.279-7.839c0-1.999-0.402-3.904-1.13-5.641L18.363,3.924l0.36-0.624l10.249,5.917\tc-2.174-4.141-6.273-7.119-11.101-7.738L6.723,7.916l-0.36-0.624l10.252-5.919C16.412,1.364,16.206,1.36,16,1.36\tc-4.741,0-8.963,2.266-11.64,5.771v12.862H3.64V8.161C2.197,10.429,1.36,13.119,1.36,16c0,1.998,0.402,3.903,1.13,5.641\tl11.147,6.436l-0.36,0.623L3.028,22.782c2.174,4.142,6.273,7.12,11.101,7.739l11.148-6.437l0.359,0.623L15.385,30.627z M11.352,23.664c-0.062,0-0.124-0.017-0.18-0.049c-0.118-0.067-0.188-0.196-0.18-0.333l0.199-3.35\tc0.003-0.056,0.02-0.11,0.048-0.158l6.5-11.258c0.048-0.083,0.126-0.143,0.219-0.167c0.09-0.025,0.189-0.012,0.272,0.036l2.598,1.5\tc0.173,0.099,0.231,0.319,0.132,0.492l-6.5,11.257c-0.028,0.049-0.066,0.09-0.113,0.12l-2.797,1.85\tC11.49,23.644,11.421,23.664,11.352,23.664z M11.905,20.061l-0.151,2.546l2.125-1.406l6.277-10.872l-1.974-1.14L11.905,20.061z"
  })), children);
});

module.exports = CloudGuidelines;
