// @carbon/pictograms-react/es/softlayer--enablement/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
import { _ as _extends } from '../_rollupPluginBabelHelpers-369b277e.js';
import React from 'react';
import Icon from '../Icon.js';
import '@carbon/icon-helpers';
import 'prop-types';

var _path;
const SoftlayerEnablement = /*#__PURE__*/React.forwardRef(function SoftlayerEnablement(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React.createElement(Icon, _extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React.createElement("path", {
    d: "M31,31.36H1c-0.199,0-0.36-0.161-0.36-0.36v-6c0-0.199,0.161-0.36,0.36-0.36h30\tc0.199,0,0.36,0.161,0.36,0.36v6C31.36,31.199,31.199,31.36,31,31.36z M1.36,30.64h29.28v-5.28H1.36V30.64z M31,19.36H1\tc-0.199,0-0.36-0.161-0.36-0.36v-6c0-0.199,0.161-0.36,0.36-0.36h30c0.199,0,0.36,0.161,0.36,0.36v6\tC31.36,19.199,31.199,19.36,31,19.36z M1.36,18.64h29.28v-5.28H1.36V18.64z M31,7.36H1C0.801,7.36,0.64,7.199,0.64,7V1\tc0-0.199,0.161-0.36,0.36-0.36h30c0.199,0,0.36,0.161,0.36,0.36v6C31.36,7.199,31.199,7.36,31,7.36z M1.36,6.64h29.28V1.36H1.36\tV6.64z"
  })), children);
});

export { SoftlayerEnablement as default };
