// @carbon/pictograms-react/lib/stairs--plan--view/index.js
/**
 * Copyright IBM Corp. 2019, 2023
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * Code generated by @carbon/icon-build-helpers. DO NOT EDIT.
 */
'use strict';

var _rollupPluginBabelHelpers = require('../_rollupPluginBabelHelpers-e2de4def.js');
var React = require('react');
var Icon_js = require('../Icon.js');
require('@carbon/icon-helpers');
require('prop-types');

function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefaultLegacy(React);

var _path;
const StairsPlanView = /*#__PURE__*/React__default["default"].forwardRef(function StairsPlanView(_ref, ref) {
  let {
    children,
    ...rest
  } = _ref;
  return /*#__PURE__*/React__default["default"].createElement(Icon_js, _rollupPluginBabelHelpers._extends({
    width: 64,
    height: 64,
    viewBox: "0 0 32 32",
    xmlns: "http://www.w3.org/2000/svg",
    fill: "currentColor",
    ref: ref
  }, rest), _path || (_path = /*#__PURE__*/React__default["default"].createElement("path", {
    d: "M16.36,30h-0.72v-5.64H2c-0.199,0-0.36-0.161-0.36-0.36V4c0-0.199,0.161-0.36,0.36-0.36h13.64V2\th0.72v5.64H30c0.199,0,0.36,0.161,0.36,0.36v20c0,0.199-0.161,0.36-0.36,0.36H16.36V30z M16.36,27.64h13.28v-3.28H16.36V27.64z M16.36,23.64h13.28v-3.28H16.36V23.64z M2.36,23.64h13.28v-3.28H2.36V23.64z M16.36,19.64h13.28v-3.28H16.36V19.64z M2.36,19.64\th13.28v-3.28H2.36V19.64z M16.36,15.64h13.28v-3.28H16.36V15.64z M2.36,15.64h13.28v-3.28H2.36V15.64z M16.36,11.64h13.28V8.36\tH16.36V11.64z M2.36,11.64h13.28V8.36H2.36V11.64z M2.36,7.64h13.28V4.36H2.36V7.64z"
  })), children);
});

module.exports = StairsPlanView;
